<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    
    <head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <!-- AppResources meta begin -->
        <meta name="paf-app-resources" content="" />
                 <script type="text/javascript">var ncbi_startTime = new Date();</script>
            
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="" />

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="books" /><meta name="ncbi_pdid" content="book-part" /><meta name="ncbi_acc" content="NBK1434" /><meta name="ncbi_domain" content="gene" /><meta name="ncbi_report" content="record" /><meta name="ncbi_type" content="fulltext" /><meta name="ncbi_objectid" content="" /><meta name="ncbi_pcid" content="/NBK1434/" /><meta name="ncbi_pagename" content="Hereditary Hearing Loss and Deafness Overview - GeneReviews® - NCBI Bookshelf" /><meta name="ncbi_bookparttype" content="chapter" /><meta name="ncbi_app" content="bookshelf" />
        <!-- Logger end -->
        
        <title>Hereditary Hearing Loss and Deafness Overview - GeneReviews® - NCBI Bookshelf</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css" /><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,FOLLOW,NOARCHIVE" /><meta name="citation_inbook_title" content="GeneReviews® [Internet]" /><meta name="citation_title" content="Hereditary Hearing Loss and Deafness Overview" /><meta name="citation_publisher" content="University of Washington, Seattle" /><meta name="citation_date" content="2017/07/27" /><meta name="citation_author" content="A Eliot Shearer" /><meta name="citation_author" content="Michael S Hildebrand" /><meta name="citation_author" content="Richard JH Smith" /><meta name="citation_pmid" content="20301607" /><meta name="citation_fulltext_html_url" content="https://www.ncbi.nlm.nih.gov/books/NBK1434/" /><meta name="citation_keywords" content="Nonsyndromic Hearing Loss and Deafness" /><meta name="citation_keywords" content="Actin, cytoplasmic 2" /><meta name="citation_keywords" content="Alpha-tectorin" /><meta name="citation_keywords" content="Barttin" /><meta name="citation_keywords" content="Cadherin-23" /><meta name="citation_keywords" content="Claudin-14" /><meta name="citation_keywords" content="Cochlin" /><meta name="citation_keywords" content="Coiled-coil domain-containing protein 50" /><meta name="citation_keywords" content="Collagen alpha-2(XI) chain" /><meta name="citation_keywords" content="Cytochrome c oxidase subunit 1" /><meta name="citation_keywords" content="Dentin sialophosphoprotein" /><meta name="citation_keywords" content="Espin" /><meta name="citation_keywords" content="Eyes absent homolog 4" /><meta name="citation_keywords" content="Gap junction beta-2 protein" /><meta name="citation_keywords" content="Gap junction beta-3 protein" /><meta name="citation_keywords" content="Gap junction beta-6 protein" /><meta name="citation_keywords" content="Gasdermin-E" /><meta name="citation_keywords" content="Glutaredoxin domain-containing cysteine-rich protein 1" /><meta name="citation_keywords" content="G-protein-signaling modulator 2" /><meta name="citation_keywords" content="Grainyhead-like protein 2 homolog" /><meta name="citation_keywords" content="Harmonin" /><meta name="citation_keywords" content="Hepatocyte growth factor" /><meta name="citation_keywords" content="Homeobox protein SIX1" /><meta name="citation_keywords" content="LHFPL tetraspan subfamily member 5 protein" /><meta name="citation_keywords" content="Lipoxygenase homology domain-containing protein 1" /><meta name="citation_keywords" content="MARVEL domain-containing protein 2" /><meta name="citation_keywords" content="Myosin-14" /><meta name="citation_keywords" content="Myosin-9" /><meta name="citation_keywords" content="Myosin-IIIa" /><meta name="citation_keywords" content="Nesprin-4" /><meta name="citation_keywords" content="Otoancorin" /><meta name="citation_keywords" content="Otoferlin" /><meta name="citation_keywords" content="P2X purinoceptor 2" /><meta name="citation_keywords" content="Pejvakin" /><meta name="citation_keywords" content="Pendrin" /><meta name="citation_keywords" content="Phosphatidylinositol phosphatase PTPRQ" /><meta name="citation_keywords" content="Potassium voltage-gated channel subfamily KQT member 4" /><meta name="citation_keywords" content="POU domain, class 3, transcription factor 4" /><meta name="citation_keywords" content="POU domain, class 4, transcription factor 3" /><meta name="citation_keywords" content="Prestin" /><meta name="citation_keywords" content="Protein diaphanous homolog 1" /><meta name="citation_keywords" content="Protein FAM189A2" /><meta name="citation_keywords" content="Protocadherin-15" /><meta name="citation_keywords" content="Radixin" /><meta name="citation_keywords" content="Ribose-phosphate pyrophosphokinase 1" /><meta name="citation_keywords" content="Small muscular protein" /><meta name="citation_keywords" content="Stereocilin" /><meta name="citation_keywords" content="Steroid hormone receptor ERR2" /><meta name="citation_keywords" content="Taperin" /><meta name="citation_keywords" content="Tight junction protein ZO-2" /><meta name="citation_keywords" content="Transmembrane channel-like protein 1" /><meta name="citation_keywords" content="Transmembrane inner ear expressed protein" /><meta name="citation_keywords" content="Transmembrane O-methyltransferase" /><meta name="citation_keywords" content="Transmembrane protease serine 3" /><meta name="citation_keywords" content="TRIO and F-actin-binding protein" /><meta name="citation_keywords" content="Unconventional myosin-Ia" /><meta name="citation_keywords" content="Unconventional myosin-VI" /><meta name="citation_keywords" content="Unconventional myosin-VIIa" /><meta name="citation_keywords" content="Unconventional myosin-XV" /><meta name="citation_keywords" content="Vesicular glutamate transporter 3" /><meta name="citation_keywords" content="Whirlin" /><meta name="citation_keywords" content="Wolframin" /><meta name="citation_keywords" content="ACTG1" /><meta name="citation_keywords" content="BSND" /><meta name="citation_keywords" content="CCDC50" /><meta name="citation_keywords" content="CDH23" /><meta name="citation_keywords" content="CLDN14" /><meta name="citation_keywords" content="COCH" /><meta name="citation_keywords" content="COL11A2" /><meta name="citation_keywords" content="DIAPH1" /><meta name="citation_keywords" content="DSPP" /><meta name="citation_keywords" content="ESPN" /><meta name="citation_keywords" content="ESRRB" /><meta name="citation_keywords" content="EYA4" /><meta name="citation_keywords" content="FAM189A2" /><meta name="citation_keywords" content="GJB2" /><meta name="citation_keywords" content="GJB3" /><meta name="citation_keywords" content="GJB6" /><meta name="citation_keywords" content="GPSM2" /><meta name="citation_keywords" content="GRHL2" /><meta name="citation_keywords" content="GRXCR1" /><meta name="citation_keywords" content="GSDME" /><meta name="citation_keywords" content="HGF" /><meta name="citation_keywords" content="KCNQ4" /><meta name="citation_keywords" content="LHFPL5" /><meta name="citation_keywords" content="LOXHD1" /><meta name="citation_keywords" content="LRTOMT" /><meta name="citation_keywords" content="MARVELD2" /><meta name="citation_keywords" content="MIR96" /><meta name="citation_keywords" content="MT-CO1" /><meta name="citation_keywords" content="MT-RNR1" /><meta name="citation_keywords" content="MT-TS1" /><meta name="citation_keywords" content="MYH14" /><meta name="citation_keywords" content="MYH9" /><meta name="citation_keywords" content="MYO15A" /><meta name="citation_keywords" content="MYO1A" /><meta name="citation_keywords" content="MYO3A" /><meta name="citation_keywords" content="MYO6" /><meta name="citation_keywords" content="MYO7A" /><meta name="citation_keywords" content="OTOA" /><meta name="citation_keywords" content="OTOF" /><meta name="citation_keywords" content="P2RX2" /><meta name="citation_keywords" content="PCDH15" /><meta name="citation_keywords" content="PJVK" /><meta name="citation_keywords" content="POU3F4" /><meta name="citation_keywords" content="POU4F3" /><meta name="citation_keywords" content="PRPS1" /><meta name="citation_keywords" content="PTPRQ" /><meta name="citation_keywords" content="RDX" /><meta name="citation_keywords" content="SIX1" /><meta name="citation_keywords" content="SLC17A8" /><meta name="citation_keywords" content="SLC26A4" /><meta name="citation_keywords" content="SLC26A5" /><meta name="citation_keywords" content="SMPX" /><meta name="citation_keywords" content="STRC" /><meta name="citation_keywords" content="SYNE4" /><meta name="citation_keywords" content="TECTA" /><meta name="citation_keywords" content="TJP2" /><meta name="citation_keywords" content="TMC1" /><meta name="citation_keywords" content="TMIE" /><meta name="citation_keywords" content="TMPRSS3" /><meta name="citation_keywords" content="TPRN" /><meta name="citation_keywords" content="TRIOBP" /><meta name="citation_keywords" content="USH1C" /><meta name="citation_keywords" content="WFS1" /><meta name="citation_keywords" content="WHRN" /><meta name="citation_keywords" content="Hereditary Hearing Loss and Deafness" /><meta name="citation_keywords" content="Overview" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="DC.Title" content="Hereditary Hearing Loss and Deafness Overview" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="University of Washington, Seattle" /><meta name="DC.Contributor" content="A Eliot Shearer" /><meta name="DC.Contributor" content="Michael S Hildebrand" /><meta name="DC.Contributor" content="Richard JH Smith" /><meta name="DC.Date" content="2017/07/27" /><meta name="DC.Identifier" content="https://www.ncbi.nlm.nih.gov/books/NBK1434/" /><meta name="description" content="This overview focuses on the clinical features and molecular genetics of common syndromic and nonsyndromic types of hereditary hearing loss." /><meta name="og:title" content="Hereditary Hearing Loss and Deafness Overview" /><meta name="og:type" content="book" /><meta name="og:description" content="This overview focuses on the clinical features and molecular genetics of common syndromic and nonsyndromic types of hereditary hearing loss." /><meta name="og:url" content="https://www.ncbi.nlm.nih.gov/books/NBK1434/" /><meta name="og:site_name" content="NCBI Bookshelf" /><meta name="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" /><meta name="twitter:card" content="summary" /><meta name="twitter:site" content="@ncbibooks" /><meta name="bk-non-canon-loc" content="/books/n/gene/deafness-overview/" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/books/NBK1434/" /><link rel="stylesheet" href="/corehtml/pmc/css/figpopup.css" type="text/css" media="screen" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books_print.min.css" type="text/css" media="print" /><style type="text/css">p a.figpopup{display:inline !important} .bk_tt {font-family: monospace}  .first-line-outdent .bk_ref {display: inline}  .body-content h2, .body-content .h2  {border-bottom: 1px solid #97B0C8} .body-content h2.inline {border-bottom: none} a.page-toc-label , .jig-ncbismoothscroll a {text-decoration:none;border:0 !important} .temp-labeled-list  .graphic {display:inline-block !important} .temp-labeled-list  img{width:100%}</style><script type="text/javascript" src="/corehtml/pmc/js/jquery.hoverIntent.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"> </script><script type="text/javascript">window.name="mainwindow";</script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/book-toc.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/books.min.js"> </script><meta name="book-collection" content="NONE" />

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="CE8D6575E0403E510000000006D40276.m_12" />
<meta name='referrer' content='origin-when-cross-origin'/><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4098876/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/4008682/3881636/3579733/4062871/12930/3964959/3854974/36029/4128070/9685/3549676/3609192/3609193/3609213/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3411343/3882866.css" media="print" /></head>
    <body class="book-part">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo"><h1 class="res_name"><a href="/books/" title="Bookshelf home">Bookshelf</a></h1><h2 class="res_tagline"></h2></div>
    <div class="search"><form method="get" action="/books/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="books" selected="selected" class="last" data-ac_dict="bookshelf-search">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books" data-ac_dict="bookshelf-search">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search Books. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'bookshelf-search',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'no'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/books/browse/">Browse Titles</a>
                    </li><li>
                        <a href="/books/advanced/">Advanced</a>
                    </li><li class="help">
                        <a href="/books/NBK3833/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <!-- Custom content 1 -->
<div class="col1">
    
</div>

<div class="container">
    <div id="maincontent" class="content eight_col col">
        <!-- Custom content in the left column above book nav -->
        <div class="col2">
            
        </div>
        
        <!-- Book content -->
        

        <!-- Custom content between navigation and content -->
        <div class="col3">
            
        </div>
        
        <div class="document">
            <div class="pre-content"><div><div class="bk_prnt"><p class="small">NCBI Bookshelf. A service of the National Library of Medicine, National Institutes of Health.</p><p>Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. </p></div><div class="iconblock clearfix whole_rhythm no_top_margin bk_noprnt"><a class="img_link icnblk_img" title="All GeneReviews" href="/books/n/gene/"><img class="source-thumb" src="/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" alt="Cover of GeneReviews®" height="100px" width="80px" /></a><div class="icnblk_cntnt eight_col"><h2>GeneReviews<sup>®</sup> [Internet].</h2><a data-jig="ncbitoggler" href="#__NBK1434_dtls__">Show details</a><div style="display:none" class="ui-widget" id="__NBK1434_dtls__"><div>Adam MP, Ardinger HH, Pagon RA, et al., editors.</div><div>Seattle (WA): <a href="http://www.washington.edu" ref="pagearea=page-banner&amp;targetsite=external&amp;targetcat=link&amp;targettype=publisher">University of Washington, Seattle</a>; 1993-2019.</div></div><div class="half_rhythm"><ul class="inline_list"><li style="margin-right:1em"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a></li></ul></div><div class="bk_noprnt"><form method="get" action="/books/n/gene/" id="bk_srch"><div class="bk_search"><label for="bk_term" class="offscreen_noflow">Search term</label><input type="text" title="Search GeneReviews" id="bk_term" name="term" value="" data-jig="ncbiclearbutton" /> <input type="submit" class="jig-ncbibutton" value="Search GeneReviews" submit="false" style="padding: 0.1em 0.4em;" /></div></form><div><ul class="inline_list"><li><a href="/books/n/gene/advanced/">GeneReviews Advanced Search</a></li><li style="margin-left:.5em"><a href="/books/n/gene/helpadvsearch/">Help</a></li></ul></div></div></div><div class="icnblk_cntnt two_col"><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/hfi/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/hht/" title="Next page in this title">Next &gt;</a></div></div></div></div></div>
            <div class="main-content lit-style" itemscope="itemscope" itemtype="http://schema.org/CreativeWork"><div class="meta-content fm-sec"><h1 id="_NBK1434_"><span class="title" itemprop="name">Hereditary Hearing Loss and Deafness Overview</span></h1><p class="contrib-group"><span itemprop="author">A Eliot Shearer</span>, MD, PhD, <span itemprop="author">Michael S Hildebrand</span>, PhD, and <span itemprop="author">Richard JH Smith</span>, MD.</p><a data-jig="ncbitoggler" href="#__NBK1434_ai__" style="border:0;text-decoration:none">Author Information</a><div style="display:none" class="ui-widget" id="__NBK1434_ai__"><div class="contrib half_rhythm"><span itemprop="author">A Eliot Shearer</span>, MD, PhD<div class="affiliation small">Department of Otolaryngology<br />University of Iowa Hospitals and Clinics<br />Iowa City, Iowa<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.awoiu@reraehs-nedia" class="oemail">ude.awoiu@reraehs-nedia</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Michael S Hildebrand</span>, PhD<div class="affiliation small">Epilepsy Research Centre<br />Department of Medicine<br />University of Melbourne<br />Melbourne, Australia<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ua.ude.bleminu@dnarbedlih.leahcim" class="oemail">ua.ude.bleminu@dnarbedlih.leahcim</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Richard JH Smith</span>, MD<div class="affiliation small">Department of Otolaryngology<br />University of Iowa Hospitals and Clinics<br />Iowa City, Iowa<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.awoiu@htims-drahcir" class="oemail">ude.awoiu@htims-drahcir</a></div></div></div></div><p class="small">Initial Posting: <span itemprop="datePublished">February 14, 1999</span>; Last Update: <span itemprop="dateModified">July 27, 2017</span>.</p><p><em>Estimated reading time: 31 minutes</em></p></div><div class="jig-ncbiinpagenav body-content whole_rhythm" data-jigconfig="allHeadingLevels: ['h2'],smoothScroll: false" itemprop="text"><div id="deafness-overview.Summary" itemprop="description"><h2 id="_deafness-overview_Summary_">Summary</h2><p>This overview focuses on the clinical features and molecular genetics of common <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> and nonsyndromic types of hereditary hearing loss.</p><p>The goals of this overview on hereditary hearing loss and deafness are the following:</p><div><h4 class="inline">Goal 1.</h4><p>Describe the <a href="#deafness-overview.Clinical_Characteristi">clinical characteristics</a> of hereditary hearing loss and deafness.</p></div><div><h4 class="inline">Goal 2.</h4><p>Review the <a href="#deafness-overview.Causes_of_Hereditary_H">causes</a> of hereditary hearing loss and deafness.</p></div><div><h4 class="inline">Goal 3.</h4><p>Provide an <a href="#deafness-overview.Evaluation_Strategy">evaluation strategy</a> to identify the genetic cause of hereditary hearing loss and deafness in a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> (when possible).</p></div><div><h4 class="inline">Goal 4.</h4><p>Inform <a href="#deafness-overview.Genetic_Counseling">genetic counseling</a> of family members of an individual with hereditary hearing loss and deafness.</p></div><div><h4 class="inline">Goal 5.</h4><p>Review <a href="#deafness-overview.Management">management</a> of hereditary hearing loss and deafness.</p></div></div><div id="deafness-overview.Clinical_Characteristi"><h2 id="_deafness-overview_Clinical_Characteristi_">Clinical Characteristics of Hereditary Hearing Loss and Deafness</h2><p><b>Type</b></p><ul><li class="half_rhythm"><div>Conductive hearing loss results from abnormalities of the external ear and/or the ossicles of the middle ear.</div></li><li class="half_rhythm"><div>Sensorineural hearing loss results from malfunction of inner ear structures (i.e., cochlea or auditory nerve).</div></li><li class="half_rhythm"><div>Mixed hearing loss is a combination of conductive and sensorineural hearing loss.</div></li><li class="half_rhythm"><div>Central auditory dysfunction results from damage or dysfunction at the level of the eighth cranial nerve, auditory brain stem, or cerebral cortex.</div></li></ul><p><b>Onset</b></p><ul><li class="half_rhythm"><div>Prelingual hearing loss is present before speech develops. All <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> (present at birth) hearing loss is prelingual, but not all prelingual hearing loss is congenital.</div></li><li class="half_rhythm"><div>Postlingual hearing loss occurs after the development of normal speech.</div></li></ul><p><b>Severity of hearing loss.</b> Hearing is measured in <b>decibels</b> (dB). The threshold or 0 dB mark for each frequency refers to the level at which normal young adults perceive a tone burst 50% of the time. Hearing is considered normal if an individual's thresholds are within 15 dB of normal thresholds. Severity of hearing loss is graded as shown in <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.severity_of_hearing/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTseverityofhearing" rid-ob="figobdeafnessoverviewTseverityofhearing">Table 1</a>.</p><div id="deafness-overview.T.severity_of_hearing" class="table"><h3><span class="label">Table 1. </span></h3><div class="caption"><p>Severity of Hearing Loss in Decibels (dB)</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1434/table/deafness-overview.T.severity_of_hearing/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__deafness-overview.T.severity_of_hearing_lrgtbl__"><table><thead><tr><th id="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severity</th><th id="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Hearing Threshold in Decibels</th></tr></thead><tbody><tr><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Mild</td><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">26-40 dB</td></tr><tr><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate</td><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">41-55 dB</td></tr><tr><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderately severe</td><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">56-70 dB</td></tr><tr><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe</td><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">71-90 dB</td></tr><tr><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Profound</td><td headers="hd_h_deafness-overview.T.severity_of_hearing_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">90 dB</td></tr></tbody></table></div></div><p>To calculate the percent hearing impairment, 25 dB is subtracted from the pure tone average of 500 Hz, 1000 Hz, 2000 Hz, 3000 Hz. The result is multiplied by 1.5 to obtain an ear-specific level. Impairment is determined by weighting the better ear five times the poorer ear [<a class="bk_pop" href="#deafness-overview.REF.jama.1979.2055">JAMA 1979</a>] (see <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.percent_hearing_impa/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTpercenthearingimpa" rid-ob="figobdeafnessoverviewTpercenthearingimpa">Table 2</a>).</p><p>Note: (1) Because conversational speech is at approximately 50-60 dB HL (hearing level), calculating <b>functional</b> impairment based on pure tone averages can be misleading. For example, a 45-dB hearing loss is functionally much more significant than 30% implies. (2) A different rating scale is appropriate for young children, for whom even limited hearing loss can have a great impact on language development [<a class="bk_pop" href="#deafness-overview.REF.northern.2002">Northern &#x00026; Downs 2002</a>].</p><div id="deafness-overview.T.percent_hearing_impa" class="table"><h3><span class="label">Table 2. </span></h3><div class="caption"><p>Percent Hearing Impairment</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1434/table/deafness-overview.T.percent_hearing_impa/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__deafness-overview.T.percent_hearing_impa_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">% Impairment</th><th id="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Pure Tone Average (dB)&#x000a0;<sup>1</sup></th><th id="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">% Residual Hearing</th></tr></thead><tbody><tr><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">100%</td><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">91 dB</td><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">0%</td></tr><tr><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">80%</td><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">78 dB</td><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">20%</td></tr><tr><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">60%</td><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">65 dB</td><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">40%</td></tr><tr><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">30%</td><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">45 dB</td><td headers="hd_h_deafness-overview.T.percent_hearing_impa_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">70%</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="deafness-overview.TF.2.1"><p class="no_margin">Pure tone average of 500 Hz, 1000 Hz, 2000 Hz, 3000 Hz</p></div></dd></dl></div></div></div><p><b>The frequency of hearing loss</b> is designated as:</p><ul><li class="half_rhythm"><div>Low (&#x0003c;500 Hz)</div></li><li class="half_rhythm"><div>Middle (501-2000 Hz)</div></li><li class="half_rhythm"><div>High (&#x0003e;2000 Hz)</div></li></ul><p><b>Other terminology</b></p><p><b>"Hearing impairment" and "hearing loss"</b> are often used interchangeably by health care professionals to refer to hearing determined by audiometry to be below threshold levels for normal hearing.</p><p><b>Deaf (small "d")</b> is a colloquial term that implies hearing thresholds in the severe-to-profound range by audiometry.</p><p><b>Deaf culture (always a capital "D").</b> Members of the Deaf community in the US are deaf and use American Sign Language. As in other cultures, members are characterized by unique social and societal attributes. Members of the Deaf community (i.e., the Deaf) do NOT consider themselves to be hearing "impaired," nor do they feel that they have a hearing "loss." Rather, they consider themselves deaf. Their deafness is not considered to be a pathology or disease to be treated or cured.</p><p><b>"Hard of hearing"</b> is more functional than audiologic. It is used by the Deaf to signify that a person has some usable hearing &#x02013; anything from mild to severe hearing loss. In the Deaf community persons who are deaf do not use oral language, while those who are hard of hearing usually have some oral language.</p><div id="deafness-overview.Diagnosing_Deafness_an"><h3>Diagnosing Deafness and Hearing Loss</h3><p><b>Physiologic tests</b> objectively determine the functional status of the auditory system and can be performed at any age. They include the following:</p><ul><li class="half_rhythm"><div><b>Auditory brain stem response testing (ABR, also known as BAER, BSER)</b> uses a stimulus (clicks) to evoke electrophysiologic responses, which originate in the eighth cranial nerve and auditory brain stem and are recorded with surface electrodes. ABR "wave V detection threshold" correlates best with hearing sensitivity in the 1500- to 4000-Hz region in neurologically normal individuals; ABR does not assess low frequency (&#x0003c;1500 Hz) sensitivity.</div></li><li class="half_rhythm"><div><b>Auditory steady-state response testing (ASSR)</b> is like ABR in that both are auditory evoked potentials and they are measured in similar ways. ASSR uses an objective, statistics-based mathematical detection algorithm to detect and define hearing thresholds. ASSR can be obtained using broadband or frequency-specific stimuli and can offer hearing threshold differentiation in the severe-to-profound range. It is frequently used to give frequency-specific information that ABR does not give. Test frequencies of 500, 1000, 2000, and 4000 Hz are commonly used.</div></li><li class="half_rhythm"><div><b>Evoked otoacoustic emissions (EOAEs)</b> are sounds originating within the cochlea that are measured in the external auditory canal using a probe with a microphone and transducer. EOAEs reflect primarily the activity of the outer hair cells of the cochlea across a broad frequency range and are present in ears with hearing sensitivity better than 40-50 dB HL.</div></li><li class="half_rhythm"><div><b>Immittance testing (tympanometry, acoustic reflex thresholds, acoustic reflex decay)</b> assesses the peripheral auditory system, including middle ear pressure, tympanic membrane mobility, Eustachian tube function, and mobility of the middle ear ossicles.</div></li></ul><p><b>Audiometry</b> subjectively determines how the individual processes auditory information (i.e., hears). Audiometry consists of behavioral testing and pure tone audiometry:</p><ul><li class="half_rhythm"><div><b>Behavioral testing</b> includes behavioral observation audiometry (BOA) and visual reinforcement audiometry (VRA). BOA is used in infants from birth to age six months, is highly dependent on the skill of the tester, and is subject to error. VRA is used in children from age six months to 2.5 years and can provide a reliable, complete audiogram, but is dependent on the child's maturational age and the skill of the tester.</div></li><li class="half_rhythm"><div><b>Pure-tone audiometry</b> (air and bone conduction) involves determination of the lowest intensity at which an individual "hears" a pure tone, as a function of frequency (or pitch). Octave frequencies from 250 (close to middle C) to 8000 Hz are tested using earphones. Intensity or loudness is measured in decibels (dB), defined as the ratio between two sound pressures. 0 dB HL is the average threshold for a normal hearing adult; 120 dB HL is so loud as to cause pain. Speech reception thresholds (SRTs) and speech discrimination are assessed.</div><ul><li class="half_rhythm"><div><b>Air conduction audiometry</b> presents sounds through earphones; thresholds depend on the condition of the external ear canal, middle ear, and inner ear.</div></li><li class="half_rhythm"><div><b>Bone conduction audiometry</b> presents sounds through a vibrator placed on the mastoid bone or forehead, thus bypassing the external and middle ears; thresholds depend on the condition of the inner ear.</div></li></ul></li><li class="half_rhythm"><div><b>Conditioned play audiometry</b> (CPA) is used to test children from age 2.5 to five years. A complete frequency-specific audiogram for each ear can be obtained from a cooperative child.</div></li><li class="half_rhythm"><div><b>Conventional audiometry</b> is used to test individuals age five years and older; the individual indicates when the sound is heard.</div></li><li class="half_rhythm"><div><b>Audioprofile</b> refers to the recording of several audiograms on a single graph (<a class="figpopup" href="/books/NBK1434/figure/deafness-overview.F1/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewF1" rid-ob="figobdeafnessoverviewF1">Figure 1</a>). These audiograms may be from one individual at different times, but more frequently they are from different members of the same family segregating deafness usually in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> fashion. By plotting numerous audiograms with age on the same graph, the age-related progression of hearing loss can be appreciated within these families. Often the composite picture is characteristic of specific genetic causes of autosomal dominant nonsyndromic hearing loss. One of the most characteristic audioprofiles is associated with DFNA6/14/38 hearing loss caused by a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>WFS1</i>.</div></li></ul><div class="iconblock whole_rhythm clearfix ten_col fig" id="figdeafnessoverviewF1" co-legend-rid="figlgnddeafnessoverviewF1"><a href="/books/NBK1434/figure/deafness-overview.F1/?report=objectonly" target="object" title="Figure 1. " class="img_link icnblk_img figpopup" rid-figpopup="figdeafnessoverviewF1" rid-ob="figobdeafnessoverviewF1"><img class="small-thumb" src="/books/NBK1434/bin/deafness-overview-Image001.gif" src-large="/books/NBK1434/bin/deafness-overview-Image001.jpg" alt="Figure 1. " /></a><div class="icnblk_cntnt" id="figlgnddeafnessoverviewF1"><h4 id="deafness-overview.F1"><a href="/books/NBK1434/figure/deafness-overview.F1/?report=objectonly" target="object" rid-ob="figobdeafnessoverviewF1">Figure 1. </a></h4><p class="float-caption no_bottom_margin">Audioprofile </p></div></div><p><b>Other</b></p><ul><li class="half_rhythm"><div>Congenital hearing loss can be identified by newborn hearing screening (NBHS), which has been advocated by the National Institutes of Health. NBHS is universally required by law or rule in 43 states plus the District of Columbia. In the remaining states, newborn hearing screening is offered but not required. The result is that in the United States 95% of all newborns undergo newborn hearing screening [<a href="https://report.nih.gov/NIHfactsheets/ViewFactSheet.aspx?csid=104" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NIH <i>RePORT</i></a>].</div></li><li class="half_rhythm"><div>Parental concerns about possible hearing loss or observed delays in speech development require auditory screening in any child.</div></li></ul></div><div id="deafness-overview.Differential_Diagnosis"><h3>Differential Diagnosis of Hereditary Hearing Loss and Deafness</h3><p>In children with delayed speech development, the auditory system should be assessed.</p><p>In the presence of normal audiometry associated with progressive loss of speech and temporal lobe seizures, the diagnosis of Landau-Kleffner syndrome should be considered.</p><p>Delayed speech suggesting possible hearing loss can also be seen in young children with autism spectrum disorder or specific speech and language disorders.</p><p>In developed countries approximately 80% of <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> hearing loss is due to genetic causes and the remainder to environmental (acquired) causes (<a class="figpopup" href="/books/NBK1434/figure/deafness-overview.F2/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewF2" rid-ob="figobdeafnessoverviewF2">Figure 2</a>). Acquired causes should be differentiated from genetic causes to inform the evaluation and required ancillary testing (i.e., CT, MRI, and consultation with specialists) and to inform prognosis and treatment recommendations.</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figdeafnessoverviewF2" co-legend-rid="figlgnddeafnessoverviewF2"><a href="/books/NBK1434/figure/deafness-overview.F2/?report=objectonly" target="object" title="Figure 2. " class="img_link icnblk_img figpopup" rid-figpopup="figdeafnessoverviewF2" rid-ob="figobdeafnessoverviewF2"><img class="small-thumb" src="/books/NBK1434/bin/deafness-overview-Image002.gif" src-large="/books/NBK1434/bin/deafness-overview-Image002.jpg" alt="Figure 2. " /></a><div class="icnblk_cntnt" id="figlgnddeafnessoverviewF2"><h4 id="deafness-overview.F2"><a href="/books/NBK1434/figure/deafness-overview.F2/?report=objectonly" target="object" rid-ob="figobdeafnessoverviewF2">Figure 2. </a></h4><p class="float-caption no_bottom_margin">Causes of prelingual hearing loss in developed countries </p></div></div><p><b>Acquired hearing loss in children</b> commonly results from prenatal infections from "TORCH" organisms (i.e., <i>to</i>xoplasmosis, <i>r</i>ubella, <i>c</i>ytomegalovirus, and <i>h</i>erpes), or postnatal infections, particularly bacterial meningitis caused by <i>Neisseria meningitidis, Haemophilus influenzae,</i> or <i>Streptococcus pneumoniae</i>. Meningitis from many other organisms including <i>Escherichia coli, Listeria monocytogenes</i>, <i>Streptococcus agalactiae</i>, and <i>Enterobacter cloacae</i> can also cause hearing loss.</p><p>In developed countries, however, the most common environmental, non-genetic cause of <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> hearing loss is congenital cytomegalovirus (cCMV) infection. Its overall birth prevalence is approximately 0.64%; 10% of this number have symptomatic CMV, which is characterized by a variable number and degree of findings including neurologic deficits (death, seizures, cerebral palsy), hepatic insufficiency, and characteristic rash. Hearing loss affects approximately 50% of symptomatic individuals with cCMV. The remaining 90% of individuals with cCMV are considered "asymptomatic"; of these up to 15% develop unilateral or bilateral hearing loss. Thus, the majority of individuals with hearing loss due to cCMV are classified as "asymptomatic."</p><p>The diagnosis of CMV hearing loss can be difficult to make, often can go unrecognized, and is characterized by variable-severity bilateral, asymmetric, or unilateral sensorineural hearing loss [<a class="bk_pop" href="#deafness-overview.REF.kenneson.2007.253">Kenneson &#x00026; Cannon 2007</a>]. Testing for cCMV requires a high degree of suspicion and should be done within 21 days of birth given the ubiquity of the virus in the environment. Several states have introduced targeted testing for cCMV for newborns who fail their newborn hearing screen. Recognizing cCMV hearing loss is increasingly important given new studies that show improvement of hearing loss with antiviral therapy for persons with symptomatic CMV [<a class="bk_pop" href="#deafness-overview.REF.kimberlin.2015.933">Kimberlin et al 2015</a>]. To date, however, the use of antivirals to treat hearing loss in persons with cCMV whose only manifestation is hearing loss is experimental.</p><p><b>Acquired hearing loss in adults,</b> most often attributed to environmental factors, most likely reflects environmental-genetic interactions, the most frequent of which are age-related and noise-induced hearing loss. Although both of these types of hearing loss reflect complex "environmental-genetic" hearing loss, to date variants in only a few genes have been associated with these traits [<a class="bk_pop" href="#deafness-overview.REF.yamasoba.2013.30">Yamasoba et al 2013</a>].</p><p>An environmental interaction pertinent to medical care is the observation that aminoglycoside-induced hearing loss is more likely in persons with specific variants in the mitochondrial genome (mtDNA) (see <a href="/books/n/gene/mt-deafness/">Nonsyndromic Hearing Loss and Deafness, Mitochondrial</a>).</p></div></div><div id="deafness-overview.Causes_of_Hereditary_H"><h2 id="_deafness-overview_Causes_of_Hereditary_H_">Causes of Hereditary Hearing Loss and Deafness</h2><p>Hereditary hearing loss and deafness can be regarded as <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> or nonsyndromic (<a class="figpopup" href="/books/NBK1434/figure/deafness-overview.F2/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewF2" rid-ob="figobdeafnessoverviewF2">Figure 2</a>). Syndromic hearing impairment is associated with malformations of the external ear, with malformations in other organs, or with medical problems involving other organ systems. Nonsyndromic hearing impairment has no associated visible abnormalities of the external ear or any related medical problems; however, it can be associated with abnormalities of the middle ear and/or inner ear.</p><p>Approximately 80% of prelingual deafness is genetic, most often <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> and nonsyndromic. The most common cause of severe-to-profound autosomal recessive nonsyndromic hearing loss in most populations is mutation of <i>GJB2</i>. The most common cause of mild-to-moderate autosomal recessive hearing loss is mutation of <i>STRC</i>; of note, there is ethnic-based variability [<a class="bk_pop" href="#deafness-overview.REF.sloanheggen.2016.441">Sloan-Heggen et al 2016</a>].</p><div id="deafness-overview.Syndromic_Hearing_Impa"><h3>Syndromic Hearing Impairment</h3><p>More than 400 genetic syndromes that include hearing loss have been described [<a class="bk_pop" href="#deafness-overview.REF.toriello.2004">Toriello et al 2004</a>]. Although <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> hearing impairment accounts for up to 30% of prelingual deafness, its relative contribution to all deafness is much smaller, commensurate with the occurrence and diagnosis of postlingual hearing loss. Syndromic hearing loss discussed here is categorized by <a class="def" href="/books/n/gene/glossary/def-item/mode-of-inheritance/">mode of inheritance</a> (<a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.causes_of_syndromic/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTcausesofsyndromic" rid-ob="figobdeafnessoverviewTcausesofsyndromic">Table 3</a>).</p><div id="deafness-overview.T.causes_of_syndromic" class="table"><h3><span class="label">Table 3. </span></h3><div class="caption"><p>Causes of Syndromic Hearing Impairment</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1434/table/deafness-overview.T.causes_of_syndromic/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__deafness-overview.T.causes_of_syndromic_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_1" rowspan="2" scope="col" colspan="1" headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_1" style="text-align:left;vertical-align:middle;">MOI</th><th id="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" rowspan="2" scope="col" colspan="1" headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" style="text-align:left;vertical-align:middle;">Syndrome</th><th id="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="2" scope="col" colspan="1" headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" style="text-align:left;vertical-align:middle;">Gene(s)</th><th id="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4" colspan="3" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">Hearing Impairment</th><th id="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="2" scope="col" colspan="1" headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" style="text-align:left;vertical-align:middle;">Other Clinical Features</th><th id="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="2" scope="col" colspan="1" headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" style="text-align:left;vertical-align:middle;">Diagnosis/<br />Screening/Other</th></tr><tr><th headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4" id="hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" colspan="1" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">Type</th><th headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4" id="hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Onset</th><th headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4" id="hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severity</th></tr></thead><tbody><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_1" rowspan="4" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><b>AD</b></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/ws1/">Waardenburg syndrome</a> (WS)</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>PAX3</i><br /><i>MITF</i><br /><i>EDNRB</i><br /><i>EDN3</i><br /><i>SOX10</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">WS1: SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Congenital</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Variable</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Most common type of AD SHL</div></li><li class="half_rhythm"><div>Pigmentary abnormalities of skin, hair (white forelock&#x000a0;<sup>1</sup>), &#x00026; eyes (heterochromia iridis)</div></li><li class="half_rhythm"><div>Subtype characteristics:</div><ul><li class="half_rhythm"><div>WS1. Dystopia canthorum&#x000a0;<sup>2</sup></div></li><li class="half_rhythm"><div>WS2. Absence of dystopia canthorum; many other features shared w/WS1</div></li><li class="half_rhythm"><div>WS3. Upper-limb abnormalities</div></li><li class="half_rhythm"><div>WS4. Hirschsprung disease</div></li></ul></li></ul></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"></td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/bor/">Branchiootorenal spectrum disorders</a></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>EYA1</i><br /><i>SIX1</i><br /><i>SIX5</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Conductive;<br />SN; mixed</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Variable</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Variable</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>2nd most common type of AD SHL</div></li><li class="half_rhythm"><div>BOR&#x000a0;<sup>3</sup>: Branchial cleft cysts or fistulae, malformations of external ear incl preauricular pits, &#x00026; renal anomalies</div></li><li class="half_rhythm"><div>BOS&#x000a0;<sup>3</sup>: Same features as BOR syndrome but w/out renal involvement</div></li><li class="half_rhythm"><div>High <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a>; extremely <a class="def" href="/books/n/gene/glossary/def-item/variable-expressivity/">variable expressivity</a></div></li></ul></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"></td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/nf2/">Neurofibromatosis 2</a> (NF2)</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>NF2</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">~3rd decade</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Generally<br />unilateral<br />&#x00026; gradual;<br />can be<br />bilateral &#x00026;<br />sudden</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>HL secondary to bilateral vestibular schwannomas; a rare, potentially treatable type of deafness</div></li><li class="half_rhythm"><div>Risk for a variety of other tumors incl meningiomas, astrocytomas, ependymomas, &#x00026; meningioangiomatosis</div></li></ul></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">A retrocochlear lesion can often be diagnosed by audiologic evaluation, although definitive diagnosis requires MRI w/gadolinium contrast</td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" rowspan="2" colspan="1" scope="row" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/stickler/">Stickler syndrome</a></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"><i>COL2A1 COL11A1 COL11A2 COL9A1 COL9A2 COL9A3</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Conductive;<br />SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Variable</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Variable</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Connective tissue disorder that can include myopia, cataract, &#x00026; retinal detachment</div></li><li class="half_rhythm"><div>Midfacial underdevelopment &#x00026; cleft palate (either alone or as part of Robin sequence)</div></li><li class="half_rhythm"><div>Mild spondyloepiphyseal dysplasia &#x00026;/or precocious arthritis</div></li></ul></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"></td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_1" rowspan="9" colspan="1" style="text-align:left;vertical-align:middle;"><b>AR</b></td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/usher1/">Usher syndrome type I</a></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYO7A</i><br /><i>USH1C</i><br /><i>CDH23</i><br /><i>PCDH15</i><br /><i>USH1G</i><br /><i>CIB2</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="3" colspan="1" style="text-align:left;vertical-align:middle;">SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="3" colspan="1" style="text-align:left;vertical-align:middle;">Congenital</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Abnormal vestibular function</div></li><li class="half_rhythm"><div>Affected persons find traditional amplification ineffective &#x00026; usually communicate manually</div></li><li class="half_rhythm"><div>Due to vestibular deficit, developmental motor milestones for sitting &#x00026; walking always reached at later-than-normal ages</div></li></ul></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="3" colspan="1" style="text-align:left;vertical-align:middle;">Most common type of AR SHL<br />Dual sensory impairments: <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals are born w/SHL, then develop RP<br />Affects more than 50% of the deaf-blind in the U.S.</td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/usher2/">Usher syndrome type II</a></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ADGRV1 WHRN</i><br /><i>USH2A</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" colspan="1" rowspan="1" style="text-align:left;vertical-align:middle;">Mild to severe</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Normal vestibular function</div></li><li class="half_rhythm"><div>Hearing aids provide effective amplification; communication usually oral</div></li></ul></td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Usher syndrome type III<br />(OMIM <a href="http://omim.org/entry/276902" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">276902</a>, <a href="http://omim.org/entry/614504" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">614504</a>)</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CLRN1</i><br /><i>HARS1</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" colspan="1" rowspan="1" style="text-align:left;vertical-align:middle;">Progressive</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Progressive deterioration of vestibular function</td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/pendred/">Pendred syndrome</a></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SLC26A4</i>&#x000a0;<sup>4,&#x000a0;5</sup></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Congenital</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Usually<br />(but not<br />invariably)<br />severe to<br />profound</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>2nd most common type of AR SHL</div></li><li class="half_rhythm"><div>Hearing &#x00026; euthyroid goiter</div></li><li class="half_rhythm"><div>Deafness assocd w/an abnormality of the bony labyrinth (Mondini dysplasia or dilated [enlarged] vestibular aqueduct)</div></li><li class="half_rhythm"><div>Goiter not present at birth; develops in early puberty (40%) or adulthood (60%)</div></li></ul></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Mondini dysplasia or dilated vestibular aqueduct can be diagnosed by CT examination of temporal bones</td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/jln/">Jervell and Lange-Nielsen syndrome</a></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>KCNQ1</i><br /><i>KCNE1</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Congenital</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Profound</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>3rd most common type of AR HL</div></li><li class="half_rhythm"><div>Deafness &#x00026; prolongation of QT interval as detected by ECG (abnormal QTc [c=corrected] &#x0003e;440 msec)</div></li><li class="half_rhythm"><div>Syncopal episodes; sudden death</div></li></ul></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Although a screening ECG is not highly sensitive, it may be suitable for screening deaf children. High-risk children (i.e., those w/family history positive for sudden death, SIDS, syncopal episodes, or long QT syndrome) should have a thorough cardiac evaluation.</td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/biotin/">Biotinidase deficiency</a></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>BTD</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Variable</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Variable</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>If not recognized &#x00026; corrected by daily addition of biotin to diet, <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> persons develop neurologic features (e.g., seizures, hypertonia, developmental delay, ataxia) &#x00026; visual problems.</div></li><li class="half_rhythm"><div>Some degree of HL is present in &#x02265;75% of children who become symptomatic.</div></li><li class="half_rhythm"><div>Cutaneous features (e.g., skin rash, alopecia, conjunctivitis)</div></li></ul></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Neurologic &#x00026; cutaneous manifestations resolve w/biotin treatment; hearing loss &#x00026; optic atrophy are usually irreversible<br />When a child presents w/episodic or progressive ataxia &#x00026; progressive sensorineural deafness &#x000b1; neurologic or cutaneous symptoms, consider biotinidase deficiency.<br />To prevent metabolic coma, initiate diet &#x00026; treatment ASAP.</td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><a href="https://www.ncbi.nlm.nih.gov/books/n/gene/refsum/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Refsum disease</a></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>PHYH</i><br /><i>PEX7</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Variable</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe;<br />progressive</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Anosmia &#x00026; early-onset retinitis pigmentosa &#x02013; both universal findings w/variable combinations of neuropathy, deafness, ataxia, &#x00026; ichthyosis</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Although it is extremely rare, consider Refsum disease in evaluation of a deaf person as it can be treated w/dietary modification &#x00026; plasmapharesis.<br />Diagnosis is established by determining serum concentration of phytanic acid&#x000a0;<sup>6</sup>.</td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" rowspan="2" colspan="1" scope="row" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/alport/">Alport syndrome</a>&#x000a0;<sup>7</sup></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"><i>COL4A5</i><br /><i>COL4A3</i>&#x000a0;<sup>7</sup><br /><i>COL4A4</i>&#x000a0;<sup>7</sup></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Typically after age 10 yrs</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Varying<br />severity;<br />progressive</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Renal, cochlear, ocular involvement</div></li><li class="half_rhythm"><div>W/out treatment, renal disease progresses from microscopic hematuria to proteinuria, progressive renal insufficiency, ESRD.</div></li></ul></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"></td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_1" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"><b>XL</b></td></tr><tr><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/ddon/">Deafness-dystonia-optic neuronopathy syndrome</a><br />(Mohr-Tranebjaerg syndrome)</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TIMM8A</i></td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">SN</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Early childhood</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_4 hd_h_deafness-overview.T.causes_of_syndromic_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Progressive;<br />pre- or<br />postlingual</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Visual disability, dystonia, fractures, intellectual disability</td><td headers="hd_h_deafness-overview.T.causes_of_syndromic_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">AD = <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a>; AR = <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a>; ESRD = end-stage renal disease; HL = hearing loss; MOI = <a class="def" href="/books/n/gene/glossary/def-item/mode-of-inheritance/">mode of inheritance</a>; RP = retinitis pigmentosa; SHL = <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> hearing loss; SN = sensorineural; XL = <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a></p></div></dd><dt>1. </dt><dd><div id="deafness-overview.TF.3.1"><p class="no_margin">Because <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> persons may dye their hair, the presence of a white forelock should be specifically sought in the history and physical examination.</p></div></dd><dt>2. </dt><dd><div id="deafness-overview.TF.3.2"><p class="no_margin">Dystopia canthorum: lateral displacement of the inner canthus of the eye</p></div></dd><dt>3. </dt><dd><div id="deafness-overview.TF.3.3"><p class="no_margin">Branchiootorenal spectrum disorders comprise branchiootorenal (BOR) syndrome and branchiootic syndrome (BOS).</p></div></dd><dt>4. </dt><dd><div id="deafness-overview.TF.3.4"><p class="no_margin">Digenic inheritance, in which an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individual has double heterozygosity for a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>SLC26A4</i> and a pathogenic variant in <i>FOXI1</i> [<a class="bk_pop" href="#deafness-overview.REF.yang.2007.1055">Yang et al 2007</a>] or double heterozygosity for a pathogenic variant in <i>SLC26A4</i> and a pathogenic variant in KCNJ10, has also been observed in Pendred syndrome.</p></div></dd><dt>5. </dt><dd><div id="deafness-overview.TF.3.5"><p class="no_margin">Mutation of <i>SLC26A4</i> is also associated with nonsyndromic hearing loss (DFNB4).</p></div></dd><dt>6. </dt><dd><div id="deafness-overview.TF.3.6"><p class="no_margin">See also <a href="/books/n/gene/pbd/">Peroxisome Biogenesis Disorders, Zellweger Syndrome Spectrum</a>.</p></div></dd><dt>7. </dt><dd><div id="deafness-overview.TF.3.7"><p class="no_margin"><a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> inheritance accounts for approximately 85% of <a href="/books/n/gene/alport/">Alport syndrome</a>; <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> inheritance accounts for approximately 15% of cases; <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> inheritance has also been reported on occasion.</p></div></dd></dl></div></div></div><div id="deafness-overview.Mitochondrial_Syndromi"><h4>Mitochondrial Syndromic Hearing Impairment</h4><p><b>Mitochondrial DNA pathogenic variants</b> have been implicated in a variety of diseases ranging from rare neuromuscular syndromes such as Kearns-Sayre syndrome (see <a href="/books/n/gene/kss/">Mitochondrial DNA Deletion Syndromes</a>), <a href="/books/n/gene/melas/">MELAS</a>, <a href="/books/n/gene/merrf/">MERRF</a>, and <a href="/books/n/gene/narp/">NARP</a> (see <a href="/books/n/gene/mt-overview/">Mitochondrial Disorders Overview</a>), to common conditions such as diabetes mellitus, <a href="/books/n/gene/parkinson-overview/">Parkinson disease</a>, and <a href="/books/n/gene/alzheimer/">Alzheimer disease</a>. One <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>, the 3243 A-to-G transition in <i>MTTL1,</i> has been found in 2%-6% of individuals with diabetes mellitus in Japan; 61% of persons with diabetes mellitus and this pathogenic variant have hearing loss. The hearing loss is sensorineural and develops only after the onset of the diabetes mellitus. The same pathogenic variant is associated with MELAS, raising questions of <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> and tissue specificity &#x02013; issues further confounded by <a class="def" href="/books/n/gene/glossary/def-item/heteroplasmy/">heteroplasmy</a>.</p></div></div><div id="deafness-overview.Nonsyndromic_Hearing_I"><h3>Nonsyndromic Hearing Impairment</h3><p><b>Nomenclature.</b> For historical reasons, nonsyndromic hearing impairment may be referred to by the <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> involved (e.g., <i>OTOF</i>-related deafness) or by the genetic <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> (e.g., DFNB9). Nonsyndromic deafness loci are designated DFN (for DeaFNess) and further classified by <a class="def" href="/books/n/gene/glossary/def-item/mode-of-inheritance/">mode of inheritance</a> (DFN<b>A</b>: <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a>; DFN<b>B</b>: <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a>; DFN<b>X</b>: <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a>) and a number indicating the order of gene mapping and/or discovery).</p><p><b>Inheritance.</b> The inheritance pattern among the disorders with prelingual nonsyndromic hearing loss is 80% <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a>, 20% <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a>, and 1%-1.5% <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a>, mitochondrial, or other (<a class="figpopup" href="/books/NBK1434/figure/deafness-overview.F2/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewF2" rid-ob="figobdeafnessoverviewF2">Figure 2</a>) [<a class="bk_pop" href="#deafness-overview.REF.smith.2005.879">Smith et al 2005</a>]. Although similar data are not available for the disorders with postlingual nonsyndromic hearing impairment, most reported families demonstrate autosomal dominant inheritance.</p><p><b>Genetic heterogeneity.</b> Nonsyndromic hereditary hearing loss is characterized by extreme genetic heterogeneity: to date, more than 6,000 causative variants have been identified in more than 110 genes. In the largest study to date in which a <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a> was used for comprehensive genetic testing, more than 40 causative genes were identified among the 440 individuals in whom a genetic diagnosis was established [<a class="bk_pop" href="#deafness-overview.REF.sloanheggen.2016.441">Sloan-Heggen et al 2016</a>]. This extreme genetic heterogeneity underscores the importance of use of multigene sequencing panels for genetic diagnosis (see <a href="#deafness-overview.Evaluation_Strategy">Evaluation Strategy</a>).</p><div id="deafness-overview.Autosomal_Dominant_Non"><h4>Autosomal Dominant Nonsyndromic Hearing Impairment</h4><p>More than 25 genes have been associated with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> nonsyndromic hearing loss. The genes implicated in autosomal dominant nonsyndromic hearing impairment and their clinical manifestations are summarized in <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.autosomal_dominant_n/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTautosomaldominantn" rid-ob="figobdeafnessoverviewTautosomaldominantn">Table 4</a>. Note that the audioprofile can be distinctive and therefore guide <a class="def" href="/books/n/gene/glossary/def-item/genotype-phenotype-correlations/">genotype-phenotype correlations</a> [<a class="bk_pop" href="#deafness-overview.REF.taylor.2013.539">Taylor et al 2013</a>] (see <a href="#deafness-overview.Evaluation_Strategy">Evaluation Strategy</a>).</p><p>Most <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> loci cause postlingual hearing impairment.</p><ul><li class="half_rhythm"><div>Exceptions in which hearing loss is prelingual include <i>GJB2</i> and <i>GJB6</i> (DFNA3), <i>TECTA</i> (DFNA8/12), and DFNA19.</div></li><li class="half_rhythm"><div><i>WFS1</i> (DFNA6/14/38) is noteworthy as the hearing loss primarily affects the low frequencies.</div></li></ul><div id="deafness-overview.T.autosomal_dominant_n" class="table"><h3><span class="label">Table 4. </span></h3><div class="caption"><p>Autosomal Dominant Nonsyndromic Hearing Impairment: Genes and Their Clinical Manifestations</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1434/table/deafness-overview.T.autosomal_dominant_n/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__deafness-overview.T.autosomal_dominant_n_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene</th><th id="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Locus</th><th id="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Onset/Decade</th><th id="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Audioprofile</th></tr></thead><tbody><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ACTG1</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA20/26</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CCDC50</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA440</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Low to mild frequencies; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CD164</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA66</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat or mid-frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CEACAM16</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA4B</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>COCH</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA9</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/2nd</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>COL11A2</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA13</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/2nd</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Mid-frequency loss</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>GSDME</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA5</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/1st</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>DIAPH1</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA1</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/1st</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Low frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>DMXL2</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">-</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/2nd</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>DSPP</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA39</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>EYA4</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA10</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/3rd, 4th</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat/gently downsloping</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/dfna3/"><i>GJB2</i></a>&#x000a0;<sup>1</sup></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA3</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>GJB3</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA2B</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/4th</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/dfna3/"><i>GJB6</i></a>&#x000a0;<sup>1</sup></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA3</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>GRHL2</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA28</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat/gently downsloping</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>HOMER2</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA68</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/1st</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/dfna2/"><i>KCNQ4</i></a></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA2</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/2nd</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MIR96</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA50</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/2nd</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MCM2</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA70</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYH14</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA4</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat/gently downsloping</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYH9</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA17</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYO1A</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA48</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYO6</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA22</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYO7A</i>&#x000a0;<sup>2</sup></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA11</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/1st</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat/gently downsloping</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>OSBPL2</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA67</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>P2RX2</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA41</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>POU4F3</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA15</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SIX1</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA23</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Downsloping</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SLC17A8</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA25</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/2nd-6th</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TBC1D24</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA65</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TECTA</i>&#x000a0;<sup>3,&#x000a0;4</sup></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA8/12</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Mid-frequency loss</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TJP2 &#x00026; FAM189A2</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA51</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual/4th</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TMC1</i></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA36</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Flat/gently downsloping</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/wfs/"><i>WFS1</i></a>&#x000a0;<sup>5</sup></td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNA6/14/38</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_dominant_n_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Low frequency; progressive</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">From <a href="http://hereditaryhearingloss.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">hereditaryhearingloss.org</a> [<a class="bk_pop" href="#deafness-overview.REF.van_camp.2017">Van Camp &#x00026; Smith 2017</a>]</p></div></dd><dt></dt><dd><div><p class="no_margin">See <a href="http://omim.org/phenotypicSeries/PS124900" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Deafness, Autosomal Dominant: Phenotypic Series</a> to view genes associated with this <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> in OMIM.</p></div></dd><dt>1. </dt><dd><div id="deafness-overview.TF.4.1"><p class="no_margin"><i>GJB2</i> and <i>GJB6</i> pathogenic variants are also associated with DFNB1 (<a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic hearing impairment).</p></div></dd><dt>2. </dt><dd><div id="deafness-overview.TF.4.2"><p class="no_margin"><i>MYO7A</i> pathogenic variants are also associated with DFNB2 (<a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic hearing impairment) and Usher syndrome 1B.</p></div></dd><dt>3. </dt><dd><div id="deafness-overview.TF.4.3"><p class="no_margin"><i>TECTA</i> pathogenic variants are also associated with DFNB21 (<a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic hearing impairment).</p></div></dd><dt>4. </dt><dd><div id="deafness-overview.TF.4.4"><p class="no_margin">In DFNA8/12, the pathogenic variants in <i>TECTA</i> are <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants, with the audioprofile dependent on the location of the variant. Missense variants in the ZP <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a> cause stable or progressive hearing loss involving the mid frequencies, while missense variants in the ZA domain result in progressive hearing loss in the high frequencies.</p></div></dd><dt>5. </dt><dd><div id="deafness-overview.TF.4.5"><p class="no_margin"><i>WFS1</i> pathogenic variants are also associated with Wolfram syndrome (see <a href="/books/n/gene/wfs/"><i>WFS1</i>-Related Disorders</a>).</p></div></dd></dl></div></div></div></div><div id="deafness-overview.Autosomal_Recessive_No"><h4>Autosomal Recessive Nonsyndromic Hearing Impairment</h4><p>Large studies have underscored the extreme genetic heterogeneity of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic deafness. The 70 genes implicated in autosomal recessive nonsyndromic hearing impairment and their clinical manifestations are summarized in <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.autosomal_recessive/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTautosomalrecessive" rid-ob="figobdeafnessoverviewTautosomalrecessive">Table 5</a>.</p><p>While it is true that in some world populations, up to 50% of persons with severe-to-profound <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic hearing loss have pathogenic variants in <i>GJB2</i> (see <a href="/books/n/gene/dfnb1/">DFNB1</a>), recent studies have shown that the contribution of pathogenic variants in <i>GJB2</i> to deafness varies considerably by ethnicity [<a class="bk_pop" href="#deafness-overview.REF.sloanheggen.2016.441">Sloan-Heggen et al 2016</a>]. For example, among individuals of African descent, pathogenic variants in <i>GJB2</i> are very rare [<a class="bk_pop" href="#deafness-overview.REF.rudman.2017.285">Rudman et al 2017</a>].</p><div id="deafness-overview.T.autosomal_recessive" class="table"><h3><span class="label">Table 5. </span></h3><div class="caption"><p>Autosomal Recessive Nonsyndromic Hearing Impairment: Genes and Their Clinical Manifestations</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1434/table/deafness-overview.T.autosomal_recessive/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__deafness-overview.T.autosomal_recessive_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene</th><th id="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Locus</th><th id="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Onset</th><th id="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Type</th></tr></thead><tbody><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ADCY1</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB44</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Mild to moderate; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>BDP1</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB49</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>BSND</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB73</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CABP2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB93</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate to severe; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CDC14A</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB105</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CDH23</i>&#x000a0;<sup>1</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB12</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CIB2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB48</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CLDN14</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB29</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CLIC5</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB103</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>COL11A2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB53</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>DCDC2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB66</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>PJVK</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB59</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ELMOD3</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB88</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; mixed</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>EPS8</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB102</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>EPS8L2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">-</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ESPN</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB36</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">&#x02014;</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ESRRB</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB35</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Unknown</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>GIPC3</i>&#x000a0;<sup>2</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB15/72/95</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/dfna3/"><i>GJB2</i></a>&#x000a0;<sup>3</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB1</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual&#x000a0;<sup>4</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Usually stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/dfna3/"><i>GJB6</i></a>&#x000a0;<sup>3</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB1</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual&#x000a0;<sup>4</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Usually stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>GPSM2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB32/82</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>GRXCR1</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB25</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate to profound; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>GRXCR2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB101</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>HGF</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB39</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; downsloping</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ILDR1</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB42</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate to severe</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>KARS1</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB89</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate to severe; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>LHFPL5</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB67</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>LOXHD1</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB77</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate to profound; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>LRTOMT</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB63</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MARVELD2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB49</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MET</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB97</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MSRB3</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB74</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYO15A</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB3</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYO3A</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB30</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYO6</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB37</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">&#x02014;</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MYO7A</i>&#x000a0;<sup>5</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB2</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual, postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Unspecified</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>NARS2</i>&#x000a0;<sup>6</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB94</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>OTOG</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB18B</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Mild to moderate; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>OTOGL</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB84</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>OTOA</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB22</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/dfnb9/"><i>OTOF</i></a></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB9</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Usually severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>PCDH15</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB23</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>PNPT1</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB70</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>PTPRQ</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB84</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate to profound; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>RDX</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB24</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>RIPOR2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB104</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ROR1</i>&#x000a0;<sup>7</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">-</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>S1PR2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB68</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SERPINB6</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB91</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate to severe</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SLC22A4</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB60</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SLC26A4</i>&#x000a0;<sup>8</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB4</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual, postlingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Stable; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SLC26A5</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB61</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>STRC</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB16</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SYNE4</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB76</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TECTA</i>&#x000a0;<sup>9,&#x000a0;10</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB21</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TBC1D24</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB86</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TMC1</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB7/11</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TMEM132E</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB99</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TMIE</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB6</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TMPRSS3</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB8/10</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual&#x000a0;<sup>11</sup>, prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Progressive; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TPRN</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB79</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TRIOBP</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB28</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TSPEAR</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB98</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>USH1C</i>&#x000a0;<sup>12</sup></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB18</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound; stable</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>WBP2</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">-</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">High frequency; progressive</td></tr><tr><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>WHRN</i></td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNB31</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.autosomal_recessive_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">&#x02014;</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">From <a href="http://hereditaryhearingloss.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">hereditaryhearingloss.org</a> [<a class="bk_pop" href="#deafness-overview.REF.van_camp.2017">Van Camp &#x00026; Smith 2017</a>]</p></div></dd><dt></dt><dd><div><p class="no_margin">See <a href="http://omim.org/phenotypicSeries/PS220290" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Deafness, Autosomal Recessive: Phenotypic Series</a> to view genes associated with this <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> in OMIM.</p></div></dd><dt>1. </dt><dd><div id="deafness-overview.TF.5.1"><p class="no_margin"><i>CDH23</i> pathogenic variants are also associated with <a href="/books/n/gene/usher1/">Usher syndrome type 1D</a>.</p></div></dd><dt>2. </dt><dd><div id="deafness-overview.TF.5.2"><p class="no_margin"><i>GIPC3</i> pathogenic variants are associated with audiogenic seizures.</p></div></dd><dt>3. </dt><dd><div id="deafness-overview.TF.5.3"><p class="no_margin"><i>GJB2</i> and <i>GJB6</i> pathogenic variants are also associated with DFNA3 (<a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> nonsyndromic hearing impairment).</p></div></dd><dt>4. </dt><dd><div id="deafness-overview.TF.5.4"><p class="no_margin">Prelingual deafness also includes <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> deafness.</p></div></dd><dt>5. </dt><dd><div id="deafness-overview.TF.5.5"><p class="no_margin"><i>MYO7A</i> pathogenic variants are also associated with DFNA11 (<a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> nonsyndromic hearing impairment) and <a href="/books/n/gene/usher1/">Usher syndrome 1B</a>.</p></div></dd><dt>6. </dt><dd><div id="deafness-overview.TF.5.6"><p class="no_margin"><i>NARS2</i> pathogenic variants also cause Leigh syndrome.</p></div></dd><dt>7. </dt><dd><div id="deafness-overview.TF.5.7"><p class="no_margin"><i>ROR1</i> pathogenic variants cause recessive nonsyndromic hearing loss associated with common cavity inner ear malformations and auditory neuropathy.</p></div></dd><dt>8. </dt><dd><div id="deafness-overview.TF.5.8"><p class="no_margin"><i>SLC26A4</i> pathogenic variants are also associated with Pendred syndrome.</p></div></dd><dt>9. </dt><dd><div id="deafness-overview.TF.5.9"><p class="no_margin"><i>TECTA</i> pathogenic variants are also associated with DFNA8/12 (<a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> nonsyndromic hearing impairment).</p></div></dd><dt>10. </dt><dd><div id="deafness-overview.TF.5.10"><p class="no_margin">In DFNB21, the pathogenic variants in <i>TECTA</i> result in premature protein truncation and act like <a class="def" href="/books/n/gene/glossary/def-item/null/">null</a> alleles. Examples include frameshift variants, <a class="def" href="/books/n/gene/glossary/def-item/nonsense-variant/">nonsense</a> variants, and deletions. In all cases, the hearing loss is prelingual, symmetric, and moderate to severe in degree.</p></div></dd><dt>11. </dt><dd><div id="deafness-overview.TF.5.11"><p class="no_margin">The onset of DFNB8 hearing loss is postlingual (age 10-12 years), while the onset of DFNB10 hearing loss is prelingual (<a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a>). This phenotypic difference reflects a genotypic difference: the DFNB8-causing variant is a <a class="def" href="/books/n/gene/glossary/def-item/splice-site/">splice site</a> variant, suggesting that inefficient <a class="def" href="/books/n/gene/glossary/def-item/splicing/">splicing</a> is associated with a reduced amount of normal protein that is sufficient to prevent prelingual deafness but not sufficient to prevent eventual hearing loss.</p></div></dd><dt>12. </dt><dd><div id="deafness-overview.TF.5.12"><p class="no_margin"><i>USH1C</i> pathogenic variants are also associated with <a href="/books/n/gene/usher1/">Usher syndrome type 1C</a>.</p></div></dd></dl></div></div></div><p><b>X-Linked Nonsyndromic Hearing Impairment</b></p><p>The genes implicated in <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> nonsyndromic hearing impairment and their clinical manifestations are summarized in <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.xlinked_nonsyndromic/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTxlinkednonsyndromic" rid-ob="figobdeafnessoverviewTxlinkednonsyndromic">Table 6</a>.</p><p><a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> nonsyndromic hearing loss can be either pre- or postlingual; one disorder, DFNX3, has mixed hearing loss.</p><div id="deafness-overview.T.xlinked_nonsyndromic" class="table"><h3><span class="label">Table 6. </span></h3><div class="caption"><p><a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> Nonsyndromic Hearing Impairment: Genes and Their Clinical Manifestations</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1434/table/deafness-overview.T.xlinked_nonsyndromic/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__deafness-overview.T.xlinked_nonsyndromic_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene</th><th id="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Locus</th><th id="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Onset</th><th id="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Type / Degree</th><th id="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_5" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Frequencies</th></tr></thead><tbody><tr><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/dfnx1/"><i>PRPS1</i></a></td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNX1</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Progressive sensorineural / severe to profound</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">All</td></tr><tr><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>POU3F4</i></td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNX2</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Progressive, mixed / variable, but progresses to profound</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">All</td></tr><tr><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SMPX</i></td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNX4</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Progressive sensorineural / mild to profound</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">All</td></tr><tr><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>AIFM1</i></td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNX5</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Progressive auditory neuropathy</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Low frequency</td></tr><tr><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>COL4A6</i></td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DFNX6</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Prelingual</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Progressive sensorineural / severe to profound</td><td headers="hd_h_deafness-overview.T.xlinked_nonsyndromic_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">All</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">From <a href="http://hereditaryhearingloss.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">hereditaryhearingloss.org</a> [<a class="bk_pop" href="#deafness-overview.REF.van_camp.2017">Van Camp &#x00026; Smith 2017</a>]</p></div></dd><dt></dt><dd><div><p class="no_margin">See <a href="http://omim.org/phenotypicSeries/PS304500" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Deafness, X-linked: Phenotypic Series</a> to view genes associated with this <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> in OMIM.</p></div></dd></dl></div></div></div></div><div id="deafness-overview.Nonsyndromic_Hearing_L"><h4>Nonsyndromic Hearing Loss and Deafness, Mitochondrial</h4><p>The majority of pathogenic variants in mitochondrial genes cause a broad spectrum of maternally inherited multisystem disorders; however, variants in a subset of genes, mainly <i>MT-RNR1</i> and <i>MT-TS1</i>, cause nonsyndromic hearing loss by a currently unknown mechanism [<a class="bk_pop" href="#deafness-overview.REF.fischelghodsian.1998.15">Fischel-Ghodsian 1998</a>] (see <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.mitochondrial_nonsyn/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTmitochondrialnonsyn" rid-ob="figobdeafnessoverviewTmitochondrialnonsyn">Table 7</a> and <a href="/books/n/gene/mt-deafness/">Nonsyndromic Hearing Loss and Deafness, Mitochondrial</a>).</p><p><i>MT-RNR1</i> encodes for the 12S ribosomal RNA. One variant in this <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>, 1555G&#x0003e;A, is a frequent cause of maternally inherited nonsyndromic hearing loss. In some individuals with the 1555G&#x0003e;A variant, the hearing loss is induced by the administration of appropriate doses of aminoglycosides; however, phenotypic variation is great, consistent with the effect of modifier genes [<a class="bk_pop" href="#deafness-overview.REF.kokotas.2007.379">Kokotas et al 2007</a>].</p><p><i>MT-TS1</i> encodes for the transfer RNA<sup>Ser(UCN)</sup>. Two families with <a class="def" href="/books/n/gene/glossary/def-item/heteroplasmy/">heteroplasmy</a> for an A-to-G transition at nt7445 of this <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> have been identified; however, <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> of hearing loss was low, and it has been suggested that <i>MT-TS1</i> pathogenic variants on their own play an insignificant role in hearing loss.</p><p><i>MT-CO1</i> encodes cytochrome c oxidase subunit 1. Six individuals with severe-to-profound deafness showed cosegregation of a <a class="def" href="/books/n/gene/glossary/def-item/homoplasmic/">homoplasmic</a> G-to-A transition at nt7444 of <i>MT-CO1</i> and the 1555A&#x0003e;G <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>MT-RNR1</i> [<a class="bk_pop" href="#deafness-overview.REF.pandya.1999.1803">Pandya et al 1999</a>]. Five of the six individuals showed maternal inheritance and two had a previous history of aminoglycoside use. As opposed to the variable hearing loss associated with <i>MT-RNR1</i> 1555A&#x0003e;G, all individuals with this double variant showed severe-to-profound impairment and <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> was complete.</p><div id="deafness-overview.T.mitochondrial_nonsyn" class="table"><h3><span class="label">Table 7. </span></h3><div class="caption"><p>Mitochondrial Nonsyndromic Hearing Impairment: Genes and Their Clinical Manifestations</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1434/table/deafness-overview.T.mitochondrial_nonsyn/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__deafness-overview.T.mitochondrial_nonsyn_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene</th><th id="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Pathogenic Variant</th><th id="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severity</th><th id="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Penetrance</th></tr></thead><tbody><tr><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_1" rowspan="3" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><i>MT-RNR1</i></td><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">961 different variants</td><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_3" rowspan="7" colspan="1" style="text-align:left;vertical-align:middle;">Variable</td><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_4" rowspan="3" colspan="1" style="text-align:left;vertical-align:middle;">Highly variable, aminoglycoside induced</td></tr><tr><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_2" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;">1494C&#x0003e;T</td></tr><tr><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_2" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;">1555A&#x0003e;G</td></tr><tr><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_1" rowspan="4" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><i>MT-TS1</i></td><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">7445A&#x0003e;G</td><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_4" rowspan="4" colspan="1" style="text-align:left;vertical-align:middle;">Highly variable</td></tr><tr><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_2" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;">7472insC</td></tr><tr><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_2" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;">7510T&#x0003e;C</td></tr><tr><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_2" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;">7511T</td></tr><tr><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>MT-CO1</i></td><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">7444G&#x0003e;A</td><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe to profound</td><td headers="hd_h_deafness-overview.T.mitochondrial_nonsyn_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Complete, aminoglycoside associated; associated with <i>MT-RNR1</i> 1555A&#x0003e;G</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">From <a href="http://hereditaryhearingloss.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">hereditaryhearingloss.org</a> [<a class="bk_pop" href="#deafness-overview.REF.van_camp.2017">Van Camp &#x00026; Smith 2017</a>]</p></div></dd></dl></div></div></div></div></div></div><div id="deafness-overview.Evaluation_Strategy"><h2 id="_deafness-overview_Evaluation_Strategy_">Evaluation Strategy</h2><p>Recent guidelines have underscored the importance of genetic testing in the evaluation of patients with hearing loss. See <a class="bk_pop" href="#deafness-overview.REF.alford.2014">Alford et al [2014]</a> (<a href="http://www.nature.com/gim/journal/v16/n4/full/gim20142a.html?foxtrotcallback=true" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">full text</a>), <a class="bk_pop" href="#deafness-overview.REF.liming.2016.251">Liming et al [2016]</a> (<a href="http://www.sciencedirect.com/science/article/pii/S0165587616303160?via%3Dihub" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">full text</a>), and <a class="figpopup" href="/books/NBK1434/figure/deafness-overview.F3/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewF3" rid-ob="figobdeafnessoverviewF3">Figure 3</a>.</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figdeafnessoverviewF3" co-legend-rid="figlgnddeafnessoverviewF3"><a href="/books/NBK1434/figure/deafness-overview.F3/?report=objectonly" target="object" title="Figure 3. " class="img_link icnblk_img figpopup" rid-figpopup="figdeafnessoverviewF3" rid-ob="figobdeafnessoverviewF3"><img class="small-thumb" src="/books/NBK1434/bin/deafness-overview-Image003.gif" src-large="/books/NBK1434/bin/deafness-overview-Image003.jpg" alt="Figure 3. " /></a><div class="icnblk_cntnt" id="figlgnddeafnessoverviewF3"><h4 id="deafness-overview.F3"><a href="/books/NBK1434/figure/deafness-overview.F3/?report=objectonly" target="object" rid-ob="figobdeafnessoverviewF3">Figure 3. </a></h4><p class="float-caption no_bottom_margin">Diagnostic algorithm for presumed hereditary nonsyndromic sensorineural hearing loss </p></div></div><p>Because <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> is the single type of test with the highest diagnostic rate, it should be offered first in evaluation of individuals with presumed hereditary sensorineural hearing loss and deafness unless past medical history, physical examination, and/or audiometric testing indicates a specific <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> form of hearing loss (<a class="figpopup" href="/books/NBK1434/figure/deafness-overview.F3/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewF3" rid-ob="figobdeafnessoverviewF3">Figure 3</a> and <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.causes_of_syndromic/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTcausesofsyndromic" rid-ob="figobdeafnessoverviewTcausesofsyndromic">Table 3</a>).</p><p><b>Initial evaluation</b> should always include the following:</p><ul><li class="half_rhythm"><div><b>Family history.</b> Effort should be made to obtain a three-generation family history with attention to other relatives with hearing loss and associated findings. Documentation of relevant findings in relatives can be accomplished either through direct examination of those individuals or through review of their medical records, including audiograms, otologic examinations, and <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a>.</div></li><li class="half_rhythm"><div><b>Clinical examination.</b> All persons with hearing loss of unknown cause should be evaluated for features associated with <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> deafness (see <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.causes_of_syndromic/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTcausesofsyndromic" rid-ob="figobdeafnessoverviewTcausesofsyndromic">Table 3</a>). Because <a class="def" href="/books/n/gene/glossary/def-item/variable-expressivity/">variable expressivity</a> is common among <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> syndromes with deafness, the correct diagnosis may depend on <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a>.</div></li><li class="half_rhythm"><div><b>Audiometric testing.</b> Hearing status can be determined at any age (see <a href="#deafness-overview.Diagnosing_Deafness_an">Diagnosing Deafness and Hearing Loss</a>).</div><ul><li class="half_rhythm"><div>Individuals with progressive hearing loss should be evaluated for <a href="/books/n/gene/alport/">Alport syndrome</a>, <a href="/books/n/gene/pendred/">Pendred syndrome</a>, and <a href="/books/n/gene/stickler/">Stickler syndrome</a>. In addition to genetic testing, temporal bone-computed tomography should be considered to evaluate for enlarged vestibular aqueduct.</div></li><li class="half_rhythm"><div>Sudden or rapidly progressive hearing loss can be seen with temporal bone anomalies (as in Pendred syndrome and <a href="/books/n/gene/bor/">BOR syndrome</a>), neoplasms (associated with <a href="/books/n/gene/nf2/">NF2</a>), and immunologic-related deafness, as well as trauma, infections (syphilis, Lyme disease), and metabolic, neurologic, or circulatory disturbances.</div></li></ul></li></ul><p><b>Subsequent evaluation</b> should be directed based on findings from above. In cases of apparent nonsyndromic sensorineural hearing loss, genetic testing should be obtained prior to other (ancillary) testing (<a class="figpopup" href="/books/NBK1434/figure/deafness-overview.F3/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewF3" rid-ob="figobdeafnessoverviewF3">Figure 3</a>).</p><p><b>Molecular genetic testing.</b> Genes known to cause common syndromes with hereditary hearing loss and deafness are included in <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.causes_of_syndromic/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTcausesofsyndromic" rid-ob="figobdeafnessoverviewTcausesofsyndromic">Table 3</a>. Because <a class="def" href="/books/n/gene/glossary/def-item/variable-expressivity/">variable expressivity</a> is common among <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> syndromes with deafness, the correct diagnosis may depend on <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a>.</p><p>The diagnosis of nonsyndromic hereditary hearing loss and deafness is established in a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> by identification of <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>(s) in a given <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> (see <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.autosomal_dominant_n/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTautosomaldominantn" rid-ob="figobdeafnessoverviewTautosomaldominantn">Tables 4</a>, <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.autosomal_recessive/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTautosomalrecessive" rid-ob="figobdeafnessoverviewTautosomalrecessive">5</a>, <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.xlinked_nonsyndromic/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTxlinkednonsyndromic" rid-ob="figobdeafnessoverviewTxlinkednonsyndromic">6</a>, and <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.mitochondrial_nonsyn/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTmitochondrialnonsyn" rid-ob="figobdeafnessoverviewTmitochondrialnonsyn">7</a>).</p><p>Molecular genetic testing for hereditary hearing loss and deafness historically relied on single-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> testing; however, this testing approach has largely been supplanted by multigene panels, which are comprehensive (i.e., include all genes known to cause deafness or all genes known to cause either <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> or <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> hearing loss and deafness). Multigene testing panels have greatly improved the diagnostic rate regardless of presumed inheritance or ethnicity [<a class="bk_pop" href="#deafness-overview.REF.shearer.2015.175">Shearer &#x00026; Smith 2015</a>]. In select instances, <a class="def" href="/books/n/gene/glossary/def-item/exome-sequencing/">exome sequencing</a> has been used in a diagnostic setting and can provide a means for comprehensive genetic testing for hereditary hearing loss and deafness, although small copy number variants can be easily missed [<a class="bk_pop" href="#deafness-overview.REF.zazo_seco.2017.308">Zazo Seco et al 2017</a>].</p><p><b>A multigene hearing loss and deafness panel</b> that includes most of the genes listed in <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.causes_of_syndromic/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTcausesofsyndromic" rid-ob="figobdeafnessoverviewTcausesofsyndromic">Tables 3</a>, <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.autosomal_dominant_n/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTautosomaldominantn" rid-ob="figobdeafnessoverviewTautosomaldominantn">4</a>, <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.autosomal_recessive/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTautosomalrecessive" rid-ob="figobdeafnessoverviewTautosomalrecessive">5</a>, <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.xlinked_nonsyndromic/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTxlinkednonsyndromic" rid-ob="figobdeafnessoverviewTxlinkednonsyndromic">6</a>, and <a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.mitochondrial_nonsyn/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTmitochondrialnonsyn" rid-ob="figobdeafnessoverviewTmitochondrialnonsyn">7</a> is recommended. Note: (1) The genes included in the panel and the diagnostic <a class="def" href="/books/n/gene/glossary/def-item/sensitivity/">sensitivity</a> of the testing used for each <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> vary by laboratory and are likely to change over time. (2) Some multigene panels may include genes not associated with the condition discussed in this <i>GeneReview</i>; thus, clinicians need to determine which <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a> is most likely to identify the genetic cause of the condition at the most reasonable cost while limiting identification of variants of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a> and pathogenic variants in genes that do not explain the underlying <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>. (3) In some laboratories, panel options may include a custom laboratory-designed panel and/or custom phenotype-focused <a class="def" href="/books/n/gene/glossary/def-item/exome/">exome</a> analysis that includes genes specified by the clinician. (4) Methods used in a panel may include <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a>, <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>, and/or other non-sequencing-based tests. In the evaluation of hearing loss/deafness, a multigene panel that includes the identification of deletions and duplications is mandatory as copy number variants are a common cause of hereditary nonsyndromic hearing loss [<a class="bk_pop" href="#deafness-overview.REF.shearer.2014.37">Shearer et al 2014</a>]. For an introduction to multigene panels click <a href="/books/n/gene/app5/#app5.Multigene_Panels">here</a>. More detailed information for clinicians ordering genetic tests can be found <a href="/books/n/gene/app5/#app5.Multigene_Panels_FAQs">here</a>.</p><p>Individuals with the distinctive findings of <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> hearing loss (<a class="figpopup" href="/books/NBK1434/table/deafness-overview.T.causes_of_syndromic/?report=objectonly" target="object" rid-figpopup="figdeafnessoverviewTcausesofsyndromic" rid-ob="figobdeafnessoverviewTcausesofsyndromic">Table 3</a>) may be diagnosed using <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted testing. Of note, many multigene panels for hereditary hearing loss and deafness now include the most common causes of syndromic hearing loss, including those that masquerade as nonsyndromic hearing loss until secondary signs and symptoms present (e.g., Usher syndrome [blindness]; Pendred syndrome [thyroid goiter]).</p><p><b>Temporal bone CT or MRI.</b> A dedicated thin cut CT or MRI of the temporal bones is useful for detecting malformations of the inner ear (i.e., Mondini deformity, Michel aplasia, enlarged/dilated vestibular aqueduct, dilation of the internal auditory canal), which should be considered in persons with progressive hearing loss.</p><p><b>Ancillary testing.</b> Further testing including cardiac, renal, or ophthalmologic evaluation should not be routinely ordered unless there are concerning clinical findings or the results of genetic testing raise concerns about other organ system involvement.</p></div><div id="deafness-overview.Genetic_Counseling"><h2 id="_deafness-overview_Genetic_Counseling_">Genetic Counseling</h2><p>
<i>Genetic counseling is the process of providing individuals and families with
information on the nature, inheritance, and implications of genetic disorders to help them
make informed medical and personal decisions. The following section deals with genetic
risk assessment and the use of family history and genetic testing to clarify genetic
status for family members. This section is not meant to address all personal, cultural, or
ethical issues that individuals may face or to substitute for consultation with a genetics
professional</i>. &#x02014;ED.</p><div id="deafness-overview.Mode_of_Inheritance"><h3>Mode of Inheritance</h3><p>Hereditary hearing loss may be transmitted in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a>, <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a>, or <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> manner or by maternal inheritance.</p><p>(See <a href="/books/n/gene/mt-deafness/">Nonsyndromic Hearing Loss and Deafness, Mitochondrial</a> for <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> information about maternal inheritance.)</p></div><div id="deafness-overview.Autosomal_Dominant_Inh"><h3>Autosomal Dominant Inheritance &#x02013; Risk to Family Members</h3><p><b>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>Most individuals diagnosed as having <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> hereditary hearing loss have a deaf parent; the family history is rarely negative.</div></li><li class="half_rhythm"><div>A <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> hereditary hearing loss may have the disorder as the result of a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>. The proportion of cases caused by <i>de novo</i> pathogenic variants is unknown but thought to be small.</div></li><li class="half_rhythm"><div>Recommendations for the evaluation of parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with an apparent <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> include audiometry and <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a>.</div></li><li class="half_rhythm"><div>Note: Although most individuals diagnosed with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> hereditary hearing loss have a deaf parent, the family history may appear to be negative because of failure to recognize hereditary hearing loss in family members, late onset in a parent, reduced <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> of the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in an asymptomatic parent, or a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> variant for hereditary hearing loss.</div></li></ul><p><b>Sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>The risk to sibs depends on the genetic status of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s parents.</div></li><li class="half_rhythm"><div>If one of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s parents has a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>, the risk to the sibs of inheriting the variant is 50%.</div></li><li class="half_rhythm"><div>Depending on the specific diagnosis, clinical severity and <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> may differ between individuals with the same variant; thus, age of onset and/or progression may not be predictable.</div></li></ul><p><b>Offspring of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>Individuals with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> hereditary hearing loss have a 50% chance of transmitting the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> to each child.</div></li><li class="half_rhythm"><div>Depending on the specific diagnosis, clinical severity and <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> may differ between individuals with the same variant; thus, age of onset and/or progression may not be predictable.</div></li></ul><p><b>Other family members.</b> The risk to other family members depends on the status of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s parents: if a parent has the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>, his or her family members may be at risk.</p><p><b>Considerations in families with an apparent <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a></b>. When neither parent of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> condition has the causative variant identified in the proband or clinical evidence of the disorder, the variant is likely <i>de novo</i>. However, non-medical explanations including <a class="def" href="/books/n/gene/glossary/def-item/alternate-paternity/">alternate paternity</a> or maternity (e.g., with assisted reproduction) and undisclosed adoption could also be explored.</p></div><div id="deafness-overview.Autosomal_Recessive_In"><h3>Autosomal Recessive Inheritance</h3><div id="deafness-overview.Risk_to_Family_Members"><h4>Risk to Family Members</h4><p><b>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>The parents of an individual diagnosed as having <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> hereditary hearing loss are obligate heterozygotes (i.e., carriers a single copy of a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>).</div></li><li class="half_rhythm"><div>Heterozygotes are asymptomatic.</div></li></ul><p><b>Sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>At conception, each sib has a 25% chance of being deaf, a 50% chance of having normal hearing and being a <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a>, and a 25% chance of having normal hearing and not being a carrier.</div></li><li class="half_rhythm"><div>Heterozygotes are asymptomatic.</div></li><li class="half_rhythm"><div>Depending on the specific diagnosis, clinical severity and <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> may differ between individuals with the same variants; thus, age of onset and/or progression of hearing loss may not be predictable. (Note: One exception is <i>GJB2</i>-related hearing loss; studies have shown that it is possible to predict phenotype based on <i>GJB2</i> <a class="def" href="/books/n/gene/glossary/def-item/genotype/">genotype</a> (see <a href="/books/n/gene/dfnb1/">Nonsyndromic Hearing Loss and Deafness, DFNB1</a>).</div></li></ul><p><b>Offspring of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</b> All offspring are obligate carriers.</p><p><b>Other family members.</b> The sibs of obligate heterozygotes have a 50% chance of being heterozygotes.</p></div><div id="deafness-overview.Heterozygote_Carrier_D"><h4>Heterozygote (Carrier) Detection</h4><p>Carrier testing for at-risk relatives requires prior identification of the pathogenic variants in the family.</p></div></div><div id="deafness-overview.XLinked_Inheritance"><h3>X-Linked Inheritance</h3><div id="deafness-overview.Risk_to_Family_Members_1"><h4>Risk to Family Members</h4><p><b>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div><b>Male <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></div><ul><li class="half_rhythm"><div>The father of a male <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> will not be <a class="def" href="/books/n/gene/glossary/def-item/hemizygous/">hemizygous</a> for the <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> identified in his son.</div></li><li class="half_rhythm"><div>In a family with more than one individual with <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> deafness, the mother of a deaf male is an <a class="def" href="/books/n/gene/glossary/def-item/obligate-heterozygote/">obligate heterozygote</a> (<a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a>). Note: If a woman has more than one deaf child and no other <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> relatives and if the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> cannot be detected in her leukocyte DNA, she most likely has <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a>.</div></li><li class="half_rhythm"><div>If a male is the only deaf family member, the mother may be a <a class="def" href="/books/n/gene/glossary/def-item/heterozygote/">heterozygote</a> (<a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a>) or the deaf male may have a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>, in which case the mother is not a carrier.</div></li></ul></li><li class="half_rhythm"><div><b>Female <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></div><ul><li class="half_rhythm"><div>A female <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> may have inherited the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> from either her mother or her father, or the variant may be <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a>.</div></li><li class="half_rhythm"><div>Detailed evaluation of the parents and review of the extended family history may help distinguish probands with a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> variant from those with an inherited variant. Molecular genetic testing of the mother (and possibly the father, or subsequently the father) can determine if the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> was inherited.</div></li></ul></li></ul><p><b>Sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div><b>Male <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</b> The risk to sibs depends on the genetic status of the mother: if the mother of the proband has a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>, the chance of transmitting it in each pregnancy is 50%. Males who inherit the variant will be deaf; females who inherit the variant will be heterozygotes (carriers) and are likely to have normal hearing.</div></li><li class="half_rhythm"><div><b>Female <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</b> The risk to sibs depends on the genetic status of the parents:</div><ul><li class="half_rhythm"><div>If the mother of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> has a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>, the chance of transmitting it in each pregnancy is 50%. Males who inherit the variant will be deaf; females who inherit the variant will be heterozygotes (carriers) and are likely to have normal hearing.</div></li><li class="half_rhythm"><div>If the father of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> has a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>, he will transmit it to all his daughters and none of his sons.</div></li></ul></li><li class="half_rhythm"><div>If the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> represents a <a class="def" href="/books/n/gene/glossary/def-item/simplex/">simplex</a> case (i.e., a single occurrence in a family) and if the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> cannot be detected in the leukocyte DNA of either parent, the risk to sibs is slightly greater than that of the general population (though still &#x0003c;1%) because of the theoretic possibility of parental <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a>.</div></li><li class="half_rhythm"><div>Depending on the specific syndrome, clinical severity and <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> may differ between individuals with the same <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>; thus, age of onset and/or progression may not be predictable.</div></li></ul><p><b>Offspring of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div><b>Males</b> with <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> hereditary hearing loss will transmit the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> to all of their daughters and none of their sons.</div></li><li class="half_rhythm"><div><b>Females</b> with <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> hereditary hearing loss have a 50% chance of transmitting the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> to each child.</div></li></ul><p><b>Other family members.</b> The <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s maternal aunts may be at risk of being carriers and the aunt's offspring, depending on their gender, may be at risk of being carriers or of being deaf.</p></div><div id="deafness-overview.Heterozygote_Carrier_D_1"><h4>Heterozygote (Carrier) Detection</h4><p>Carrier testing for at-risk relatives requires prior identification of the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in the family.</p></div></div><div id="deafness-overview.Empiric_Risk_to_Family"><h3>Empiric Risk to Family Members</h3><p>If a specific diagnosis cannot be established (and/or the <a class="def" href="/books/n/gene/glossary/def-item/mode-of-inheritance/">mode of inheritance</a> cannot be established in a person with a positive family history of deafness or hearing loss), the following empiric figures can be used.</p><p><b>The subsequent offspring of a hearing couple with one deaf child and an otherwise negative family history of deafness</b> have an 18% empiric probability of deafness in future children [<a class="bk_pop" href="#deafness-overview.REF.green.1999.2211">Green et al 1999</a>].</p><ul><li class="half_rhythm"><div>If the deaf child does not have DFNB1 based on <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> of <i>GJB2</i> and <i>GJB6</i>, the <a class="def" href="/books/n/gene/glossary/def-item/recurrence-risk/">recurrence risk</a> is 14% for deafness unrelated to connexin 26 [<a class="bk_pop" href="#deafness-overview.REF.green.1999.2211">Green et al 1999</a>].</div></li><li class="half_rhythm"><div>If the hearing couple is <a class="def" href="/books/n/gene/glossary/def-item/consanguineous/">consanguineous</a> or comes from a highly consanguineous community, the subsequent offspring have close to a 25% probability of deafness because of the high likelihood of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> inheritance.</div></li></ul><p><b>The offspring of a deaf person and a hearing person</b> have a 10% empiric risk of deafness [<a class="bk_pop" href="#deafness-overview.REF.green.1999.2211">Green et al 1999</a>].</p><ul><li class="half_rhythm"><div>Most of the risk is attributed to <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> deafness.</div></li><li class="half_rhythm"><div>If both <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> deafness and a family history of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> inheritance can be excluded, the risk of deafness is chiefly related to <a class="def" href="/books/n/gene/glossary/def-item/pseudodominant-inheritance/">pseudodominant inheritance</a> of recessive deafness. <i>GJB2</i> testing can identify much of this risk (see <a href="/books/n/gene/dfnb1/">DFNB1</a>).</div></li></ul><p><b>The child of a nonconsanguineous deaf couple in whom <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> deafness has been excluded</b> has an approximately 15% empiric risk for deafness [<a class="bk_pop" href="#deafness-overview.REF.green.1999.2211">Green et al 1999</a>].</p><ul><li class="half_rhythm"><div>If both parents have <i>GJB2</i>-related deafness, the risk to their offspring is 100%.</div></li><li class="half_rhythm"><div>If the couple has <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> deafness known to be caused by deafness-causing variants at two different loci, the chance of deafness in their offspring is lower than that of the general population.</div></li></ul><p><b>The child of a hearing sib of a deaf <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> (presumed to have <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic deafness) and a deaf person</b> has a 1/200 (0.5%) empiric risk for deafness, or five times the general population risk.</p><p><i>GJB2</i> and <i>GJB6</i> <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> can clarify if the risks are higher. If the hearing sib is a <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a> of a <i>GJB2</i> deafness-causing variant or a <i>GJB6</i> deafness-causing variant and his/her reproductive partner has DFNB1 deafness, the chance of having a deaf child is 50%.</p></div><div id="deafness-overview.Related_Genetic_Counse"><h3>Related Genetic Counseling Issues</h3><p>See Management <a href="#deafness-overview.Evaluation_of_Relative">Evaluation of Relatives at Risk</a> for information on evaluating at-risk relatives for the purpose of early diagnosis and treatment.</p><p>The following points are noteworthy:</p><ul><li class="half_rhythm"><div>Communication with individuals who are members of the Deaf community and who sign requires the services of a skilled interpreter.</div></li><li class="half_rhythm"><div>Members of the Deaf community may view deafness as a distinguishing characteristic and not as a handicap, impairment, or medical condition requiring a "treatment" or "cure," or to be "prevented."</div></li><li class="half_rhythm"><div>Many deaf people are interested in obtaining information about the cause of their own deafness, including information on medical, educational, and social services, rather than information about prevention, reproduction, or family planning. It is, therefore, important to ascertain and address the questions and concerns of the family/individual.</div></li><li class="half_rhythm"><div>The use of certain terms is preferred: probability or chance vs risk; deaf and hard-of-hearing vs hearing impaired. Terms such as "abnormal" should be avoided.</div></li></ul><p><b>Family planning</b></p><ul><li class="half_rhythm"><div>The optimal time for determination of genetic status and discussion of the availability of prenatal testing is before pregnancy.</div></li><li class="half_rhythm"><div>It is appropriate to offer <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> (including discussion of the probability of deafness in offspring and reproductive options) to young adults who are deaf.</div></li></ul><p><b>DNA banking</b> is the storage of DNA (typically extracted from white blood cells) for possible future use. Because it is likely that testing methodology and our understanding of genes, allelic variants, and hereditary hearing loss will improve in the future, consideration should be given to banking DNA of <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals.</p></div><div id="deafness-overview.Prenatal_Testing_and_P"><h3>Prenatal Testing and Preimplantation Genetic Diagnosis</h3><p>Once the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>(s) have been identified in the family, <a class="def" href="/books/n/gene/glossary/def-item/prenatal-diagnosis/">prenatal diagnosis</a> for a pregnancy at increased risk and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> for deafness or hearing loss are possible.</p><p>Differences in perspective may exist among medical professionals and within families regarding the use of prenatal testing, particularly if the testing is being considered for the purpose of pregnancy termination rather than early diagnosis. While most centers would consider decisions regarding prenatal testing to be the choice of the parents, discussion of these issues is appropriate.</p></div></div><div id="deafness-overview.Resources"><h2 id="_deafness-overview_Resources_">Resources</h2><p>
<i>GeneReviews staff has selected the following disease-specific and/or umbrella
support organizations and/or registries for the benefit of individuals with this disorder
and their families. GeneReviews is not responsible for the information provided by other
organizations. For information on selection criteria, click <a href="/books/n/gene/app4/">here</a>.</i></p><ul><li class="half_rhythm"><div><b>American Society for Deaf Children (ASDC)</b></div><div>800 Florida Avenue Northeast</div><div>Suite 2047</div><div>Washington DC 20002-3695</div><div><b>Phone:</b> 800-942-2732 (Toll-free Parent Hotline); 866-895-4206 (toll free voice/TTY)</div><div><b>Fax:</b> 410-795-0965</div><div><b>Email:</b> info@deafchildren.org; asdc@deafchildren.org</div><div><a href="http://www.deafchildren.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.deafchildren.org</a></div></li><li class="half_rhythm"><div><b>My46 Trait Profile</b></div><div><a href="https://www.my46.org/trait-document?trait=Nonsyndromic%20congenital%20hearing%20loss&#x00026;parent=Carrier%20Status&#x00026;type=profile" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Nonsyndromic congenital hearing loss</a></div></li><li class="half_rhythm"><div><b>National Association of the Deaf (NAD)</b></div><div>8630 Fenton Street</div><div>Suite 820</div><div>Silver Spring MD 20910</div><div><b>Phone:</b> 301-587-1788; 301-587-1789 (TTY)</div><div><b>Fax:</b> 301-587-1791</div><div><b>Email:</b> nad.info@nad.org</div><div><a href="http://www.nad.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.nad.org</a></div></li><li class="half_rhythm"><div><b>National Library of Medicine Genetics Home Reference</b></div><div><a href="http://ghr.nlm.nih.gov/condition=nonsyndromicdeafness" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Nonsyndromic deafness</a></div></li><li class="half_rhythm"><div><b>NCBI Genes and Disease</b></div><div><a href="/books/NBK22204/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Deafness</a></div></li><li class="half_rhythm"><div><b>Alexander Graham Bell Association for the Deaf and Hard of Hearing</b></div><div>3417 Volta Place Northwest</div><div>Washington DC 20007</div><div><b>Phone:</b> 866-337-5220 (toll-free); 202-337-5220; 202-337-5221 (TTY)</div><div><b>Fax:</b> 202-337-8314</div><div><b>Email:</b> info@agbell.org</div><div><a href="http://www.agbell.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Listening and Spoken Language Knowledge Center</a></div></li><li class="half_rhythm"><div><b>BabyHearing.org</b></div><div><i>This site, developed with support from the National Institute on Deafness and Other Communication Disorders, provides information about newborn hearing screening and hearing loss.</i></div><div><a href="http://www.babyhearing.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.babyhearing.org</a></div></li></ul></div><div id="deafness-overview.Management"><h2 id="_deafness-overview_Management_">Management</h2><div id="deafness-overview.Treatment_of_Manifesta"><h3>Treatment of Manifestations</h3><p>Ideally, the team evaluating and treating the deaf individual should consist of an otolaryngologist with expertise in the management of early childhood otologic disorders, an audiologist experienced in the assessment of hearing loss in children, a clinical geneticist, and a pediatrician. The expertise of an educator of the Deaf, a neurologist, and a pediatric ophthalmologist may also be required.</p><p>An important part of the evaluation is determining the appropriate habilitation option. Possibilities include hearing aids, vibrotactile devices, and cochlear implantation. Cochlear implantation can be considered in children older than age 12 months with severe-to-profound hearing loss.</p><p>The ultimate goal in the evaluation and treatment of a child with hereditary hearing loss and deafness is mainstream schooling. Research shows that diagnosis by age three months and habilitation by six months makes this goal possible for children with mild-moderate hearing loss. Cochlear implantation in children with severe-to-profound deafness who are part of mainstream education leads to social functioning and educational attainment that is indistinguishable from normal-hearing peers [<a class="bk_pop" href="#deafness-overview.REF.loy.2010.247">Loy et al 2010</a>, <a class="bk_pop" href="#deafness-overview.REF.langereis.2015.834">Langereis &#x00026; Vermeulen 2015</a>].</p><p>Recent research has focused on cochlear implant performance based on the <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> involved. Due to the genetic heterogeneity of deafness, large sample sizes are difficult to obtain for performance on a per-gene basis. However, the data are clear that individuals with <i>GJB2</i>-related hearing loss (see <a href="/books/n/gene/dfnb1/">Nonsyndromic Hearing Loss and Deafness, DFNB1</a>) have excellent cochlear implant outcomes that are significantly better than those of individuals with environmental causes of deafness [<a class="bk_pop" href="#deafness-overview.REF.yoshida.2013.435">Yoshida et al 2013</a>, <a class="bk_pop" href="#deafness-overview.REF.abdurehim.2017.16">Abdurehim et al 2017</a>].</p><p>In adults, cochlear implant performance may be compromised when the genetic defect affects the auditory nerve itself; however, this hypothesis requires further research [<a class="bk_pop" href="#deafness-overview.REF.shearer.2017.138">Shearer et al 2017</a>].</p><p>DFNX3 is characterized by a mixed conductive-sensorineural hearing loss, the conductive component of which is caused by stapedial fixation. In contrast to other types of conductive hearing loss, surgical correction of DFNX3-related hearing loss can compromise hearing. An abnormal communication between the cerebrospinal fluid and perilymph can lead to fluid leakage ("perilymphatic gusher") at surgery and complete loss of hearing upon fenestration or removal of the stapes footplate.</p></div><div id="deafness-overview.Prevention_of_Primary"><h3>Prevention of Primary Manifestations</h3><p>Whenever a child presents with progressive sensorineural hearing loss and progressive ataxia, with or without neurologic or cutaneous symptoms, <a href="/books/n/gene/biotin/">biotinidase deficiency</a> should be considered, with initiation of treatment as early as possible to prevent irreversible sequelae.</p></div><div id="deafness-overview.Prevention_of_Secondar"><h3>Prevention of Secondary Complications</h3><p>Regardless of its etiology, uncorrected hearing loss has consistent sequelae. Auditory deprivation through age two years is associated with poor reading performance, poor communication skills, and poor speech production. Educational intervention is insufficient to completely remediate these deficiencies. In contrast, early auditory intervention is effective &#x02013; whether through amplification, otologic surgery, or cochlear implantation [<a class="bk_pop" href="#deafness-overview.REF.smith.2005.879">Smith et al 2005</a>].</p><p>Although decreased cognitive skills and performance in mathematics and reading are associated with deafness, examination of persons with hereditary hearing loss has shown that these deficiencies are not intrinsically linked to the cause of the deafness. For example, assessment of cognitive skills in individuals with <i>GJB2</i>-related hearing loss reveals a normal Hiskey IQ and normal reading performance after cochlear implantation [<a class="bk_pop" href="#deafness-overview.REF.bauer.2003.2135">Bauer et al 2003</a>]. Thus, early identification and timely intervention is essential for optimal cognitive development in children with prelingual deafness.</p></div><div id="deafness-overview.Surveillance"><h3>Surveillance</h3><p>Sequential audiologic examinations are essential to:</p><ul><li class="half_rhythm"><div>Document the stability or progression of the hearing loss;</div></li><li class="half_rhythm"><div>Identify and treat superimposed hearing losses, such as middle ear effusion.</div></li></ul><p>In a person with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic hearing loss caused by pathogenic variants in <i>SLC26A4</i>, the hearing loss can progress and annual audiometric testing may be warranted. Additionally, thyroid function should be followed if the diagnosis is consistent with <a href="/books/n/gene/pendred/">Pendred syndrome</a>.</p></div><div id="deafness-overview.AgentsCircumstances_to"><h3>Agents/Circumstances to Avoid</h3><p>Noise exposure is a well-recognized environmental cause of hearing loss. Since this risk can be minimized by avoidance, persons with documented hearing loss should be counseled appropriately.</p></div><div id="deafness-overview.Evaluation_of_Relative"><h3>Evaluation of Relatives at Risk</h3><p>At a minimum, all children at risk for hereditary deafness and hearing loss should receive screening audiometry.</p><p>See <a href="#deafness-overview.Related_Genetic_Counse">Genetic Counseling</a> for issues related to testing of at-risk relatives for <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> purposes.</p></div></div><div id="deafness-overview.References"><h2 id="_deafness-overview_References_">References</h2><div id="deafness-overview.Published_Guidelines"><h3>Published Guidelines / Consensus Statements</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.alford.2014">Alford RL, Arnos KS, Fox M, Lin JW, Palmer CG, Pandya A, Rehm HL, Robin NH, Scott DA, Yoshinaga-Itano C; ACMG Working Group on Update of Genetics Evaluation Guidelines for the Etiologic Diagnosis of Congenital Hearing Loss; Professional Practice and Guidelines Committee. American College of Medical Genetics and Genomics guideline for the clinical evaluation and etiologic diagnosis of hearing loss. Available <a href="http://www.nature.com/gim/journal/v16/n4/full/gim20142a.html?foxtrotcallback=true" ref="pagearea=cite-ref&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">online</a>. 2014. Accessed 7-18-19. [<a href="/pubmed/24651602" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24651602</span></a>]</div></li></ul></div><div id="deafness-overview.Literature_Cited"><h3>Literature Cited</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.abdurehim.2017.16">Abdurehim Y, Lehmann A, Zeitouni AG. Predictive value of GJB2 mutation status for hearing outcomes of pediatric cochlear implantation. <span><span class="ref-journal">Otolaryngol Head Neck Surg. </span>2017;<span class="ref-vol">157</span>:16–24.</span> [<a href="/pubmed/28322114" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 28322114</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.bauer.2003.2135">Bauer PW, Geers AE, Brenner C, Moog JS, Smith RJ. The effect of GJB2 allele variants on performance after cochlear implantation. <span><span class="ref-journal">Laryngoscope. </span>2003;<span class="ref-vol">113</span>:2135–40.</span> [<a href="/pubmed/14660916" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 14660916</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.fischelghodsian.1998.15">Fischel-Ghodsian N. Mitochondrial mutations and hearing loss: paradigm for mitochondrial genetics. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">62</span>:15–9.</span> [<a href="/pmc/articles/PMC1376819/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1376819</span></a>] [<a href="/pubmed/9443888" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9443888</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.green.1999.2211">Green GE, Scott DA, McDonald JM, Woodworth GG, Sheffield VC, Smith RJ. Carrier rates in the Midwestern United States for GJB2 mutations causing inherited deafness. <span><span class="ref-journal">JAMA. </span>1999;<span class="ref-vol">281</span>:2211–6.</span> [<a href="/pubmed/10376574" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10376574</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.jama.1979.2055">JAMA.  Guide for the evaluation of hearing handicap. <span><span class="ref-journal">JAMA. </span>1979;<span class="ref-vol">241</span>:2055–9.</span> [<a href="/pubmed/430800" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 430800</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.kenneson.2007.253">Kenneson A, Cannon MJ. Review and meta-analysis of the epidemiology of congenital cytomegalovirus (CMV) infection. <span><span class="ref-journal">Rev Med Virol. </span>2007;<span class="ref-vol">17</span>:253–76.</span> [<a href="/pubmed/17579921" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17579921</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.kimberlin.2015.933">Kimberlin DW, Jester PM, S&#x000e1;nchez PJ, Ahmed A, Arav-Boger R, Michaels MG, Ashouri N, Englund JA, Estrada B, Jacobs RF, Romero JR, Sood SK, Whitworth MS, Abzug MJ, Caserta MT, Fowler S, Lujan-Zilbermann J, Storch GA, DeBiasi RL, Han JY, Palmer A, Weiner LB, Bocchini JA, Dennehy PH, Finn A, Griffiths PD, Luck S, Gutierrez K, Halasa N, Homans J, Shane AL, Sharland M, Simonsen K, Vanchiere JA, Woods CR, Sabo DL, Aban I, Kuo H, James SH, Prichard MN, Griffin J, Giles D, Acosta EP, Whitley RJ, et al.  Valganciclovir for symptomatic congenital cytomegalovirus disease. <span><span class="ref-journal">NEJM. </span>2015;<span class="ref-vol">372</span>:933–43.</span> [<a href="/pmc/articles/PMC4401811/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4401811</span></a>] [<a href="/pubmed/25738669" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 25738669</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.kokotas.2007.379">Kokotas H, Petersen MB, Willems PJ. Mitochondrial deafness. <span><span class="ref-journal">Clin Genet. </span>2007;<span class="ref-vol">71</span>:379–91.</span> [<a href="/pubmed/17489842" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17489842</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.langereis.2015.834">Langereis M, Vermeulen A. School performance and wellbeing of children with CI in different communicative&#x02013;educational environments. <span><span class="ref-journal">Int J Pediatr Otorhinolaryngol. </span>2015;<span class="ref-vol">79</span>:834–9.</span> [<a href="/pubmed/25840945" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 25840945</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.liming.2016.251">Liming BJ, Carter J, Cheng A, Choo D, Curotta J, Carvalho D, Germiller JA, Hone S, Kenna MA, Loundon N, Preciado D, Schilder A, Reilly BJ, Roman S, Strychowsky J, Triglia JM, Young N, Smith RJ. International Pediatric Otolaryngology Group (IPOG) consensus recommendations: hearing loss in the pediatric patient. <span><span class="ref-journal">Int J Pediatr Otorhinolaryngol. </span>2016;<span class="ref-vol">90</span>:251–8.</span> [<a href="/pubmed/27729144" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 27729144</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.loy.2010.247">Loy B, Warner-Czyz AD, Tong L, Tobey EA, Roland PS. The children speak: an examination of the quality of life of pediatric cochlear implant users. <span><span class="ref-journal">Otolaryngol Head Neck Surg. </span>2010;<span class="ref-vol">142</span>:247–53.</span> [<a href="/pmc/articles/PMC2852181/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2852181</span></a>] [<a href="/pubmed/20115983" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20115983</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.northern.2002">Northern JL, Downs M. <em>Hearing in Children.</em> Baltimore, MD: Lippincott, Williams, and Wilkins; 2002.</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.pandya.1999.1803">Pandya A, Xia X-J, Erdenetungalag R, Amendola M, Landa B, Radnaabazar J, Dangaasuren B, Van Tuyle G, Nance WE. Heterozygous point mutations in the mitochondrial tRNA Ser(UCN) precursor coexisting with the A1555G mutation in deaf students from Mongolia. <span><span class="ref-journal">Am J Hum Genet. </span>1999;<span class="ref-vol">65</span>:1803–6.</span> [<a href="/pmc/articles/PMC1288397/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1288397</span></a>] [<a href="/pubmed/10577941" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10577941</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.rudman.2017.285">Rudman JR, Kabahuma RI, Bressler SE, Feng Y, Blanton SH, Yan D, Liu XZ. The genetic basis of deafness in populations of African descent. <span><span class="ref-journal">J Genet Genomics. </span>2017;<span class="ref-vol">44</span>:285–94.</span> [<a href="/pubmed/28642064" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 28642064</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.zazo_seco.2017.308">Zazo Seco C, Wesdorp M, Feenstra I, Pfundt R, Hehir-Kwa JY, Lelieveld SH, Castelein S, Gilissen C, de Wijs IJ, Admiraal RJ, Pennings RJ, Kunst HP, van de Kamp JM, Tamminga S, Houweling AC, Plomp AS, Maas SM, de Koning Gans PA, Kant SG, de Geus CM, Frints SG, Vanhoutte EK, van Dooren MF, van den Boogaard MH, Scheffer H, Nelen M, Kremer H, Hoefsloot L, Schraders M, Yntema HG. The diagnostic yield of whole-exome sequencing targeting a gene panel for hearing impairment in The Netherlands. <span><span class="ref-journal">Eur J Hum Genet. </span>2017;<span class="ref-vol">25</span>:308–14.</span> [<a href="/pmc/articles/PMC5315517/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC5315517</span></a>] [<a href="/pubmed/28000701" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 28000701</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.shearer.2017.138">Shearer AE, Eppsteiner RW, Frees K, Tejani V, Sloan-Heggen CM, Brown C, Abbas P, Dunn C, Hansen MR, Gantz BJ, Smith RJH. Genetic variants in the peripheral auditory system significantly affect adult cochlear implant performance. <span><span class="ref-journal">Hear Res. </span>2017;<span class="ref-vol">348</span>:138–42.</span> [<a href="/pmc/articles/PMC5527292/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC5527292</span></a>] [<a href="/pubmed/28213135" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 28213135</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.shearer.2014.37">Shearer AE, Kolbe DL, Azaiez H, Sloan CM, Frees KL, Weaver AE, Clark ET, Nishimura CJ, Black-Ziegelbein EA, Smith RJ. Copy number variants are a common cause of non-syndromic hearing loss. <span><span class="ref-journal">Genome Med. </span>2014;<span class="ref-vol">6</span>:37.</span> [<a href="/pmc/articles/PMC4067994/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4067994</span></a>] [<a href="/pubmed/24963352" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24963352</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.shearer.2015.175">Shearer AE, Smith RJH. Massively parallel sequencing for genetic diagnosis of hearing loss: the new standard of care. <span><span class="ref-journal">Otolaryngol Head Neck Surg. </span>2015;<span class="ref-vol">153</span>:175–82.</span> [<a href="/pmc/articles/PMC4743024/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4743024</span></a>] [<a href="/pubmed/26084827" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26084827</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.sloanheggen.2016.441">Sloan-Heggen CM, Bierer AO, Shearer AE, Kolbe DL, Nishimura CJ, Frees KL, Ephraim SS, Shibata SB, Booth KT, Campbell CA, Ranum PT, Weaver AE, Black-Ziegelbein EA, Wang D, Azaiez H, Smith RJ. Comprehensive genetic testing in the clinical evaluation of 1119 patients with hearing loss. <span><span class="ref-journal">Hum Genet. </span>2016;<span class="ref-vol">135</span>:441–50.</span> [<a href="/pmc/articles/PMC4796320/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4796320</span></a>] [<a href="/pubmed/26969326" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26969326</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.smith.2005.879">Smith RJH, Bale JF, White KR. Sensorineural hearing loss in children. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">365</span>:879–90.</span> [<a href="/pubmed/15752533" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15752533</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.taylor.2013.539">Taylor KR, Deluca AP, Shearer AE, Hildebrand MS, Black-Ziegelbein EA, Anand VN, Sloan CM, Eppsteiner RW, Scheetz TE, Huygen PL, Smith RJ, Braun TA, Casavant TL. AudioGene: predicting hearing loss genotypes from phenotypes to guide genetic screening. <span><span class="ref-journal">Hum Mutat. </span>2013;<span class="ref-vol">34</span>:539–45.</span> [<a href="/pmc/articles/PMC3753227/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3753227</span></a>] [<a href="/pubmed/23280582" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23280582</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.toriello.2004">Toriello HV, Reardon W, Gorlin RJ, eds. <em>Hereditary Hearing Loss and Its Syndromes</em>. New York: Oxford University Press; 2004.</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.van_camp.2017">Van Camp G, Smith RJH. The Hereditary Hearing Loss Homepage. Available <a href="http://hereditaryhearingloss.org/" ref="pagearea=cite-ref&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">online</a>. 2017. Accessed 7-18-19.</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.yamasoba.2013.30">Yamasoba T, Lin FR, Someya S, Kashio A, Sakamoto T, Kondo K. Current concepts in age-related hearing loss: epidemiology and mechanistic pathways. <span><span class="ref-journal">Hear Res. </span>2013;<span class="ref-vol">303</span>:30–8.</span> [<a href="/pmc/articles/PMC3723756/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3723756</span></a>] [<a href="/pubmed/23422312" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23422312</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.yang.2007.1055">Yang T, Vidarsson H, Rodrigo-Blomqvist S, Rosengren SS, Enerback S, Smith RJ. Transcriptional control of SLC26A4 is involved in Pendred syndrome and nonsyndromic enlargement of vestibular aqueduct (DFNB4). <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>:1055–63.</span> [<a href="/pmc/articles/PMC1867094/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1867094</span></a>] [<a href="/pubmed/17503324" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17503324</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="deafness-overview.REF.yoshida.2013.435">Yoshida H, Takahashi H, Kanda Y, Usami S. Long term speech perception after cochlear implant in pediatric patients with GJB2 mutations. <span><span class="ref-journal">Auris Nasus Larynx. </span>2013;<span class="ref-vol">40</span>:435–9.</span> [<a href="/pubmed/23477838" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23477838</span></a>]</div></li></ul></div></div><div id="deafness-overview.Chapter_Notes"><h2 id="_deafness-overview_Chapter_Notes_">Chapter Notes</h2><div id="deafness-overview.Author_History"><h3>Author History</h3><p>Glenn Edward Green, MD; University of Arizona (1999-2005) <br />Michael S Hildebrand, PhD (2010-present)<br />A Eliot Shearer (2012-present)<br />Richard JH Smith, MD (1999-present) <br />Guy Van Camp, PhD; University of Antwerp (1999-2017)</p></div><div id="deafness-overview.Revision_History"><h3>Revision History</h3><ul><li class="half_rhythm"><div>27 July 2017 (bp) Comprehensive update posted live</div></li><li class="half_rhythm"><div>9 January 2014 (rjhs) Revision: DFNA41 and DFNB76 added</div></li><li class="half_rhythm"><div>3 January 2013 (cd) Revision: clinical testing available for <i>DFNB79</i> and <i>DFNX4</i> (<i>DFN6</i>)</div></li><li class="half_rhythm"><div>5 January 2012 (cd) Revision: clinical testing for mutations in <i>MT-CO1</i> associated with hearing loss and multigene hearing loss/deafness panels now listed in the GeneTests&#x02122; Laboratory Directory</div></li><li class="half_rhythm"><div>14 October 2010 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>2 December 2008 (rjs) Revision: DFNB23 added</div></li><li class="half_rhythm"><div>28 October 2008 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>30 January 2007 (rjs) Revision: clinical testing and <a class="def" href="/books/n/gene/glossary/def-item/prenatal-diagnosis/">prenatal diagnosis</a> available for DFNB9</div></li><li class="half_rhythm"><div>4 December 2006 (rjs) Revision: clinical testing available for DFNB21 and DFNA8/12</div></li><li class="half_rhythm"><div>22 August 2006 (rjs) Revision: to incorporate concerns of reader regarding hearing impairment scales</div></li><li class="half_rhythm"><div>30 December 2005 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>18 February 2005 (rjs) Revision: clinical availability of testing, <i>KCNQ4</i>-related DFNA2</div></li><li class="half_rhythm"><div>15 July 2004 (rjs) Revision: use of an interpreter</div></li><li class="half_rhythm"><div>18 December 2003 (cd,rjs) Revision: change in test availability</div></li><li class="half_rhythm"><div>3 November 2003 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>13 January 2003 (cd) Revision: test availability</div></li><li class="half_rhythm"><div>24 April 2001 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>14 February 1999 (pb) Overview posted live</div></li><li class="half_rhythm"><div>30 October 1998 (rjs) Original overview submission [Supported in part by grants 1RO1DC02842 and 1RO1DC03544 (RJHS) and Belgian National Fonds voor Wetenschappelijk Onderzoek (GVC).]</div></li></ul></div></div><div id="bk_toc_contnr"></div></div></div>
            <div class="post-content"><div><div class="half_rhythm"><a href="/books/about/copyright/">Copyright</a> © 1993-2019, University of Washington, Seattle. GeneReviews is
a registered trademark of the University of Washington, Seattle. All rights
reserved.<p class="small">GeneReviews® chapters are owned by the University of Washington. Permission is
hereby granted to reproduce, distribute, and translate copies of content materials for
noncommercial research purposes only, provided that (i) credit for source (<a href="http://www.genereviews.org/" ref="pagearea=meta&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">http://www.genereviews.org/</a>) and copyright (© 1993-2019 University of
Washington) are included with each copy; (ii) a link to the original material is provided
whenever the material is published elsewhere on the Web; and (iii) reproducers,
distributors, and/or translators comply with the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>. No further modifications are allowed. For clarity, excerpts
of GeneReviews chapters for use in lab reports and clinic notes are a permitted
use.</p><p class="small">For more information, see the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>.</p><p class="small">For questions regarding permissions or whether a specified use is allowed,
contact: <a href="mailto:dev@null" data-email="ude.wu@tssamda" class="oemail">ude.wu@tssamda</a>.</p></div><div class="small"><span class="label">Bookshelf ID: NBK1434</span><span class="label">PMID: <a href="/pubmed/20301607" title="PubMed record of this page" ref="pagearea=meta&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">20301607</a></span></div><div style="margin-top:2em" class="bk_noprnt"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/hfi/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/hht/" title="Next page in this title">Next &gt;</a></div></div></div></div>
            
        </div>

        <!-- Custom content below content -->
        <div class="col4">
            
        </div>
        
        
        <!-- Book content -->
        
        <!-- Custom contetnt below bottom nav -->
        <div class="col5">
            
        </div>
    </div>

    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            <div id="ncbi_share_book"><a href="#" class="ncbi_share" data-ncbi_share_config="popup:false,shorten:true" ref="id=NBK1434&amp;db=books">Share</a></div>

        </div>
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Views</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PDF_download" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/NBK1434/?report=reader">PubReader</a></li><li><a href="/books/NBK1434/?report=printable">Print View</a></li><li><a data-jig="ncbidialog" href="#_ncbi_dlg_citbx_NBK1434" data-jigconfig="width:400,modal:true">Cite this Page</a><div id="_ncbi_dlg_citbx_NBK1434" style="display:none" title="Cite this Page"><div class="bk_tt">Shearer AE, Hildebrand MS, Smith RJH. Hereditary Hearing Loss and Deafness Overview. 1999 Feb 14 [Updated 2017 Jul 27]. In: Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. <span class="bk_cite_avail"></span></div></div></li><li><a href="/books/NBK1434/pdf/Bookshelf_NBK1434.pdf">PDF version of this page</a> (718K)</li><li><a href="#" class="toggle-glossary-link" title="Enable/disable links to the glossary">Disable Glossary Links</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>In this GeneReview</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="page-toc" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="#deafness-overview.Summary" ref="log$=inpage&amp;link_id=inpage">Summary</a></li><li><a href="#deafness-overview.Clinical_Characteristi" ref="log$=inpage&amp;link_id=inpage">Clinical Characteristics of Hereditary Hearing Loss and Deafness</a></li><li><a href="#deafness-overview.Causes_of_Hereditary_H" ref="log$=inpage&amp;link_id=inpage">Causes of Hereditary Hearing Loss and Deafness</a></li><li><a href="#deafness-overview.Evaluation_Strategy" ref="log$=inpage&amp;link_id=inpage">Evaluation Strategy</a></li><li><a href="#deafness-overview.Genetic_Counseling" ref="log$=inpage&amp;link_id=inpage">Genetic Counseling</a></li><li><a href="#deafness-overview.Resources" ref="log$=inpage&amp;link_id=inpage">Resources</a></li><li><a href="#deafness-overview.Management" ref="log$=inpage&amp;link_id=inpage">Management</a></li><li><a href="#deafness-overview.References" ref="log$=inpage&amp;link_id=inpage">References</a></li><li><a href="#deafness-overview.Chapter_Notes" ref="log$=inpage&amp;link_id=inpage">Chapter Notes</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>GeneReviews Links</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="source-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/n/gene/advanced/"><i>GeneReviews</i> Advanced Search</a></li><li><a href="/books/n/gene/glossary/"><i>GeneReviews</i> Glossary</a></li><li><a href="/books/n/gene/resource_mats/">Resource Materials</a> <span class="bk_hlight1">NEW FEATURE</span></li><li><a href="/books/n/gene/updates/">New in <i>GeneReviews</i></a></li><li><a href="/books/n/gene/authors/">Author List</a></li><li><a href="/books/n/gene/prospective_authors/">For Current/Prospective Authors</a></li><li><a href="/books/n/gene/GRpersonnel/"><i>GeneReviews</i> Personnel</a></li><li><a href="/books/n/gene/howto_linkin/">Download/Link to <i>GeneReviews</i></a></li><li><a href="/books/n/gene/contact_us/">Contact Us</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Tests in GTR by Gene</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="document-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=10083[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">USH1C</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=79977[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GRHL2</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=71[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">ACTG1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=5456[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">POU3F4</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=65217[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">PCDH15</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=53904[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MYO3A</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=4647[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MYO7A</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=4627[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MYH9</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=407053[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MIR96</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=23676[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">SMPX</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=3766[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">KCNJ10</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=117531[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">TMC1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=4574[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MT-TS1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=2070[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">EYA4</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=10804[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GJB6</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=1687[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GSDME</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=7007[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">TECTA</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=6495[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">SIX1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=152137[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">CCDC50</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=5459[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">POU4F3</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=2299[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">FOXI1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=153562[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MARVELD2</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=125336[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">LOXHD1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=4512[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MT-CO1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=55687[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">TRMU</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=222662[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">LHFPL5</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=9132[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">KCNQ4</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=83715[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">ESPN</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=5962[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">RDX</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=2707[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GJB3</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=259236[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">TMIE</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=79784[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MYH14</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=4549[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MT-RNR1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=4646[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MYO6</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=5631[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">PRPS1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=1690[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">COCH</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=1302[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">COL11A2</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=220074[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">LRTOMT</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=64072[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">CDH23</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=2706[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GJB2</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=491[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">ATP2B2</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=246213[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">SLC17A8</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=161497[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">STRC</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=51168[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">MYO15A</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=25861[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">WHRN</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=1729[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">DIAPH1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=2103[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">ESRRB</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=64699[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">TMPRSS3</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=7466[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">WFS1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=494513[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">PJVK</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=11078[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">TRIOBP</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=146183[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">OTOA</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=3082[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">HGF</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=5172[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">SLC26A4</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=9381[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">OTOF</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Related information</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="discovery_db_links" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pmc&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pmc_refs&amp;IdsFromResult=1459514" ref="log$=recordlinks">PMC</a><div class="brieflinkpop offscreen_noflow">PubMed Central citations</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pubmed&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pubmed_refs&amp;IdsFromResult=1459514" ref="log$=recordlinks">PubMed</a><div class="brieflinkpop offscreen_noflow">Links to PubMed</div></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/23965379" ref="ordinalpos=1&amp;linkpos=1&amp;log$=relatedarticles&amp;logdbfrom=pubmed">Adaptive evolution of tight junction protein claudin-14 in echolocating whales.</a><span class="source">[Gene. 2013]</span><div class="brieflinkpop offscreen_noflow">Adaptive evolution of tight junction protein claudin-14 in echolocating whales.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Xu H, Liu Y, He G, Rossiter SJ, Zhang S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Gene. 2013 Nov 10; 530(2):208-14. Epub 2013 Aug 18.</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/15850684" ref="ordinalpos=1&amp;linkpos=2&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nuclear and mitochondrial genes mutated in nonsyndromic impaired hearing.</a><span class="source">[Int J Pediatr Otorhinolaryngol...]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nuclear and mitochondrial genes mutated in nonsyndromic impaired hearing.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Finsterer J, Fellinger J. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Int J Pediatr Otorhinolaryngol. 2005 May; 69(5):621-47. </em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/25279224" ref="ordinalpos=1&amp;linkpos=3&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Strategy for the customized mass screening of genetic sensorineural hearing loss in koreans.</a><span class="source">[Korean J Audiol. 2014]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Strategy for the customized mass screening of genetic sensorineural hearing loss in koreans.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Chang MY, Choi BY. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Korean J Audiol. 2014 Sep; 18(2):45-9. Epub 2014 Sep 16.</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/12324385" ref="ordinalpos=1&amp;linkpos=4&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Hereditary deafness and phenotyping in humans.</a><span class="source">[Br Med Bull. 2002]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Hereditary deafness and phenotyping in humans.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Bitner-Glindzicz M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Br Med Bull. 2002; 63:73-94. </em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/31389194" ref="ordinalpos=1&amp;linkpos=5&amp;log$=relatedarticles&amp;logdbfrom=pubmed">Sequence variants in genes causing nonsyndromic hearing loss in a Pakistani cohort.</a><span class="source">[Mol Genet Genomic Med. 2019]</span><div class="brieflinkpop offscreen_noflow">Sequence variants in genes causing nonsyndromic hearing loss in a Pakistani cohort.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Khan A, Han S, Wang R, Ansar M, Ahmad W, Zhang X. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Mol Genet Genomic Med. 2019 Sep; 7(9):e917. Epub 2019 Aug 6.</em></div></div></li></ul><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=20301607" ref="ordinalpos=1&amp;log$=relatedreviews_seeall&amp;logdbfrom=pubmed">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=20301607" ref="ordinalpos=1&amp;log$=relatedarticles_seeall&amp;logdbfrom=pubmed">See all...</a></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Recent Activity</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="Shutter"></a></div><div class="portlet_content"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class=""><div class="action"><a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a></div><ul id="activity"><li class="ra_rcd ralinkpopper two_line"><a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5e041455dde0891169ceb9d9">Hereditary Hearing Loss and Deafness Overview - GeneReviews®</a><div class="ralinkpop offscreen_noflow">Hereditary Hearing Loss and Deafness Overview - GeneReviews®<div class="brieflinkpopdesc"></div></div><div class="tertiary"></div></li></ul><p class="HTOn">Your browsing activity is empty.</p><p class="HTOff">Activity recording is turned off.</p><p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p><a class="seemore" href="/sites/myncbi/recentactivity">See more...</a></div></div></div>
        
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script type="text/javascript" src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script type="text/javascript">
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-12-25T21:00:53-05:00&amp;Snapshot=%2Fprojects%2Fbooks%2FPBooks@5.22&amp;Host=portal106&amp;ncbi_phid=CE8D6575E0403E510000000006D40276&amp;ncbi_session=CE8D6575E0414541_1748SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK1434%2F&amp;Db=pmc&amp;folderID=134&amp;Ncbi_App=bookshelf&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        </div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=books&amp;ncbi_pdid=book-part&amp;ncbi_acc=NBK1434&amp;ncbi_domain=gene&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/NBK1434/&amp;ncbi_pagename=Hereditary Hearing Loss and Deafness Overview - GeneReviews® - NCBI Bookshelf&amp;ncbi_bookparttype=chapter&amp;ncbi_app=bookshelf" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8D6575E0414541_1748SID /projects/books/PBooks@5.22 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/js/3879255/4121861/3501987/4008961/3893018/3821238/4062932/4123007/4087685/4076480/3921943/3400083/3426610.js" snapshot="books"></script></body>
</html>