<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    
    <head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <!-- AppResources meta begin -->
        <meta name="paf-app-resources" content="" />
                 <script type="text/javascript">var ncbi_startTime = new Date();</script>
            
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="" />

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="books" /><meta name="ncbi_pdid" content="book-part" /><meta name="ncbi_acc" content="NBK1272" /><meta name="ncbi_domain" content="gene" /><meta name="ncbi_report" content="record" /><meta name="ncbi_type" content="fulltext" /><meta name="ncbi_objectid" content="" /><meta name="ncbi_pcid" content="/NBK1272/" /><meta name="ncbi_pagename" content="Nonsyndromic Hearing Loss and Deafness, DFNB1 - GeneReviews® - NCBI Bookshelf" /><meta name="ncbi_bookparttype" content="chapter" /><meta name="ncbi_app" content="bookshelf" />
        <!-- Logger end -->
        
        <title>Nonsyndromic Hearing Loss and Deafness, DFNB1 - GeneReviews® - NCBI Bookshelf</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css" /><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,FOLLOW,NOARCHIVE" /><meta name="citation_inbook_title" content="GeneReviews® [Internet]" /><meta name="citation_title" content="Nonsyndromic Hearing Loss and Deafness, DFNB1" /><meta name="citation_publisher" content="University of Washington, Seattle" /><meta name="citation_date" content="2016/08/18" /><meta name="citation_author" content="Richard JH Smith" /><meta name="citation_author" content="Mary-Kayt N Jones" /><meta name="citation_pmid" content="20301449" /><meta name="citation_fulltext_html_url" content="https://www.ncbi.nlm.nih.gov/books/NBK1272/" /><meta name="citation_keywords" content="GJB2-Related DFNB 1 Nonsyndromic Hearing Loss and Deafness" /><meta name="citation_keywords" content="GJB2-Related DFNB1 Nonsyndromic Hearing Loss and Deafness" /><meta name="citation_keywords" content="Gap junction beta-2 protein" /><meta name="citation_keywords" content="GJB2" /><meta name="citation_keywords" content="Nonsyndromic Hearing Loss and Deafness, DFNB1" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="DC.Title" content="Nonsyndromic Hearing Loss and Deafness, DFNB1" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="University of Washington, Seattle" /><meta name="DC.Contributor" content="Richard JH Smith" /><meta name="DC.Contributor" content="Mary-Kayt N Jones" /><meta name="DC.Date" content="2016/08/18" /><meta name="DC.Identifier" content="https://www.ncbi.nlm.nih.gov/books/NBK1272/" /><meta name="description" content="Nonsyndromic hearing loss and deafness (DFNB1) is characterized by congenital non-progressive mild-to-profound sensorineural hearing impairment. No other associated medical findings are present." /><meta name="og:title" content="Nonsyndromic Hearing Loss and Deafness, DFNB1" /><meta name="og:type" content="book" /><meta name="og:description" content="Nonsyndromic hearing loss and deafness (DFNB1) is characterized by congenital non-progressive mild-to-profound sensorineural hearing impairment. No other associated medical findings are present." /><meta name="og:url" content="https://www.ncbi.nlm.nih.gov/books/NBK1272/" /><meta name="og:site_name" content="NCBI Bookshelf" /><meta name="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" /><meta name="twitter:card" content="summary" /><meta name="twitter:site" content="@ncbibooks" /><meta name="bk-non-canon-loc" content="/books/n/gene/dfnb1/" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/books/NBK1272/" /><link rel="stylesheet" href="/corehtml/pmc/css/figpopup.css" type="text/css" media="screen" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books_print.min.css" type="text/css" media="print" /><style type="text/css">p a.figpopup{display:inline !important} .bk_tt {font-family: monospace}  .first-line-outdent .bk_ref {display: inline}  .body-content h2, .body-content .h2  {border-bottom: 1px solid #97B0C8} .body-content h2.inline {border-bottom: none} a.page-toc-label , .jig-ncbismoothscroll a {text-decoration:none;border:0 !important} .temp-labeled-list  .graphic {display:inline-block !important} .temp-labeled-list  img{width:100%}</style><script type="text/javascript" src="/corehtml/pmc/js/jquery.hoverIntent.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"> </script><script type="text/javascript">window.name="mainwindow";</script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/book-toc.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/books.min.js"> </script><meta name="book-collection" content="NONE" />

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="CE8D4892E04013710000000006870206.m_12" />
<meta name='referrer' content='origin-when-cross-origin'/><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4098876/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/4008682/3881636/3579733/4062871/12930/3964959/3854974/36029/4128070/9685/3549676/3609192/3609193/3609213/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3411343/3882866.css" media="print" /></head>
    <body class="book-part">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo"><h1 class="res_name"><a href="/books/" title="Bookshelf home">Bookshelf</a></h1><h2 class="res_tagline"></h2></div>
    <div class="search"><form method="get" action="/books/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="books" selected="selected" class="last" data-ac_dict="bookshelf-search">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books" data-ac_dict="bookshelf-search">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search Books. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'bookshelf-search',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'no'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/books/browse/">Browse Titles</a>
                    </li><li>
                        <a href="/books/advanced/">Advanced</a>
                    </li><li class="help">
                        <a href="/books/NBK3833/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <!-- Custom content 1 -->
<div class="col1">
    
</div>

<div class="container">
    <div id="maincontent" class="content eight_col col">
        <!-- Custom content in the left column above book nav -->
        <div class="col2">
            
        </div>
        
        <!-- Book content -->
        

        <!-- Custom content between navigation and content -->
        <div class="col3">
            
        </div>
        
        <div class="document">
            <div class="pre-content"><div><div class="bk_prnt"><p class="small">NCBI Bookshelf. A service of the National Library of Medicine, National Institutes of Health.</p><p>Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. </p></div><div class="iconblock clearfix whole_rhythm no_top_margin bk_noprnt"><a class="img_link icnblk_img" title="All GeneReviews" href="/books/n/gene/"><img class="source-thumb" src="/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" alt="Cover of GeneReviews®" height="100px" width="80px" /></a><div class="icnblk_cntnt eight_col"><h2>GeneReviews<sup>®</sup> [Internet].</h2><a data-jig="ncbitoggler" href="#__NBK1272_dtls__">Show details</a><div style="display:none" class="ui-widget" id="__NBK1272_dtls__"><div>Adam MP, Ardinger HH, Pagon RA, et al., editors.</div><div>Seattle (WA): <a href="http://www.washington.edu" ref="pagearea=page-banner&amp;targetsite=external&amp;targetcat=link&amp;targettype=publisher">University of Washington, Seattle</a>; 1993-2019.</div></div><div class="half_rhythm"><ul class="inline_list"><li style="margin-right:1em"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a></li></ul></div><div class="bk_noprnt"><form method="get" action="/books/n/gene/" id="bk_srch"><div class="bk_search"><label for="bk_term" class="offscreen_noflow">Search term</label><input type="text" title="Search GeneReviews" id="bk_term" name="term" value="" data-jig="ncbiclearbutton" /> <input type="submit" class="jig-ncbibutton" value="Search GeneReviews" submit="false" style="padding: 0.1em 0.4em;" /></div></form><div><ul class="inline_list"><li><a href="/books/n/gene/advanced/">GeneReviews Advanced Search</a></li><li style="margin-left:.5em"><a href="/books/n/gene/helpadvsearch/">Help</a></li></ul></div></div></div><div class="icnblk_cntnt two_col"><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/dfna3/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/mt-deafness/" title="Next page in this title">Next &gt;</a></div></div></div></div></div>
            <div class="main-content lit-style" itemscope="itemscope" itemtype="http://schema.org/CreativeWork"><div class="meta-content fm-sec"><h1 id="_NBK1272_"><span class="title" itemprop="name">Nonsyndromic Hearing Loss and Deafness, DFNB1</span></h1><div itemprop="alternativeHeadline" class="subtitle whole_rhythm">Synonym: <i>GJB2</i>-Related DFNB 1 Nonsyndromic Hearing Loss and Deafness</div><p class="contrib-group"><span itemprop="author">Richard JH Smith</span>, MD and <span itemprop="author">Mary-Kayt N Jones</span>, BA.</p><a data-jig="ncbitoggler" href="#__NBK1272_ai__" style="border:0;text-decoration:none">Author Information</a><div style="display:none" class="ui-widget" id="__NBK1272_ai__"><div class="contrib half_rhythm"><span itemprop="author">Richard JH Smith</span>, MD<div class="affiliation small">Department of Otolaryngology<br />University of Iowa Hospitals and Clinics<br />Iowa City, Iowa<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.awoiu@htims-drahcir" class="oemail">ude.awoiu@htims-drahcir</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Mary-Kayt N Jones</span>, BA<div class="affiliation small">Department of Otolaryngology<br />University of Iowa Hospitals and Clinics<br />Iowa City, Iowa<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="moc.liamg@1senojnkm" class="oemail">moc.liamg@1senojnkm</a></div></div></div></div><p class="small">Initial Posting: <span itemprop="datePublished">September 28, 1998</span>; Last Update: <span itemprop="dateModified">August 18, 2016</span>.</p><p><em>Estimated reading time: 20 minutes</em></p></div><div class="jig-ncbiinpagenav body-content whole_rhythm" data-jigconfig="allHeadingLevels: ['h2'],smoothScroll: false" itemprop="text"><div id="dfnb1.Summary" itemprop="description"><h2 id="_dfnb1_Summary_">Summary</h2><div><h4 class="inline">Clinical characteristics.</h4><p>Nonsyndromic hearing loss and deafness (DFNB1) is characterized by <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> non-progressive mild-to-profound sensorineural hearing impairment. No other associated medical findings are present.</p></div><div><h4 class="inline">Diagnosis/testing.</h4><p>Diagnosis of DFNB1 depends on <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> to identify <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in <i>GJB2</i> (sequence variants as well as variants in upstream <a class="def" href="/books/n/gene/glossary/def-item/cis/"><i>cis</i></a>-regulatory elements that alter expression of the gap junction beta-2 protein [connexin 26]).</p></div><div><h4 class="inline">Management.</h4><p><i>Treatment of manifestations:</i> Hearing aids; enrollment in appropriate educational programs; consideration of cochlear implantation for individuals with profound deafness.</p><p><i>Surveillance:</i> Surveillance includes annual examinations and repeat audiometry to confirm stability of hearing loss.</p><p><i>Evaluation of relatives at risk:</i> If both pathogenic variants have been identified in an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> family member, <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> can be used to clarify the genetic status of an at-risk relative in childhood so that appropriate early support and management can be provided.</p></div><div><h4 class="inline">Genetic counseling.</h4><p>DFNB1 is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> manner. In each pregnancy, the parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> have a 25% chance of having a deaf child, a 50% chance of having a hearing child who is a <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a>, and a 25% chance of having a hearing child who is not a carrier. When the <i>GJB2</i> pathogenic variants causing DFNB1 are detected in an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> family member, <a class="def" href="/books/n/gene/glossary/def-item/carrier-testing/">carrier testing</a> for at-risk relatives, prenatal testing for pregnancies at increased risk, and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> are possible.</p></div></div><div id="dfnb1.Diagnosis"><h2 id="_dfnb1_Diagnosis_">Diagnosis</h2><div id="dfnb1.Suggestive_Findings"><h3>Suggestive Findings</h3><p>Nonsyndromic hearing loss and deafness caused by <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic <i>GJB2</i> variants (DFNB1) <b>should be suspected</b> in individuals with the following:</p><ul><li class="half_rhythm"><div class="half_rhythm"><b>Congenital, generally non-progressive sensorineural hearing impairment</b> that is mild to profound by auditory brain stem response testing (ABR) or pure tone audiometry</div><div class="half_rhythm">Note: (1) Hearing is measured in <b>decibels</b> (dB). The threshold or 0-dB mark for each frequency refers to the level at which normal young adults perceive a tone burst 50% of the time. Hearing is considered normal if an individual's thresholds are within 25 dB of normal thresholds. (2) Severity of hearing loss is graded as mild (26-40 dB), moderate (41-55 dB), moderately severe (56-70 dB), severe (71-90 dB), or profound (90 dB). The frequency of hearing loss is designated as low (&#x0003c;500Hz), middle (501-2,000 Hz), or high (&#x0003e;2,000 Hz) (see <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a>).</div></li><li class="half_rhythm"><div class="half_rhythm"><b>No related systemic findings</b> identified by medical history and physical examination</div></li><li class="half_rhythm"><div class="half_rhythm"><b>A family history of nonsyndromic hearing loss</b> consistent with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> inheritance</div></li></ul></div><div id="dfnb1.Establishing_the_Diagnosis"><h3>Establishing the Diagnosis</h3><p>The diagnosis of DFNB1 <b>is established</b> in a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with mild to profound <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a>, generally non-progressive sensorineural hearing impairment and identification of <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in <i>GJB2</i> (encoding connexin 26) on <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> (see <a class="figpopup" href="/books/NBK1272/table/dfnb1.T.molecular_genetic_testing_used_i/?report=objectonly" target="object" rid-figpopup="figdfnb1Tmoleculargenetictestingusedi" rid-ob="figobdfnb1Tmoleculargenetictestingusedi">Table 1</a>).</p><p>Individuals with DFNB1 are EITHER:</p><ul><li class="half_rhythm"><div>Homozygous or <a class="def" href="/books/n/gene/glossary/def-item/compound-heterozygous/">compound heterozygous</a> for <i>GJB2</i> pathogenic variants (99%); OR</div></li><li class="half_rhythm"><div>Compound <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> for one <i>GJB2</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> and one of three large deletions that includes sequences upstream of <i>GJB2</i> and a portion of <i>GJB6</i> (&#x0003c;1%)<i>.</i></div></li></ul><p>Molecular genetic testing approaches can include a combination of <b><a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted testing</b> (single-gene testing and/or a <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a>) and <b><a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing</b> (comprehensive genomic sequencing).</p><p>Gene-targeted testing requires the clinician to determine which <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>(s) are likely involved based on phenotypic data, while comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing does not. Because of the overlapping phenotypes of the many causes of hereditary hearing loss and deafness, most individuals with hereditary hearing loss and deafness are diagnosed by one of two approaches: comprehensive genomic sequencing (<a href="#dfnb1.Recommended_Testing">recommended</a>) or gene-targeted testing (<a href="#dfnb1.Testing_to_Consider">to consider</a>).</p><div id="dfnb1.Recommended_Testing"><h4>Recommended Testing</h4><p><b>A comprehensive deafness-specific genetic panel</b> that includes all genes implicated in nonsyndromic hearing loss and nonsyndromic hearing loss mimics (see <a href="#dfnb1.Differential_Diagnosis">Differential Diagnosis</a> and <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a>) is recommended as the initial genetic test (<a class="figpopup" href="/books/NBK1272/figure/dfnb1.F1/?report=objectonly" target="object" rid-figpopup="figdfnb1F1" rid-ob="figobdfnb1F1">Figure 1</a>).</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figdfnb1F1" co-legend-rid="figlgnddfnb1F1"><a href="/books/NBK1272/figure/dfnb1.F1/?report=objectonly" target="object" title="Figure 1. " class="img_link icnblk_img figpopup" rid-figpopup="figdfnb1F1" rid-ob="figobdfnb1F1"><img class="small-thumb" src="/books/NBK1272/bin/dfnb1-Image001.gif" src-large="/books/NBK1272/bin/dfnb1-Image001.jpg" alt="Figure 1. . Genetic diagnostic rates in 1,119 sequentially accrued persons with hearing loss." /></a><div class="icnblk_cntnt" id="figlgnddfnb1F1"><h4 id="dfnb1.F1"><a href="/books/NBK1272/figure/dfnb1.F1/?report=objectonly" target="object" rid-ob="figobdfnb1F1">Figure 1. </a></h4><p class="float-caption no_bottom_margin">Genetic diagnostic rates in 1,119 sequentially accrued persons with hearing loss. No person was excluded based on phenotype, inheritance, or previous testing. Testing resulted in identification of the underlying genetic cause for hearing loss in 440 individuals <a href="/books/NBK1272/figure/dfnb1.F1/?report=objectonly" target="object" rid-ob="figobdfnb1F1">(more...)</a></p></div></div><p>Note: (1) Genes included in available panels and the diagnostic <a class="def" href="/books/n/gene/glossary/def-item/sensitivity/">sensitivity</a> of the test used for each <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> vary by laboratory and are likely to change over time [<a class="bk_pop" href="#dfnb1.REF.shearer.2015.175">Shearer &#x00026; Smith 2015</a>]. (2) Some multigene panels may include genes not associated with the condition discussed in this <i>GeneReview</i>; thus, clinicians need to determine which <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a> provides the best opportunity to identify the genetic cause of the condition at the most reasonable cost while limiting identification of variants of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a> and pathogenic variants in genes that do not explain the underlying <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>. (3) Methods used in a panel may include <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a>, <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>, and/or other non-sequencing-based tests; detection of copy number variants must be included in hearing loss panels [<a class="bk_pop" href="#dfnb1.REF.shearer.2014.37">Shearer et al 2014</a>].</p><p>For an introduction to multigene panels click <a href="/books/n/gene/app5/#app5.Multigene_Panels">here</a>. More detailed information for clinicians ordering genetic tests can be found <a href="/books/n/gene/app5/#app5.Multigene_Panels_FAQs">here</a>.</p></div><div id="dfnb1.Testing_to_Consider"><h4>Testing to Consider</h4><p><b>Single-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> testing</b> can be considered if a deafness-specific <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a> is not available. However, performing <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> of <i>GJB2</i> alone is not cost-effective unless it is limited to persons with severe-to-profound <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> nonsyndromic hearing loss. Offering single-gene testing of <i>GJB2</i> reflexively to everyone with congenital hearing loss without regard to the degree of hearing loss is not evidence based and not cost effective [<a class="bk_pop" href="#dfnb1.REF.jayawardena.2015.843">Jayawardena et al 2015</a>, <a class="bk_pop" href="#dfnb1.REF.shearer.2015.175">Shearer &#x00026; Smith 2015</a>].</p><p><b>Comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing</b> including <a class="def" href="/books/n/gene/glossary/def-item/exome-sequencing/">exome sequencing</a> and <a class="def" href="/books/n/gene/glossary/def-item/genome-sequencing/">genome sequencing</a> may be considered if the <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> alone is insufficient to warrant <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted testing. Both exome sequencing and genome sequencing should be complemented with appropriate <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> before and after testing.</p><p>For an introduction to comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing click <a href="/books/n/gene/app5/#app5.Comprehensive_Genomic_Testing">here</a>. More detailed information for clinicians ordering genomic testing can be found <a href="/books/n/gene/app5/#app5.Comprehensive_Genomic_Testing_1">here</a>.</p><div id="dfnb1.T.molecular_genetic_testing_used_i" class="table"><h3><span class="label">Table 1. </span></h3><div class="caption"><p>Molecular Genetic Testing Used in DFNB1</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1272/table/dfnb1.T.molecular_genetic_testing_used_i/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfnb1.T.molecular_genetic_testing_used_i_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfnb1.T.molecular_genetic_testing_used_i_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene&#x000a0;<sup>1</sup></th><th id="hd_h_dfnb1.T.molecular_genetic_testing_used_i_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Method</th><th id="hd_h_dfnb1.T.molecular_genetic_testing_used_i_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Proportion of Probands with Pathogenic Variants&#x000a0;<sup>2,</sup>&#x000a0;<sup>3</sup> Detectable by Method</th></tr></thead><tbody><tr><td headers="hd_h_dfnb1.T.molecular_genetic_testing_used_i_1_1_1_1" rowspan="2" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><i>GJB2</i></td><td headers="hd_h_dfnb1.T.molecular_genetic_testing_used_i_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Sequence analysis&#x000a0;<sup>4</sup></td><td headers="hd_h_dfnb1.T.molecular_genetic_testing_used_i_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">&#x0003e; 99%</td></tr><tr><td headers="hd_h_dfnb1.T.molecular_genetic_testing_used_i_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>&#x000a0;<sup>5,</sup>&#x000a0;<sup>6</sup></td><td headers="hd_h_dfnb1.T.molecular_genetic_testing_used_i_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">&#x0003c;1%</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="dfnb1.TF.1.1"><p class="no_margin">See <a href="/books/NBK1272/#dfnb1.molgen.TA">Table A. Genes and Databases</a> for <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> and protein.</p></div></dd><dt>2. </dt><dd><div id="dfnb1.TF.1.2"><p class="no_margin">See <a href="#dfnb1.Molecular_Genetics">Molecular Genetics</a> for information on allelic variants detected in this <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>.</p></div></dd><dt>3. </dt><dd><div id="dfnb1.TF.1.3"><p class="no_margin">Percentages vary depending on ethnicity. Numbers in table reflect screening of a US population primarily of northern European ancestry.</p></div></dd><dt>4. </dt><dd><div id="dfnb1.TF.1.4"><p class="no_margin">Sequence analysis detects variants that are benign, likely benign, of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a>, likely pathogenic, or pathogenic. Pathogenic variants may include small intragenic deletions/insertions and <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a>, <a class="def" href="/books/n/gene/glossary/def-item/nonsense-variant/">nonsense</a>, and <a class="def" href="/books/n/gene/glossary/def-item/splice-site/">splice site</a> variants; typically, <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> or whole-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> deletions/duplications are not detected. For issues to consider in interpretation of <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> results, click <a href="/books/n/gene/app2/">here</a>.</p></div></dd><dt>5. </dt><dd><div id="dfnb1.TF.1.5"><p class="no_margin">Gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a> detects intragenic deletions or duplications. Methods used may include <a class="def" href="/books/n/gene/glossary/def-item/quantitative-pcr/">quantitative PCR</a>, long-range PCR, multiplex ligation-dependent probe amplification (MLPA), and a <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted microarray designed to detect single-<a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> deletions or duplications.</p></div></dd><dt>6. </dt><dd><div id="dfnb1.TF.1.6"><p class="no_margin">Gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a>/<a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a> testing will detect deletions ranging from a single <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> to the whole <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>; however, breakpoints of large deletions and/or deletion of adjacent genes (e.g., those described by <a class="bk_pop" href="#dfnb1.REF.feldmann.2009.195">Feldmann et al [2009]</a>) may not be detected by these methods.</p></div></dd></dl></div></div></div></div></div></div><div id="dfnb1.Clinical_Characteristics"><h2 id="_dfnb1_Clinical_Characteristics_">Clinical Characteristics</h2><div id="dfnb1.Clinical_Description"><h3>Clinical Description</h3><p>Nonsyndromic hearing loss and deafness (DFNB1) is characterized by <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> (present at birth) non-progressive sensorineural hearing impairment. Intrafamilial variability in the degree of deafness is seen.</p><ul><li class="half_rhythm"><div>If an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> person has severe-to-profound deafness, an affected sib with the same <i>GJB2</i> pathogenic variants has a 91% chance of having severe-to-profound deafness and a 9% chance of having mild-to-moderate deafness [<a class="bk_pop" href="#dfnb1.REF39">Tennessee Department of Health 2005</a>].</div></li><li class="half_rhythm"><div>If an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> person has mild-to-moderate deafness, an affected sib with the same <i>GJB2</i> pathogenic variants has a 66% chance of having mild-to-moderate deafness and a 34% chance of having severe-to-profound deafness [<a class="bk_pop" href="#dfnb1.REF39">Tennessee Department of Health 2005</a>].</div></li><li class="half_rhythm"><div>A few reports describe children with <i>GJB2</i> variants who pass the newborn hearing screen and have somewhat later-onset hearing loss [<a class="bk_pop" href="#dfnb1.REF.norris.2006.732">Norris et al 2006</a>, <a class="bk_pop" href="#dfnb1.REF.orzan.2007.501">Orzan &#x00026; Murgia 2007</a>].</div></li></ul><p>In a large cross-sectional analysis of <i>GJB2</i> <a class="def" href="/books/n/gene/glossary/def-item/genotype/">genotype</a> and audiometric data from 1,531 individuals with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> mild-to-profound nonsyndromic deafness (median age 8 years; 90% within age 0-26 years) from 16 countries, linear regression analysis of hearing thresholds on age in the entire study and in subsets defined by genotype did not show significant progression of hearing loss in any individual [<a class="bk_pop" href="#dfnb1.REF.snoeckx.2005.945">Snoeckx et al 2005</a>]. This finding is in concordance with prior studies [<a class="bk_pop" href="#dfnb1.REF.orzan.1999.291">Orzan et al 1999</a>, <a class="bk_pop" href="#dfnb1.REF.l_ffler.2001.226">L&#x000f6;ffler et al 2001</a>]; however, progression of hearing loss cannot be excluded definitively given the cross-sectional nature of the regression analysis.</p><p>Although <a class="bk_pop" href="#dfnb1.REF.snoeckx.2005.945">Snoeckx et al [2005]</a> found a slight degree of asymmetry, the difference in pure tone average at 0.5, 1.0, and 2.0 kHz between ears was less than 15 dB in 90% of individuals.</p><p>Vestibular function is normal; <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> infants and young children do not experience balance problems and learn to sit and walk at age-appropriate times.</p><p>Except for the hearing impairment, <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals are healthy; life span is normal.</p></div><div id="dfnb1.GenotypePhenotype_Correlations"><h3>Genotype-Phenotype Correlations</h3><p>Numerous studies have shown that it is possible to predict <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> based on <a class="def" href="/books/n/gene/glossary/def-item/genotype/">genotype</a>. The largest study to date involved a cross-sectional analysis of <i>GJB2</i> genotype and audiometric data from 1,531 persons from 16 different countries with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> mild-to-profound nonsyndromic deafness [<a class="bk_pop" href="#dfnb1.REF.snoeckx.2005.945">Snoeckx et al 2005</a>]. Of the 83 different variants identified, 47 were predicted non-truncating (e.g., <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants) and 36 were predicted truncating (e.g., premature stop codons). By classifying variants this way, the authors defined three genotype classes: <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> truncating (T/T) variants, biallelic non-truncating (NT/NT) variants, and <a class="def" href="/books/n/gene/glossary/def-item/compound-heterozygous/">compound heterozygous</a> truncating/non-truncating (T/NT) variants (<a class="figpopup" href="/books/NBK1272/table/dfnb1.T.genotypephenotype_correlations_b/?report=objectonly" target="object" rid-figpopup="figdfnb1Tgenotypephenotypecorrelationsb" rid-ob="figobdfnb1Tgenotypephenotypecorrelationsb">Table 2</a>).</p><div id="dfnb1.T.genotypephenotype_correlations_b" class="table"><h3><span class="label">Table 2. </span></h3><div class="caption"><p>Genotype-Phenotype Correlations by Variant Type in 1531 Persons with Biallelic <i>GJB2</i> Pathogenic Variants</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1272/table/dfnb1.T.genotypephenotype_correlations_b/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfnb1.T.genotypephenotype_correlations_b_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_1" rowspan="2" scope="col" colspan="1" headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_1" style="text-align:left;vertical-align:middle;"><i>GJB2</i> Genotype Class</th><th id="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_2" rowspan="2" scope="col" colspan="1" headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_2" style="text-align:left;vertical-align:middle;">Total of All DFNB1</th><th id="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3" colspan="4" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">Hearing Loss</th></tr><tr><th headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3" id="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_1" colspan="1" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">Mild</th><th headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3" id="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Moderate</th><th headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3" id="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Severe</th><th headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3" id="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Profound</th></tr></thead><tbody><tr><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">T/T</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">77.3%&#x000a0;<sup>1</sup></td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">0%-3%</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">10%-12%</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">25%-28%</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">59%-64%</td></tr><tr><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">NT/NT</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">6.2%&#x000a0;<sup>2</sup></td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">53%</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">26%</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">8%</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">13%</td></tr><tr><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">T/NT</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">16.5%&#x000a0;<sup>3</sup></td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">29%-37%</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">24%-29%</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">10%-17%</td><td headers="hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_1_3 hd_h_dfnb1.T.genotypephenotype_correlations_b_1_1_2_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">24%-30%</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Based on <a class="bk_pop" href="#dfnb1.REF.snoeckx.2005.945">Snoeckx et al [2005]</a>. See <a href="http://www.sciencedirect.com/science/article/pii/S0002929707633801" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">full text</a>, Figure 3 for scatter diagrams showing the binaural mean pure tone average (PTA) at 0.5, 1, and 2 kHz (PTA<sub>0.5,1,2kHz</sub>) for each person within each <a class="def" href="/books/n/gene/glossary/def-item/genotype/">genotype</a> class, using individuals <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a> for the c.35delG <a class="def" href="/books/n/gene/glossary/def-item/allele/">allele</a> as a reference group.</p></div></dd><dt></dt><dd><div><p class="no_margin">NT/NT = <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> predicted non-truncating variants; T/NT = <a class="def" href="/books/n/gene/glossary/def-item/compound-heterozygous/">compound heterozygous</a> predicted truncating/non-truncating variants; T/T = biallelic predicted truncating variants</p></div></dd><dt>1. </dt><dd><div id="dfnb1.TF.2.1"><p class="no_margin">64 different genotypes (36% of all genotypes)</p></div></dd><dt>2. </dt><dd><div id="dfnb1.TF.2.2"><p class="no_margin">42 different genoytpes (24% of all genotypes)</p></div></dd><dt>3. </dt><dd><div id="dfnb1.TF.2.3"><p class="no_margin">71 different genotypes (40% of all genotypes)</p></div></dd></dl></div></div></div></div><div id="dfnb1.Nomenclature"><h3>Nomenclature</h3><p>DFNB with a suffix integer is used to designate loci for <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic deafness.</p></div><div id="dfnb1.Prevalence"><h3>Prevalence</h3><p>DFNB1 accounts for approximately 50% of <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> severe-to-profound <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic hearing loss in the United States, France, Britain, and New Zealand/Australia [<a class="bk_pop" href="#dfnb1.REF.green.1999.2211">Green et al 1999</a>, <a class="bk_pop" href="#dfnb1.REF.azaiez.2004.305">Azaiez et al 2004</a>, <a class="bk_pop" href="#dfnb1.REF.angeli.2008.2014">Angeli 2008</a>]. Its approximate prevalence in the general population is 14:100,000, based on the following calculation: the incidence of congenital hereditary hearing impairment is 1:2,000 neonates, of which 70% have nonsyndromic hearing loss. Seventy-five to 80% of cases of nonsyndromic hearing loss are autosomal recessive; of these, 50% result from <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in <i>GJB2</i>. Thus, 5:10,000 x 0.7 x 0.8 x 0.5 = 14:100,000.</p><p>Given the extreme heterogeneity of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic hearing impairment, it is not surprising that epidemiologic studies in other populations have shown that the frequency of <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> <i>GJB2</i> pathogenic variants as a cause of hearing impairment is highly variable. For example, among families segregating autosomal recessive nonsyndromic hearing impairment, <i>GJB2</i> variants are causally related to <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> hereditary hearing impairment in:</p><ul><li class="half_rhythm"><div>25% of Palestinian families [<a class="bk_pop" href="#dfnb1.REF.shahin.2002.284">Shahin et al 2002</a>]</div></li><li class="half_rhythm"><div>An estimated 24% of Altaians from Siberia [<a class="bk_pop" href="#dfnb1.REF.posukh.2005.12">Posukh et al 2005</a>]</div></li><li class="half_rhythm"><div>18% of Han Chinese [<a class="bk_pop" href="#dfnb1.REF.duan.2015.586">Duan et al 2015</a>]</div></li><li class="half_rhythm"><div>16% of the Iranian deaf population [<a class="bk_pop" href="#dfnb1.REF.bazazzadegan.2012.1164">Bazazzadegan et al 2012</a>]</div></li><li class="half_rhythm"><div>15% of Hui people [<a class="bk_pop" href="#dfnb1.REF.duan.2015.586">Duan et al 2015</a>]</div></li><li class="half_rhythm"><div>11% of Tibetan families [<a class="bk_pop" href="#dfnb1.REF.duan.2015.586">Duan et al 2015</a>]</div></li></ul></div></div><div id="dfnb1.Genetically_Related_Disorders"><h2 id="_dfnb1_Genetically_Related_Disorders_">Genetically Related Disorders</h2><p><b>Allelic disorders.</b> The following <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> disorders are caused by <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> pathogenic variants in <i>GJB2</i>:</p><ul><li class="half_rhythm"><div><a href="/books/n/gene/dfna3/"><b>DFNA3</b></a> is characterized by progressive, moderate-to-severe sensorineural impairment. It is less commonly caused by <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> pathogenic variants in <i>GJB6</i>.</div></li><li class="half_rhythm"><div><b>Palmoplantar keratoderma with deafness</b> (OMIM <a href="http://omim.org/entry/148350" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">148350</a>) is characterized by diffuse hyperkeratosis of the hands and feet [<a class="bk_pop" href="#dfnb1.REF.richard.1998.393">Richard et al 1998</a>, <a class="bk_pop" href="#dfnb1.REF.heathcote.2000.50">Heathcote et al 2000</a>].</div></li><li class="half_rhythm"><div><b>Keratitis-ichthyosis-deafness (KID) syndrome</b> (OMIM <a href="http://omim.org/entry/148210" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">148210</a>) is an ectodermal dysplasia in which <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals have vascularizing keratitis, progressive erythrokeratoderma, and profound sensorineural hearing loss, as well as scarring alopecia and predisposition to squamous cell carcinoma [<a class="bk_pop" href="#dfnb1.REF.richard.2002.1341">Richard et al 2002</a>, <a class="bk_pop" href="#dfnb1.REF.van_geel.2002.938">van Geel et al 2002</a>, <a class="bk_pop" href="#dfnb1.REF.van_steensel.2002.724">van Steensel et al 2002</a>].</div></li><li class="half_rhythm"><div><b>Hystrix-like ichthyosis-deafness (HID) syndrome</b> (OMIM <a href="http://omim.org/entry/602540" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">602540</a>) is a keratinizing disorder characterized by sensorineural hearing loss and hyperkeratosis of the skin. Shortly after birth, erythroderma develops, with spiky and cobblestone-like hyperkeratosis of the entire skin surface appearing by age one year. Severe palmoplantar keratoderma and scarring alopecia occur in some. HID syndrome is considered to differ from KID syndrome in: (1) the extent and time of occurrence of skin symptoms; (2) the severity of keratitis; and (3) electron microscopic features [<a class="bk_pop" href="#dfnb1.REF.van_geel.2002.938">van Geel et al 2002</a>].</div></li><li class="half_rhythm"><div><b>Vohwinkel syndrome</b> (OMIM <a href="http://omim.org/entry/124500" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">124500</a>) is a "mutilating" diffuse keratoderma because circumferential hyperkeratosis of the digits can lead to autoamputation. Mild-to-moderate sensorineural hearing loss is often associated with the disease [<a class="bk_pop" href="#dfnb1.REF.maestrini.1999.1237">Maestrini et al 1999</a>].</div></li><li class="half_rhythm"><div>With some variants in <i>GJB2</i>, the epidermal disease and hearing loss cosegregate, while with other variants, the severity of the disease <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> varies, suggesting that other factors modify <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> expression [<a class="bk_pop" href="#dfnb1.REF.kutkowskaka_mierczak.2015.329">Kutkowska-Ka&#x0017a;mierczak et al 2015</a>].</div></li></ul><p><b>Other.</b> A large <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> encompassing <i>GJB2</i>, the adjacent genes <i>GJA3</i> and <i>GJB6</i>, and a portion of <i>CRYL1</i> in <a class="def" href="/books/n/gene/glossary/def-item/trans/"><i>trans</i></a> with a <i>GJB2</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> was identified in an individual with a contiguous <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> syndrome characterized by profound prelingual hearing loss, mental and psychomotor development delay, clinodactyly of the second toes, and a frontal tuft [<a class="bk_pop" href="#dfnb1.REF.feldmann.2009.195">Feldmann et al 2009</a>].</p></div><div id="dfnb1.Differential_Diagnosis"><h2 id="_dfnb1_Differential_Diagnosis_">Differential Diagnosis</h2><p>See <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a> for information on the genes causing nonsyndromic hereditary hearing loss and deafness.</p><div id="dfnb1.T.autosomal_recessive_syndromes_in" class="table"><h3><span class="label">Table 3. </span></h3><div class="caption"><p>Autosomal Recessive Syndromes Involving Hearing Loss</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1272/table/dfnb1.T.autosomal_recessive_syndromes_in/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfnb1.T.autosomal_recessive_syndromes_in_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Syndrome</th><th id="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Distinctive Feature in Addition to Hearing Loss</th><th id="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene(s)</th><th id="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Hearing Loss</th><th id="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_5" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Comments</th></tr></thead><tbody><tr><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/usher1/">Usher syndrome type I</a></td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_2" rowspan="3" colspan="1" style="text-align:left;vertical-align:middle;">Retinitis pigmentosa&#x000a0;<sup>1</sup></td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">See footnote 2</td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Congenital bilateral profound sensorineural hearing loss</td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Vestibular areflexia w/delay in motor milestones (delayed sitting &#x00026; walking)</div></li><li class="half_rhythm"><div>Unless fitted w/a cochlear implant, individuals w/Usher syndrome type 1 do not typically develop speech.</div></li><li class="half_rhythm"><div>RP develops in adolescence, resulting in progressively constricted visual fields &#x00026; impaired visual acuity</div></li></ul>
</td></tr><tr><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/usher2/">Usher syndrome type II</a></td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_3" colspan="1" rowspan="1" style="text-align:left;vertical-align:middle;"><i>USH2A</i><br /><i>ADGRV1</i><br /><i>DFNB31</i></td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Congenital bilateral sensorineural hearing loss: mild-moderate in low frequencies; severe-profound in higher frequencies</td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Intact vestibular responses w/no delay in motor milestones</div></li><li class="half_rhythm"><div>Clinical distinction between Usher syndrome types I &#x00026; II: children w/type I usually delayed in walking until age 18 mos &#x02013; 2 yrs (because of vestibular involvement); children w/type II usually begin walking at ~1 yr</div></li></ul>
</td></tr><tr><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Usher syndrome type III</td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_3" colspan="1" rowspan="1" style="text-align:left;vertical-align:middle;"><i>CLRN1</i></td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Postlingual progressive sensorineural hearing loss</td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Late-onset RP</div></li><li class="half_rhythm"><div>Variable impairment of vestibular function</div></li><li class="half_rhythm"><div>Older individuals w/Usher syndrome type III may have profound hearing loss &#x00026; vestibular disturbance resembling Usher syndrome type I</div></li></ul>
</td></tr><tr><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/pendred/">Pendred syndrome</a></td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Thyroid enlargement</td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>SLC26A4</i>&#x000a0;<sup>3</sup></td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Hearing impairment usually <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> &#x00026; often severe-profound (mild-moderate progressive hearing impairment also occurs)</td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Bilateral dilation of the vestibular aqueduct&#x000a0;<sup>4</sup> w/ or w/out cochlear hypoplasia&#x000a0;<sup>5</sup></div></li><li class="half_rhythm"><div>Either an abnormal perchlorate discharge test or goiter</div></li><li class="half_rhythm"><div>Thyroid abnormality variable&#x000a0;<sup>6</sup></div></li><li class="half_rhythm"><div>Vestibular function usually abnormal</div></li></ul>
</td></tr><tr><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/jln/">Jervell and Lange-Nielsen syndrome</a></td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Cardiac conduction defects; long QTc, usually &#x0003e;500 msec</td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>KCNQ1</i><br /><i>KCNE1</i></td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Congenital profound bilateral sensorineural hearing loss</td><td headers="hd_h_dfnb1.T.autosomal_recessive_syndromes_in_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Long QTc is associated w/tachyarrhythmias, which may culminate in syncope or sudden death; &#x0003e;50% of untreated children w/JLNS die before age 15 years&#x000a0;<sup>7</sup></div></li><li class="half_rhythm"><div>Classic presentation: deaf child who experiences syncopal episodes during periods of stress, exercise, or fright</div></li><li class="half_rhythm"><div>Consider JLNS in any child w/<a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> sensorineural deafness &#x00026; negative DFNB1 testing &#x02013; esp. w/a history of syncope or seizure or family history of sudden death before age 40 years</div></li></ul>
</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="dfnb1.TF.3.1"><p class="no_margin">Retinitis pigmentosa is a progressive bilateral symmetric degeneration of rod and cone functions of the retina.</p></div></dd><dt>2. </dt><dd><div id="dfnb1.TF.3.2"><p class="no_margin">Pathogenic variants in genes at a minimum of nine different loci cause Usher syndrome type I. Genes at six of these loci &#x02013; <i>MYO7A</i> (USH1B), <i>USH1C</i>, <i>CDH23</i> (USH1D), <i>PCDH15</i> (USH1F), <i>USH1G</i>, and <i>CIB2</i> (USH1J) &#x02013; have been identified.</p></div></dd><dt>3. </dt><dd><div id="dfnb1.TF.3.3"><p class="no_margin">Pendred syndrome and DFNB4 comprise a phenotypic spectrum caused by <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in <i>SLC26A4</i> (the most common cause), or double heterozygosity in either <i>SLC26A4</i> and <i>FOXI1</i> or <i>SLC26A4</i> and <i>KCNJ10</i>.</p></div></dd><dt>4. </dt><dd><div id="dfnb1.TF.3.4"><p class="no_margin">Also called enlarged vestibular aqueduct (EVA)</p></div></dd><dt>5. </dt><dd><div id="dfnb1.TF.3.5"><p class="no_margin">DVA with cochlear hypoplasia is known as Mondini malformation or dysplasia.</p></div></dd><dt>6. </dt><dd><div id="dfnb1.TF.3.6"><p class="no_margin">Goitrous changes are typically not present at birth but do develop in early puberty (40%) or adulthood (60%).</p></div></dd><dt>7. </dt><dd><div id="dfnb1.TF.3.7"><p class="no_margin">Treatment involves use of beta-adrenergic blockers, cardiac pacemakers, and implantable defibrillators as well as avoidance of drugs that cause further prolongation of the QT interval and of activities known to precipitate syncopal events.</p></div></dd></dl></div></div></div><p><b>Autosomal recessive nonsyndromic hearing loss</b> without an identifiable <i>GJB2</i> variant and with progression of hearing loss:</p><ul><li class="half_rhythm"><div>With a dilated vestibular aqueduct on thin-cut computed tomography (CT) of the temporal bones suggests <a href="/books/n/gene/pendred/">DFNB4</a> [<a class="bk_pop" href="#dfnb1.REF.azaiez.2007.451">Azaiez et al 2007</a>];</div></li><li class="half_rhythm"><div>With a Mondini malformation on thin-cut CT of the temporal bones suggests <a href="/books/n/gene/pendred/">Pendred syndrome</a>. A perchlorate test and <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> of <i>SLC26A4</i> should be considered [<a class="bk_pop" href="#dfnb1.REF.azaiez.2007.451">Azaiez et al 2007</a>].</div></li></ul><p><b>Other causes of <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> severe-to-profound hearing loss</b> should be considered in children who represent single cases in a family:</p><ul><li class="half_rhythm"><div>Congenital CMV (cytomegalovirus), the most common cause of <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> non-hereditary hearing loss</div></li><li class="half_rhythm"><div>Prematurity, low birth weight, low Apgar scores, infection, and any illness requiring care in a neonatal intensive care unit</div></li></ul></div><div id="dfnb1.Management"><h2 id="_dfnb1_Management_">Management</h2><div id="dfnb1.Evaluations_Following_Initial_Diag"><h3>Evaluations Following Initial Diagnosis</h3><p>To establish the extent of involvement and needs in an individual diagnosed with nonsyndromic hearing loss and deafness caused by <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in <i>GJB2</i> (DFNB1), the following evaluations are recommended:</p><ul><li class="half_rhythm"><div>Complete assessment of auditory acuity using age-appropriate tests such as ABR testing, auditory steady-state response (ASSR) testing, and pure tone audiometry</div></li><li class="half_rhythm"><div>Ophthalmologic evaluation for refractive errors</div></li><li class="half_rhythm"><div>Consultation with a clinical geneticist and/or genetic counselor</div></li></ul></div><div id="dfnb1.Treatment_of_Manifestations"><h3>Treatment of Manifestations</h3><p>The following are indicated:</p><ul><li class="half_rhythm"><div>Fitting with appropriate hearing aids</div></li><li class="half_rhythm"><div>Enrollment in an appropriate educational program for the hearing impaired</div></li><li class="half_rhythm"><div>Consideration of cochlear implantation (CI), an excellent habilitation option for persons with profound deafness</div></li><li class="half_rhythm"><div>Recognition that, unlike with many clinical conditions, the management and treatment of severe-to-profound <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> deafness falls largely within the purview of the social welfare and educational systems rather than the medical care system [<a class="bk_pop" href="#dfnb1.REF.smith.2005.879">Smith et al 2005</a>]</div></li></ul><p>Also see <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness</a> for more detailed discussion of management issues.</p></div><div id="dfnb1.Surveillance"><h3>Surveillance</h3><p>The following are appropriate:</p><ul><li class="half_rhythm"><div>Annual examination by a physician familiar with hereditary hearing impairment</div></li><li class="half_rhythm"><div>Repeat audiometry to confirm stability of hearing loss</div></li></ul></div><div id="dfnb1.AgentsCircumstances_to_Avoid"><h3>Agents/Circumstances to Avoid</h3><p>Individuals with hearing loss should avoid environmental exposures known to cause hearing loss. Most important among these for persons with DFNB1 and mild-to-moderate hearing loss is avoidance of repeated overexposure to loud noises.</p></div><div id="dfnb1.Evaluation_of_Relatives_at_Risk"><h3>Evaluation of Relatives at Risk</h3><p>If both <i>GJB2</i> pathogenic variants have been identified in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>, it is appropriate to clarify the genetic status of at-risk sibs shortly after birth so that appropriate early support and management can be provided to the child and family.</p><p>See <a href="#dfnb1.Related_Genetic_Counseling_Issues">Genetic Counseling</a> for issues related to testing of at-risk relatives for <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> purposes.</p></div><div id="dfnb1.Therapies_Under_Investigation"><h3>Therapies Under Investigation</h3><p>Search <a href="http://clinicaltrials.gov/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">ClinicalTrials.gov</a> in the US and <a href="http://www.clinicaltrialsregister.eu/ctr-search/search" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">EU Clinical Trials Register</a> in Europe for information on clinical studies for a wide range of diseases and conditions. Note: There may not be clinical trials for this condition.</p></div></div><div id="dfnb1.Genetic_Counseling"><h2 id="_dfnb1_Genetic_Counseling_">Genetic Counseling</h2><p>
<i>Genetic counseling is the process of providing individuals and families with
information on the nature, inheritance, and implications of genetic disorders to help them
make informed medical and personal decisions. The following section deals with genetic
risk assessment and the use of family history and genetic testing to clarify genetic
status for family members. This section is not meant to address all personal, cultural, or
ethical issues that individuals may face or to substitute for consultation with a genetics
professional</i>. &#x02014;ED.</p><div id="dfnb1.Mode_of_Inheritance"><h3>Mode of Inheritance</h3><p>Nonsyndromic hearing loss and deafness caused by <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> <i>GJB2</i> pathogenic variants (DFNB1) is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> manner.</p></div><div id="dfnb1.Risk_to_Family_Members"><h3>Risk to Family Members</h3><p><b>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>The parents of a child with DFNB1 are obligate heterozygotes (i.e., carriers of one <i>GJB2</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>).</div></li><li class="half_rhythm"><div>Heterozygotes are asymptomatic.</div></li></ul><p><b>Sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>At conception, each sib of an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individual has a 25% chance of being affected, a 50% chance of being an asymptomatic <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a>, and a 25% chance of being unaffected and not a carrier.</div></li><li class="half_rhythm"><div>Heterozygotes are asymptomatic.</div></li></ul><p><b>Offspring of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</b> The offspring of an individual with DFNB1 are obligate heterozygotes (carriers) for a <i>GJB2</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>.</p><p><b>Other family members.</b> Each sib of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>&#x02019;s parents has a 50% chance of being a <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a> of a <i>GJB2</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>.</p></div><div id="dfnb1.Carrier_Heterozygote_Detection"><h3>Carrier (Heterozygote) Detection</h3><p>Carrier testing for relatives of an individual with DFNB1 requires prior identification of the <i>GJB2</i> pathogenic variants in the family.</p></div><div id="dfnb1.Related_Genetic_Counseling_Issues"><h3>Related Genetic Counseling Issues</h3><p>See Management, <a href="#dfnb1.Evaluation_of_Relatives_at_Risk">Evaluation of Relatives at Risk</a> for information on evaluating at-risk relatives for the purpose of early diagnosis and treatment.</p><p>The following points are noteworthy:</p><ul><li class="half_rhythm"><div>Communication with individuals who are members of the Deaf community and who sign requires the services of a skilled interpreter.</div></li><li class="half_rhythm"><div>Members of the Deaf community may view deafness as a distinguishing characteristic and not as a handicap, impairment, or medical condition requiring a "treatment" or "cure," or to be "prevented."</div></li><li class="half_rhythm"><div>Many deaf people are interested in obtaining information about the cause of their own deafness, including information on medical, educational, and social services, rather than information about prevention, reproduction, or family planning.</div></li><li class="half_rhythm"><div>The use of certain terms is preferred: probability or chance vs risk; deaf and hard-of-hearing vs hearing impaired. Terms such as "abnormal" should be avoided.</div></li></ul><p><b>Family planning</b></p><ul><li class="half_rhythm"><div>The optimal time for clarification of <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a> status and discussion of the availability of prenatal testing is before pregnancy.</div></li><li class="half_rhythm"><div>It is appropriate to offer <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> to young adults who are deaf or who may be carriers.</div></li></ul><p><b>DNA banking</b> is the storage of DNA (typically extracted from white blood cells) for possible future use. Because it is likely that testing methodology and our understanding of genes and allelic variants will improve in the future, consideration should be given to banking DNA of <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals.</p></div><div id="dfnb1.Prenatal_Testing_and_Preimplantati"><h3>Prenatal Testing and Preimplantation Genetic Diagnosis</h3><p>Once the <i>GJB2</i> pathogenic variants have been identified in a family member with DFNB1, prenatal testing for a pregnancy at increased risk and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> for DFNB1 are possible.</p><p>Many deaf individuals are interested in obtaining information about the underlying etiology of their hearing loss rather than information about reproductive risks. It is, therefore, important to ascertain and address the questions and concerns of the family/individual. "In contrast to the medical model which considers deafness to be a pathologic condition, many deaf people do not consider themselves to be handicapped but define themselves as being part of a distinct cultural group with its own language, customs, and beliefs. Strategies for effective <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> to deaf people include the recognition that perception of risk is very subjective and that some deaf individuals may prefer to have deaf children." &#x02014; from <a class="bk_pop" href="#dfnb1.REF.arnos.1991.212">Arnos et al [1991]</a></p><p>Differences in perspective may exist among medical professionals and within families regarding the use of prenatal testing, particularly if testing is being considered for the purpose of pregnancy termination rather than early diagnosis. While most centers would consider decisions regarding prenatal testing to be the choice of the parents, discussion of these issues is appropriate.</p></div></div><div id="dfnb1.Resources"><h2 id="_dfnb1_Resources_">Resources</h2><p>
<i>GeneReviews staff has selected the following disease-specific and/or umbrella
support organizations and/or registries for the benefit of individuals with this disorder
and their families. GeneReviews is not responsible for the information provided by other
organizations. For information on selection criteria, click <a href="/books/n/gene/app4/">here</a>.</i></p><ul><li class="half_rhythm"><div><b>Alexander Graham Bell Association for the Deaf and Hard of Hearing</b></div><div>3417 Volta Place Northwest</div><div>Washington DC 20007</div><div><b>Phone:</b> 866-337-5220 (toll-free); 202-337-5220; 202-337-5221 (TTY)</div><div><b>Fax:</b> 202-337-8314</div><div><b>Email:</b> info@agbell.org</div><div><a href="http://www.agbell.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Listening and Spoken Language Knowledge Center</a></div></li><li class="half_rhythm"><div><b>American Society for Deaf Children (ASDC)</b></div><div>800 Florida Avenue Northeast</div><div>Suite 2047</div><div>Washington DC 20002-3695</div><div><b>Phone:</b> 800-942-2732 (Toll-free Parent Hotline); 866-895-4206 (toll free voice/TTY)</div><div><b>Fax:</b> 410-795-0965</div><div><b>Email:</b> info@deafchildren.org; asdc@deafchildren.org</div><div><a href="http://www.deafchildren.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.deafchildren.org</a></div></li><li class="half_rhythm"><div><b>BabyHearing.org</b></div><div><i>This site, developed with support from the National Institute on Deafness and Other Communication Disorders, provides information about newborn hearing screening and hearing loss.</i></div><div><a href="http://www.babyhearing.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.babyhearing.org</a></div></li><li class="half_rhythm"><div><b>National Association of the Deaf (NAD)</b></div><div>8630 Fenton Street</div><div>Suite 820</div><div>Silver Spring MD 20910</div><div><b>Phone:</b> 301-587-1788; 301-587-1789 (TTY)</div><div><b>Fax:</b> 301-587-1791</div><div><b>Email:</b> nad.info@nad.org</div><div><a href="http://www.nad.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.nad.org</a></div></li><li class="half_rhythm"><div><b>National Library of Medicine Genetics Home Reference</b></div><div><a href="http://ghr.nlm.nih.gov/condition=nonsyndromicdeafness" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Nonsyndromic deafness</a></div></li></ul></div><div id="dfnb1.Molecular_Genetics"><h2 id="_dfnb1_Molecular_Genetics_">Molecular Genetics</h2><p><i>Information in the Molecular Genetics and OMIM tables may differ from that elsewhere in the GeneReview: tables may contain more recent information. &#x02014;</i>ED.</p><div id="dfnb1.molgen.TA" class="table"><h3><span class="label">Table A.</span></h3><div class="caption"><p>Nonsyndromic Hearing Loss and Deafness, DFNB1: Genes and Databases</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1272/table/dfnb1.molgen.TA/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfnb1.molgen.TA_lrgtbl__"><table class="no_bottom_margin"><tbody><tr><th id="hd_b_dfnb1.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;">Locus Name</th><th id="hd_b_dfnb1.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;">Gene</th><th id="hd_b_dfnb1.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;">Chromosome Locus</th><th id="hd_b_dfnb1.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;">Protein</th><th id="hd_b_dfnb1.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;">Locus-Specific Databases</th><th id="hd_b_dfnb1.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;">HGMD</th><th id="hd_b_dfnb1.molgen.TA_1_1_1_7" rowspan="1" colspan="1" style="vertical-align:top;">ClinVar</th></tr><tr><td headers="hd_b_dfnb1.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;">DFNB1</td><td headers="hd_b_dfnb1.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/2706" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>GJB2</i></a></td><td headers="hd_b_dfnb1.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=2706" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">13q12<wbr style="display:inline-block"></wbr>​.11</a></td><td headers="hd_b_dfnb1.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/P29033" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Gap junction beta-2 protein</a></td><td headers="hd_b_dfnb1.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://davinci.crg.es/deafness/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">The Connexin-deafness homepage (GJB2)</a><br /><a href="http://hereditaryhearingloss.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Hereditary Hearing Loss Homepage (GJB2)</a><br /><a href="https://research.cchmc.org/LOVD2/home.php?select_db=GJB2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">CCHMC - Human Genetics Mutation Database (GJB2)</a></td><td headers="hd_b_dfnb1.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=GJB2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">GJB2</a></td><td headers="hd_b_dfnb1.molgen.TA_1_1_1_7" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=GJB2[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">GJB2</a></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div id="dfnb1.TFA.1"><p class="no_margin">Data are compiled from the following standard references: <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> from
<a href="http://www.genenames.org/index.html" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HGNC</a>;
<a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> from
<a href="http://www.omim.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">OMIM</a>;
protein from <a href="http://www.uniprot.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">UniProt</a>.
For a description of databases (Locus Specific, HGMD, ClinVar) to which links are provided, click
<a href="/books/n/gene/app1/">here</a>.</p></div></dd></dl></div></div></div><div id="dfnb1.molgen.TB" class="table"><h3><span class="label">Table B.</span></h3><div class="caption"><p>OMIM Entries for Nonsyndromic Hearing Loss and Deafness, DFNB1 (<a href="/omim/121011,220290" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">View All in OMIM</a>) </p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1272/table/dfnb1.molgen.TB/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfnb1.molgen.TB_lrgtbl__"><table><tbody><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/121011" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">121011</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">GAP JUNCTION PROTEIN, BETA-2; GJB2</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/220290" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">220290</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">DEAFNESS, AUTOSOMAL RECESSIVE 1A; DFNB1A</td></tr></tbody></table></div></div><p><b>Introduction.</b> Gap junction channels are permeable to ions and small metabolites with relative molecular masses up to approximately 1.2 kd [<a class="bk_pop" href="#dfnb1.REF.harris.2001.357">Harris &#x00026; Bevans 2001</a>]. Differences in ionic selectivity and gating mechanisms among gap junctions reflect the existence of more than 20 different connexin <a class="def" href="/books/n/gene/glossary/def-item/isoforms/">isoforms</a> in humans. Most connexin genes have a common architecture, with the entire <a class="def" href="/books/n/gene/glossary/def-item/coding-region/">coding region</a> contained in a single large <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> separated from a noncoding exon by an <a class="def" href="/books/n/gene/glossary/def-item/intron/">intron</a> of variable size. The gap junction protein connexin 26 is encoded by <i>GJB2.</i></p><p><b>Gene structure.</b> The coding sequence of <i>GJB2</i> (<a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> 2) encodes a 226-amino-acid protein. For a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information, see <a href="/books/NBK1272/#dfnb1.molgen.TA">Table A</a>, <b>Gene</b>.</p><p><b>Pathogenic variants.</b> See <a class="figpopup" href="/books/NBK1272/table/dfnb1.T.selected_gjb2_pathogenic_variant/?report=objectonly" target="object" rid-figpopup="figdfnb1Tselectedgjb2pathogenicvariant" rid-ob="figobdfnb1Tselectedgjb2pathogenicvariant">Table 4</a>. Numerous <i>GJB2</i> pathogenic variants that result in <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic hearing loss are listed on the <a href="http://davinci.crg.es/deafness/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Connexin-deafness Home page</a>.</p><p>Common <i>GJB2</i> pathogenic variants:</p><ul><li class="half_rhythm"><div>c.35delG</div><ul><li class="half_rhythm"><div>Carrier rate of 2%-4% in individuals of northern European descent [<a class="bk_pop" href="#dfnb1.REF.snoeckx.2005.945">Snoeckx et al 2005</a>]</div></li><li class="half_rhythm"><div>Also reported in individuals of Arabic, Bedouin, Indian, and Pakistani descent</div></li></ul></li><li class="half_rhythm"><div>c.235delC, the most common variant in the Japanese population (<a class="def" href="/books/n/gene/glossary/def-item/carrier-rate/">carrier rate</a>: 1%-2%) [<a class="bk_pop" href="#dfnb1.REF.abe.2000.41">Abe et al 2000</a>, <a class="bk_pop" href="#dfnb1.REF.kudo.2000.141">Kudo et al 2000</a>, <a class="bk_pop" href="#dfnb1.REF.snoeckx.2005.945">Snoeckx et al 2005</a>]</div></li><li class="half_rhythm"><div>c.167delT, the most common variant in the <a class="def" href="/books/n/gene/glossary/def-item/ashkenazi-jewish/">Ashkenazi Jewish</a> population (<a class="def" href="/books/n/gene/glossary/def-item/carrier-rate/">carrier rate</a>: 7.5%) [<a class="bk_pop" href="#dfnb1.REF.morell.1998.1500">Morell et al 1998</a>]</div></li><li class="half_rhythm"><div>p.Val37Ile, the most common variant in Thailand (<a class="def" href="/books/n/gene/glossary/def-item/carrier-rate/">carrier rate</a>: 11.6%) [<a class="bk_pop" href="#dfnb1.REF.hwa.2003.161">Hwa et al 2003</a>]; associated with mild hearing loss [<a class="bk_pop" href="#dfnb1.REF.abe.2000.41">Abe et al 2000</a>, <a class="bk_pop" href="#dfnb1.REF.wilcox.2000.399">Wilcox et al 2000</a>, <a class="bk_pop" href="#dfnb1.REF.kenna.2001.1037">Kenna et al 2001</a>, <a class="bk_pop" href="#dfnb1.REF.lin.2001.42">Lin et al 2001</a>, <a class="bk_pop" href="#dfnb1.REF.marlin.2001.927">Marlin et al 2001</a>, <a class="bk_pop" href="#dfnb1.REF.snoeckx.2005.945">Snoeckx et al 2005</a>]</div></li><li class="half_rhythm"><div>p.Met34Thr, which has a minor <a class="def" href="/books/n/gene/glossary/def-item/allele-frequency/">allele frequency</a> of 1%-2% and is associated with mild hearing loss [<a class="bk_pop" href="#dfnb1.REF.snoeckx.2005.945">Snoeckx et al 2005</a>]</div></li></ul><p>The following three large deletions, which include sequences upstream of <i>GJB2</i> and a portion of <i>GJB6</i>, result in reduced <i>GJB2</i> expression, presumably due to <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> of a <a class="def" href="/books/n/gene/glossary/def-item/cis/"><i>cis</i></a>-regulatory element.</p><ul><li class="half_rhythm"><div>~309-kb <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> (known as &#x02206;<i>GJB6</i>-D13S1830) (see Note) that is common in the Spanish population (67% DFNB1 alleles in deaf Spanish individuals not detected by <i>GJB2</i> <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a>) [<a class="bk_pop" href="#dfnb1.REF.wu.2002.279">Wu et al 2002</a>, <a class="bk_pop" href="#dfnb1.REF.del_castillo.2003.1452">Del Castillo et al 2003</a>, <a class="bk_pop" href="#dfnb1.REF.stevenson.2003.151">Stevenson et al 2003</a>]</div></li><li class="half_rhythm"><div>~232-kb <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> (known as &#x02206;<i>GJB6</i>-D13S1854) (see Note) that is also relatively common in the Spanish population (25% of DFNB1 alleles in deaf Spanish individuals not detected by <i>GJB2</i> <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> or screening for the 309-kb deletion) [<a class="bk_pop" href="#dfnb1.REF.del_castillo.2005.588">del Castillo et al 2005</a>]</div></li><li class="half_rhythm"><div>~131.4-kb <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> with a proximal breakpoint more than 100 kb upstream of the transcriptional start sites of <i>GJB2</i> and <i>GJB6</i>; identified in a large German-American family with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> severe-to-profound nonsyndromic sensorineural hearing loss [<a class="bk_pop" href="#dfnb1.REF.wilch.2010.267">Wilch et al 2010</a>]. This family is an example of a common and a rare <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> segregating in a non-<a class="def" href="/books/n/gene/glossary/def-item/consanguineous/">consanguineous</a> family comprising many individuals with an autosomal recessive disorder.</div></li></ul><p>Note: These two deletions have also been identified at lower frequencies in other populations of European origin [<a class="bk_pop" href="#dfnb1.REF.wu.2002.279">Wu et al 2002</a>, <a class="bk_pop" href="#dfnb1.REF.del_castillo.2003.1452">Del Castillo et al 2003</a>, <a class="bk_pop" href="#dfnb1.REF.stevenson.2003.151">Stevenson et al 2003</a>].</p><div id="dfnb1.T.selected_gjb2_pathogenic_variant" class="table"><h3><span class="label">Table 4. </span></h3><div class="caption"><p>Selected <i>GJB2</i> Pathogenic Variants</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1272/table/dfnb1.T.selected_gjb2_pathogenic_variant/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfnb1.T.selected_gjb2_pathogenic_variant_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change<br />(Alias&#x000a0;<sup>1</sup>)</th><th id="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change</th><th id="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.35delG</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Gly12ValfsTer1</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_3" rowspan="18" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/entrez/viewer.fcgi?db=nuccore&#x00026;id=118572604" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_004004<wbr style="display:inline-block"></wbr>​.4</a><br /><a href="/entrez/viewer.fcgi?db=protein&#x00026;id=42558283" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_003995<wbr style="display:inline-block"></wbr>​.2</a></td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.35G&#x0003e;T</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Gly12Val</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.-3179G&#x0003e;A&#x000a0;<sup>2</sup><br />(IVS1+1G&#x0003e;A)</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">--</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.56G&#x0003e;C</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Ser19Thr</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.101T&#x0003e;C</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Met34Thr&#x000a0;<sup>3</sup></td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.109G&#x0003e;A</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Val37Ile&#x000a0;<sup>3</sup></td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.167delT</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Leu56ArgfsTer26</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.235delC</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Leu79CysfsTer3</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.231G&#x0003e;A</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Trp77Arg</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.269T&#x0003e;C</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Leu90Pro</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.339T&#x0003e;G</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Ser113Arg</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.358_360delGAG</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Glu120del</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.427C&#x0003e;T</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg143Trp</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.487A&#x0003e;G</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Met163Val</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.551G&#x0003e;C</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg184Pro</td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">&#x02206;<i>GJB6</i>-D13S1830</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"></td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">&#x02206;<i>GJB6</i>-D13S1854</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"></td></tr><tr><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">del(chr13:19,837,344-19,968,698)</td><td headers="hd_h_dfnb1.T.selected_gjb2_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin"><i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd><dt>1. </dt><dd><div id="dfnb1.TF.4.1"><p class="no_margin">Variant designation that does not conform to current naming conventions</p></div></dd><dt>2. </dt><dd><div id="dfnb1.TF.4.2"><p class="no_margin">IVS1+1G&#x0003e;A is -3179 nucleotides from the beginning of <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> 2 in the <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> sequence (Reference Sequence <a href="/entrez/viewer.fcgi?db=nuccore&#x00026;id=51511729" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NC_000013<wbr style="display:inline-block"></wbr>​.9</a>)</p></div></dd><dt>3. </dt><dd><div id="dfnb1.TF.4.3"><p class="no_margin">p.Met34Thr and p.Val37Ile are associated with normal hearing or mild hearing loss. See discussion in <b>Pathogenic variants</b>.</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b>
<i>GJB2</i> encodes connexin 26, a beta-2 gap junction protein composed of 226 amino acids. Connexins aggregate in groups of six around a central 2.3-nm pore to form a connexon. Connexons from adjoining cells covalently bond forming a channel between cells. Large aggregations of connexons called plaques are the constituents of gap junctions. Gap junctions permit direct intercellular exchange of ions and molecules through their central aqueous pores and permit synchronization of activity in excitable tissues and the exchange of metabolites and signal molecules in non-excitable tissues. Connexin 26 forms functional combinations with itself, connexin 32, connexin 46, and connexin 50.</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b>
<i>GJB2</i> pathogenic variants result in one of the following:</p><ul><li class="half_rhythm"><div>Loss of connexin 26 function (p.Gly12Val, p.Ser19Thr, c.35delG, p.Leu90Pro)</div></li><li class="half_rhythm"><div>Inability to induce formation of homotypic gap junction channels (p.Val37Ile, p.Trp77Arg, p.Ser113Arg, p.Glu120del, p.Met163Val, p.Arg184Pro, and c.235delC)</div></li><li class="half_rhythm"><div>Interference with translation (p.Arg184Pro) [<a class="bk_pop" href="#dfnb1.REF.snoeckx.2005.945">Snoeckx et al 2005</a>]</div></li></ul></div><div id="dfnb1.References"><h2 id="_dfnb1_References_">References</h2><div id="dfnb1.Published_Guidelines__Consensus_St"><h3>Published Guidelines / Consensus Statements</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF1">American College of Medical Genetics. Statement on universal newborn hearing screening. Available <a href="http://www.acmg.net/PDFLibrary/Universal-Newborn-Hearing-Screening.pdf" ref="pagearea=cite-ref&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">online</a>. 2000. Accessed 7-12-19.</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF2">Genetics Evaluation of Congenital Hearing Loss Expert Panel, American College of Medical Genetics. Genetics evaluation guidelines for the etiologic diagnosis of congenital hearing loss. Available <a href="/pmc/articles/PMC3110944/" ref="pagearea=cite-ref&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">online</a>. 2002. Accessed 7-12-19.</div></li></ul></div><div id="dfnb1.Literature_Cited"><h3>Literature Cited</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.abe.2000.41">Abe S, Usami S, Shinkawa H, Kelley PM, Kimberling WJ. Prevalent connexin 26 gene (GJB2) mutations in Japanese. <span><span class="ref-journal">J Med Genet. </span>2000;<span class="ref-vol">37</span>:41–3.</span> [<a href="/pmc/articles/PMC1734448/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1734448</span></a>] [<a href="/pubmed/10633133" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10633133</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.arnos.1991.212">Arnos KS, Israel J, Cunningham M. Genetic counseling of the deaf. Medical and cultural considerations. <span><span class="ref-journal">Ann N Y Acad Sci. </span>1991;<span class="ref-vol">630</span>:212–22.</span> [<a href="/pubmed/1952592" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 1952592</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.angeli.2008.2014">Angeli SI. Phenotype/genotype correlations in a DFNB1 cohort with ethnical diversity. <span><span class="ref-journal">Laryngoscope. </span>2008;<span class="ref-vol">118</span>:2014–23.</span> [<a href="/pubmed/18758381" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 18758381</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.azaiez.2004.305">Azaiez H, Chamberlin GP, Fischer SM, Welp CL, Prasad SD, Taggart RT, del Castillo I, Van Camp G, Smith RJ. GJB2: the spectrum of deafness-causing allele variants and their phenotype. <span><span class="ref-journal">Hum Mutat. </span>2004;<span class="ref-vol">24</span>:305–11.</span> [<a href="/pubmed/15365987" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15365987</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.azaiez.2007.451">Azaiez H, Yang T, Prasad S, Sorensen JL, Nishimura CJ, Kimberling WJ, Smith RJ. Genotype-phenotype correlations for SLC26A4-related deafness. <span><span class="ref-journal">Hum Genet. </span>2007;<span class="ref-vol">122</span>:451–7.</span> [<a href="/pubmed/17690912" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17690912</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.bazazzadegan.2012.1164">Bazazzadegan N, Nikzat N, Fattahi Z, Nishimura C, Meyer N, Sahraian S, Jamali P, Babanejad M, Kashef A, Yazdan H, Sabbagh Kermani F, Taghdiri M, Azadeh B, Mojahedi F, Khoshaeen A, Habibi H, Reyhanifar F, Nouri N, Smith RJ, Kahrizi K, Najmabadi H. The spectrum of GJB2 mutations in the Iranian population with non-syndromic hearing loss--a twelve year study. <span><span class="ref-journal">Int J Pediatr Otorhinolaryngol. </span>2012;<span class="ref-vol">76</span>:1164–74.</span> [<a href="/pubmed/22695344" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 22695344</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.del_castillo.2005.588">del Castillo FJ, Rodriguez-Ballesteros M, Alvarez A, Hutchin T, Leonardi E, de Oliveira CA, Azaiez H, Brownstein Z, Avenarius MR, Marlin S, Pandya A, Shahin H, Siemering KR, Weil D, Wuyts W, Aguirre LA, Martin Y, Moreno-Pelayo MA, Villamar M, Avraham KB, Dahl HH, Kanaan M, Nance WE, Petit C, Smith RJ, Van Camp G, Sartorato EL, Murgia A, Moreno F, del Castillo I. A novel deletion involving the connexin-30 gene, del(GJB6-d13s1854), found in trans with mutations in the GJB2 gene (connexin-26) in subjects with DFNB1 non-syndromic hearing impairment. <span><span class="ref-journal">J Med Genet. </span>2005;<span class="ref-vol">42</span>:588–94.</span> [<a href="/pmc/articles/PMC1736094/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1736094</span></a>] [<a href="/pubmed/15994881" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15994881</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.del_castillo.2003.1452">Del Castillo I, Moreno-Pelayo MA, Del Castillo FJ, Brownstein Z, Marlin S, Adina Q, Cockburn DJ, Pandya A, Siemering KR, Chamberlin GP, Ballana E, Wuyts W, Maciel-Guerra AT, Alvarez A, Villamar M, Shohat M, Abeliovich D, Dahl HH, Estivill X, Gasparini P, Hutchin T, Nance WE, Sartorato EL, Smith RJ, Van Camp G, Avraham KB, Petit C, Moreno F. Prevalence and evolutionary origins of the del(GJB6-D13S1830) mutation in the DFNB1 locus in hearing-impaired subjects: a multicenter study. <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">73</span>:1452–8.</span> [<a href="/pmc/articles/PMC1180408/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1180408</span></a>] [<a href="/pubmed/14571368" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 14571368</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.duan.2015.586">Duan SH, Zhu YM, Wang YL, Guo YF. Common molecular etiology of nonsyndromic hearing loss in 484 patients of 3 ethnicities in northwest China. <span><span class="ref-journal">Acta Otolaryngol. </span>2015;<span class="ref-vol">135</span>:586–91.</span> [<a href="/pubmed/25761933" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 25761933</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.feldmann.2009.195">Feldmann D, Le Mar&#x000e9;chal C, Jonard L, Thierry P, Czajka C, Couderc R, Ferec C, Denoyelle F, Marlin S, Fellmann F. A new large deletion in the DFNB1 locus causes nonsyndromic hearing loss. <span><span class="ref-journal">Eur J Med Genet. </span>2009;<span class="ref-vol">52</span>:195–200.</span> [<a href="/pubmed/19101659" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19101659</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.green.1999.2211">Green GE, Scott DA, McDonald JM, Woodworth GG, Sheffield VC, Smith RJ. Carrier rates in the midwestern United States for GJB2 mutations causing inherited deafness. <span><span class="ref-journal">JAMA. </span>1999;<span class="ref-vol">281</span>:2211–6.</span> [<a href="/pubmed/10376574" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10376574</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.harris.2001.357">Harris AL, Bevans CG. Exploring hemichannel permeability in vitro. <span><span class="ref-journal">Methods Mol Biol. </span>2001;<span class="ref-vol">154</span>:357–77.</span> [<a href="/pubmed/11218659" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11218659</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.heathcote.2000.50">Heathcote K, Syrris P, Carter ND, Patton MA. A connexin 26 mutation causes a syndrome of sensorineural hearing loss and palmoplantar hyperkeratosis (MIM 148350). <span><span class="ref-journal">J Med Genet. </span>2000;<span class="ref-vol">37</span>:50–1.</span> [<a href="/pmc/articles/PMC1734451/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1734451</span></a>] [<a href="/pubmed/10633135" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10633135</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.hwa.2003.161">Hwa HL, Ko TM, Hsu CJ, Huang CH, Chiang YL, Oong JL, Chen CC, Hsu CK. Mutation spectrum of the connexin 26 (GJB2) gene in Taiwanese patients with prelingual deafness. <span><span class="ref-journal">Genet Med. </span>2003;<span class="ref-vol">5</span>:161–5.</span> [<a href="/pubmed/12792423" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12792423</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.jayawardena.2015.843">Jayawardena AD, Shearer AE, Smith RJH. Sensorineural hearing loss &#x02013; a changing paradigm for its evaluation. <span><span class="ref-journal">Otolaryngol Head Neck Surg. </span>2015;<span class="ref-vol">153</span>:843–50.</span> [<a href="/pmc/articles/PMC4730883/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4730883</span></a>] [<a href="/pubmed/26216887" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26216887</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.kenna.2001.1037">Kenna MA, Wu BL, Cotanche DA, Korf BR, Rehm HL. Connexin 26 studies in patients with sensorineural hearing loss. <span><span class="ref-journal">Arch Otolaryngol Head Neck Surg. </span>2001;<span class="ref-vol">127</span>:1037–42.</span> [<a href="/pubmed/11556849" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11556849</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.kudo.2000.141">Kudo T, Ikeda K, Kure S, Matsubara Y, Oshima T, Watanabe K, Kawase T, Narisawa K, Takasaka T. Novel mutations in the connexin 26 gene (GJB2) responsible for childhood deafness in the Japanese population. <span><span class="ref-journal">Am J Med Genet. </span>2000;<span class="ref-vol">90</span>:141–5.</span> [<a href="/pubmed/10607953" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10607953</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.kutkowskaka_mierczak.2015.329">Kutkowska-Ka&#x0017a;mierczak A, Niepok&#x000f3;j K, Wertheim-Tysarowska K, Giza A, Mordasewicz-Goliszewska M, Bal J, Obersztyn E. Phenotypic variability in gap junction syndromic skin disorders: experience from KID and Clouston syndromes' clinical diagnostics. <span><span class="ref-journal">J Appl Genet. </span>2015;<span class="ref-vol">56</span>:329–37.</span> [<a href="/pmc/articles/PMC4543413/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4543413</span></a>] [<a href="/pubmed/25575739" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 25575739</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.lin.2001.42">Lin D, Goldstein JA, Mhatre AN, Lustig LR, Pfister M, Lalwani AK. Assessment of denaturing high-performance liquid chromatography (DHPLC) in screening for mutations in connexin 26 (GJB2). <span><span class="ref-journal">Hum Mutat. </span>2001;<span class="ref-vol">18</span>:42–51.</span> [<a href="/pubmed/11438992" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11438992</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.l_ffler.2001.226">L&#x000f6;ffler J, Nekahm D, Hirst-Stadlmann A, Gunther B, Menzel HJ, Utermann G, Janecke AR. Sensorineural hearing loss and the incidence of Cx26 mutations in Austria. <span><span class="ref-journal">Eur J Hum Genet. </span>2001;<span class="ref-vol">9</span>:226–30.</span> [<a href="/pubmed/11313763" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11313763</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.maestrini.1999.1237">Maestrini E, Korge BP, Ocana-Sierra J, Calzolari E, Cambiaghi S, Scudder PM, Hovnanian A, Monaco AP, Munro CS. A missense mutation in connexin26, D66H, causes mutilating keratoderma with sensorineural deafness (Vohwinkel's syndrome) in three unrelated families. <span><span class="ref-journal">Hum Mol Genet. </span>1999;<span class="ref-vol">8</span>:1237–43.</span> [<a href="/pubmed/10369869" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10369869</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.marlin.2001.927">Marlin S, Garabedian EN, Roger G, Moatti L, Matha N, Lewin P, Petit C, Denoyelle F. Connexin 26 gene mutations in congenitally deaf children: pitfalls for genetic counseling. <span><span class="ref-journal">Arch Otolaryngol Head Neck Surg. </span>2001;<span class="ref-vol">127</span>:927–33.</span> [<a href="/pubmed/11493200" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11493200</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.morell.1998.1500">Morell RJ, Kim HJ, Hood LJ, Goforth L, Friderici K, Fisher R, Van Camp G, Berlin CI, Oddoux C, Ostrer H, Keats B, Friedman TB. Mutations in the connexin 26 gene (GJB2) among Ashkenazi Jews with nonsyndromic recessive deafness. <span><span class="ref-journal">N Engl J Med. </span>1998;<span class="ref-vol">339</span>:1500–5.</span> [<a href="/pubmed/9819448" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9819448</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.norris.2006.732">Norris VW, Arnos KS, Hanks WD, Xia X, Nance WE, Pandya A. Does universal newborn hearing screening identify all children with GJB2 (Connexin 26) deafness? <span><span class="ref-journal">Ear Hear. </span>2006;<span class="ref-vol">27</span>:732–41.</span> [<a href="/pubmed/17086082" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17086082</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.orzan.2007.501">Orzan E, Murgia A. Connexin 26 deafness is not always congenital. <span><span class="ref-journal">Int J Pediatr Otorhinolaryngol. </span>2007;<span class="ref-vol">71</span>:501–7.</span> [<a href="/pubmed/17222463" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17222463</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.orzan.1999.291">Orzan E, Polli R, Martella M, Vinanzi C, Leonardi M, Murgia A. Molecular genetics applied to clinical practice: the Cx26 hearing impairment. <span><span class="ref-journal">Br J Audiol. </span>1999;<span class="ref-vol">33</span>:291–5.</span> [<a href="/pubmed/10890143" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10890143</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.posukh.2005.12">Posukh O, Pallares-Ruiz N, Tadinova V, Osipova L, Claustres M, Roux AF. First molecular screening of deafness in the Altai Republic population. <span><span class="ref-journal">BMC Med Genet. </span>2005;<span class="ref-vol">6</span>:12.</span> [<a href="/pmc/articles/PMC1079841/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1079841</span></a>] [<a href="/pubmed/15790391" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15790391</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.richard.2002.1341">Richard G, Rouan F, Willoughby CE, Brown N, Chung P, Ryynanen M, Jabs EW, Bale SJ, DiGiovanna JJ, Uitto J, Russell L. Missense mutations in GJB2 encoding connexin-26 cause the ectodermal dysplasia keratitis-ichthyosis-deafness syndrome. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">70</span>:1341–8.</span> [<a href="/pmc/articles/PMC447609/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC447609</span></a>] [<a href="/pubmed/11912510" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11912510</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.richard.1998.393">Richard G, White TW, Smith LE, Bailey RA, Compton JG, Paul DL, Bale SJ. Functional defects of Cx26 resulting from a heterozygous missense mutation in a family with dominant deaf-mutism and palmoplantar keratoderma. <span><span class="ref-journal">Hum Genet. </span>1998;<span class="ref-vol">103</span>:393–9.</span> [<a href="/pubmed/9856479" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9856479</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.shahin.2002.284">Shahin H, Walsh T, Sobe T, Lynch E, King MC, Avraham KB, Kanaan M. Genetics of congenital deafness in the Palestinian population: multiple connexin 26 alleles with shared origins in the Middle East. <span><span class="ref-journal">Hum Genet. </span>2002;<span class="ref-vol">110</span>:284–9.</span> [<a href="/pubmed/11935342" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11935342</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.shearer.2014.37">Shearer AE, Kolbe DL, Azaiez H, Sloan CM, Frees KL, Weaver AE, Clark ET, Nishimura CJ, Black-Ziegelbein EA, Smith RJ. Copy number variants are a common cause of non-syndromic hearing loss. <span><span class="ref-journal">Genome Med. </span>2014;<span class="ref-vol">6</span>:37.</span> [<a href="/pmc/articles/PMC4067994/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4067994</span></a>] [<a href="/pubmed/24963352" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24963352</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.shearer.2015.175">Shearer AE, Smith RJ. Massively parallel sequencing for genetic diagnosis of hearing loss: the new standard of care. <span><span class="ref-journal">Otolaryngol Head Neck Surg. </span>2015;<span class="ref-vol">153</span>:175–82.</span> [<a href="/pmc/articles/PMC4743024/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4743024</span></a>] [<a href="/pubmed/26084827" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26084827</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.sloanheggen.2016.441">Sloan-Heggen CM, Bierer AO, Shearer AE, Kolbe DL, Nishimura CJ, Frees KL, Ephraim SS, Shibata SB, Booth KT, Campbell CA, Ranum PT, Weaver AE, Black-Ziegelbein EA, Wang D, Azaiez H, Smith RJ. Comprehensive genetic testing in the clinical evaluation of 1119 patients with hearing loss. <span><span class="ref-journal">Hum Genet. </span>2016;<span class="ref-vol">135</span>:441–50.</span> [<a href="/pmc/articles/PMC4796320/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4796320</span></a>] [<a href="/pubmed/26969326" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26969326</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.smith.2005.879">Smith RJ, Bale JF Jr, White KR. Sensorineural hearing loss in children. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">365</span>:879–90.</span> [<a href="/pubmed/15752533" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15752533</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.snoeckx.2005.945">Snoeckx RL, Huygen PL, Feldmann D, Marlin S, Denoyelle F, Waligora J, Mueller-Malesinska M, Pollak A, Ploski R, Murgia A, Orzan E, Castorina P, Ambrosetti U, Nowakowska-Szyrwinska E, Bal J, Wiszniewski W, Janecke AR, Nekahm-Heis D, Seeman P, Bendova O, Kenna MA, Frangulov A, Rehm HL, Tekin M, Incesulu A, Dahl HH, du Sart D, Jenkins L, Lucas D, Bitner-Glindzicz M, Avraham KB, Brownstein Z, Del Castillo I, Moreno F, Blin N, Pfister M, Sziklai I, Toth T, Kelley PM, Cohn ES, Van Maldergem L, Hilbert P, Roux AF, Mondain M, Hoefsloot LH, Cremers CW, Lopponen T, Lopponen H, Parving A, Gronskov K, Schrijver I, Roberson J, Gualandi F, Martini A, Lina-Granade G, Pallares-Ruiz N, Correia C, Fialho G, Cryns K, Hilgert N, Van de Heyning P, Nishimura CJ, Smith RJ, Van Camp G. GJB2 Mutations and Degree of Hearing Loss: A Multicenter Study. <span><span class="ref-journal">Am J Hum Genet. </span>2005;<span class="ref-vol">77</span>:945–57.</span> [<a href="/pmc/articles/PMC1285178/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1285178</span></a>] [<a href="/pubmed/16380907" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 16380907</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.stevenson.2003.151">Stevenson VA, Ito M, Milunsky JM. Connexin-30 deletion analysis in connexin-26 heterozygotes. <span><span class="ref-journal">Genet Test. </span>2003;<span class="ref-vol">7</span>:151–4.</span> [<a href="/pubmed/12885339" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12885339</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF39">Tennessee Department of Health. Pediatric Audiology Guidelines &#x02013; Tennessee Genetic Centers Referral Pattern for Hearing Loss. Appendix 4. Available <a href="http://www.infanthearing.org/stateguidelines/Tennessee/Genetics.pdf" ref="pagearea=cite-ref&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">online</a>. 2005. Accessed 7-12-19.</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.van_geel.2002.938">van Geel M, van Steensel MA, Kuster W, Hennies HC, Happle R, Steijlen PM, Konig A. HID and KID syndromes are associated with the same connexin 26 mutation. <span><span class="ref-journal">Br J Dermatol. </span>2002;<span class="ref-vol">146</span>:938–42.</span> [<a href="/pubmed/12072059" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12072059</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.van_steensel.2002.724">van Steensel MA, van Geel M, Nahuys M, Smitt JH, Steijlen PM. A novel connexin 26 mutation in a patient diagnosed with keratitis- ichthyosis-deafness syndrome. <span><span class="ref-journal">J Invest Dermatol. </span>2002;<span class="ref-vol">118</span>:724–7.</span> [<a href="/pubmed/11918723" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11918723</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.wilch.2010.267">Wilch E, Azaiez H, Fisher RA, Elfenbein J, Murgia A, Birkenh&#x000e4;ger R, Bolz H, Da Silva-Costa SM, Del Castillo I, Haaf T, Hoefsloot L, Kremer H, Kubisch C, Le Marechal C, Pandya A, Sartorato EL, Schneider E, Van Camp G, Wuyts W, Smith RJ, Friderici KH. A novel DFNB1 deletion allele supports the existence of a distant cis-regulatory region that controls GJB2 and GJB6 expression. <span><span class="ref-journal">Clin Genet. </span>2010;<span class="ref-vol">78</span>:267–74.</span> [<a href="/pmc/articles/PMC2919588/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2919588</span></a>] [<a href="/pubmed/20236118" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20236118</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.wilcox.2000.399">Wilcox SA, Saunders K, Osborn AH, Arnold A, Wunderlich J, Kelly T, Collins V, Wilcox LJ, McKinlay Gardner RJ, Kamarinos M, Cone-Wesson B, Williamson R, Dahl HH. High frequency hearing loss correlated with mutations in the GJB2 gene. <span><span class="ref-journal">Hum Genet. </span>2000;<span class="ref-vol">106</span>:399–405.</span> [<a href="/pubmed/10830906" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10830906</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfnb1.REF.wu.2002.279">Wu BL, Lindeman N, Lip V, Adams A, Amato RS, Cox G, Irons M, Kenna M, Korf B, Raisen J, Platt O. Effectiveness of sequencing connexin 26 (GJB2) in cases of familial or sporadic childhood deafness referred for molecular diagnostic testing. <span><span class="ref-journal">Genet Med. </span>2002;<span class="ref-vol">4</span>:279–88.</span> [<a href="/pubmed/12172394" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12172394</span></a>]</div></li></ul></div></div><div id="dfnb1.Chapter_Notes"><h2 id="_dfnb1_Chapter_Notes_">Chapter Notes</h2><div id="dfnb1.Acknowledgments"><h3>Acknowledgments</h3><p>Supported in part by grant RO1-DC02842 from the NIDCD (RJHS)</p></div><div id="dfnb1.Author_History"><h3>Author History</h3><p>Mary-Kayt N Jones, BA (2016-present)<br />Daryl A Scott, MD, PhD; University of Iowa (1998-2001)<br />Val C Sheffield, MD, PhD; University of Iowa (1998-2001)<br />Richard JH Smith, MD (1998-present)<br />Guy Van Camp, PhD; University of Antwerp (1998-2016)</p></div><div id="dfnb1.Revision_History"><h3>Revision History</h3><ul><li class="half_rhythm"><div>18 August 2016 (bp) Comprehensive update posted live</div></li><li class="half_rhythm"><div>2 January 2014 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>14 July 2011 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>11 July 2008 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>21 December 2005 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>14 March 2005 (rjs) Revision: information on <i>GJB6</i> deletions</div></li><li class="half_rhythm"><div>15 July 2004 (rjs) Revision: use of an interpreter</div></li><li class="half_rhythm"><div>27 October 2003 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>24 April 2001 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>28 September 1998 (pb) Review posted live</div></li><li class="half_rhythm"><div>4 April 1998 (rjs) Original submission</div></li></ul></div></div><div id="bk_toc_contnr"></div></div></div>
            <div class="post-content"><div><div class="half_rhythm"><a href="/books/about/copyright/">Copyright</a> © 1993-2019, University of Washington, Seattle. GeneReviews is
a registered trademark of the University of Washington, Seattle. All rights
reserved.<p class="small">GeneReviews® chapters are owned by the University of Washington. Permission is
hereby granted to reproduce, distribute, and translate copies of content materials for
noncommercial research purposes only, provided that (i) credit for source (<a href="http://www.genereviews.org/" ref="pagearea=meta&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">http://www.genereviews.org/</a>) and copyright (© 1993-2019 University of
Washington) are included with each copy; (ii) a link to the original material is provided
whenever the material is published elsewhere on the Web; and (iii) reproducers,
distributors, and/or translators comply with the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>. No further modifications are allowed. For clarity, excerpts
of GeneReviews chapters for use in lab reports and clinic notes are a permitted
use.</p><p class="small">For more information, see the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>.</p><p class="small">For questions regarding permissions or whether a specified use is allowed,
contact: <a href="mailto:dev@null" data-email="ude.wu@tssamda" class="oemail">ude.wu@tssamda</a>.</p></div><div class="small"><span class="label">Bookshelf ID: NBK1272</span><span class="label">PMID: <a href="/pubmed/20301449" title="PubMed record of this page" ref="pagearea=meta&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">20301449</a></span></div><div style="margin-top:2em" class="bk_noprnt"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/dfna3/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/mt-deafness/" title="Next page in this title">Next &gt;</a></div></div></div></div>
            
        </div>

        <!-- Custom content below content -->
        <div class="col4">
            
        </div>
        
        
        <!-- Book content -->
        
        <!-- Custom contetnt below bottom nav -->
        <div class="col5">
            
        </div>
    </div>

    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            <div id="ncbi_share_book"><a href="#" class="ncbi_share" data-ncbi_share_config="popup:false,shorten:true" ref="id=NBK1272&amp;db=books">Share</a></div>

        </div>
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Views</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PDF_download" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/NBK1272/?report=reader">PubReader</a></li><li><a href="/books/NBK1272/?report=printable">Print View</a></li><li><a data-jig="ncbidialog" href="#_ncbi_dlg_citbx_NBK1272" data-jigconfig="width:400,modal:true">Cite this Page</a><div id="_ncbi_dlg_citbx_NBK1272" style="display:none" title="Cite this Page"><div class="bk_tt">Smith RJH, Jones MKN. Nonsyndromic Hearing Loss and Deafness, DFNB1. 1998 Sep 28 [Updated 2016 Aug 18]. In: Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. <span class="bk_cite_avail"></span></div></div></li><li><a href="/books/NBK1272/pdf/Bookshelf_NBK1272.pdf">PDF version of this page</a> (630K)</li><li><a href="#" class="toggle-glossary-link" title="Enable/disable links to the glossary">Disable Glossary Links</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>In this GeneReview</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="page-toc" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="#dfnb1.Summary" ref="log$=inpage&amp;link_id=inpage">Summary</a></li><li><a href="#dfnb1.Diagnosis" ref="log$=inpage&amp;link_id=inpage">Diagnosis</a></li><li><a href="#dfnb1.Clinical_Characteristics" ref="log$=inpage&amp;link_id=inpage">Clinical Characteristics</a></li><li><a href="#dfnb1.Genetically_Related_Disorders" ref="log$=inpage&amp;link_id=inpage">Genetically Related Disorders</a></li><li><a href="#dfnb1.Differential_Diagnosis" ref="log$=inpage&amp;link_id=inpage">Differential Diagnosis</a></li><li><a href="#dfnb1.Management" ref="log$=inpage&amp;link_id=inpage">Management</a></li><li><a href="#dfnb1.Genetic_Counseling" ref="log$=inpage&amp;link_id=inpage">Genetic Counseling</a></li><li><a href="#dfnb1.Resources" ref="log$=inpage&amp;link_id=inpage">Resources</a></li><li><a href="#dfnb1.Molecular_Genetics" ref="log$=inpage&amp;link_id=inpage">Molecular Genetics</a></li><li><a href="#dfnb1.References" ref="log$=inpage&amp;link_id=inpage">References</a></li><li><a href="#dfnb1.Chapter_Notes" ref="log$=inpage&amp;link_id=inpage">Chapter Notes</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>GeneReviews Links</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="source-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/n/gene/advanced/"><i>GeneReviews</i> Advanced Search</a></li><li><a href="/books/n/gene/glossary/"><i>GeneReviews</i> Glossary</a></li><li><a href="/books/n/gene/resource_mats/">Resource Materials</a> <span class="bk_hlight1">NEW FEATURE</span></li><li><a href="/books/n/gene/updates/">New in <i>GeneReviews</i></a></li><li><a href="/books/n/gene/authors/">Author List</a></li><li><a href="/books/n/gene/prospective_authors/">For Current/Prospective Authors</a></li><li><a href="/books/n/gene/GRpersonnel/"><i>GeneReviews</i> Personnel</a></li><li><a href="/books/n/gene/howto_linkin/">Download/Link to <i>GeneReviews</i></a></li><li><a href="/books/n/gene/contact_us/">Contact Us</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Tests in GTR by Gene</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="document-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=2707[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GJB3</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=2706[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GJB2</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=10804[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GJB6</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Variations in ClinVar</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="document-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=NBK1272+AND+genereviews[submitter]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Variations from this GeneReview in ClinVar</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Related information</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="discovery_db_links" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=omim&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_omim&amp;IdsFromResult=1495302" ref="log$=recordlinks">OMIM</a><div class="brieflinkpop offscreen_noflow">Related OMIM records</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pmc&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pmc_refs&amp;IdsFromResult=1495302" ref="log$=recordlinks">PMC</a><div class="brieflinkpop offscreen_noflow">PubMed Central citations</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pubmed&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pubmed_refs&amp;IdsFromResult=1495302" ref="log$=recordlinks">PubMed</a><div class="brieflinkpop offscreen_noflow">Links to PubMed</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=gene&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_gene&amp;IdsFromResult=1495302" ref="log$=recordlinks">Gene</a><div class="brieflinkpop offscreen_noflow">Locus Links</div></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301708" ref="ordinalpos=1&amp;linkpos=1&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, DFNA3</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, DFNA3<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Smith RJH, Ranum PT. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301429" ref="ordinalpos=1&amp;linkpos=2&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">OTOF</i>-Related Deafness</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">OTOF</i>-Related Deafness<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Shearer AE, Smith RJH. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/25719194" ref="ordinalpos=1&amp;linkpos=3&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">TBC1D24</i>-Related Disorders</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">TBC1D24</i>-Related Disorders<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Mucha BE, Hennekam RCM, Sisodiya S, Campeau PM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301595" ref="ordinalpos=1&amp;linkpos=4&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, Mitochondrial</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, Mitochondrial<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Usami S, Nishio S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/10704187" ref="ordinalpos=1&amp;linkpos=5&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Clinical phenotype and mutations in connexin 26 (DFNB1/GJB2), the most common cause of childhood hearing loss.</a><span class="source">[Am J Med Genet. 1999]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Clinical phenotype and mutations in connexin 26 (DFNB1/GJB2), the most common cause of childhood hearing loss.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Cohn ES, Kelley PM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Am J Med Genet. 1999 Sep 24; 89(3):130-6. </em></div></div></li></ul><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=20301449" ref="ordinalpos=1&amp;log$=relatedreviews_seeall&amp;logdbfrom=pubmed">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=20301449" ref="ordinalpos=1&amp;log$=relatedarticles_seeall&amp;logdbfrom=pubmed">See all...</a></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Recent Activity</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="Shutter"></a></div><div class="portlet_content"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class=""><div class="action"><a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a></div><ul id="activity"><li class="ra_rcd ralinkpopper two_line"><a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5e0410efdde089116992fbc2">Nonsyndromic Hearing Loss and Deafness, DFNB1 - GeneReviews®</a><div class="ralinkpop offscreen_noflow">Nonsyndromic Hearing Loss and Deafness, DFNB1 - GeneReviews®<div class="brieflinkpopdesc"></div></div><div class="tertiary"></div></li></ul><p class="HTOn">Your browsing activity is empty.</p><p class="HTOff">Activity recording is turned off.</p><p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p><a class="seemore" href="/sites/myncbi/recentactivity">See more...</a></div></div></div>
        
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script type="text/javascript" src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script type="text/javascript">
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-12-25T20:46:23-05:00&amp;Snapshot=%2Fprojects%2Fbooks%2FPBooks@5.22&amp;Host=portal106&amp;ncbi_phid=CE8D4892E04013710000000006870206&amp;ncbi_session=CE8D4892E0410EE1_1671SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK1272%2F&amp;Db=pmc&amp;folderID=134&amp;Ncbi_App=bookshelf&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        </div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=books&amp;ncbi_pdid=book-part&amp;ncbi_acc=NBK1272&amp;ncbi_domain=gene&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/NBK1272/&amp;ncbi_pagename=Nonsyndromic Hearing Loss and Deafness, DFNB1 - GeneReviews® - NCBI Bookshelf&amp;ncbi_bookparttype=chapter&amp;ncbi_app=bookshelf" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8D4892E0410EE1_1671SID /projects/books/PBooks@5.22 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/js/3879255/4121861/3501987/4008961/3893018/3821238/4062932/4123007/4087685/4076480/3921943/3400083/3426610.js" snapshot="books"></script></body>
</html>