<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    
    <head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <!-- AppResources meta begin -->
        <meta name="paf-app-resources" content="" />
                 <script type="text/javascript">var ncbi_startTime = new Date();</script>
            
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="" />

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="books" /><meta name="ncbi_pdid" content="book-part" /><meta name="ncbi_acc" content="NBK1536" /><meta name="ncbi_domain" content="gene" /><meta name="ncbi_report" content="record" /><meta name="ncbi_type" content="fulltext" /><meta name="ncbi_objectid" content="" /><meta name="ncbi_pcid" content="/NBK1536/" /><meta name="ncbi_pagename" content="Nonsyndromic Hearing Loss and Deafness, DFNA3 - GeneReviews® - NCBI Bookshelf" /><meta name="ncbi_bookparttype" content="chapter" /><meta name="ncbi_app" content="bookshelf" />
        <!-- Logger end -->
        
        <title>Nonsyndromic Hearing Loss and Deafness, DFNA3 - GeneReviews® - NCBI Bookshelf</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css" /><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,FOLLOW,NOARCHIVE" /><meta name="citation_inbook_title" content="GeneReviews® [Internet]" /><meta name="citation_title" content="Nonsyndromic Hearing Loss and Deafness, DFNA3" /><meta name="citation_publisher" content="University of Washington, Seattle" /><meta name="citation_date" content="2016/12/22" /><meta name="citation_author" content="Richard JH Smith" /><meta name="citation_author" content="Paul T Ranum" /><meta name="citation_pmid" content="20301708" /><meta name="citation_fulltext_html_url" content="https://www.ncbi.nlm.nih.gov/books/NBK1536/" /><meta name="citation_keywords" content="GJB2-Related DFNA 3 Nonsyndromic Hearing Loss and Deafness" /><meta name="citation_keywords" content="GJB6-Related DFNA 3 Nonsyndromic Hearing Loss and Deafness" /><meta name="citation_keywords" content="Gap junction beta-2 protein" /><meta name="citation_keywords" content="Gap junction beta-6 protein" /><meta name="citation_keywords" content="GJB2" /><meta name="citation_keywords" content="GJB6" /><meta name="citation_keywords" content="Nonsyndromic Hearing Loss and Deafness, DFNA3" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="DC.Title" content="Nonsyndromic Hearing Loss and Deafness, DFNA3" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="University of Washington, Seattle" /><meta name="DC.Contributor" content="Richard JH Smith" /><meta name="DC.Contributor" content="Paul T Ranum" /><meta name="DC.Date" content="2016/12/22" /><meta name="DC.Identifier" content="https://www.ncbi.nlm.nih.gov/books/NBK1536/" /><meta name="description" content="Nonsyndromic hearing loss and deafness, DFNA3 is characterized by pre- or postlingual, mild-to-profound progressive high-frequency sensorineural hearing impairment. Affected individuals have no other associated medical findings." /><meta name="og:title" content="Nonsyndromic Hearing Loss and Deafness, DFNA3" /><meta name="og:type" content="book" /><meta name="og:description" content="Nonsyndromic hearing loss and deafness, DFNA3 is characterized by pre- or postlingual, mild-to-profound progressive high-frequency sensorineural hearing impairment. Affected individuals have no other associated medical findings." /><meta name="og:url" content="https://www.ncbi.nlm.nih.gov/books/NBK1536/" /><meta name="og:site_name" content="NCBI Bookshelf" /><meta name="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" /><meta name="twitter:card" content="summary" /><meta name="twitter:site" content="@ncbibooks" /><meta name="bk-non-canon-loc" content="/books/n/gene/dfna3/" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/books/NBK1536/" /><link rel="stylesheet" href="/corehtml/pmc/css/figpopup.css" type="text/css" media="screen" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books_print.min.css" type="text/css" media="print" /><style type="text/css">p a.figpopup{display:inline !important} .bk_tt {font-family: monospace}  .first-line-outdent .bk_ref {display: inline}  .body-content h2, .body-content .h2  {border-bottom: 1px solid #97B0C8} .body-content h2.inline {border-bottom: none} a.page-toc-label , .jig-ncbismoothscroll a {text-decoration:none;border:0 !important} .temp-labeled-list  .graphic {display:inline-block !important} .temp-labeled-list  img{width:100%}</style><script type="text/javascript" src="/corehtml/pmc/js/jquery.hoverIntent.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"> </script><script type="text/javascript">window.name="mainwindow";</script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/book-toc.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/books.min.js"> </script><meta name="book-collection" content="NONE" />

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="CE88DFEDE040C1410000000006610264.m_12" />
<meta name='referrer' content='origin-when-cross-origin'/><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4098876/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/4008682/3881636/3579733/4062871/12930/3964959/3854974/36029/4128070/9685/3549676/3609192/3609193/3609213/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3411343/3882866.css" media="print" /></head>
    <body class="book-part">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo"><h1 class="res_name"><a href="/books/" title="Bookshelf home">Bookshelf</a></h1><h2 class="res_tagline"></h2></div>
    <div class="search"><form method="get" action="/books/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="books" selected="selected" class="last" data-ac_dict="bookshelf-search">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books" data-ac_dict="bookshelf-search">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search Books. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'bookshelf-search',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'no'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/books/browse/">Browse Titles</a>
                    </li><li>
                        <a href="/books/advanced/">Advanced</a>
                    </li><li class="help">
                        <a href="/books/NBK3833/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <!-- Custom content 1 -->
<div class="col1">
    
</div>

<div class="container">
    <div id="maincontent" class="content eight_col col">
        <!-- Custom content in the left column above book nav -->
        <div class="col2">
            
        </div>
        
        <!-- Book content -->
        

        <!-- Custom content between navigation and content -->
        <div class="col3">
            
        </div>
        
        <div class="document">
            <div class="pre-content"><div><div class="bk_prnt"><p class="small">NCBI Bookshelf. A service of the National Library of Medicine, National Institutes of Health.</p><p>Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. </p></div><div class="iconblock clearfix whole_rhythm no_top_margin bk_noprnt"><a class="img_link icnblk_img" title="All GeneReviews" href="/books/n/gene/"><img class="source-thumb" src="/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" alt="Cover of GeneReviews®" height="100px" width="80px" /></a><div class="icnblk_cntnt eight_col"><h2>GeneReviews<sup>®</sup> [Internet].</h2><a data-jig="ncbitoggler" href="#__NBK1536_dtls__">Show details</a><div style="display:none" class="ui-widget" id="__NBK1536_dtls__"><div>Adam MP, Ardinger HH, Pagon RA, et al., editors.</div><div>Seattle (WA): <a href="http://www.washington.edu" ref="pagearea=page-banner&amp;targetsite=external&amp;targetcat=link&amp;targettype=publisher">University of Washington, Seattle</a>; 1993-2019.</div></div><div class="half_rhythm"><ul class="inline_list"><li style="margin-right:1em"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a></li></ul></div><div class="bk_noprnt"><form method="get" action="/books/n/gene/" id="bk_srch"><div class="bk_search"><label for="bk_term" class="offscreen_noflow">Search term</label><input type="text" title="Search GeneReviews" id="bk_term" name="term" value="" data-jig="ncbiclearbutton" /> <input type="submit" class="jig-ncbibutton" value="Search GeneReviews" submit="false" style="padding: 0.1em 0.4em;" /></div></form><div><ul class="inline_list"><li><a href="/books/n/gene/advanced/">GeneReviews Advanced Search</a></li><li style="margin-left:.5em"><a href="/books/n/gene/helpadvsearch/">Help</a></li></ul></div></div></div><div class="icnblk_cntnt two_col"><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/gonad-dys-46xy/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/dfnb1/" title="Next page in this title">Next &gt;</a></div></div></div></div></div>
            <div class="main-content lit-style" itemscope="itemscope" itemtype="http://schema.org/CreativeWork"><div class="meta-content fm-sec"><h1 id="_NBK1536_"><span class="title" itemprop="name">Nonsyndromic Hearing Loss and Deafness, DFNA3</span></h1><p class="contrib-group"><span itemprop="author">Richard JH Smith</span>, MD and <span itemprop="author">Paul T Ranum</span>, BA.</p><a data-jig="ncbitoggler" href="#__NBK1536_ai__" style="border:0;text-decoration:none">Author Information</a><div style="display:none" class="ui-widget" id="__NBK1536_ai__"><div class="contrib half_rhythm"><span itemprop="author">Richard JH Smith</span>, MD<div class="affiliation small">Professor of Internal Medicine, Division of Nephrology<br />Sterba Hearing Research Professor of Otolaryngology<br />Director, Molecular Otolaryngology and Renal Research Laboratories<br />University of Iowa<br />Iowa City, Iowa<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.awoiu@htims-drahcir" class="oemail">ude.awoiu@htims-drahcir</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Paul T Ranum</span>, BA<div class="affiliation small">Molecular Otolaryngology and Renal Research Laboratories<br />University of Iowa<br />Iowa City, Iowa<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.awoiu@munar-luap" class="oemail">ude.awoiu@munar-luap</a></div></div></div></div><p class="small">Initial Posting: <span itemprop="datePublished">September 28, 1998</span>; Last Update: <span itemprop="dateModified">December 22, 2016</span>.</p><p><em>Estimated reading time: 21 minutes</em></p></div><div class="jig-ncbiinpagenav body-content whole_rhythm" data-jigconfig="allHeadingLevels: ['h2'],smoothScroll: false" itemprop="text"><div id="dfna3.Summary" itemprop="description"><h2 id="_dfna3_Summary_">Summary</h2><div><h4 class="inline">Clinical characteristics.</h4><p>Nonsyndromic hearing loss and deafness, DFNA3 is characterized by pre- or postlingual, mild-to-profound progressive high-frequency sensorineural hearing impairment. Affected individuals have no other associated medical findings.</p></div><div><h4 class="inline">Diagnosis/testing.</h4><p>Diagnosis of DFNA3 depends on <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> to identify a <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>GJB2</i> (encoding connexin 26 [Cx26]) or <i>GJB6</i> (encoding connexin 30 [Cx30]).</p></div><div><h4 class="inline">Management.</h4><p><i>Treatment of manifestations:</i> Early diagnosis, habilitation with hearing aids or cochlear implantation, and educational programming diminishes the likelihood of long-term speech or educational delay.</p><p><i>Surveillance:</i> Semiannual examination by a physician who is familiar with hereditary hearing impairment; repeat audiometry to confirm stability of hearing loss</p><p><i>Agents/circumstances to avoid:</i> Environmental exposures known to cause hearing loss, such as repeated loud noises.</p><p><i>Evaluation of relatives at risk:</i> Once the <i>GJB2</i> or <i>GJB6</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> has been identified in an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> family member, <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> can be used to clarify the genetic status of at-risk relatives in infancy or early childhood so that appropriate early support and management can be provided.</p></div><div><h4 class="inline">Genetic counseling.</h4><p>DFNA3 is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> manner. Most individuals diagnosed as having DFNA3 have a deaf parent. Each child of an individual with DFNA3 has a 50% chance of inheriting the <i>GJB2</i> or <i>GJB6</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>. Once the <i>GJB2</i> or <i>GJB6</i> pathogenic variant has been identified in a family member with DFNA3, prenatal testing for a pregnancy at increased risk and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> are possible.</p></div></div><div id="dfna3.Diagnosis"><h2 id="_dfna3_Diagnosis_">Diagnosis</h2><p>No formal diagnostic criteria have been published for DFNA3-related hearing loss.</p><div id="dfna3.Suggestive_Findings"><h3>Suggestive Findings</h3><p>Nonsyndromic hearing loss and deafness, DFNA3 <b>should be suspected</b> in individuals with the following:</p><ul><li class="half_rhythm"><div class="half_rhythm">Pre- or postlingual, mild-to-profound progressive sensorineural hearing impairment [<a class="bk_pop" href="#dfna3.REF.denoyelle.2002.47">Denoyelle et al 2002</a>]</div><div class="half_rhythm">Note: (1) Hearing is measured in decibels (dB). The threshold or 0 dB mark for each frequency refers to the level at which normal young adults perceive a tone burst 50% of the time. Hearing is considered normal if an individual's thresholds are within 15 dB of normal thresholds. (2) Severity of hearing loss is graded as mild (26-40 dB), moderate (41-55 dB), moderately severe (56-70 dB), severe (71-90dB), or profound (&#x0003e;90dB). The frequency of hearing loss is designated as low (&#x0003c;500Hz), middle (501-2000Hz), or high (&#x0003e;2000Hz) (see <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a>).</div></li><li class="half_rhythm"><div class="half_rhythm"><b>No related systemic findings</b> identified by medical history and physical examination</div></li><li class="half_rhythm"><div class="half_rhythm"><b>A family history of nonsyndromic hearing loss</b> consistent with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> inheritance</div></li></ul></div><div id="dfna3.Establishing_the_Diagnosis"><h3>Establishing the Diagnosis</h3><p>The diagnosis of DFNA3 <b>is established</b> in a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> by detection of a <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>GJB2</i> (encoding connexin 26) or <i>GJB6</i> (encoding connexin 30) (see <a class="figpopup" href="/books/NBK1536/table/dfna3.T.molecular_genetic_testing_used_i/?report=objectonly" target="object" rid-figpopup="figdfna3Tmoleculargenetictestingusedi" rid-ob="figobdfna3Tmoleculargenetictestingusedi">Table 1</a>).</p><p>Molecular genetic testing approaches in a person with hearing loss documented on audiogram can include a combination of <b><a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted testing</b> (single-gene testing), multigene panels (including all genes implicated in nonsyndromic hearing loss), or <b><a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing</b> (e.g., <a class="def" href="/books/n/gene/glossary/def-item/exome/">exome</a> or <a class="def" href="/books/n/gene/glossary/def-item/genome-sequencing/">genome sequencing</a>).</p><p>Gene-targeted testing requires the clinician to determine which <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>(s) are likely involved based on phenotypic data, while comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing does not. Because of the overlapping phenotypes of the many causes of hereditary hearing loss and deafness, most individuals with hereditary hearing loss and deafness are diagnosed by one of two approaches: a <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a> of ALL genes implicated in nonsyndromic hearing loss and nonsyndromic mimics (<a href="#dfna3.Recommended_Testing">recommended</a>) or <a class="def" href="/books/n/gene/glossary/def-item/exome/">exome</a>/<a class="def" href="/books/n/gene/glossary/def-item/genome-sequencing/">genome sequencing</a> (<a href="#dfna3.Testing_to_Consider">to consider</a>).</p><div id="dfna3.Recommended_Testing"><h4>Recommended Testing</h4><p><b>A comprehensive deafness-specific <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a></b> that includes all genes implicated in nonsyndromic hearing loss and nonsyndromic hearing-loss mimics (see <a href="#dfna3.Differential_Diagnosis">Differential Diagnosis</a> and <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a>) is recommended as the initial test in the evaluation of hearing loss following an audiogram that has documented the hearing loss [<a class="bk_pop" href="#dfna3.REF.sloanheggen.2016.441">Sloan-Heggen et al 2016</a>] (<a class="figpopup" href="/books/NBK1536/figure/dfna3.F1/?report=objectonly" target="object" rid-figpopup="figdfna3F1" rid-ob="figobdfna3F1">Figure 1</a>).</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figdfna3F1" co-legend-rid="figlgnddfna3F1"><a href="/books/NBK1536/figure/dfna3.F1/?report=objectonly" target="object" title="Figure 1. " class="img_link icnblk_img figpopup" rid-figpopup="figdfna3F1" rid-ob="figobdfna3F1"><img class="small-thumb" src="/books/NBK1536/bin/dfna3-Image001.gif" src-large="/books/NBK1536/bin/dfna3-Image001.jpg" alt="Figure 1. . Genetic diagnostic rates in 1119 sequentially accrued persons with hearing loss." /></a><div class="icnblk_cntnt" id="figlgnddfna3F1"><h4 id="dfna3.F1"><a href="/books/NBK1536/figure/dfna3.F1/?report=objectonly" target="object" rid-ob="figobdfna3F1">Figure 1. </a></h4><p class="float-caption no_bottom_margin">Genetic diagnostic rates in 1119 sequentially accrued persons with hearing loss. No person was excluded based on phenotype, inheritance, or previous testing. Testing resulted in identification of the underlying genetic cause for hearing loss in 440 individuals <a href="/books/NBK1536/figure/dfna3.F1/?report=objectonly" target="object" rid-ob="figobdfna3F1">(more...)</a></p></div></div><p>Note: (1) Genes included in available panels and the diagnostic <a class="def" href="/books/n/gene/glossary/def-item/sensitivity/">sensitivity</a> of the test used for each <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> vary by laboratory and are likely to change over time [<a class="bk_pop" href="#dfna3.REF.shearer.2015.175">Shearer &#x00026; Smith 2015</a>]. (2) Some multigene panels may include genes not associated with the condition discussed in this <i>GeneReview</i>; thus, clinicians need to determine which <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a> is most likely to identify the genetic cause of the condition at the most reasonable cost while limiting identification of variants of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a> and pathogenic variants in genes that do not explain the underlying <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>. (3) Methods used in a panel may include <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a>, <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>, and/or other non-sequencing based tests. (4) All platforms used to identify deafness-causing genetic variants should include detection of copy number variants in the testing algorithm [<a class="bk_pop" href="#dfna3.REF.shearer.2014b.37">Shearer et al 2014b</a>].</p><p>For an introduction to multigene panels click <a href="/books/n/gene/app5/#app5.Multigene_Panels">here</a>. More detailed information for clinicians ordering genetic tests can be found <a href="/books/n/gene/app5/#app5.Multigene_Panels_FAQs">here</a>.</p></div><div id="dfna3.Testing_to_Consider"><h4>Testing to Consider</h4><p><b>Single-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> testing</b> can be considered if a deafness-specific <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a> is not available. However, performing <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> of <i>GJB2</i> and <i>GJB6</i> alone is not cost effective unless it is limited to persons with severe-to-profound <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> nonsyndromic hearing loss. Offering single-gene testing of <i>GJB2</i> and <i>GJB6</i> reflexively to everyone with congenital hearing loss without regard to the degree of hearing loss is not evidence based and not cost effective [<a class="bk_pop" href="#dfna3.REF.jayawardena.2015.843">Jayawardena et al 2015</a>, <a class="bk_pop" href="#dfna3.REF.shearer.2015.175">Shearer &#x00026; Smith 2015</a>].</p><p><b>Comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing</b> (i.e., <a class="def" href="/books/n/gene/glossary/def-item/exome-sequencing/">exome sequencing</a> and <a class="def" href="/books/n/gene/glossary/def-item/genome-sequencing/">genome sequencing</a>) <i>may be</i> considered; however, given the huge number of variants that will be identified by these sequencing strategies, even if a trio (i.e., parents and child) is studied, it is <i>extremely</i> unlikely that either strategy will identify a cause for hearing loss. With exome and genome sequencing, appropriate <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> is mandatory before and after testing. For an introduction to comprehensive genomic testing click <a href="/books/n/gene/app5/#app5.Comprehensive_Genomic_Testing">here</a>. More detailed information for clinicians ordering genomic testing can be found <a href="/books/n/gene/app5/#app5.Comprehensive_Genomic_Testing_1">here</a>.</p><div id="dfna3.T.molecular_genetic_testing_used_i" class="table"><h3><span class="label">Table 1. </span></h3><div class="caption"><p>Molecular Genetic Testing Used in Nonsyndromic Hearing Loss and Deafness, DFNA3</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1536/table/dfna3.T.molecular_genetic_testing_used_i/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna3.T.molecular_genetic_testing_used_i_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_1" rowspan="2" scope="col" colspan="1" headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_1" style="text-align:left;vertical-align:middle;">Gene&#x000a0;<sup>1</sup></th><th id="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_2" rowspan="2" scope="col" colspan="1" headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_2" style="text-align:left;vertical-align:middle;">Proportion of DFNA3 Attributed to Pathogenic Variants in This Gene</th><th id="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_3" colspan="2" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">Proportion of Pathogenic Variants&#x000a0;<sup>2</sup> Detectable by This Method</th></tr><tr><th headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_3" id="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_2_1" colspan="1" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">Sequence analysis&#x000a0;<sup>3</sup></th><th headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_3" id="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_2_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>&#x000a0;<sup>4</sup></th></tr></thead><tbody><tr><td headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>GJB2</i></td><td headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">&#x0003e;90%</td><td headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_3 hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">100%&#x000a0;<sup>5</sup></td><td headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_3 hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">See footnote 5</td></tr><tr><td headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>GJB6</i></td><td headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">&#x0003e;10%</td><td headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_3 hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">100%<sup>6</sup></td><td headers="hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_1_3 hd_h_dfna3.T.molecular_genetic_testing_used_i_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">See footnote 5</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="dfna3.TF.1.1"><p class="no_margin">See <a href="/books/NBK1536/#dfna3.molgen.TA">Table A. Genes and Databases</a> for <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> and protein.</p></div></dd><dt>2. </dt><dd><div id="dfna3.TF.1.2"><p class="no_margin">See <a href="#dfna3.Molecular_Genetics">Molecular Genetics</a> for information on allelic variants detected in this <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>.</p></div></dd><dt>3. </dt><dd><div id="dfna3.TF.1.3"><p class="no_margin">Sequence analysis detects variants that are benign, likely benign, of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a>, likely pathogenic, or pathogenic. Pathogenic variants may include small intragenic deletions/insertions and <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a>, <a class="def" href="/books/n/gene/glossary/def-item/nonsense-variant/">nonsense</a>, and <a class="def" href="/books/n/gene/glossary/def-item/splice-site/">splice site</a> variants; typically, <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> or whole-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> deletions/duplications are not detected. For issues to consider in interpretation of <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> results, click <a href="/books/n/gene/app2/">here</a>.</p></div></dd><dt>4. </dt><dd><div id="dfna3.TF.1.4"><p class="no_margin">Gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a> detects intragenic deletions or duplications. Methods used may include: <a class="def" href="/books/n/gene/glossary/def-item/quantitative-pcr/">quantitative PCR</a>, long-range PCR, multiplex ligation-dependent probe amplification (MLPA), and a <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted microarray designed to detect single-<a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> deletions or duplications.</p></div></dd><dt>5. </dt><dd><div id="dfna3.TF.1.5"><p class="no_margin">No data on detection rate of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a> are available.</p></div></dd></dl></div></div></div></div></div></div><div id="dfna3.Clinical_Characteristics"><h2 id="_dfna3_Clinical_Characteristics_">Clinical Characteristics</h2><div id="dfna3.Clinical_Description"><h3>Clinical Description</h3><p>Nonsyndromic hearing loss and deafness, DFNA3 is characterized by progressive, mild-to-severe high-frequency sensorineural hearing impairment (<a class="figpopup" href="/books/NBK1536/figure/dfna3.F2/?report=objectonly" target="object" rid-figpopup="figdfna3F2" rid-ob="figobdfna3F2">Figure 2</a>). <i>GJB2</i>-related DFNA3 is associated with prelingual and postlingual onset of hearing loss while <i>GJB6</i>-related DFNA3 is associated with prelingual hearing loss [<a class="bk_pop" href="#dfna3.REF.weegerink.2013">Weegerink 2013</a>].</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figdfna3F2" co-legend-rid="figlgnddfna3F2"><a href="/books/NBK1536/figure/dfna3.F2/?report=objectonly" target="object" title="Figure 2. " class="img_link icnblk_img figpopup" rid-figpopup="figdfna3F2" rid-ob="figobdfna3F2"><img class="small-thumb" src="/books/NBK1536/bin/dfna3-Image002.gif" src-large="/books/NBK1536/bin/dfna3-Image002.jpg" alt="Figure 2. . This audioprofile surface shows the anticipated progression of DFNA3 hearing loss with age." /></a><div class="icnblk_cntnt" id="figlgnddfna3F2"><h4 id="dfna3.F2"><a href="/books/NBK1536/figure/dfna3.F2/?report=objectonly" target="object" rid-ob="figobdfna3F2">Figure 2. </a></h4><p class="float-caption no_bottom_margin">This audioprofile surface shows the anticipated progression of DFNA3 hearing loss with age. Note that the hearing loss is typically congenital and slowly becomes more severe with age [Taylor et al 2016]. </p></div></div><p>DFNA3 audioprofiles (visual plots of hearing loss severity across a range of frequencies measured by pure tone audiometry) may vary significantly, even within a family. Note that when the hearing loss is postlingual, individuals with DFNA3 may pass the newborn hearing screen.</p><p>Tests of vestibular function and computed tomography of the temporal bones in persons with DFNA3 are normal [<a class="bk_pop" href="#dfna3.REF.denoyelle.2002.47">Denoyelle et al 2002</a>].</p><p>Individuals with DFNA3 have no other associated medical findings.</p></div><div id="dfna3.Phenotype_Correlations_by_Gene"><h3>Phenotype Correlations by Gene</h3><p><b><i>GJB2.</i></b> The twelve pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants of <i>GJB2</i> (<a class="figpopup" href="/books/NBK1536/table/dfna3.T.selected_gjb2_variants/?report=objectonly" target="object" rid-figpopup="figdfna3Tselectedgjb2variants" rid-ob="figobdfna3Tselectedgjb2variants">Table 3</a>) that cause deafness at the DFNA3 <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> are associated with at least two different audioprofiles based on age of onset.</p><p>The majority of pathogenic variants cause progressive prelingual hearing loss (<a class="figpopup" href="/books/NBK1536/figure/dfna3.F2/?report=objectonly" target="object" rid-figpopup="figdfna3F2" rid-ob="figobdfna3F2">Figure 2</a>):</p><ul><li class="half_rhythm"><div><b>p.Trp44Cys.</b> The audioprofile is characterized by a bilaterally symmetric sensorineural loss that varies from mild to profound, beginning with high-frequency hearing loss and progressing to loss at all frequencies.</div></li><li class="half_rhythm"><div><b>p.Pro58Ala.</b> Hearing loss is progressive, ranging from mild to severe.</div></li><li class="half_rhythm"><div><b>p.Arg75Gln and p.Arg75Trp.</b> Hearing loss is usually profound (average threshold for p.Arg75Gln is 105 dbHL).</div></li><li class="half_rhythm"><div><b>p.Arg143Gln.</b> Progressive and profound high-frequency hearing loss is observed.</div></li><li class="half_rhythm"><div><b>c.551G&#x0003e;A (p.Arg184Gln).</b> Audioprofiles are downsloping and consistent with severe-to-profound prelingual hearing loss [<a class="bk_pop" href="#dfna3.REF.janecke.2001.269">Janecke et al 2001</a>, <a class="bk_pop" href="#dfna3.REF.l_ffler.2001.226">L&#x000f6;ffler et al 2001</a>, <a class="bk_pop" href="#dfna3.REF.tekin.2001.269">Tekin et al 2001</a>, <a class="bk_pop" href="#dfna3.REF.denoyelle.2002.47">Denoyelle et al 2002</a>, <a class="bk_pop" href="#dfna3.REF.feldmann.2005.225">Feldmann et al 2005</a>, <a class="bk_pop" href="#dfna3.REF.primignani.2007.821">Primignani et al 2007</a>, <a class="bk_pop" href="#dfna3.REF.weegerink.2011.191">Weegerink et al 2011</a>].</div></li></ul><p>In contrast, deafness related to the pathogenic variants resulting in the substitutions p.Thr55Asn, p.Asp179Asn, and p.Cys202Phe is postlingual:</p><ul><li class="half_rhythm"><div><b>c.164C&#x0003e;A (p.Thr55Asn).</b> Audioprofiles have a downsloping pattern and are consistent with a severe-to-profound postlingual hearing loss.</div></li><li class="half_rhythm"><div><b>p.Asp179Asn.</b> Age of onset for hearing loss ranges from the first to the third decade. The audioprofile shows a mild-to-moderate hearing loss, particularly at high frequencies.</div></li><li class="half_rhythm"><div><b>p.Cys202Phe.</b> Hearing loss is usually not detected until the second decade. Initially, the loss preferentially affects the high frequencies but progresses to affect the middle frequencies by middle age [<a class="bk_pop" href="#dfna3.REF.morl_.2000.368">Morl&#x000e9; et al 2000</a>, <a class="bk_pop" href="#dfna3.REF.denoyelle.2002.47">Denoyelle et al 2002</a>, <a class="bk_pop" href="#dfna3.REF.primignani.2003.516">Primignani et al 2003</a>, <a class="bk_pop" href="#dfna3.REF.melchionda.2005.799">Melchionda et al 2005</a>].</div></li></ul><p>Other:</p><ul><li class="half_rhythm"><div><b>p.Trp44Ser.</b> Audioprofiles are not available.</div></li><li class="half_rhythm"><div><b>p.Asp46Asn.</b> Audioprofiles show <a class="def" href="/books/n/gene/glossary/def-item/intrafamilial-variability/">intrafamilial variability</a> with some individuals showing postlingual progressive hearing loss with onset in the first decade of life and some showing prelingual hearing loss.</div></li><li class="half_rhythm"><div><b>p.Met163Leu.</b> A mild-to-moderate high-frequency hearing loss is observed; age of onset was not reported [<a class="bk_pop" href="#dfna3.REF.hamelmann.2001.84">Hamelmann et al 2001</a>, <a class="bk_pop" href="#dfna3.REF.marziano.2003.805">Marziano et al 2003</a>, <a class="bk_pop" href="#dfna3.REF.matos.2008.87">Matos et al 2008</a>, <a class="bk_pop" href="#dfna3.REF.bazazzadegan.2011.1202">Bazazzadegan et al 2011</a>].</div></li></ul><p><b><i>GJB6</i> (</b><a class="figpopup" href="/books/NBK1536/table/dfna3.T.selected_gjb6_pathogenic_variant/?report=objectonly" target="object" rid-figpopup="figdfna3Tselectedgjb6pathogenicvariant" rid-ob="figobdfna3Tselectedgjb6pathogenicvariant"><b>Table 4</b></a><b>)</b></p><ul><li class="half_rhythm"><div><b>p.Thr5Met.</b> The audioprofile of the one family reported [<a class="bk_pop" href="#dfna3.REF.grifa.1999.16">Grifa et al 1999</a>] is characterized by middle- to high-frequency hearing loss. The degree of hearing loss is progressive and variable, ranging from mild to profound. The age of onset of hearing loss was not reported - and in the absence of linkage data, these results should be interpreted with caution [<a class="bk_pop" href="#dfna3.REF.friedman.2003.341">Friedman &#x00026; Griffith 2003</a>].</div></li><li class="half_rhythm"><div><b>p.Ala40Val.</b> The audioprofile and age of onset were not included in the one reported individual [<a class="bk_pop" href="#dfna3.REF.yang.2007.198">Yang et al 2007</a>, <a class="bk_pop" href="#dfna3.REF.wang.2011.e21473">Wang et al 2011</a>].</div></li></ul></div><div id="dfna3.Penetrance"><h3>Penetrance</h3><p><i>GJB2</i> and <i>GJB6</i> pathogenic variants that cause DFNA3 are fully penetrant.</p></div><div id="dfna3.Nomenclature"><h3>Nomenclature</h3><p>The different <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> loci for nonsyndromic deafness are designated DFN (for <i>d</i>ea<i>fn</i>ess).</p><p>Loci are named based on <a class="def" href="/books/n/gene/glossary/def-item/mode-of-inheritance/">mode of inheritance</a>:</p><ul><li class="half_rhythm"><div><b>DFNA.</b> Autosomal dominant</div></li><li class="half_rhythm"><div><b>DFNB.</b> Autosomal recessive</div></li><li class="half_rhythm"><div><b>DFN.</b> <a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a></div></li></ul><p>The number following the above designations reflects the order of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> mapping and/or discovery.</p></div><div id="dfna3.Prevalence"><h3>Prevalence</h3><p>The relative prevalence of DFNA3 as a cause of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> nonsyndromic hearing loss is not known, but it is extremely rare [<a class="bk_pop" href="#dfna3.REF.sloanheggen.2016.441">Sloan-Heggen et al 2016</a>]: 14 pathogenic variants have been described worldwide. The majority of these pathogenic variants are described only in single families or <a class="def" href="/books/n/gene/glossary/def-item/simplex/">simplex</a> cases (i.e., a single occurrence in a family) [<a class="bk_pop" href="#dfna3.REF.denoyelle.2002.47">Denoyelle et al 2002</a>, <a class="bk_pop" href="#dfna3.REF.hilgert.2009.189">Hilgert et al 2009</a>].</p><p>Prevalence for different pathogenic variants varies by population [<a class="bk_pop" href="#dfna3.REF.abe.2000.41">Abe et al 2000</a>, <a class="bk_pop" href="#dfna3.REF.hamelmann.2001.84">Hamelmann et al 2001</a>, <a class="bk_pop" href="#dfna3.REF.l_ffler.2001.226">L&#x000f6;ffler et al 2001</a>, <a class="bk_pop" href="#dfna3.REF.liu.2002.688">Liu et al 2002</a>, <a class="bk_pop" href="#dfna3.REF.shearer.2014a.445">Shearer et al 2014a</a>, <a class="bk_pop" href="#dfna3.REF.xiao.2004.1797">Xiao &#x00026; Xie 2004</a>].</p></div></div><div id="dfna3.Genetically_Related_Allelic_Disord"><h2 id="_dfna3_Genetically_Related_Allelic_Disord_">Genetically Related (Allelic) Disorders</h2><div id="dfna3.T.allelic_disorders" class="table"><h3><span class="label">Table 2. </span></h3><div class="caption"><p>Allelic Disorders</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1536/table/dfna3.T.allelic_disorders/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna3.T.allelic_disorders_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfna3.T.allelic_disorders_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene</th><th id="hd_h_dfna3.T.allelic_disorders_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Phenotype&#x000a0;<sup>1</sup></th><th id="hd_h_dfna3.T.allelic_disorders_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">MOI</th></tr></thead><tbody><tr><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_1" rowspan="6" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><b><i>GJB2</i></b></td><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/dfnb1/">DFNB1</a></td><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">AR</td></tr><tr><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Palmoplantar keratoderma with deafness (OMIM <a href="http://omim.org/entry/148350" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">148350</a>)</td><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">AD</td></tr><tr><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Keratitis-ichthyosis-deafness (KID) syndrome (OMIM <a href="http://omim.org/entry/148210" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">148210</a>)</td><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">AD</td></tr><tr><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Hystrix-like ichthyosis-deafness (HID) syndrome (OMIM <a href="http://omim.org/entry/602540" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">602540</a>)</td><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">AD</td></tr><tr><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Vohwinkel syndrome (OMIM <a href="http://omim.org/entry/124500" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">124500</a>)</td><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">AD</td></tr><tr><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Bart-Pumphrey syndrome (OMIM <a href="http://omim.org/entry/149200" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">149200</a>)</td><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">AD</td></tr><tr><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b><i>GJB6</i></b></td><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/ed2/">Hidrotic ectodermal dysplasia 2</a> (Clouston syndrome)</td><td headers="hd_h_dfna3.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">AD</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">AD= <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a>; AR=<a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a>; MOI = mode of Inheritance</p></div></dd><dt>1. </dt><dd><div id="dfna3.TF.2.1"><p class="no_margin">See hyperlinked <i>GeneReview</i> or OMIM <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> entry for more information.</p></div></dd></dl></div></div></div><p>The other phenotypes associated with mutation of <i>GJB2</i> and <i>GJB6</i> are described below:</p><p><b><i>GJB2</i></b></p><ul><li class="half_rhythm"><div class="half_rhythm"><a href="/books/n/gene/dfnb1/"><b>DFNB1</b></a> is characterized by (generally) moderate-to-severe sensorineural impairment.</div><div class="half_rhythm"><i>GJB2</i> is shared by an overlapping <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a>, DFNB1, which is associated with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> nonsyndromic hearing loss. Variants in <i>GJB2</i> may cause either <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> DFNA3-associated or autosomal recessive DFNB1-associated nonsyndromic hearing loss and deafness. The <a class="def" href="/books/n/gene/glossary/def-item/mode-of-inheritance/">mode of inheritance</a> is ultimately determined by how the variant affects the expression and function of the connexin 26 protein.</div></li><li class="half_rhythm"><div class="half_rhythm"><b>Palmoplantar keratoderma with deafness</b> is characterized by diffuse hyperkeratosis of the hands and feet [<a class="bk_pop" href="#dfna3.REF.richard.1998.393">Richard et al 1998</a>, <a class="bk_pop" href="#dfna3.REF.heathcote.2000.50">Heathcote et al 2000</a>, <a class="bk_pop" href="#dfna3.REF.feldmann.2005.225">Feldmann et al 2005</a>, <a class="bk_pop" href="#dfna3.REF.de_zwartstorm.2008.1113">de Zwart-Storm et al 2008</a>].</div></li><li class="half_rhythm"><div class="half_rhythm"><b>Keratitis-ichthyosis-deafness (KID)</b> and <b>hystrix-like ichthyosis-deafness (HID) syndromes</b> are associated with the same <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> (p.Asp50Asn) in the first extracellular <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a> of the <i>GJB2</i>-encoded protein connexin 26 [<a class="bk_pop" href="#dfna3.REF.van_geel.2002.938">van Geel et al 2002</a>].</div><ul><li class="half_rhythm"><div><b>Keratitis-ichthyosis-deafness (KID) syndrome</b> is an ectodermal dysplasia characterized by vascularizing keratitis, progressive erythrokeratoderma, and profound sensorineural hearing loss as well as scarring alopecia and predisposition to squamous cell carcinoma [<a class="bk_pop" href="#dfna3.REF.richard.2002.1341">Richard et al 2002</a>, <a class="bk_pop" href="#dfna3.REF.van_geel.2002.938">van Geel et al 2002</a>, <a class="bk_pop" href="#dfna3.REF.van_steensel.2002.724">van Steensel et al 2002</a>].</div></li><li class="half_rhythm"><div><b>Hystrix-like ichthyosis-deafness (HID) syndrome</b> is a keratinizing disorder characterized by sensorineural hearing loss and hyperkeratosis of the skin. Shortly after birth, erythroderma develops, with spiky and cobblestone-like hyperkeratosis of the entire skin surface appearing by age one year. Severe palmoplantar keratoderma and scarring alopecia occur in some. HID syndrome is considered to differ from KID syndrome in: (1) the extent and time of occurrence of skin manifestations; (2) the severity of keratitis; and (3) electron microscopic features.</div></li></ul></li><li class="half_rhythm"><div class="half_rhythm"><b>Vohwinkel syndrome</b> is classified as a "mutilating" diffuse keratoderma because circumferential hyperkeratosis of the digits can lead to autoamputation (termed "pseudoainhum"). Mild-to-moderate sensorineural hearing loss is often associated with the disease.</div></li><li class="half_rhythm"><div class="half_rhythm"><b>Bart-Pumphrey syndrome</b> is characterized by palmoplantar keratoderma, knuckle pads, leukonychia, and sensorineural hearing loss. The clinical features partially overlap with Vohwinkel syndrome and KID syndrome.</div></li></ul><p>With the <i>GJB2</i> pathogenic variants in which the epidermal disease and hearing loss cosegregate, the severity of the skin disease <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> is highly variable, suggesting that other factors modify <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> expression [<a class="bk_pop" href="#dfna3.REF.kelsell.2001.559">Kelsell et al 2001</a>, <a class="bk_pop" href="#dfna3.REF.feldmann.2005.225">Feldmann et al 2005</a>]. It is important to clinically assess each patient with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> hearing loss for any <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> features, which may be subtle or easily overlooked in <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> relatives.</p><p><b><i>GJB6</i></b></p><ul><li class="half_rhythm"><div><a href="/books/n/gene/ed2/"><b>Hidrotic ectodermal dysplasia 2</b></a>
<b>(Clouston syndrome)</b> is characterized by hidrotic ectodermal dysplasia, alopecia, and palmoplantar hyperkeratosis [<a class="bk_pop" href="#dfna3.REF.smith.2002.530">Smith et al 2002</a>].</div></li></ul></div><div id="dfna3.Differential_Diagnosis"><h2 id="_dfna3_Differential_Diagnosis_">Differential Diagnosis</h2><p>Other causes of postlingual, acquired forms of hearing loss need to be considered (see <a href="/books/n/gene/deafness-overview/">Deafness and Hereditary Hearing Loss Overview</a>). Because the diagnosis of <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> forms of hearing loss can be challenging, many multigene panels now include the more frequently diagnosed forms of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> syndromic hearing loss [<a class="bk_pop" href="#dfna3.REF.sloanheggen.2016.441">Sloan-Heggen et al 2016</a>].</p><p>Autosomal dominant <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> forms of hearing loss with:</p><ul><li class="half_rhythm"><div class="half_rhythm"><b>Malformations of the head and neck.</b>
<a href="/books/n/gene/bor/">Branchiootorenal (BOR) syndrome</a> is characterized by malformations of the outer, middle, and inner ear associated with: conductive, sensorineural, or mixed hearing impairment; branchial fistulae and cysts; and renal malformations ranging from mild renal hypoplasia to bilateral renal agenesis [<a class="bk_pop" href="#dfna3.REF.chang.2004.582">Chang et al 2004</a>]. Pathogenic variants in <i>EYA1</i>, <i>SIX5</i>, or <i>SIX1</i> are causative.</div></li><li class="half_rhythm"><div class="half_rhythm"><b>Pigmentary anomalies.</b>
<a href="/books/n/gene/ws1/">Waardenburg syndrome type 1</a> (WS1) is characterized by <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> sensorineural hearing loss and pigmentary disturbances of the iris, hair, and skin, along with dystopia canthorum (lateral displacement of the inner canthi) [<a class="bk_pop" href="#dfna3.REF.destefano.1998.499">DeStefano et al 1998</a>].</div><div class="half_rhythm">Hearing loss occurs in approximately 57% and is <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a>, sensorineural, typically non-progressive, and either unilateral or bilateral. Most commonly, hearing loss is bilateral and profound (&#x0003e;100 dB). The majority of individuals with WS1 have either a white forelock (45%) or graying of the scalp hair before age 30 years. Affected individuals may have complete heterochromia iridium, partial/segmental heterochromia, or hypoplastic or brilliant blue irides. The diagnosis is established by clinical findings. Diagnostic criteria rely on the presence of sensorineural hearing loss, pigmentary changes, and calculation of the W index to identify dystopia canthorum. Pathogenic variants in <i>PAX3</i> are causative.</div></li></ul></div><div id="dfna3.Management"><h2 id="_dfna3_Management_">Management</h2><div id="dfna3.Evaluations_Following_Initial_Diag"><h3>Evaluations Following Initial Diagnosis</h3><p>To establish the extent of involvement and needs in an individual diagnosed with nonsyndromic hearing loss, DFNA3, the following evaluations are recommended:</p><ul><li class="half_rhythm"><div>Complete assessment of auditory acuity using age-appropriate tests including ABR testing, auditory steady-state response (ASSR) testing, and/or pure tone audiometry (see <a href="/books/n/gene/deafness-overview/">Deafness and Hereditary Hearing Loss Overview</a>).</div></li><li class="half_rhythm"><div>Consultation with a clinical geneticist and/or genetic counselor</div></li></ul></div><div id="dfna3.Treatment_of_Manifestations"><h3>Treatment of Manifestations</h3><p>Early diagnosis, habilitation with hearing aids or cochlear implantation, and educational programming will diminish the likelihood of long-term speech or educational delay.</p><p>The following are indicated:</p><ul><li class="half_rhythm"><div>Fitting with appropriate hearing aids</div></li><li class="half_rhythm"><div>Enrollment in an appropriate educational program for the hearing impaired</div></li><li class="half_rhythm"><div>Consideration of cochlear implantation, an effective habilitation option for persons with preserved residual hearing [<a class="bk_pop" href="#dfna3.REF.gantz.2016.e118">Gantz et al 2016</a>]</div></li><li class="half_rhythm"><div>Recognition that unlike with many clinical conditions, management and treatment of mild-to-profound deafness fall largely within the purview of the social welfare and educational systems rather than the medical care system [<a class="bk_pop" href="#dfna3.REF.smith.2005.879">Smith et al 2005</a>]</div></li></ul><p>See <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness</a> for more detailed discussion of management issues.</p></div><div id="dfna3.Surveillance"><h3>Surveillance</h3><p>The following are appropriate:</p><ul><li class="half_rhythm"><div>Semiannual examination by a physician who is familiar with hereditary hearing impairment</div></li><li class="half_rhythm"><div>Repeat audiometry to confirm stability of hearing loss</div></li></ul></div><div id="dfna3.AgentsCircumstances_to_Avoid"><h3>Agents/Circumstances to Avoid</h3><p>Individuals with hearing loss should avoid environmental exposures known to cause hearing loss. Most important is avoidance of repeated exposure to loud noises.</p></div><div id="dfna3.Evaluation_of_Relatives_at_Risk"><h3>Evaluation of Relatives at Risk</h3><p>If a <i>GJB2</i> or <i>GJB6</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> has been identified in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>, it is appropriate to clarify the genetic status of at-risk sibs shortly after birth so that appropriate early support and management can be provided to the child and family.</p><p>See <a href="#dfna3.Related_Genetic_Counseling_Issues">Genetic Counseling</a> for issues related to testing of at-risk relatives for <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> purposes.</p></div><div id="dfna3.Therapies_Under_Investigation"><h3>Therapies Under Investigation</h3><p>Search <a href="http://clinicaltrials.gov/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">ClinicalTrials.gov</a> in the US and <a href="http://www.ClinicalTrialsRegister.eu" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.ClinicalTrialsRegister.eu</a> in Europe for access to information on clinical studies for a wide range of diseases and conditions. Note: There may not be human trials for this condition.</p></div></div><div id="dfna3.Genetic_Counseling"><h2 id="_dfna3_Genetic_Counseling_">Genetic Counseling</h2><p>
<i>Genetic counseling is the process of providing individuals and families with
information on the nature, inheritance, and implications of genetic disorders to help them
make informed medical and personal decisions. The following section deals with genetic
risk assessment and the use of family history and genetic testing to clarify genetic
status for family members. This section is not meant to address all personal, cultural, or
ethical issues that individuals may face or to substitute for consultation with a genetics
professional</i>. &#x02014;ED.</p><div id="dfna3.Mode_of_Inheritance"><h3>Mode of Inheritance</h3><p>Nonsyndromic hearing loss and deafness, DFNA3 is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> manner.</p></div><div id="dfna3.Risk_to_Family_Members"><h3>Risk to Family Members</h3><p><b>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>Most individuals diagnosed as having DFNA3 have a deaf parent.</div></li><li class="half_rhythm"><div>The proportion of individuals with DFNA3 caused by a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a>
<i>GJB2</i> or <i>GJB6</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> is very small.</div></li><li class="half_rhythm"><div>Recommendations for the evaluation of parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with an apparent <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> include (a) genetic testing to validate the <i>de novo</i> nature of the identified variant; (b) assessment of auditory acuity in the parents; and (c) medical history and physical examination to rule out other systemic findings in the family.</div></li><li class="half_rhythm"><div>If the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> found in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> cannot be detected in leukocyte DNA of either parent, possible explanations include a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> pathogenic variant in the proband or <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a> in a parent. While theoretically possible, germline mosaicism has not been reported.</div></li></ul><p><b>Sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>The likelihood that a sib will have DFNA3 depends on the genetic status of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s parents: if one of the proband's parents has a <i>GJB2</i> or <i>GJB6</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>, each sib has a 50% chance of inheriting the pathogenic variant.</div></li><li class="half_rhythm"><div>When the parents are clinically unaffected, the risk to the sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> appears to be low.</div></li><li class="half_rhythm"><div>If the <i>GJB2</i> or <i>GJB6</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> found in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> cannot be detected in the leukocyte DNA of either parent, the risk to sibs is presumed to be slightly greater than that of the general population (though still &#x0003c;1%) because of the theoretic possibility of parental <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a>.</div></li></ul><p><b>Offspring of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</b> Each child of an individual with DFNA3 has a 50% chance of inheriting the <i>GJB2</i> or <i>GJB6</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>.</p><p><b>Other family members.</b> The risk to other family members depends on the status of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s parents: if a parent is deaf, his or her family members may also have DFNA3.</p></div><div id="dfna3.Related_Genetic_Counseling_Issues"><h3>Related Genetic Counseling Issues</h3><p>See Management, <a href="#dfna3.Evaluation_of_Relatives_at_Risk">Evaluation of Relatives at Risk</a> for information on evaluating at-risk relatives for the purpose of early diagnosis and treatment.</p><p>The following points are noteworthy:</p><ul><li class="half_rhythm"><div>Communication with individuals who are members of the Deaf community and who sign requires the services of a skilled interpreter.</div></li><li class="half_rhythm"><div>Members of the Deaf community may view deafness as a distinguishing characteristic and not as a handicap, impairment, or medical condition requiring a "treatment" or "cure," or to be "prevented."</div></li><li class="half_rhythm"><div>Many deaf people are interested in obtaining information about the cause of their own deafness, including information on medical, educational, and social services, rather than information about prevention, reproduction, or family planning. It is, therefore, important to ascertain and address the questions and concerns of the family/individual.</div></li><li class="half_rhythm"><div>The use of certain terms is preferred: probability or chance vs risk; deaf and hard-of-hearing vs hearing impaired. Terms such as "abnormal" should be avoided.</div></li></ul><p><b>Family planning</b></p><ul><li class="half_rhythm"><div>The optimal time for determination of genetic status and discussion of the availability of prenatal testing is before pregnancy.</div></li><li class="half_rhythm"><div>It is appropriate to offer <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> (including discussion of the probability of deafness in offspring and reproductive options) to young adults who are deaf.</div></li></ul><p><b>Considerations in families with an apparent <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>.</b> When neither parent of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> condition has the pathogenic variant identified in the proband or clinical evidence of the condition, the pathogenic variant is likely <i>de novo</i>. However, non-medical explanations including <a class="def" href="/books/n/gene/glossary/def-item/alternate-paternity/">alternate paternity</a> or maternity (e.g., with assisted reproduction) and undisclosed adoption could also be explored.</p><p><b>DNA banking</b> is the storage of DNA (typically extracted from white blood cells) for possible future use. Because it is likely that testing methodology and our understanding of genes, allelic variants, and diseases will improve in the future, consideration should be given to banking DNA of <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals.</p></div><div id="dfna3.Prenatal_Testing_and_Preimplantati"><h3>Prenatal Testing and Preimplantation Genetic Diagnosis</h3><p>Once the <i>GJB2</i> or <i>GJB6</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> has been identified in a family member with DFNA3, prenatal testing for a pregnancy at increased risk and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> are possible.</p><p>Differences in perspective may exist among medical professionals and within families regarding the use of prenatal testing, particularly if the testing is being considered for the purpose of pregnancy termination rather than early diagnosis. While most centers would consider decisions regarding prenatal testing to be the choice of the parents, discussion of these issues is appropriate.</p></div></div><div id="dfna3.Resources"><h2 id="_dfna3_Resources_">Resources</h2><p>
<i>GeneReviews staff has selected the following disease-specific and/or umbrella
support organizations and/or registries for the benefit of individuals with this disorder
and their families. GeneReviews is not responsible for the information provided by other
organizations. For information on selection criteria, click <a href="/books/n/gene/app4/">here</a>.</i></p><ul><li class="half_rhythm"><div><b>National Library of Medicine Genetics Home Reference</b></div><div><a href="http://ghr.nlm.nih.gov/condition=nonsyndromicdeafness" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Nonsyndromic deafness</a></div></li><li class="half_rhythm"><div><b>Alexander Graham Bell Association for the Deaf and Hard of Hearing</b></div><div>3417 Volta Place Northwest</div><div>Washington DC 20007</div><div><b>Phone:</b> 866-337-5220 (toll-free); 202-337-5220; 202-337-5221 (TTY)</div><div><b>Fax:</b> 202-337-8314</div><div><b>Email:</b> info@agbell.org</div><div><a href="http://www.agbell.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Listening and Spoken Language Knowledge Center</a></div></li><li class="half_rhythm"><div><b>American Society for Deaf Children (ASDC)</b></div><div>800 Florida Avenue Northeast</div><div>Suite 2047</div><div>Washington DC 20002-3695</div><div><b>Phone:</b> 800-942-2732 (Toll-free Parent Hotline); 866-895-4206 (toll free voice/TTY)</div><div><b>Fax:</b> 410-795-0965</div><div><b>Email:</b> info@deafchildren.org; asdc@deafchildren.org</div><div><a href="http://www.deafchildren.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.deafchildren.org</a></div></li><li class="half_rhythm"><div><b>BabyHearing.org</b></div><div><i>This site, developed with support from the National Institute on Deafness and Other Communication Disorders, provides information about newborn hearing screening and hearing loss.</i></div><div><a href="http://www.babyhearing.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.babyhearing.org</a></div></li><li class="half_rhythm"><div><b>National Association of the Deaf (NAD)</b></div><div>8630 Fenton Street</div><div>Suite 820</div><div>Silver Spring MD 20910</div><div><b>Phone:</b> 301-587-1788; 301-587-1789 (TTY)</div><div><b>Fax:</b> 301-587-1791</div><div><b>Email:</b> nad.info@nad.org</div><div><a href="http://www.nad.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.nad.org</a></div></li></ul></div><div id="dfna3.Molecular_Genetics"><h2 id="_dfna3_Molecular_Genetics_">Molecular Genetics</h2><p><i>Information in the Molecular Genetics and OMIM tables may differ from that elsewhere in the GeneReview: tables may contain more recent information. &#x02014;</i>ED.</p><div id="dfna3.molgen.TA" class="table"><h3><span class="label">Table A.</span></h3><div class="caption"><p>Nonsyndromic Hearing Loss and Deafness, DFNA3: Genes and Databases</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1536/table/dfna3.molgen.TA/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna3.molgen.TA_lrgtbl__"><table class="no_bottom_margin"><tbody><tr><th id="hd_b_dfna3.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;">Gene</th><th id="hd_b_dfna3.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;">Chromosome Locus</th><th id="hd_b_dfna3.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;">Protein</th><th id="hd_b_dfna3.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;">Locus-Specific Databases</th><th id="hd_b_dfna3.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;">HGMD</th><th id="hd_b_dfna3.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;">ClinVar</th></tr><tr><td headers="hd_b_dfna3.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/2706" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>GJB2</i></a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=2706" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">13q12<wbr style="display:inline-block"></wbr>​.11</a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/P29033" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Gap junction beta-2 protein</a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://davinci.crg.es/deafness/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">The Connexin-deafness homepage (GJB2)</a><br /><a href="http://hereditaryhearingloss.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Hereditary Hearing Loss Homepage (GJB2)</a><br /><a href="https://research.cchmc.org/LOVD2/home.php?select_db=GJB2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">CCHMC - Human Genetics Mutation Database (GJB2)</a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=GJB2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">GJB2</a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=GJB2[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">GJB2</a></td></tr><tr><td headers="hd_b_dfna3.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/10804" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>GJB6</i></a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=10804" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">13q12<wbr style="display:inline-block"></wbr>​.11</a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/O95452" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Gap junction beta-6 protein</a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://davinci.crg.es/deafness/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">The Connexin-deafness homepage (GJB6)</a><br /><a href="http://hereditaryhearingloss.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Hereditary Hearing Loss Homepage (GJB6)</a><br /><a href="https://research.cchmc.org/LOVD2/home.php?select_db=GJB6" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">CCHMC - Human Genetics Mutation Database (GJB6)</a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=GJB6" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">GJB6</a></td><td headers="hd_b_dfna3.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=GJB6[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">GJB6</a></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div id="dfna3.TFA.1"><p class="no_margin">Data are compiled from the following standard references: <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> from
<a href="http://www.genenames.org/index.html" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HGNC</a>;
<a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> from
<a href="http://www.omim.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">OMIM</a>;
protein from <a href="http://www.uniprot.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">UniProt</a>.
For a description of databases (Locus Specific, HGMD, ClinVar) to which links are provided, click
<a href="/books/n/gene/app1/">here</a>.</p></div></dd></dl></div></div></div><div id="dfna3.molgen.TB" class="table"><h3><span class="label">Table B.</span></h3><div class="caption"><p>OMIM Entries for Nonsyndromic Hearing Loss and Deafness, DFNA3 (<a href="/omim/121011,601544,604418,612643" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">View All in OMIM</a>) </p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1536/table/dfna3.molgen.TB/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna3.molgen.TB_lrgtbl__"><table><tbody><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/121011" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">121011</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">GAP JUNCTION PROTEIN, BETA-2; GJB2</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/601544" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">601544</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">DEAFNESS, AUTOSOMAL DOMINANT 3A; DFNA3A</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/604418" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">604418</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">GAP JUNCTION PROTEIN, BETA-6; GJB6</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/612643" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">612643</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">DEAFNESS, AUTOSOMAL DOMINANT 3B; DFNA3B</td></tr></tbody></table></div></div><div id="dfna3.GJB2"><h3><i>GJB2</i></h3><p><b>Gene structure.</b> Most connexin genes have a common architecture, with the entire <a class="def" href="/books/n/gene/glossary/def-item/coding-region/">coding region</a> contained in a single large <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> separated from the 5'-untranslated region by an <a class="def" href="/books/n/gene/glossary/def-item/intron/">intron</a> of variable size. The coding sequence of <i>GJB2</i> (exon 2) is 681 base pairs (including the stop codon) and is translated into a 226-amino acid protein, connexin 26 (Cx26). For a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information, see <a href="/books/NBK1536/#dfna3.molgen.TA">Table A</a>, <b>Gene</b>.</p><p><b>Pathogenic variants.</b> Twelve pathogenic variants in <i>GJB2</i> are known (see <a class="figpopup" href="/books/NBK1536/table/dfna3.T.selected_gjb2_variants/?report=objectonly" target="object" rid-figpopup="figdfna3Tselectedgjb2variants" rid-ob="figobdfna3Tselectedgjb2variants">Table 3</a>). The majority segregate in families; however, the p.Arg75Gln and p.Arg75Trp variants have also been identified as <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> variants in <a class="def" href="/books/n/gene/glossary/def-item/simplex/">simplex</a> cases (i.e., a single occurrence in a family). These two variants are implicated in both <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> nonsyndromic hearing loss and <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> hearing loss associated with skin disorders (see <a href="#dfna3.Genetically_Related_Allelic_Disord">Genetically Related Disorders</a>) [<a class="bk_pop" href="#dfna3.REF.janecke.2001.269">Janecke et al 2001</a>, <a class="bk_pop" href="#dfna3.REF.feldmann.2005.225">Feldmann et al 2005</a>].</p><p>Of note, although the pathogenicity of the <a class="figpopup" href="/books/NBK1536/table/dfna3.T.selected_gjb2_variants/?report=objectonly" target="object" rid-figpopup="figdfna3Tselectedgjb2variants" rid-ob="figobdfna3Tselectedgjb2variants">p.Arg75Trp</a> variant had been questioned as it was reported in one of 77 Egyptian controls whose hearing status was not reported [<a class="bk_pop" href="#dfna3.REF.richard.1998.393">Richard et al 1998</a>], subsequent case reports, animal models, and functional studies strongly argue for the pathogenicity of this variant [<a class="bk_pop" href="#dfna3.REF.janecke.2001.269">Janecke et al 2001</a>, <a class="bk_pop" href="#dfna3.REF.kudo.2003.995">Kudo et al 2003</a>, <a class="bk_pop" href="#dfna3.REF.maeda.2005.1641">Maeda et al 2005</a>, <a class="bk_pop" href="#dfna3.REF.maeda.2009.597">Maeda et al 2009</a>, <a class="bk_pop" href="#dfna3.REF.mani.2009.502">Mani et al 2009</a>, <a class="bk_pop" href="#dfna3.REF.yum.2010.226">Yum et al 2010</a>, <a class="bk_pop" href="#dfna3.REF.weegerink.2011.191">Weegerink et al 2011</a>, <a class="bk_pop" href="#dfna3.REF.zhang.2011.71">Zhang et al 2011</a>].</p><div id="dfna3.T.selected_gjb2_variants" class="table"><h3><span class="label">Table 3. </span></h3><div class="caption"><p>Selected <i>GJB2</i> Variants</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1536/table/dfna3.T.selected_gjb2_variants/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna3.T.selected_gjb2_variants_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Variant Classification</th><th id="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change</th><th id="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change</th><th id="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Benign</b></td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.101T&#x0003e;C</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Met34Thr&#x000a0;<sup>1</sup></td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_4" rowspan="13" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/entrez/viewer.fcgi?val=NM_004004.5" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_004004<wbr style="display:inline-block"></wbr>​.5</a><br /><a href="/entrez/viewer.fcgi?val=NP_003995.2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_003995<wbr style="display:inline-block"></wbr>​.2</a></td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_1" rowspan="12" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><b>Pathogenic</b></td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.132G&#x0003e;C</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Trp44Cys</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.136G&#x0003e;A</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Asp46Asn</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.164C&#x0003e;A</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Thr55Asn</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.131G&#x0003e;C</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Trp44Ser</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.172C&#x0003e;G</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Pro58Ala</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.223C&#x0003e;T</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg75Trp</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.224G&#x0003e;A</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg75Gln</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.428G&#x0003e;A</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg143Gln</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.487A&#x0003e;C</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Met163Leu</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.535G&#x0003e;A</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Asp179Asn</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.551G&#x0003e;A</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg184Gln</td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.605G&#x0003e;T</td><td headers="hd_h_dfna3.T.selected_gjb2_variants_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Cys202Phe</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Note on variant classification: Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin">Note on nomenclature: <i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd><dt>1. </dt><dd><div id="dfna3.TF.3.1"><p class="no_margin">Variant found in normal hearing persons and family with palmoplantar keratoderma</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> Connexin 26 (Cx26) is a beta-2 gap junction protein. Gap junctions are highly specialized organelles consisting of clustered channels that permit direct intercellular exchange of ions and molecules through central aqueous pores. Postulated roles include: the rapid propagation of electrical signals and synchronization of activity in excitable tissues; and the exchange of metabolites and signal molecules in non-excitable tissues [<a class="bk_pop" href="#dfna3.REF.evans.2002.121">Evans &#x00026; Martin 2002</a>].</p><p>Each connexin protein contains two extracellular (E1-E2), four transmembrane (M1-M4), and three cytoplasmic domains (N-terminus, C-terminus, and a cytoplasmic loop located between M2 and M3) [<a class="bk_pop" href="#dfna3.REF.maeda.2009.597">Maeda et al 2009</a>]. Each extracellular <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a> contains three cysteine residues joined between the E1 and E2 loops by at least one disulfide bond [<a class="bk_pop" href="#dfna3.REF.kovacs.2007.15">Kovacs et al 2007</a>, <a class="bk_pop" href="#dfna3.REF.yeager.2007.521">Yeager &#x00026; Harris 2007</a>]. The presumed importance of these six cysteines can be inferred from Cx32 experiments in which any Cys <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> completely blocks the development of gap-junction conductances between <i>Xenopus</i> oocyte pairs. The third transmembrane domain (M3) is amphipathic and lines the putative wall of the intercellular channel [<a class="bk_pop" href="#dfna3.REF.kovacs.2007.15">Kovacs et al 2007</a>, <a class="bk_pop" href="#dfna3.REF.yeager.2007.521">Yeager &#x00026; Harris 2007</a>], which is created by oligomerization of six connexins to form a hexameric structure called a connexon. Two connexons, one from each cell, join in the extracellular gap to complete the cell-to-cell pathway. If the connexons contributed by each cell are of identical composition, the channel is homotypic; if each connexon is formed by a different composition of connexins, it is termed heterotypic. Most connexins are phosphoproteins and undergo post-transcriptional modifications [<a class="bk_pop" href="#dfna3.REF.moreno.2005.164">Moreno 2005</a>, <a class="bk_pop" href="#dfna3.REF.locke.2006.1221">Locke et al 2006</a>]. Cx26 forms functional combinations with itself, Cx30, Cx31, Cx32, Cx46, and Cx50 [<a class="bk_pop" href="#dfna3.REF.cottrell.2005.126">Cottrell &#x00026; Burt 2005</a>, <a class="bk_pop" href="#dfna3.REF.liu.2009.53">Liu et al 2009</a>].</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> Gap-junction channels are permeable to ions and small metabolites with relative molecular masses up to approximately 1.2 kd [<a class="bk_pop" href="#dfna3.REF.harris.2001.357">Harris &#x00026; Bevans 2001</a>]. Differences in ionic selectivity and gating mechanisms among gap junctions reflect the existence of more than 20 different connexin <a class="def" href="/books/n/gene/glossary/def-item/isoforms/">isoforms</a> in humans. The abnormal gene product in DFNA3 causes deafness via a <a class="def" href="/books/n/gene/glossary/def-item/dominant-negative/">dominant-negative</a> mechanism of action.</p></div><div id="dfna3.GJB6"><h3><i>GJB6</i></h3><p><b>Gene structure.</b>
<i>GJB6</i> has three exons, of which only the third is coding. The translated protein, connexin 30 (Cx30), is 261 amino acids long. For a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information, see <a href="/books/NBK1536/#dfna3.molgen.TA">Table A</a>, <b>Gene</b>.</p><p><b>Pathogenic variants.</b> Two <i>GBJ6</i> DFNA3-causing pathogenic variants are known: p.Thr5Met and p.Ala40Val [<a class="bk_pop" href="#dfna3.REF.grifa.1999.16">Grifa et al 1999</a>, <a class="bk_pop" href="#dfna3.REF.yang.2007.198">Yang et al 2007</a>] (see <a class="figpopup" href="/books/NBK1536/table/dfna3.T.selected_gjb6_pathogenic_variant/?report=objectonly" target="object" rid-figpopup="figdfna3Tselectedgjb6pathogenicvariant" rid-ob="figobdfna3Tselectedgjb6pathogenicvariant">Table 4</a>).</p><div id="dfna3.T.selected_gjb6_pathogenic_variant" class="table"><h3><span class="label">Table 4. </span></h3><div class="caption"><p>Selected <i>GJB6</i> Pathogenic Variants</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1536/table/dfna3.T.selected_gjb6_pathogenic_variant/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna3.T.selected_gjb6_pathogenic_variant_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfna3.T.selected_gjb6_pathogenic_variant_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change</th><th id="hd_h_dfna3.T.selected_gjb6_pathogenic_variant_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change</th><th id="hd_h_dfna3.T.selected_gjb6_pathogenic_variant_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_dfna3.T.selected_gjb6_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.14C&#x0003e;T</td><td headers="hd_h_dfna3.T.selected_gjb6_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Thr5Met&#x000a0;<sup>1</sup></td><td headers="hd_h_dfna3.T.selected_gjb6_pathogenic_variant_1_1_1_3" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/entrez/viewer.fcgi?val=NM_001110219.2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_001110219<wbr style="display:inline-block"></wbr>​.2</a><br /><a href="/entrez/viewer.fcgi?val=NP_001103689.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_001103689<wbr style="display:inline-block"></wbr>​.1</a></td></tr><tr><td headers="hd_h_dfna3.T.selected_gjb6_pathogenic_variant_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.119C&#x0003e;T</td><td headers="hd_h_dfna3.T.selected_gjb6_pathogenic_variant_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Ala40Val</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Note on variant classification: Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin">Note on nomenclature: <i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd><dt>1. </dt><dd><div id="dfna3.TF.4.1"><p class="no_margin">See also <a class="figpopup" href="/books/NBK1536/table/dfna3.T.molecular_genetic_testing_used_i/?report=objectonly" target="object" rid-figpopup="figdfna3Tmoleculargenetictestingusedi" rid-ob="figobdfna3Tmoleculargenetictestingusedi">Table 1</a> and <a href="#dfna3.Phenotype_Correlations_by_Gene">Phenotype Correlations by Gene</a>, <b><i>GJB6</i></b>.</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> Connexin 30 (Cx30) is a beta-6 gap junction protein. It shares an architecture that is common to all connexins (see <a href="#dfna3.GJB2"><i>GJB2</i></a>, <b>Normal gene product</b>).</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> Like the abnormal gene products of <i>GJB2-</i>related DFNA3, the p.Thr5Met and p.Ala40Val variants of <i>GJB6-</i>related DFNA3 act via a <a class="def" href="/books/n/gene/glossary/def-item/dominant-negative/">dominant-negative</a> mechanism to inhibit activity of <a class="def" href="/books/n/gene/glossary/def-item/wild-type/">wild-type</a> Cx30 gap junction channels.</p><p>Additionally, the p.Ala40Val variant exerts a trans-<a class="def" href="/books/n/gene/glossary/def-item/dominant-negative/">dominant-negative</a> effect on Cx26, impairing gap junction formation in the cochlea.</p><p>In functional studies with a mouse model, the p.Thr5Met variant was shown to exert its pathogenic effect through diminished biochemical coupling between cochlear cells [<a class="bk_pop" href="#dfna3.REF.grifa.1999.16">Grifa et al 1999</a>, <a class="bk_pop" href="#dfna3.REF.sch_tz.2010.4759">Sch&#x000fc;tz et al 2010</a>, <a class="bk_pop" href="#dfna3.REF.wang.2011.e21473">Wang et al 2011</a>].</p></div></div><div id="dfna3.References"><h2 id="_dfna3_References_">References</h2><div id="dfna3.Published_Guidelines__Consensus_St"><h3>Published Guidelines / Consensus Statements</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF1">American College of Medical Genetics. Genetics evaluation guidelines for the etiologic diagnosis of congenital hearing loss. Genetic evaluation of congenital hearing loss expert panel. Available <a href="/pmc/articles/PMC3110944/" ref="pagearea=cite-ref&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">online</a>. 2002. Accessed 7-19-18.</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF2">American College of Medical Genetics. Statement on universal newborn hearing screening. Available <a href="http://www.acmg.net/PDFLibrary/Universal-Newborn-Hearing-Screening.pdf" ref="pagearea=cite-ref&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">online</a>. 2000. Accessed 7-19-18.</div></li></ul></div><div id="dfna3.Literature_Cited"><h3>Literature Cited</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.abe.2000.41">Abe S, Usami S, Shinkawa H, Kelley PM, Kimberling WJ. Prevalent connexin 26 gene (GJB2) mutations in Japanese. <span><span class="ref-journal">J Med Genet. </span>2000;<span class="ref-vol">37</span>:41–3.</span> [<a href="/pmc/articles/PMC1734448/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1734448</span></a>] [<a href="/pubmed/10633133" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10633133</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.bazazzadegan.2011.1202">Bazazzadegan N, Sheffield AM, Sobhani M, Kahrizi K, Meyer NC, Van Camp G, Hilgert N, Abedini SS, Habibi F, Daneshi A, Nishimura C, Avenarius MR, Farhadi M, Smith RJ, Najmabadi H. Two Iranian families with a novel mutation in GJB2 causing autosomal dominant nonsyndromic hearing loss. <span><span class="ref-journal">Am J Med Genet A. </span>2011;<span class="ref-vol">155A</span>:1202–11.</span> [<a href="/pmc/articles/PMC3080436/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3080436</span></a>] [<a href="/pubmed/21484990" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 21484990</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.chang.2004.582">Chang EH, Menezes M, Meyer NC, Cucci RA, Vervoort VS, Schwartz CE, Smith RJ. Branchio-oto-renal syndrome: the mutation spectrum in EYA1 and its phenotypic consequences. <span><span class="ref-journal">Hum Mutat. </span>2004;<span class="ref-vol">23</span>:582–9.</span> [<a href="/pubmed/15146463" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15146463</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.cottrell.2005.126">Cottrell GT, Burt JM. Functional consequences of heterogeneous gap junction channel formation and its influence in health and disease. <span><span class="ref-journal">Biochim Biophys Acta. </span>2005;<span class="ref-vol">1711</span>:126–41.</span> [<a href="/pubmed/15955298" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15955298</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.de_zwartstorm.2008.1113">de Zwart-Storm EA, van Geel M, van Neer PA, Steijlen PM, Martin PE, van Steensel MA. A novel missense mutation in the second extracellular domain of GJB2, p.Ser183Phe, causes a syndrome of focal palmoplantar keratoderma with deafness. <span><span class="ref-journal">Am J Pathol. </span>2008;<span class="ref-vol">173</span>:1113–9.</span> [<a href="/pmc/articles/PMC2543078/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2543078</span></a>] [<a href="/pubmed/18787097" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 18787097</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.denoyelle.2002.47">Denoyelle F, Lina-Granade G, Petit C. DFNA3. <span><span class="ref-journal">Adv Otorhinolaryngol. </span>2002;<span class="ref-vol">61</span>:47–52.</span> [<a href="/pubmed/12408062" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12408062</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.destefano.1998.499">DeStefano AL, Cupples LA, Arnos KS, Asher JH Jr, Baldwin CT, Blanton S, Carey ML, da Silva EO, Friedman TB, Greenberg J, Lalwani AK, Milunsky A, Nance WE, Pandya A, Ramesar RS, Read AP, Tassabejhi M, Wilcox ER, Farrer LA. Correlation between Waardenburg syndrome phenotype and genotype in a population of individuals with identified PAX3 mutations. <span><span class="ref-journal">Hum Genet. </span>1998;<span class="ref-vol">102</span>:499–506.</span> [<a href="/pubmed/9654197" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9654197</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.evans.2002.121">Evans WH, Martin PE. Gap junctions: structure and function. <span><span class="ref-journal">Mol Membr Biol. </span>2002;<span class="ref-vol">19</span>:121–36.</span> [<a href="/pubmed/12126230" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12126230</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.feldmann.2005.225">Feldmann D, Denoyelle F, Blons H, Lyonnet S, Loundon N, Rouillon I, Hadj-Rabia S, Petit C, Couderc R, Garab&#x000e9;dian EN, Marlin S. The GJB2 mutation R75Q can cause nonsyndromic hearing loss DFNA3 or hereditary palmoplantar keratoderma with deafness. <span><span class="ref-journal">Am J Med Genet A. </span>2005;<span class="ref-vol">137</span>:225–7.</span> [<a href="/pubmed/16059934" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 16059934</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.friedman.2003.341">Friedman TB, Griffith AJ. Human nonsyndromic sensorineural deafness. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2003;<span class="ref-vol">4</span>:341–402.</span> [<a href="/pubmed/14527306" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 14527306</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.gantz.2016.e118">Gantz BJ, Dunn C, Walker E, Van Voorst T, Gogel S, Hansen M. Outcomes of adolescents with a short electrode cochlear implant with preserved residual hearing. <span><span class="ref-journal">Otol Neurotol. </span>2016;<span class="ref-vol">37</span>:e118–25.</span> [<a href="/pmc/articles/PMC4712731/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4712731</span></a>] [<a href="/pubmed/26756143" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26756143</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.grifa.1999.16">Grifa A, Wagner CA, D'Ambrosio L, Melchionda S, Bernardi F, Lopez-Bigas N, Rabionet R, Arbones M, Monica MD, Estivill X, Zelante L, Lang F, Gasparini P. Mutations in GJB6 cause nonsyndromic autosomal dominant deafness at DFNA3 locus. <span><span class="ref-journal">Nat Genet. </span>1999;<span class="ref-vol">23</span>:16–8.</span> [<a href="/pubmed/10471490" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10471490</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.hamelmann.2001.84">Hamelmann C, Amedofu GK, Albrecht K, Muntau B, Gelhaus A, Brobby GW, Horstmann RD. Pattern of connexin 26 (GJB2) mutations causing sensorineural hearing impairment in Ghana. <span><span class="ref-journal">Hum Mutat. </span>2001;<span class="ref-vol">18</span>:84–5.</span> [<a href="/pubmed/11439000" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11439000</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.harris.2001.357">Harris AL, Bevans CG. Exploring hemichannel permeability in vitro. <span><span class="ref-journal">Methods Mol Biol. </span>2001;<span class="ref-vol">154</span>:357–77.</span> [<a href="/pubmed/11218659" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11218659</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.heathcote.2000.50">Heathcote K, Syrris P, Carter ND, Patton MA. A connexin 26 mutation causes a syndrome of sensorineural hearing loss and palmoplantar hyperkeratosis (MIM 148350). <span><span class="ref-journal">J Med Genet. </span>2000;<span class="ref-vol">37</span>:50–1.</span> [<a href="/pmc/articles/PMC1734451/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1734451</span></a>] [<a href="/pubmed/10633135" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10633135</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.hilgert.2009.189">Hilgert N, Smith RJ, Van Camp G. Forty-six genes causing nonsyndromic hearing impairment: which ones should be analyzed in DNA diagnostics? <span><span class="ref-journal">Mutat Res. </span>2009;<span class="ref-vol">681</span>:189–96.</span> [<a href="/pmc/articles/PMC2847850/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2847850</span></a>] [<a href="/pubmed/18804553" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 18804553</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.janecke.2001.269">Janecke AR, Nekahm D, L&#x000f6;ffler J, Hirst-Stadlmann A, M&#x000fc;ller T, Utermann G. De novo mutation of the connexin 26 gene associated with dominant non-syndromic sensorineural hearing loss. <span><span class="ref-journal">Hum Genet. </span>2001;<span class="ref-vol">108</span>:269–70.</span> [<a href="/pubmed/11354642" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11354642</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.jayawardena.2015.843">Jayawardena AD, Shearer AE, Smith RJ. Sensorineural Hearing Loss: A Changing Paradigm for Its Evaluation. <span><span class="ref-journal">Otolaryngol Head Neck Surg. </span>2015;<span class="ref-vol">153</span>:843–50.</span> [<a href="/pmc/articles/PMC4730883/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4730883</span></a>] [<a href="/pubmed/26216887" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26216887</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.kelsell.2001.559">Kelsell DP, Di WL, Houseman MJ. Connexin mutations in skin disease and hearing loss. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">68</span>:559–68.</span> [<a href="/pmc/articles/PMC1274469/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1274469</span></a>] [<a href="/pubmed/11179004" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11179004</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.kovacs.2007.15">Kovacs JA, Baker KA, Altenberg GA, Abagyan R, Yeager M. Molecular modeling and mutagenesis of gap junction channels. <span><span class="ref-journal">Prog Biophys Mol Biol. </span>2007;<span class="ref-vol">94</span>:15–28.</span> [<a href="/pmc/articles/PMC2819402/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2819402</span></a>] [<a href="/pubmed/17524457" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17524457</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.kudo.2003.995">Kudo T, Kure S, Ikeda K, Xia AP, Katori Y, Suzuki M, Kojima K, Ichinohe A, Suzuki Y, Aoki Y, Kobayashi T, Matsubara Y. Transgenic expression of a dominant-negative connexin26 causes degeneration of the organ of Corti and non-syndromic deafness. <span><span class="ref-journal">Hum Mol Genet. </span>2003;<span class="ref-vol">12</span>:995–1004.</span> [<a href="/pubmed/12700168" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12700168</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.liu.2009.53">Liu XZ, Yuan Y, Yan D, Ding EH, Ouyang XM, Fei Y, Tang W, Yuan H, Chang Q, Du LL, Zhang X, Wang G, Ahmad S, Kang DY, Lin X, Dai P. Digenic inheritance of non-syndromic deafness caused by mutations at the gap junction proteins Cx26 and Cx31. <span><span class="ref-journal">Hum Genet. </span>2009;<span class="ref-vol">125</span>:53–62.</span> [<a href="/pmc/articles/PMC2737700/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2737700</span></a>] [<a href="/pubmed/19050930" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19050930</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.liu.2002.688">Liu Y, Ke X, Qi Y, Li W, Zhu P. Connexin 26 gene (GJB2): prevalence of mutations in the Chinese population. <span><span class="ref-journal">J Hum Genet. </span>2002;<span class="ref-vol">47</span>:688–90.</span> [<a href="/pubmed/12522692" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12522692</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.locke.2006.1221">Locke D, Koreen IV, Harris AL. Isoelectric points and post-translational modifications of connexin 26 and connexin 32. <span><span class="ref-journal">FASEB J. </span>2006;<span class="ref-vol">20</span>:1221–3.</span> [<a href="/pubmed/16645047" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 16645047</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.l_ffler.2001.226">L&#x000f6;ffler J, Nekahm D, Hirst-Stadlmann A, G&#x000fc;nther B, Menzel HJ, Utermann G, Janecke AR. Sensorineural hearing loss and the incidence of Cx26 mutations in Austria. <span><span class="ref-journal">Eur J Hum Genet. </span>2001;<span class="ref-vol">9</span>:226–30.</span> [<a href="/pubmed/11313763" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11313763</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.maeda.2005.1641">Maeda Y, Fukushima K, Nishizaki K, Smith RJ. In vitro and in vivo suppression of GJB2 expression by RNA interference. <span><span class="ref-journal">Hum Mol Genet. </span>2005;<span class="ref-vol">14</span>:1641–50.</span> [<a href="/pubmed/15857852" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15857852</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.maeda.2009.597">Maeda S, Nakagawa S, Suga M, Yamashita E, Oshima A, Fujiyoshi Y, Tsukihara T. Structure of the connexin 26 gap junction channel at a 3.5 A resolution. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">458</span>:597–602.</span> [<a href="/pubmed/19340074" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19340074</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.mani.2009.502">Mani RS, Ganapathy A, Jalvi R, Srikumari Srisailapathy CR, Malhotra V, Chadha S, Agarwal A, Ramesh A, Rangasayee RR, Anand A. Functional consequences of novel connexin 26 mutations associated with hereditary hearing loss. <span><span class="ref-journal">Eur J Hum Genet. </span>2009;<span class="ref-vol">17</span>:502–9.</span> [<a href="/pmc/articles/PMC2986212/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2986212</span></a>] [<a href="/pubmed/18941476" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 18941476</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.marziano.2003.805">Marziano NK, Casalotti SO, Portelli AE, Becker DL, Forge A. Mutations in the gene for connexin 26 (GJB2) that cause hearing loss have a dominant negative effect on connexin 30. <span><span class="ref-journal">Hum Mol Genet. </span>2003;<span class="ref-vol">12</span>:805–12.</span> [<a href="/pubmed/12668604" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12668604</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.matos.2008.87">Matos TD, Caria H, Sim&#x000f5;es-Teixeira H, Aasen T, Dias O, Andrea M, Kelsell DP, Fialho G. A novel M163L mutation in connexin 26 causing cell death and associated with autosomal dominant hearing loss. <span><span class="ref-journal">Hear Res. </span>2008;<span class="ref-vol">240</span>:87–92.</span> [<a href="/pubmed/18472371" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 18472371</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.melchionda.2005.799">Melchionda S, Bicego M, Marciano E, Franz&#x000e8; A, Morgutti M, Bortone G, Zelante L, Carella M, D&#x02019;Andrea P. Functional characterization of a novel Cx26 (T55N) mutation associated to non-syndromic hearing loss. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>2005;<span class="ref-vol">337</span>:799–805.</span> [<a href="/pubmed/16226720" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 16226720</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.moreno.2005.164">Moreno AP. Connexin phosphorylation as a regulatory event linked to channel gating. <span><span class="ref-journal">Biochim Biophys Acta. </span>2005;<span class="ref-vol">1711</span>:164–71.</span> [<a href="/pubmed/15955301" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15955301</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.morl_.2000.368">Morl&#x000e9; L, Bozon M, Alloisio N, Latour P, Vandenberghe A, Plauchu H, Collet L, Edery P, Godet J, Lina-Granade G. A novel C202F mutation in the connexin26 gene (GJB2) associated with autosomal dominant isolated hearing loss. <span><span class="ref-journal">J Med Genet. </span>2000;<span class="ref-vol">37</span>:368–70.</span> [<a href="/pmc/articles/PMC1734593/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1734593</span></a>] [<a href="/pubmed/10807696" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10807696</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.primignani.2003.516">Primignani P, Castorina P, Sironi F, Curcio C, Ambrosetti U, Coviello DA. A novel dominant missense mutation--D179N--in the GJB2 gene (Connexin 26) associated with non-syndromic hearing loss. <span><span class="ref-journal">Clin Genet. </span>2003;<span class="ref-vol">63</span>:516–21.</span> [<a href="/pubmed/12786758" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12786758</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.primignani.2007.821">Primignani P, Trotta L, Castorina P, Lalatta F, Cuda D, Murri A, Ambrosetti U, Cesarani A, Curcio C, Coviello D, Travi M. A new de novo missense mutation in connexin 26 in a sporadic case of nonsyndromic deafness. <span><span class="ref-journal">Laryngoscope. </span>2007;<span class="ref-vol">117</span>:821–4.</span> [<a href="/pubmed/17473676" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17473676</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.richard.2002.1341">Richard G, Rouan F, Willoughby CE, Brown N, Chung P, Ryyn&#x000e4;nen M, Jabs EW, Bale SJ, DiGiovanna JJ, Uitto J, Russell L. Missense mutations in GJB2 encoding connexin-26 cause the ectodermal dysplasia keratitis-ichthyosis-deafness syndrome. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">70</span>:1341–8.</span> [<a href="/pmc/articles/PMC447609/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC447609</span></a>] [<a href="/pubmed/11912510" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11912510</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.richard.1998.393">Richard G, White TW, Smith LE, Bailey RA, Compton JG, Paul DL, Bale SJ. Functional defects of Cx26 resulting from a heterozygous missense mutation in a family with dominant deaf-mutism and palmoplantar keratoderma. <span><span class="ref-journal">Hum Genet. </span>1998;<span class="ref-vol">103</span>:393–9.</span> [<a href="/pubmed/9856479" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9856479</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.sch_tz.2010.4759">Sch&#x000fc;tz M, Scimemi P, Majumder P, De Siati RD, Crispino G, Rodriguez L, Bortolozzi M, Santarelli R, Seydel A, Sonntag S, Ingham N, Steel KP, Willecke K, Mammano F. The human deafness-associated connexin 30 T5M mutation causes mild hearing loss and reduces biochemical coupling among cochlear non-sensory cells in knock-in mice. <span><span class="ref-journal">Hum Mol Genet. </span>2010;<span class="ref-vol">19</span>:4759–73.</span> [<a href="/pmc/articles/PMC2989887/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2989887</span></a>] [<a href="/pubmed/20858605" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20858605</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.shearer.2014a.445">Shearer AE, Eppsteiner RW, Booth KT, Ephraim SS, Gurrola J 2nd, Simpson A, Black-Ziegelbein EA, Joshi S, Ravi H, Giuffre AC, Happe S, Hildebrand MS, Azaiez H, Bayazit YA, Erdal ME, Lopez-Escamez JA, Gazquez I, Tamayo ML, Gelvez NY, Leal GL, Jalas C, Ekstein J, Yang T, Usami S, Kahrizi K, Bazazzadegan N, Najmabadi H, Scheetz TE, Braun TA, Casavant TL, LeProust EM, Smith RJ. Utilizing ethnic-specific differences in minor allele frequency to recategorize reported pathogenic deafness variants. <span><span class="ref-journal">Am J Hum Genet. </span>2014a;<span class="ref-vol">95</span>:445–53.</span> [<a href="/pmc/articles/PMC4185121/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4185121</span></a>] [<a href="/pubmed/25262649" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 25262649</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.shearer.2014b.37">Shearer AE, Kolbe DL, Azaiez H, Sloan CM, Frees KL, Weaver AE, Clark ET, Nishimura CJ, Black-Ziegelbein EA, Smith RJ. Copy number variants are a common cause of non-syndromic hearing loss. <span><span class="ref-journal">Genome Med. </span>2014b;<span class="ref-vol">6</span>:37.</span> [<a href="/pmc/articles/PMC4067994/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4067994</span></a>] [<a href="/pubmed/24963352" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24963352</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.shearer.2015.175">Shearer AE, Smith RJ. Massively Parallel Sequencing for Genetic Diagnosis of Hearing Loss: The New Standard of Care. <span><span class="ref-journal">Otolaryngol Head Neck Surg. </span>2015;<span class="ref-vol">153</span>:175–82.</span> [<a href="/pmc/articles/PMC4743024/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4743024</span></a>] [<a href="/pubmed/26084827" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26084827</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.sloanheggen.2016.441">Sloan-Heggen CM, Bierer AO, Shearer AE, Kolbe DL, Nishimura CJ, Frees KL, Ephraim SS, Shibata SB, Booth KT, Campbell CA, Ranum PT, Weaver AE, Black-Ziegelbein EA, Wang D, Azaiez H, Smith RJ. Comprehensive genetic testing in the clinical evaluation of 1119 patients with hearing loss. <span><span class="ref-journal">Hum Genet. </span>2016;<span class="ref-vol">135</span>:441–50.</span> [<a href="/pmc/articles/PMC4796320/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4796320</span></a>] [<a href="/pubmed/26969326" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26969326</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.smith.2002.530">Smith FJ, Morley SM, McLean WH. A novel connexin 30 mutation in Clouston syndrome. <span><span class="ref-journal">J Invest Dermatol. </span>2002;<span class="ref-vol">118</span>:530–2.</span> [<a href="/pubmed/11874494" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11874494</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.smith.2005.879">Smith RJ, Bale JF Jr, White KR. Sensorineural hearing loss in children. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">365</span>:879–90.</span> [<a href="/pubmed/15752533" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15752533</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.taylor.2016.361">Taylor KR, Booth KT, Azaiez H, Sloan CM, Kolbe DL, Glanz EN, Shearer AE, DeLuca AP, Anand VN, Hildebrand MS, Simpson AC, Eppsteiner RW, Scheetz TE, Braun TA, Huygen PLM, Smith RJH, Casavant TL. Audioprofile surfaces: the 21<sup>st</sup> century audiogram. <span><span class="ref-journal">Ann Otol Rhinol Laryngol. </span>2016;<span class="ref-vol">125</span>:361–8.</span> [<a href="/pmc/articles/PMC4821702/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4821702</span></a>] [<a href="/pubmed/26530094" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26530094</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.tekin.2001.269">Tekin M, Arnos KS, Xia XJ, Oelrich MK, Liu XZ, Nance WE, Pandya A. W44C mutation in the connexin 26 gene associated with dominant non-syndromic deafness. <span><span class="ref-journal">Clin Genet. </span>2001;<span class="ref-vol">59</span>:269–73.</span> [<a href="/pubmed/11298683" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11298683</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.van_geel.2002.938">van Geel M, van Steensel MA, K&#x000fc;ster W, Hennies HC, Happle R, Steijlen PM, K&#x000f6;nig A. HID and KID syndromes are associated with the same connexin 26 mutation. <span><span class="ref-journal">Br J Dermatol. </span>2002;<span class="ref-vol">146</span>:938–42.</span> [<a href="/pubmed/12072059" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12072059</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.van_steensel.2002.724">van Steensel MA, van Geel M, Nahuys M, Smitt JH, Steijlen PM. A novel connexin 26 mutation in a patient diagnosed with keratitis- ichthyosis-deafness syndrome. <span><span class="ref-journal">J Invest Dermatol. </span>2002;<span class="ref-vol">118</span>:724–7.</span> [<a href="/pubmed/11918723" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11918723</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.wang.2011.e21473">Wang WH, Liu YF, Su CC, Su MC, Li SY, Yang JJ. A novel missense mutation in the connexin30 causes nonsyndromic hearing loss. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>:e21473.</span> [<a href="/pmc/articles/PMC3123352/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3123352</span></a>] [<a href="/pubmed/21731760" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 21731760</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.weegerink.2013">Weegerink NJ. Hereditary hearing impairment. Clinical and genetic aspects of DFNA3, DFNB8/10, DFNX4, Muckle-Wells syndrome and otosclerosis. Nijmegen, Netherlands: Radboud Universiteit Nijmegen; 2013.</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.weegerink.2011.191">Weegerink NJ, Pennings RJ, Huygen PL, Hoefsloot LH, Cremers CW, Kunst HP. Phenotypes of two Dutch DFNA3 families with mutations in GJB2. <span><span class="ref-journal">Ann Otol Rhinol Laryngol. </span>2011;<span class="ref-vol">120</span>:191–7.</span> [<a href="/pubmed/21510145" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 21510145</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.xiao.2004.1797">Xiao ZA, Xie DH. GJB2 (Cx26) gene mutations in Chinese patients with congenital sensorineural deafness and a report of one novel mutation. <span><span class="ref-journal">Chin Med J (Engl). </span>2004;<span class="ref-vol">117</span>:1797–801.</span> [<a href="/pubmed/15603707" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15603707</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.yang.2007.198">Yang JJ, Huang SH, Chou KH, Liao PJ, Su CC, Li SY. Identification of mutations in members of the connexin gene family as a cause of nonsyndromic deafness in Taiwan. <span><span class="ref-journal">Audiol Neurootol. </span>2007;<span class="ref-vol">12</span>:198–208.</span> [<a href="/pubmed/17259707" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17259707</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.yeager.2007.521">Yeager M, Harris AL. Gap junction channel structure in the early 21st century: facts and fantasies. <span><span class="ref-journal">Curr Opin Cell Biol. </span>2007;<span class="ref-vol">19</span>:521–8.</span> [<a href="/pmc/articles/PMC2819411/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2819411</span></a>] [<a href="/pubmed/17945477" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17945477</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.yum.2010.226">Yum SW, Zhang J, Scherer SS. Dominant connexin26 mutants associated with human hearing loss have trans-dominant effects on connexin30. <span><span class="ref-journal">Neurobiol Dis. </span>2010;<span class="ref-vol">38</span>:226–36.</span> [<a href="/pmc/articles/PMC2868926/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2868926</span></a>] [<a href="/pubmed/20096356" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20096356</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna3.REF.zhang.2011.71">Zhang J, Scherer SS, Yum SW. Dominant Cx26 mutants associated with hearing loss have dominant-negative effects on wild type Cx26. <span><span class="ref-journal">Mol Cell Neurosci. </span>2011;<span class="ref-vol">47</span>:71–8.</span> [<a href="/pmc/articles/PMC3132585/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3132585</span></a>] [<a href="/pubmed/21040787" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 21040787</span></a>]</div></li></ul></div></div><div id="dfna3.Chapter_Notes"><h2 id="_dfna3_Chapter_Notes_">Chapter Notes</h2><div id="dfna3.Acknowledgments"><h3>Acknowledgments</h3><p>This work was originally supported in part by research grants 1RO1 DC02842 and DC003544 (RJHS), HG00457 and P50HG00835 (VCS), and Belgian National Fonds voor Wetenschappelijk Onderzoek (GVC).</p></div><div id="dfna3.Author_History"><h3>Author History</h3><p>Paul T Ranum, BA (2014-present)<br />Daryl A Scott, MD, PhD; University of Iowa (1998-2001)<br />Abraham M Sheffield, MD; University of Iowa (2009-2014)<br />Val C Sheffield, MD, PhD; University of Iowa (1998-2001)<br />Richard JH Smith, MD (1998-present)<br />Guy Van Camp, PhD; University of Antwerp (1998-2016)</p></div><div id="dfna3.Revision_History"><h3>Revision History</h3><ul><li class="half_rhythm"><div>22 December 2016 (bp) Comprehensive update posted live</div></li><li class="half_rhythm"><div>12 June 2014 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>19 April 2012 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>30 April 2009 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>29 December 2005 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>15 July 2004 (rjs) Revision: use of an interpreter</div></li><li class="half_rhythm"><div>27 October 2003 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>24 April 2001 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>28 September 1998 (pb) Review posted live</div></li><li class="half_rhythm"><div>4 April 1998 (rjs) Original submission</div></li></ul></div></div><div id="bk_toc_contnr"></div></div></div>
            <div class="post-content"><div><div class="half_rhythm"><a href="/books/about/copyright/">Copyright</a> © 1993-2019, University of Washington, Seattle. GeneReviews is
a registered trademark of the University of Washington, Seattle. All rights
reserved.<p class="small">GeneReviews® chapters are owned by the University of Washington. Permission is
hereby granted to reproduce, distribute, and translate copies of content materials for
noncommercial research purposes only, provided that (i) credit for source (<a href="http://www.genereviews.org/" ref="pagearea=meta&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">http://www.genereviews.org/</a>) and copyright (© 1993-2019 University of
Washington) are included with each copy; (ii) a link to the original material is provided
whenever the material is published elsewhere on the Web; and (iii) reproducers,
distributors, and/or translators comply with the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>. No further modifications are allowed. For clarity, excerpts
of GeneReviews chapters for use in lab reports and clinic notes are a permitted
use.</p><p class="small">For more information, see the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>.</p><p class="small">For questions regarding permissions or whether a specified use is allowed,
contact: <a href="mailto:dev@null" data-email="ude.wu@tssamda" class="oemail">ude.wu@tssamda</a>.</p></div><div class="small"><span class="label">Bookshelf ID: NBK1536</span><span class="label">PMID: <a href="/pubmed/20301708" title="PubMed record of this page" ref="pagearea=meta&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">20301708</a></span></div><div style="margin-top:2em" class="bk_noprnt"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/gonad-dys-46xy/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/dfnb1/" title="Next page in this title">Next &gt;</a></div></div></div></div>
            
        </div>

        <!-- Custom content below content -->
        <div class="col4">
            
        </div>
        
        
        <!-- Book content -->
        
        <!-- Custom contetnt below bottom nav -->
        <div class="col5">
            
        </div>
    </div>

    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            <div id="ncbi_share_book"><a href="#" class="ncbi_share" data-ncbi_share_config="popup:false,shorten:true" ref="id=NBK1536&amp;db=books">Share</a></div>

        </div>
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Views</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PDF_download" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/NBK1536/?report=reader">PubReader</a></li><li><a href="/books/NBK1536/?report=printable">Print View</a></li><li><a data-jig="ncbidialog" href="#_ncbi_dlg_citbx_NBK1536" data-jigconfig="width:400,modal:true">Cite this Page</a><div id="_ncbi_dlg_citbx_NBK1536" style="display:none" title="Cite this Page"><div class="bk_tt">Smith RJH, Ranum PT. Nonsyndromic Hearing Loss and Deafness, DFNA3. 1998 Sep 28 [Updated 2016 Dec 22]. In: Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. <span class="bk_cite_avail"></span></div></div></li><li><a href="/books/NBK1536/pdf/Bookshelf_NBK1536.pdf">PDF version of this page</a> (713K)</li><li><a href="#" class="toggle-glossary-link" title="Enable/disable links to the glossary">Disable Glossary Links</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>In this GeneReview</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="page-toc" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="#dfna3.Summary" ref="log$=inpage&amp;link_id=inpage">Summary</a></li><li><a href="#dfna3.Diagnosis" ref="log$=inpage&amp;link_id=inpage">Diagnosis</a></li><li><a href="#dfna3.Clinical_Characteristics" ref="log$=inpage&amp;link_id=inpage">Clinical Characteristics</a></li><li><a href="#dfna3.Genetically_Related_Allelic_Disord" ref="log$=inpage&amp;link_id=inpage">Genetically Related (Allelic) Disorders</a></li><li><a href="#dfna3.Differential_Diagnosis" ref="log$=inpage&amp;link_id=inpage">Differential Diagnosis</a></li><li><a href="#dfna3.Management" ref="log$=inpage&amp;link_id=inpage">Management</a></li><li><a href="#dfna3.Genetic_Counseling" ref="log$=inpage&amp;link_id=inpage">Genetic Counseling</a></li><li><a href="#dfna3.Resources" ref="log$=inpage&amp;link_id=inpage">Resources</a></li><li><a href="#dfna3.Molecular_Genetics" ref="log$=inpage&amp;link_id=inpage">Molecular Genetics</a></li><li><a href="#dfna3.References" ref="log$=inpage&amp;link_id=inpage">References</a></li><li><a href="#dfna3.Chapter_Notes" ref="log$=inpage&amp;link_id=inpage">Chapter Notes</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>GeneReviews Links</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="source-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/n/gene/advanced/"><i>GeneReviews</i> Advanced Search</a></li><li><a href="/books/n/gene/glossary/"><i>GeneReviews</i> Glossary</a></li><li><a href="/books/n/gene/resource_mats/">Resource Materials</a> <span class="bk_hlight1">NEW FEATURE</span></li><li><a href="/books/n/gene/updates/">New in <i>GeneReviews</i></a></li><li><a href="/books/n/gene/authors/">Author List</a></li><li><a href="/books/n/gene/prospective_authors/">For Current/Prospective Authors</a></li><li><a href="/books/n/gene/GRpersonnel/"><i>GeneReviews</i> Personnel</a></li><li><a href="/books/n/gene/howto_linkin/">Download/Link to <i>GeneReviews</i></a></li><li><a href="/books/n/gene/contact_us/">Contact Us</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Tests in GTR by Gene</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="document-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=2706[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GJB2</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=10804[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">GJB6</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Related information</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="discovery_db_links" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=omim&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_omim&amp;IdsFromResult=1472152" ref="log$=recordlinks">OMIM</a><div class="brieflinkpop offscreen_noflow">Related OMIM records</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pmc&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pmc_refs&amp;IdsFromResult=1472152" ref="log$=recordlinks">PMC</a><div class="brieflinkpop offscreen_noflow">PubMed Central citations</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pubmed&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pubmed_refs&amp;IdsFromResult=1472152" ref="log$=recordlinks">PubMed</a><div class="brieflinkpop offscreen_noflow">Links to PubMed</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=gene&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_gene&amp;IdsFromResult=1472152" ref="log$=recordlinks">Gene</a><div class="brieflinkpop offscreen_noflow">Locus Links</div></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301449" ref="ordinalpos=1&amp;linkpos=1&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, DFNB1</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, DFNB1<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Smith RJH, Jones MKN. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301388" ref="ordinalpos=1&amp;linkpos=2&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> DFNA2 Nonsyndromic Hearing Loss</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> DFNA2 Nonsyndromic Hearing Loss<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Smith RJH, Hildebrand M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301429" ref="ordinalpos=1&amp;linkpos=3&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">OTOF</i>-Related Deafness</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">OTOF</i>-Related Deafness<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Shearer AE, Smith RJH. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/25719194" ref="ordinalpos=1&amp;linkpos=4&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">TBC1D24</i>-Related Disorders</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">TBC1D24</i>-Related Disorders<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Mucha BE, Hennekam RCM, Sisodiya S, Campeau PM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/25590127" ref="ordinalpos=1&amp;linkpos=5&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Deafness and Myopia Syndrome</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Deafness and Myopia Syndrome<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Ordonez JL, Tekin M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li></ul><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=20301708" ref="ordinalpos=1&amp;log$=relatedreviews_seeall&amp;logdbfrom=pubmed">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=20301708" ref="ordinalpos=1&amp;log$=relatedarticles_seeall&amp;logdbfrom=pubmed">See all...</a></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Recent Activity</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="Shutter"></a></div><div class="portlet_content"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class=""><div class="action"><a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a></div><ul id="activity"><li class="ra_rcd ralinkpopper two_line"><a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5e041aaf39a66ee2ba724696">Nonsyndromic Hearing Loss and Deafness, DFNA3 - GeneReviews®</a><div class="ralinkpop offscreen_noflow">Nonsyndromic Hearing Loss and Deafness, DFNA3 - GeneReviews®<div class="brieflinkpopdesc"></div></div><div class="tertiary"></div></li></ul><p class="HTOn">Your browsing activity is empty.</p><p class="HTOff">Activity recording is turned off.</p><p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p><a class="seemore" href="/sites/myncbi/recentactivity">See more...</a></div></div></div>
        
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script type="text/javascript" src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script type="text/javascript">
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-12-25T21:27:59-05:00&amp;Snapshot=%2Fprojects%2Fbooks%2FPBooks@5.22&amp;Host=portal101&amp;ncbi_phid=CE88DFEDE040C1410000000006610264&amp;ncbi_session=CE88DFEDE041AAE1_1633SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK1536%2F&amp;Db=pmc&amp;folderID=134&amp;Ncbi_App=bookshelf&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        </div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=books&amp;ncbi_pdid=book-part&amp;ncbi_acc=NBK1536&amp;ncbi_domain=gene&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/NBK1536/&amp;ncbi_pagename=Nonsyndromic Hearing Loss and Deafness, DFNA3 - GeneReviews® - NCBI Bookshelf&amp;ncbi_bookparttype=chapter&amp;ncbi_app=bookshelf" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE88DFEDE041AAE1_1633SID /projects/books/PBooks@5.22 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/js/3879255/4121861/3501987/4008961/3893018/3821238/4062932/4123007/4087685/4076480/3921943/3400083/3426610.js" snapshot="books"></script></body>
</html>