<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    
    <head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <!-- AppResources meta begin -->
        <meta name="paf-app-resources" content="" />
                 <script type="text/javascript">var ncbi_startTime = new Date();</script>
            
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="" />

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="books" /><meta name="ncbi_pdid" content="book-part" /><meta name="ncbi_acc" content="NBK1209" /><meta name="ncbi_domain" content="gene" /><meta name="ncbi_report" content="record" /><meta name="ncbi_type" content="fulltext" /><meta name="ncbi_objectid" content="" /><meta name="ncbi_pcid" content="/NBK1209/" /><meta name="ncbi_pagename" content="DFNA2 Nonsyndromic Hearing Loss - GeneReviews® - NCBI Bookshelf" /><meta name="ncbi_bookparttype" content="chapter" /><meta name="ncbi_app" content="bookshelf" />
        <!-- Logger end -->
        
        <title>DFNA2 Nonsyndromic Hearing Loss - GeneReviews® - NCBI Bookshelf</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css" /><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,FOLLOW,NOARCHIVE" /><meta name="citation_inbook_title" content="GeneReviews® [Internet]" /><meta name="citation_title" content="DFNA2 Nonsyndromic Hearing Loss" /><meta name="citation_publisher" content="University of Washington, Seattle" /><meta name="citation_date" content="2018/05/10" /><meta name="citation_author" content="Richard JH Smith" /><meta name="citation_author" content="Michael Hildebrand" /><meta name="citation_pmid" content="20301388" /><meta name="citation_fulltext_html_url" content="https://www.ncbi.nlm.nih.gov/books/NBK1209/" /><meta name="citation_keywords" content="Potassium voltage-gated channel subfamily KQT member 4" /><meta name="citation_keywords" content="KCNQ4" /><meta name="citation_keywords" content="DFNA2 Nonsyndromic Hearing Loss" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="DC.Title" content="DFNA2 Nonsyndromic Hearing Loss" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="University of Washington, Seattle" /><meta name="DC.Contributor" content="Richard JH Smith" /><meta name="DC.Contributor" content="Michael Hildebrand" /><meta name="DC.Date" content="2018/05/10" /><meta name="DC.Identifier" content="https://www.ncbi.nlm.nih.gov/books/NBK1209/" /><meta name="description" content="DFNA2 nonsyndromic hearing loss is characterized by symmetric, predominantly high-frequency sensorineural hearing loss (SNHL) that is progressive across all frequencies. At younger ages, hearing loss tends to be mild in the low frequencies and moderate in the high frequencies; in older persons, the hearing loss is moderate in the low frequencies and severe to profound in the high frequencies. Although the hearing impairment is often detected during routine hearing assessment of a school-age child, it is likely that hearing is impaired from birth, especially at high frequencies. Most affected persons initially require hearing aids to assist with sound amplification between ages ten and 40 years. By age 70 years, all persons with DFNA2 nonsyndromic hearing loss have severe-to-profound hearing impairment." /><meta name="og:title" content="DFNA2 Nonsyndromic Hearing Loss" /><meta name="og:type" content="book" /><meta name="og:description" content="DFNA2 nonsyndromic hearing loss is characterized by symmetric, predominantly high-frequency sensorineural hearing loss (SNHL) that is progressive across all frequencies. At younger ages, hearing loss tends to be mild in the low frequencies and moderate in the high frequencies; in older persons, the hearing loss is moderate in the low frequencies and severe to profound in the high frequencies. Although the hearing impairment is often detected during routine hearing assessment of a school-age child, it is likely that hearing is impaired from birth, especially at high frequencies. Most affected persons initially require hearing aids to assist with sound amplification between ages ten and 40 years. By age 70 years, all persons with DFNA2 nonsyndromic hearing loss have severe-to-profound hearing impairment." /><meta name="og:url" content="https://www.ncbi.nlm.nih.gov/books/NBK1209/" /><meta name="og:site_name" content="NCBI Bookshelf" /><meta name="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" /><meta name="twitter:card" content="summary" /><meta name="twitter:site" content="@ncbibooks" /><meta name="bk-non-canon-loc" content="/books/n/gene/dfna2/" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/books/NBK1209/" /><link rel="stylesheet" href="/corehtml/pmc/css/figpopup.css" type="text/css" media="screen" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books_print.min.css" type="text/css" media="print" /><style type="text/css">p a.figpopup{display:inline !important} .bk_tt {font-family: monospace}  .first-line-outdent .bk_ref {display: inline}  .body-content h2, .body-content .h2  {border-bottom: 1px solid #97B0C8} .body-content h2.inline {border-bottom: none} a.page-toc-label , .jig-ncbismoothscroll a {text-decoration:none;border:0 !important} .temp-labeled-list  .graphic {display:inline-block !important} .temp-labeled-list  img{width:100%}</style><script type="text/javascript" src="/corehtml/pmc/js/jquery.hoverIntent.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"> </script><script type="text/javascript">window.name="mainwindow";</script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/book-toc.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/books.min.js"> </script><meta name="book-collection" content="NONE" />

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="CE881E49E04174E10000000008F50360.m_12" />
<meta name='referrer' content='origin-when-cross-origin'/><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4098876/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/4008682/3881636/3579733/4062871/12930/3964959/3854974/36029/4128070/9685/3549676/3609192/3609193/3609213/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3411343/3882866.css" media="print" /></head>
    <body class="book-part">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo"><h1 class="res_name"><a href="/books/" title="Bookshelf home">Bookshelf</a></h1><h2 class="res_tagline"></h2></div>
    <div class="search"><form method="get" action="/books/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="books" selected="selected" class="last" data-ac_dict="bookshelf-search">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books" data-ac_dict="bookshelf-search">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search Books. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'bookshelf-search',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'no'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/books/browse/">Browse Titles</a>
                    </li><li>
                        <a href="/books/advanced/">Advanced</a>
                    </li><li class="help">
                        <a href="/books/NBK3833/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <!-- Custom content 1 -->
<div class="col1">
    
</div>

<div class="container">
    <div id="maincontent" class="content eight_col col">
        <!-- Custom content in the left column above book nav -->
        <div class="col2">
            
        </div>
        
        <!-- Book content -->
        

        <!-- Custom content between navigation and content -->
        <div class="col3">
            
        </div>
        
        <div class="document">
            <div class="pre-content"><div><div class="bk_prnt"><p class="small">NCBI Bookshelf. A service of the National Library of Medicine, National Institutes of Health.</p><p>Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. </p></div><div class="iconblock clearfix whole_rhythm no_top_margin bk_noprnt"><a class="img_link icnblk_img" title="All GeneReviews" href="/books/n/gene/"><img class="source-thumb" src="/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" alt="Cover of GeneReviews®" height="100px" width="80px" /></a><div class="icnblk_cntnt eight_col"><h2>GeneReviews<sup>®</sup> [Internet].</h2><a data-jig="ncbitoggler" href="#__NBK1209_dtls__">Show details</a><div style="display:none" class="ui-widget" id="__NBK1209_dtls__"><div>Adam MP, Ardinger HH, Pagon RA, et al., editors.</div><div>Seattle (WA): <a href="http://www.washington.edu" ref="pagearea=page-banner&amp;targetsite=external&amp;targetcat=link&amp;targettype=publisher">University of Washington, Seattle</a>; 1993-2019.</div></div><div class="half_rhythm"><ul class="inline_list"><li style="margin-right:1em"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a></li></ul></div><div class="bk_noprnt"><form method="get" action="/books/n/gene/" id="bk_srch"><div class="bk_search"><label for="bk_term" class="offscreen_noflow">Search term</label><input type="text" title="Search GeneReviews" id="bk_term" name="term" value="" data-jig="ncbiclearbutton" /> <input type="submit" class="jig-ncbibutton" value="Search GeneReviews" submit="false" style="padding: 0.1em 0.4em;" /></div></form><div><ul class="inline_list"><li><a href="/books/n/gene/advanced/">GeneReviews Advanced Search</a></li><li style="margin-left:.5em"><a href="/books/n/gene/helpadvsearch/">Help</a></li></ul></div></div></div><div class="icnblk_cntnt two_col"><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/depdc5-epilepsy/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/dfnx1/" title="Next page in this title">Next &gt;</a></div></div></div></div></div>
            <div class="main-content lit-style" itemscope="itemscope" itemtype="http://schema.org/CreativeWork"><div class="meta-content fm-sec"><h1 id="_NBK1209_"><span class="title" itemprop="name">DFNA2 Nonsyndromic Hearing Loss</span></h1><p class="contrib-group"><span itemprop="author">Richard JH Smith</span>, MD and <span itemprop="author">Michael Hildebrand</span>, PhD.</p><a data-jig="ncbitoggler" href="#__NBK1209_ai__" style="border:0;text-decoration:none">Author Information</a><div style="display:none" class="ui-widget" id="__NBK1209_ai__"><div class="contrib half_rhythm"><span itemprop="author">Richard JH Smith</span>, MD<div class="affiliation small">Director, Molecular Otolaryngology Research Laboratories<br />Sterba Hearing Research Professor of Otolaryngology<br />Professor of Otolaryngology, Pediatrics, and Internal Medicine, Division of Nephrology<br />Carver College of Medicine<br />University of Iowa<br />Iowa City, Iowa<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.awoiu@htims-drahcir" class="oemail">ude.awoiu@htims-drahcir</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Michael Hildebrand</span>, PhD<div class="affiliation small">Department of Medicine<br />University of Melbourne<br />Melbourne, Australia<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ua.ude.bleminu@dnarbedlih.leahcim" class="oemail">ua.ude.bleminu@dnarbedlih.leahcim</a></div></div></div></div><p class="small">Initial Posting: <span itemprop="datePublished">April 4, 2008</span>; Last Update: <span itemprop="dateModified">May 10, 2018</span>.</p><p><em>Estimated reading time: 17 minutes</em></p></div><div class="jig-ncbiinpagenav body-content whole_rhythm" data-jigconfig="allHeadingLevels: ['h2'],smoothScroll: false" itemprop="text"><div id="dfna2.Summary" itemprop="description"><h2 id="_dfna2_Summary_">Summary</h2><div><h4 class="inline">Clinical characteristics.</h4><p>DFNA2 nonsyndromic hearing loss is characterized by symmetric, predominantly high-frequency sensorineural hearing loss (SNHL) that is progressive across all frequencies. At younger ages, hearing loss tends to be mild in the low frequencies and moderate in the high frequencies; in older persons, the hearing loss is moderate in the low frequencies and severe to profound in the high frequencies. Although the hearing impairment is often detected during routine hearing assessment of a school-age child, it is likely that hearing is impaired from birth, especially at high frequencies. Most <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> persons initially require hearing aids to assist with sound amplification between ages ten and 40 years. By age 70 years, all persons with DFNA2 nonsyndromic hearing loss have severe-to-profound hearing impairment.</p></div><div><h4 class="inline">Diagnosis/testing.</h4><p>The diagnosis of DFNA2 nonsyndromic hearing loss is established in an individual with a characteristic audioprofile, a family history consistent with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> inheritance, and identification of a <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>KCNQ4</i>.</p></div><div><h4 class="inline">Management.</h4><p><i>Treatment of manifestations:</i> Hearing aids for those with mild-to-moderate hearing loss; consideration of cochlear implants when hearing loss is severe to profound; special assistance in school for hearing-impaired children and adolescents.</p><p><i>Surveillance:</i> At least annual audiogram to follow progression of hearing loss.</p><p><i>Agents/circumstances to avoid:</i> Avoiding exposure to loud noise may reduce the rate of progression of high-frequency SNHL.</p><p><i>Evaluation of relatives at risk:</i> Determining in infancy or early childhood whether a family member of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> has inherited a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>KCNQ4</i> allows for early support and management of the child and family.</p></div><div><h4 class="inline">Genetic counseling.</h4><p>DFNA2 nonsyndromic hearing loss is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> manner. Most individuals with DFNA2 nonsyndromic hearing loss have a parent with hearing loss; the proportion of individuals with a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a>
<i>KCNQ4</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> is unknown. Each child of an individual with DFNA2 nonsyndromic hearing loss has a 50% chance of inheriting the <i>KCNQ4</i> pathogenic variant. Once the <i>KCNQ4</i> pathogenic variant has been identified in a family member, prenatal testing for a pregnancy at increased risk and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> for DFNA2 nonsyndromic hearing loss are possible.</p></div></div><div id="dfna2.Diagnosis"><h2 id="_dfna2_Diagnosis_">Diagnosis</h2><div id="dfna2.Suggestive_Findings"><h3>Suggestive Findings</h3><p>DFNA2 nonsyndromic hearing loss <b>should be considered</b> in an individual with the following clinical, imaging, and family history findings:</p><p><b>Clinical findings</b></p><ul><li class="half_rhythm"><div>Symmetric, predominantly high-frequency sensorineural hearing loss (SNHL) that is progressive across all frequencies:</div><ul><li class="half_rhythm"><div>At younger ages, hearing loss tends to be mild in the low frequencies and moderate in the high frequencies.</div></li><li class="half_rhythm"><div>In older persons, the hearing loss is moderate in the low frequencies and severe to profound in the high frequencies.</div></li></ul></li><li class="half_rhythm"><div>Normal physical examination</div></li></ul><p><b>Imaging findings.</b> Temporal bone imaging (i.e., CT of the inner ears) is normal. Specifically, abnormalities such as dilation of the vestibular aqueducts (also known as enlarged vestibular aqueducts) and Mondini dysplasia should not be present.</p><p><b>Family history</b> of hearing loss is present and consistent with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> inheritance.</p></div><div id="dfna2.Establishing_the_Diagnosis"><h3>Establishing the Diagnosis</h3><p>The diagnosis of DFNA2 nonsyndromic hearing loss <b>is established</b> in a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with the above characteristic audioprofile and identification of a <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>KCNQ4</i> on <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> (see <a class="figpopup" href="/books/NBK1209/table/dfna2.T.molecular_genetic_testing_used_i/?report=objectonly" target="object" rid-figpopup="figdfna2Tmoleculargenetictestingusedi" rid-ob="figobdfna2Tmoleculargenetictestingusedi">Table 1</a>).</p><p>Because the <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> of DFNA2 nonsyndromic hearing loss is indistinguishable from many other inherited disorders with hearing loss, recommended <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> approaches include use of a <b><a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a></b> or <b>comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing</b>.</p><p>Note: Single-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> testing (<a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> of <i>KCNQ4,</i> followed by gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>) is rarely useful and typically NOT recommended.</p><p><b>A multigene hearing loss and deafness panel</b> that includes <i>KCNQ4</i> and other genes of interest (see <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a>) is most likely to identify the genetic cause of the condition at the most reasonable cost while limiting identification of variants of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a> and pathogenic variants in genes that do not explain the underlying <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>. Note: (1) The genes included in the panel and the diagnostic <a class="def" href="/books/n/gene/glossary/def-item/sensitivity/">sensitivity</a> of the testing used for each <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> vary by laboratory and are likely to change over time. (2) Some multigene panels may include genes not associated with the condition discussed in this <i>GeneReview</i>. (3) In some laboratories, panel options may include a custom laboratory-designed panel and/or custom phenotype-focused <a class="def" href="/books/n/gene/glossary/def-item/exome/">exome</a> analysis that includes genes specified by the clinician. (4) Methods used in a panel may include <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a>, <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>, and/or other non-sequencing-based tests. For this disorder, a <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a> that also includes deletion/duplication analysis is recommended (see <a class="figpopup" href="/books/NBK1209/table/dfna2.T.molecular_genetic_testing_used_i/?report=objectonly" target="object" rid-figpopup="figdfna2Tmoleculargenetictestingusedi" rid-ob="figobdfna2Tmoleculargenetictestingusedi">Table 1</a>).</p><p>For an introduction to multigene panels click <a href="/books/n/gene/app5/#app5.Multigene_Panels">here</a>. More detailed information for clinicians ordering genetic tests can be found <a href="/books/n/gene/app5/#app5.Multigene_Panels_FAQs">here</a>.</p><p><b>Comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing</b> (which does not require the clinician to determine which <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>[s] are likely involved) is another good option. <b>Exome sequencing</b> is most commonly used; <b><a class="def" href="/books/n/gene/glossary/def-item/genome-sequencing/">genome sequencing</a></b> is also possible. <b>Exome array</b> (when clinically available) may be considered if <a class="def" href="/books/n/gene/glossary/def-item/exome-sequencing/">exome sequencing</a> is not diagnostic, particularly when evidence supports <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> inheritance.</p><p>For an introduction to comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing click <a href="/books/n/gene/app5/#app5.Comprehensive_Genomic_Testing">here</a>. More detailed information for clinicians ordering genomic testing can be found <a href="/books/n/gene/app5/#app5.Comprehensive_Genomic_Testing_1">here</a>.</p><div id="dfna2.T.molecular_genetic_testing_used_i" class="table"><h3><span class="label">Table 1. </span></h3><div class="caption"><p>Molecular Genetic Testing Used in DFNA2 Nonsyndromic Hearing Loss</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1209/table/dfna2.T.molecular_genetic_testing_used_i/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna2.T.molecular_genetic_testing_used_i_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfna2.T.molecular_genetic_testing_used_i_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene&#x000a0;<sup>1</sup></th><th id="hd_h_dfna2.T.molecular_genetic_testing_used_i_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Method</th><th id="hd_h_dfna2.T.molecular_genetic_testing_used_i_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Proportion of Probands with a Pathogenic Variant&#x000a0;<sup>2</sup> Detectable by Method</th></tr></thead><tbody><tr><td headers="hd_h_dfna2.T.molecular_genetic_testing_used_i_1_1_1_1" rowspan="2" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><i>KCNQ4</i></td><td headers="hd_h_dfna2.T.molecular_genetic_testing_used_i_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Sequence analysis&#x000a0;<sup>3</sup></td><td headers="hd_h_dfna2.T.molecular_genetic_testing_used_i_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">99%&#x000a0;<sup>4</sup></td></tr><tr><td headers="hd_h_dfna2.T.molecular_genetic_testing_used_i_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>&#x000a0;<sup>5</sup></td><td headers="hd_h_dfna2.T.molecular_genetic_testing_used_i_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">1%&#x000a0;<sup>6</sup></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="dfna2.TF.1.1"><p class="no_margin">See <a href="/books/NBK1209/#dfna2.molgen.TA">Table A. Genes and Databases</a> for <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> and protein.</p></div></dd><dt>2. </dt><dd><div id="dfna2.TF.1.2"><p class="no_margin">See <a href="#dfna2.Molecular_Genetics">Molecular Genetics</a> for information on allelic variants detected in this <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>.</p></div></dd><dt>3. </dt><dd><div id="dfna2.TF.1.3"><p class="no_margin">Sequence analysis detects variants that are benign, likely benign, of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a>, likely pathogenic, or pathogenic. Pathogenic variants may include small intragenic deletions/insertions and <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a>, <a class="def" href="/books/n/gene/glossary/def-item/nonsense-variant/">nonsense</a>, and <a class="def" href="/books/n/gene/glossary/def-item/splice-site/">splice site</a> variants; typically, <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> or whole-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> deletions/duplications are not detected. For issues to consider in interpretation of <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> results, click <a href="/books/n/gene/app2/">here</a>.</p></div></dd><dt>4. </dt><dd><div id="dfna2.TF.1.4"><p class="no_margin">Sequence analysis detects pathogenic variants in <i>KCNQ4</i> in virtually all individuals with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> nonsyndromic sensorineural hearing loss mapping to the DFNA2 <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a>.</p></div></dd><dt>5. </dt><dd><div id="dfna2.TF.1.5"><p class="no_margin">Gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a> detects intragenic deletions or duplications. Methods used may include <a class="def" href="/books/n/gene/glossary/def-item/quantitative-pcr/">quantitative PCR</a>, long-range PCR, multiplex ligation-dependent probe amplification (MLPA), and a <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted microarray designed to detect single-<a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> deletions or duplications.</p></div></dd><dt>6. </dt><dd><div id="dfna2.TF.1.6"><p class="no_margin">Copy number variants (CNVs) can be detected in virtually any <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> included in targeted sequencing panels for deafness. In sequential testing of 2,506 persons with hearing loss using OtoSCOPE, one CNV of <i>KCNQ4</i> was identified [Author, personal observation].</p></div></dd></dl></div></div></div></div></div><div id="dfna2.Clinical_Characteristics"><h2 id="_dfna2_Clinical_Characteristics_">Clinical Characteristics</h2><div id="dfna2.Clinical_Description"><h3>Clinical Description</h3><p>All individuals with DFNA2 nonsyndromic hearing loss have symmetric, predominantly high-frequency hearing loss that is progressive across all frequencies. Initially, high frequencies are <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a>; later in life, hearing loss becomes severe to profound across all frequencies. A comprehensive review of the clinical presentation and prognosis of individuals diagnosed with DFNA2 nonsyndromic hearing loss has been provided by <a class="bk_pop" href="#dfna2.REF.de_leenheer.2002a.41">De Leenheer et al [2002a]</a>.</p><p>Onset of hearing loss is generally reported in early childhood or adolescence; however, it is likely that hearing is impaired from birth, especially at the high frequencies. The hearing loss is often detected during standard hearing assessment of a school-age child or less frequently during the evaluation of a child for delayed speech development.</p><p>In all <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals, the hearing loss is more severe at the high frequencies, resulting in a characteristic downsloping audioprofile with hearing thresholds between 50 and 90 dB at 500 Hz and between 90 and 120 dB at 2-4 kHz by age 50 years. A typical audiogram of an adolescent with DFNA2 nonsyndromic hearing loss is shown in <a class="figpopup" href="/books/NBK1209/figure/dfna2.F1/?report=objectonly" target="object" rid-figpopup="figdfna2F1" rid-ob="figobdfna2F1">Figure 1</a>.</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figdfna2F1" co-legend-rid="figlgnddfna2F1"><a href="/books/NBK1209/figure/dfna2.F1/?report=objectonly" target="object" title="Figure 1. " class="img_link icnblk_img figpopup" rid-figpopup="figdfna2F1" rid-ob="figobdfna2F1"><img class="small-thumb" src="/books/NBK1209/bin/dfna2-Image001.gif" src-large="/books/NBK1209/bin/dfna2-Image001.jpg" alt="Figure 1. . Audiogram from an individual age 12 years with DFNA2 hearing loss." /></a><div class="icnblk_cntnt" id="figlgnddfna2F1"><h4 id="dfna2.F1"><a href="/books/NBK1209/figure/dfna2.F1/?report=objectonly" target="object" rid-ob="figobdfna2F1">Figure 1. </a></h4><p class="float-caption no_bottom_margin">Audiogram from an individual age 12 years with DFNA2 hearing loss. Note that the loss is greater in the high frequencies at this age; with time, hearing at all frequencies progressively deteriorates. </p></div></div><p>Whereas onset age varies within families, deterioration of annual thresholds for families with DFNA2 nonsyndromic hearing loss has been calculated at a relatively uniform ~1 dB/year [<a class="bk_pop" href="#dfna2.REF.coucke.1999.1321">Coucke et al 1999</a>, <a class="bk_pop" href="#dfna2.REF.talebizadeh.1999.493">Talebizadeh et al 1999</a>, <a class="bk_pop" href="#dfna2.REF.ensink.2000.62">Ensink et al 2000</a>, <a class="bk_pop" href="#dfna2.REF.van_hauwe.2000.184">Van Hauwe et al 2000</a>, <a class="bk_pop" href="#dfna2.REF.akita.2001.355">Akita et al 2001</a>, <a class="bk_pop" href="#dfna2.REF.de_leenheer.2002a.41">De Leenheer et al 2002a</a>, <a class="bk_pop" href="#dfna2.REF.de_leenheer.2002b.267">De Leenheer et al 2002b</a>, <a class="bk_pop" href="#dfna2.REF.van_camp.2002.15">Van Camp et al 2002</a>]. Most persons with DFNA2 nonsyndromic hearing loss are first fitted with hearing aids to assist with sound amplification between ages ten and 40 years [<a class="bk_pop" href="#dfna2.REF.de_leenheer.2002a.41">De Leenheer et al 2002a</a>]. By age 70 years, all persons with hearing loss attributed to a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>KCNQ4</i> have severe-to-profound hearing loss.</p><p><b>Other findings</b></p><ul><li class="half_rhythm"><div><b>Vestibular function.</b> Thirty percent of individuals in two families with DFNA2 nonsyndromic hearing loss (Dutch families 1 and 4; <a class="figpopup" href="/books/NBK1209/table/dfna2.T.kcnq4_pathogenic_variants_discus/?report=objectonly" target="object" rid-figpopup="figdfna2Tkcnq4pathogenicvariantsdiscus" rid-ob="figobdfna2Tkcnq4pathogenicvariantsdiscus">Table 3</a>) had increased vestibulo-ocular reflex activity [<a class="bk_pop" href="#dfna2.REF.marres.1997.573">Marres et al 1997</a>, <a class="bk_pop" href="#dfna2.REF.de_leenheer.2002b.267">De Leenheer et al 2002b</a>]. Vestibular problems have not been observed in any other families with DFNA2 nonsyndromic hearing loss.</div></li><li class="half_rhythm"><div><b>Speech recognition scores.</b> When measured in several Dutch families, speech recognition scores were relatively good given the pure-tone thresholds [<a class="bk_pop" href="#dfna2.REF.de_leenheer.2002b.267">De Leenheer et al 2002b</a>, <a class="bk_pop" href="#dfna2.REF.van_camp.2002.15">Van Camp et al 2002</a>].</div></li></ul></div><div id="dfna2.GenotypePhenotype_Correlations"><h3>Genotype-Phenotype Correlations</h3><p>The <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> associated with <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> <i>KCNQ4</i> pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants is similar in all families: predominantly high-frequency sensorineural hearing loss (SNHL) that is detectable in childhood and progressive across all frequencies. At younger ages, hearing loss tends to be mild in the low frequencies and moderate in the high frequencies. In older persons, the hearing loss is moderate in the low frequencies and severe to profound in the high frequencies.</p><p>Although <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> onset of DFNA2 nonsyndromic hearing loss has been reported in one of the Dutch families with the <a class="figpopup" href="/books/NBK1209/table/dfna2.T.kcnq4_pathogenic_variants_discus/?report=objectonly" target="object" rid-figpopup="figdfna2Tkcnq4pathogenicvariantsdiscus" rid-ob="figobdfna2Tkcnq4pathogenicvariantsdiscus">p.Trp276Ser</a> variant [<a class="bk_pop" href="#dfna2.REF.de_leenheer.2002b.267">De Leenheer et al 2002b</a>, <a class="bk_pop" href="#dfna2.REF.van_camp.2002.15">Van Camp et al 2002</a>], it has not been reported in other families with this variant. The high-frequency hearing loss in this family was progressive without substantial loss of speech recognition during the first decades of life [<a class="bk_pop" href="#dfna2.REF.de_leenheer.2002b.267">De Leenheer et al 2002b</a>].</p><p>The <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> associated with <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> <i>KCNQ4</i> truncating variants differs from that associated with <i>KCNQ4</i> pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants. In two families, small frameshift deletions of <i>KCNQ4</i> (<a class="figpopup" href="/books/NBK1209/table/dfna2.T.kcnq4_pathogenic_variants_discus/?report=objectonly" target="object" rid-figpopup="figdfna2Tkcnq4pathogenicvariantsdiscus" rid-ob="figobdfna2Tkcnq4pathogenicvariantsdiscus">c.211_223del</a> and <a class="figpopup" href="/books/NBK1209/table/dfna2.T.kcnq4_pathogenic_variants_discus/?report=objectonly" target="object" rid-figpopup="figdfna2Tkcnq4pathogenicvariantsdiscus" rid-ob="figobdfna2Tkcnq4pathogenicvariantsdiscus">c.211delC</a>) are predicted to result in a profoundly truncated protein that either does not interact with normal protein translated from the normal <a class="def" href="/books/n/gene/glossary/def-item/allele/">allele</a> or may not remain in cells as a result of <a class="def" href="/books/n/gene/glossary/def-item/nonsense-variant/">nonsense</a>-mediated decay. The hearing loss associated with this dosage effect is milder in low and mid-frequencies, more severe in high frequencies, and later in onset than the hearing loss seen with pathogenic missense variants [<a class="bk_pop" href="#dfna2.REF.coucke.1999.1321">Coucke et al 1999</a>, <a class="bk_pop" href="#dfna2.REF.akita.2001.355">Akita et al 2001</a>].</p></div><div id="dfna2.Penetrance"><h3>Penetrance</h3><p>The <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> is complete. All individuals with a <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> <i>KCNQ4</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> exhibit the hearing loss <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>; onset age and severity are variable.</p></div><div id="dfna2.Prevalence"><h3>Prevalence</h3><p>In a study conducted between 2012 and 2014, the Molecular Otolaryngology and Renal Research Laboratories performed clinical diagnostic testing on a total of 1,119 individuals with hearing loss using the comprehensive genetic testing panel OtoSCOPE. It was determined that among the 440 individuals in whom hearing loss was hereditary, <i>KCNQ4</i> pathogenic variants accounted for 9.5% of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> neurosensory hearing loss (ADNSHL) [<a class="bk_pop" href="#dfna2.REF.sloanheggen.2016.441">Sloan-Heggen et al 2016</a>].</p></div></div><div id="dfna2.Genetically_Related_Allelic_Disord"><h2 id="_dfna2_Genetically_Related_Allelic_Disord_">Genetically Related (Allelic) Disorders</h2><p>No phenotypes other than those discussed in this <i>GeneReview</i> are known to be associated with <a class="def" href="/books/n/gene/glossary/def-item/germline/">germline</a> pathogenic variants in <i>KCNQ4</i>.</p></div><div id="dfna2.Differential_Diagnosis"><h2 id="_dfna2_Differential_Diagnosis_">Differential Diagnosis</h2><p>See <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a> for complete differential diagnosis. Note: <a href="/books/n/gene/deafness-overview/table/deafness-overview.T.autosomal_dominant_n/?report=objectonly" target="object">Table 4</a> has a complete listing of the genes and clinical manifestations of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> nonsyndromic hearing impairment.</p></div><div id="dfna2.Management"><h2 id="_dfna2_Management_">Management</h2><div id="dfna2.Evaluations_Following_Initial_Diag"><h3>Evaluations Following Initial Diagnosis</h3><p>To establish the extent of hearing loss and needs in an individual diagnosed with DFNA2 nonsyndromic hearing loss, the evaluations summarized in this section (if not performed as part of the evaluation that led to the diagnosis) are recommended:</p><ul><li class="half_rhythm"><div>Audiometry, including bone conduction testing</div></li><li class="half_rhythm"><div>Consultation with a clinical geneticist and/or genetic counselor</div></li></ul></div><div id="dfna2.Treatment_of_Manifestations"><h3>Treatment of Manifestations</h3><p>See <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a> for complete discussion of treatment.</p><p>When hearing loss is mild to moderate, fitting of hearing aids to provide improved amplification is warranted.</p><p>When the hearing loss becomes severe to profound, cochlear implants (CIs) can be considered. In individuals with preserved or relatively good low-frequency hearing and severe-to-profound high-frequency loss, a hybrid (short) cochlear implant may be considered. Hybrid implants combine two proven technologies &#x02013; acoustic amplification and implant technology &#x02013; to provide electroacoustic hearing. The acoustic component, which is coupled to the cochlear implant sound processor, amplifies residual low-frequency hearing; the electrical component uses cochlear implant technology to provide electrostimulation for high-frequency hearing.</p><p>For school-age children or adolescents, special assistance for the hearing impaired may be warranted and, where available, should be offered.</p></div><div id="dfna2.Surveillance"><h3>Surveillance</h3><p>Audiograms should be obtained on an annual basis to follow progression of hearing loss.</p></div><div id="dfna2.AgentsCircumstances_to_Avoid"><h3>Agents/Circumstances to Avoid</h3><p>The rate of progression of high-frequency hearing loss can be reduced by encouraging individuals with DFNA2 nonsyndromic hearing loss to avoid exposure to loud noise in the workplace and during recreation.</p></div><div id="dfna2.Evaluation_of_Relatives_at_Risk"><h3>Evaluation of Relatives at Risk</h3><p>Determining in infancy or early childhood whether a relative of a person with DFNA2 nonsyndromic hearing loss has inherited the <i>KCNQ4</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> allows for early support and management of the child and the family.</p><p>See <a href="#dfna2.Related_Genetic_Counseling_Issues">Genetic Counseling</a> for issues related to testing of at-risk relatives for <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> purposes.</p></div><div id="dfna2.Therapies_Under_Investigation"><h3>Therapies Under Investigation</h3><p>Search <a href="http://clinicaltrials.gov/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">ClinicalTrials.gov</a> in the US and <a href="http://www.clinicaltrialsregister.eu/ctr-search/search" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">EU Clinical Trials Register</a> in Europe for access to information on clinical studies for a wide range of diseases and conditions. Note: There may not be clinical trials for this disorder.</p></div></div><div id="dfna2.Genetic_Counseling"><h2 id="_dfna2_Genetic_Counseling_">Genetic Counseling</h2><p>
<i>Genetic counseling is the process of providing individuals and families with
information on the nature, inheritance, and implications of genetic disorders to help them
make informed medical and personal decisions. The following section deals with genetic
risk assessment and the use of family history and genetic testing to clarify genetic
status for family members. This section is not meant to address all personal, cultural, or
ethical issues that individuals may face or to substitute for consultation with a genetics
professional</i>. &#x02014;ED.</p><div id="dfna2.Mode_of_Inheritance"><h3>Mode of Inheritance</h3><p>DFNA2 nonsyndromic hearing loss is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> manner.</p></div><div id="dfna2.Risk_to_Family_Members"><h3>Risk to Family Members</h3><p><b>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>Most individuals diagnosed with DFNA2 nonsyndromic hearing loss have a deaf parent.</div></li><li class="half_rhythm"><div>A <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with DFNA2 nonsyndromic hearing loss may have deafness as the result of a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a>
<i>KCNQ4</i> variant. The proportion of individuals with a <i>de novo</i>
<i>KCNQ4</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> is not known. Of six clinically diagnosed individuals reported in the sequential series by <a class="bk_pop" href="#dfna2.REF.sloanheggen.2016.441">Sloan-Heggen et al [2016]</a>, five were verified to have <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> inheritance, whereas one represented a <a class="def" href="/books/n/gene/glossary/def-item/simplex/">simplex</a> case (i.e., a single occurrence in the family). Thus, it appears that in most instances <i>KCNQ4</i>-related hearing loss is inherited from an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> parent [<a class="bk_pop" href="#dfna2.REF.sloanheggen.2016.441">Sloan-Heggen et al 2016</a>].</div></li><li class="half_rhythm"><div>Recommendations for the evaluation of parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with an apparent <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> include audiometry and <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a>.</div></li><li class="half_rhythm"><div>If the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> found in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> cannot be detected in the leukocyte DNA of either parent, possible explanations include a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> pathogenic variant in the proband or <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a> in a parent. Though theoretically possible, no instances of germline mosaicism have been reported.</div></li><li class="half_rhythm"><div>The family history of some individuals with DFNA2 nonsyndromic hearing loss may appear to be negative because of failure to diagnosis a parent with a milder phenotypic presentation, early death of the parent before the onset of hearing loss, or late onset of hearing loss in a parent. Therefore, an apparently negative family history cannot be confirmed until the appropriate clinical evaluation and <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> have been performed on the parents of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</div></li><li class="half_rhythm"><div>Note: If the parent is the individual in whom the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> first occurred, s/he may have <a class="def" href="/books/n/gene/glossary/def-item/somatic-mosaicism/">somatic mosaicism</a> for the pathogenic variant and have mild hearing loss.</div></li></ul><p><b>Sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</b> The probability that the sibs of the proband will be deaf depends on the genetic status of the proband's parents:</p><ul><li class="half_rhythm"><div>If a parent of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> is deaf, each sib has a 50% chance of being deaf.</div></li><li class="half_rhythm"><div>If the <i>KCNQ4</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> found in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> cannot be detected in the leukocyte DNA of either parent, the probability that a sib will have DFNA2 nonsyndromic hearing loss is estimated to be 1% because of the theoretic possibility of parental <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a> [<a class="bk_pop" href="#dfna2.REF.rahbari.2016.126">Rahbari et al 2016</a>].</div></li><li class="half_rhythm"><div>If the parents have not been tested for the <i>KNCQ4</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> but are both hearing, the probability that a sib of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> will be deaf appears to be low. (Note: Sibs of a proband with clinically unaffected parents are presumed to have an increased probability of having DFNA2 nonsyndromic hearing loss because of the theoretic possibility of parental <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a>.)</div></li></ul><p><b>Offspring of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</b> Each child of an individual with DFNA2 nonsyndromic hearing loss has a 50% chance of inheriting the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>.</p><p><b>Other family members.</b> The probability of deafness in other family members depends on the status of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s parents: if a parent is deaf, his or her family members may also be deaf or develop deafness.</p></div><div id="dfna2.Related_Genetic_Counseling_Issues"><h3>Related Genetic Counseling Issues</h3><p>See Management, <a href="#dfna2.Evaluation_of_Relatives_at_Risk">Evaluation of Relatives at Risk</a> for information on evaluating relatives of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> for the purpose of early diagnosis and management.</p><p>The following points are noteworthy:</p><ul><li class="half_rhythm"><div>Communication with individuals who are members of the Deaf community and sign requires the services of a skilled interpreter.</div></li><li class="half_rhythm"><div>Members of the Deaf community may view deafness as a distinguishing characteristic and not as a handicap, impairment or medical condition requiring a "treatment" or "cure," or to be "prevented."</div></li><li class="half_rhythm"><div>Many deaf people are interested in obtaining information about the cause of their own deafness, including information on medical, educational and social services, rather than information about prevention, reproduction or family planning. It is important to ascertain and address the questions and concerns of the family/individual.</div></li><li class="half_rhythm"><div>The use of certain terms is preferred: probability or chance versus risk; deaf and hard-of-hearing versus hearing impaired. Terms such as "abnormal" should be avoided.</div></li></ul><p><b>Considerations in families with an apparent <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> variant.</b> When neither parent of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with DFNA2 nonsyndromic hearing loss has the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> or clinical evidence of deafness, it is likely that the proband has a <i>de novo</i> variant. However, possible non-medical explanations could be explored including <a class="def" href="/books/n/gene/glossary/def-item/alternate-paternity/">alternate paternity</a> or maternity (i.e., with assisted reproduction) or undisclosed adoption.</p><p><b>Family planning</b></p><ul><li class="half_rhythm"><div>The optimal time for determination of genetic status and discussion of the availability of prenatal testing is before pregnancy.</div></li><li class="half_rhythm"><div>It is appropriate to offer <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> (including discussion of the probability that offspring will be deaf and reproductive options) to young adults who have DFNA2 nonsyndromic hearing loss.</div></li></ul><p><b>DNA banking</b> is the storage of DNA (typically extracted from white blood cells) for possible future use. Because it is likely that testing methodology and our understanding of genes, allelic variants, and deafness will improve in the future, consideration should be given to banking DNA of individuals with DFNA2 nonsyndromic hearing loss.</p></div><div id="dfna2.Prenatal_Testing_and_Preimplantati"><h3>Prenatal Testing and Preimplantation Genetic Diagnosis</h3><p>Once the <i>KCNQ4</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> has been identified in a family member, prenatal testing for a pregnancy at increased risk and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> for DFNA2 nonsyndromic hearing loss are possible.</p><p>Differences in perspective may exist among medical professionals and within families regarding the use of prenatal testing, particularly if the testing is being considered for the purpose of pregnancy termination rather than early diagnosis. While most centers would consider decisions regarding prenatal testing to be the choice of the parents, discussion of these issues is appropriate.</p></div></div><div id="dfna2.Resources"><h2 id="_dfna2_Resources_">Resources</h2><p>
<i>GeneReviews staff has selected the following disease-specific and/or umbrella
support organizations and/or registries for the benefit of individuals with this disorder
and their families. GeneReviews is not responsible for the information provided by other
organizations. For information on selection criteria, click <a href="/books/n/gene/app4/">here</a>.</i></p><ul><li class="half_rhythm"><div><b>National Library of Medicine Genetics Home Reference</b></div><div><a href="http://ghr.nlm.nih.gov/condition=nonsyndromicdeafness" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Nonsyndromic deafness</a></div></li><li class="half_rhythm"><div><b>Alexander Graham Bell Association for the Deaf and Hard of Hearing</b></div><div>3417 Volta Place Northwest</div><div>Washington DC 20007</div><div><b>Phone:</b> 866-337-5220 (toll-free); 202-337-5220; 202-337-5221 (TTY)</div><div><b>Fax:</b> 202-337-8314</div><div><b>Email:</b> info@agbell.org</div><div><a href="http://www.agbell.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Listening and Spoken Language Knowledge Center</a></div></li><li class="half_rhythm"><div><b>American Society for Deaf Children (ASDC)</b></div><div>800 Florida Avenue Northeast</div><div>Suite 2047</div><div>Washington DC 20002-3695</div><div><b>Phone:</b> 800-942-2732 (Toll-free Parent Hotline); 866-895-4206 (toll free voice/TTY)</div><div><b>Fax:</b> 410-795-0965</div><div><b>Email:</b> info@deafchildren.org; asdc@deafchildren.org</div><div><a href="http://www.deafchildren.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.deafchildren.org</a></div></li><li class="half_rhythm"><div><b>BabyHearing.org</b></div><div><i>This site, developed with support from the National Institute on Deafness and Other Communication Disorders, provides information about newborn hearing screening and hearing loss.</i></div><div><a href="http://www.babyhearing.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.babyhearing.org</a></div></li><li class="half_rhythm"><div><b>My46 Trait Profile</b></div><div><a href="https://www.my46.org/trait-document?trait=Nonsyndromic%20congenital%20hearing%20loss&#x00026;parent=Carrier%20Status&#x00026;type=profile" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Nonsyndromic congenital hearing loss</a></div></li><li class="half_rhythm"><div><b>National Association of the Deaf (NAD)</b></div><div>8630 Fenton Street</div><div>Suite 820</div><div>Silver Spring MD 20910</div><div><b>Phone:</b> 301-587-1788; 301-587-1789 (TTY)</div><div><b>Fax:</b> 301-587-1791</div><div><b>Email:</b> nad.info@nad.org</div><div><a href="http://www.nad.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.nad.org</a></div></li><li class="half_rhythm"><div><b>NCBI Genes and Disease</b></div><div><a href="/books/NBK22204/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Deafness</a></div></li></ul></div><div id="dfna2.Molecular_Genetics"><h2 id="_dfna2_Molecular_Genetics_">Molecular Genetics</h2><p><i>Information in the Molecular Genetics and OMIM tables may differ from that elsewhere in the GeneReview: tables may contain more recent information. &#x02014;</i>ED.</p><div id="dfna2.molgen.TA" class="table"><h3><span class="label">Table A.</span></h3><div class="caption"><p>DFNA2 Nonsyndromic Hearing Loss: Genes and Databases</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1209/table/dfna2.molgen.TA/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna2.molgen.TA_lrgtbl__"><table class="no_bottom_margin"><tbody><tr><th id="hd_b_dfna2.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;">Locus Name</th><th id="hd_b_dfna2.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;">Gene</th><th id="hd_b_dfna2.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;">Chromosome Locus</th><th id="hd_b_dfna2.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;">Protein</th><th id="hd_b_dfna2.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;">Locus-Specific Databases</th><th id="hd_b_dfna2.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;">HGMD</th><th id="hd_b_dfna2.molgen.TA_1_1_1_7" rowspan="1" colspan="1" style="vertical-align:top;">ClinVar</th></tr><tr><td headers="hd_b_dfna2.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;">DFNA2</td><td headers="hd_b_dfna2.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/9132" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>KCNQ4</i></a></td><td headers="hd_b_dfna2.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=9132" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">1p34<wbr style="display:inline-block"></wbr>​.2</a></td><td headers="hd_b_dfna2.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/P56696" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Potassium voltage-gated channel subfamily KQT member 4</a></td><td headers="hd_b_dfna2.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://databases.lovd.nl/shared/genes/KCNQ4" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">KCNQ4 database</a><br /><a href="http://deafnessvariationdatabase.org/letter/k" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Deafness Variation Database - KCNQ4</a></td><td headers="hd_b_dfna2.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=KCNQ4" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">KCNQ4</a></td><td headers="hd_b_dfna2.molgen.TA_1_1_1_7" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=KCNQ4[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">KCNQ4</a></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div id="dfna2.TFA.1"><p class="no_margin">Data are compiled from the following standard references: <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> from
<a href="http://www.genenames.org/index.html" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HGNC</a>;
<a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> from
<a href="http://www.omim.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">OMIM</a>;
protein from <a href="http://www.uniprot.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">UniProt</a>.
For a description of databases (Locus Specific, HGMD, ClinVar) to which links are provided, click
<a href="/books/n/gene/app1/">here</a>.</p></div></dd></dl></div></div></div><div id="dfna2.molgen.TB" class="table"><h3><span class="label">Table B.</span></h3><div class="caption"><p>OMIM Entries for DFNA2 Nonsyndromic Hearing Loss (<a href="/omim/600101,603537" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">View All in OMIM</a>) </p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1209/table/dfna2.molgen.TB/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna2.molgen.TB_lrgtbl__"><table><tbody><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/600101" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">600101</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">DEAFNESS, AUTOSOMAL DOMINANT 2A; DFNA2A</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/603537" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">603537</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">POTASSIUM CHANNEL, VOLTAGE-GATED, KQT-LIKE SUBFAMILY, MEMBER 4; KCNQ4</td></tr></tbody></table></div></div><p><b>Gene structure.</b>
<i>KCNQ4</i> has a transcript length of 2,335 base pairs. The transcript consists of 14 exons. For a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information, see <a href="/books/NBK1209/#dfna2.molgen.TA">Table A</a>, <b>Gene</b>.</p><p><b>Variants of uncertain clinical significance.</b> Two variants that result in synonymous amino acid changes are of uncertain clinical significance (see <a class="figpopup" href="/books/NBK1209/table/dfna2.T.kcnq4_variants_of_uncertain_clin/?report=objectonly" target="object" rid-figpopup="figdfna2Tkcnq4variantsofuncertainclin" rid-ob="figobdfna2Tkcnq4variantsofuncertainclin">Table 2</a>). These nucleotide variants were detected on a screen of 185 individuals with nonsyndromic hearing loss. These individuals were reported as having nonsyndromic hearing loss; no information regarding family history was provided. To date these variants remain of uncertain clinical significance because to the authors' knowledge <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> expression has not been tested (the hypothesis being that these variants disrupt <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a>-splice enhancers and interfere with normal gene <a class="def" href="/books/n/gene/glossary/def-item/splicing/">splicing</a>).</p><div id="dfna2.T.kcnq4_variants_of_uncertain_clin" class="table"><h3><span class="label">Table 2. </span></h3><div class="caption"><p><i>KCNQ4</i> Variants of Uncertain Clinical Significance Discussed in This <i>GeneReview</i></p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1209/table/dfna2.T.kcnq4_variants_of_uncertain_clin/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna2.T.kcnq4_variants_of_uncertain_clin_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change&#x000a0;<sup>1</sup></th><th id="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change&#x000a0;<sup>1</sup></th><th id="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Protein Domain</th><th id="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Population</th><th id="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_5" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Onset of Symptoms&#x000a0;<sup>2</sup></th><th id="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_6" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference</th></tr></thead><tbody><tr><td headers="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.648C&#x0003e;T</td><td headers="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg216=&#x000a0;<sup>3</sup></td><td headers="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">S4 transmembrane <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a></td><td headers="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_4" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Taiwanese</td><td headers="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_5" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Childhood</td><td headers="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_6" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#dfna2.REF.su.2007.20">Su et al [2007]</a></td></tr><tr><td headers="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1503C&#x0003e;T</td><td headers="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Thr501=&#x000a0;<sup>3</sup></td><td headers="hd_h_dfna2.T.kcnq4_variants_of_uncertain_clin_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Distal to S6 transmembrane <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin"><i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd><dt>1. </dt><dd><div id="dfna2.TF.2.1"><p class="no_margin">Reference sequences for <i>KCNQ4</i>: <a href="/nuccore/NM_004700.2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_004700<wbr style="display:inline-block"></wbr>​.2</a>, <a href="/entrez/viewer.fcgi?db=protein&#x00026;id=26638653" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_004691<wbr style="display:inline-block"></wbr>​.2</a></p></div></dd><dt>2. </dt><dd><div id="dfna2.TF.2.2"><p class="no_margin">Pathology was high-frequency hearing impairment and tissue-specific expression was in cochlear outer hair cells and brain for all pathogenic variants described in the table.</p></div></dd><dt>3. </dt><dd><div id="dfna2.TF.2.3"><p class="no_margin">No effect on protein level is expected.</p></div></dd></dl></div></div></div><p><b>Pathogenic variants.</b> Most <i>KCNQ4</i> pathogenic variants cluster in exons 5, 6, and 7, which encode highly conserved amino acid sequences that form the channel pore. The predominant pathogenic variants are <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants that induce a <a class="def" href="/books/n/gene/glossary/def-item/dominant-negative/">dominant-negative</a> effect. The variant p.Trp276Ser appears to be most common and has been identified in four unrelated families, including three of five Dutch families with DFNA2 nonsyndromic hearing loss and one Japanese family (see <a class="figpopup" href="/books/NBK1209/table/dfna2.T.kcnq4_pathogenic_variants_discus/?report=objectonly" target="object" rid-figpopup="figdfna2Tkcnq4pathogenicvariantsdiscus" rid-ob="figobdfna2Tkcnq4pathogenicvariantsdiscus">Table 3</a>).</p><div id="dfna2.T.kcnq4_pathogenic_variants_discus" class="table"><h3><span class="label">Table 3. </span></h3><div class="caption"><p><i>KCNQ4</i> Pathogenic Variants Discussed in This <i>GeneReview</i></p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK1209/table/dfna2.T.kcnq4_pathogenic_variants_discus/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__dfna2.T.kcnq4_pathogenic_variants_discus_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change<br />(Alias&#x000a0;<sup>1</sup>)</th><th id="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change<br />(Alias&#x000a0;<sup>1</sup>)</th><th id="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Protein Domain</th><th id="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Population</th><th id="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_5" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Onset of Symptoms&#x000a0;<sup>2</sup></th><th id="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_6" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference</th></tr></thead><tbody><tr><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.211_223del13<br />(211del13)</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Gln71ProfsTer64<br />(Q71fsTer134)</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">N-terminal cytoplasmic</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Belgian</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Adolescence</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#dfna2.REF.coucke.1999.1321">Coucke et al [1999]</a></td></tr><tr><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.211delC</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Gln71SerfsTer68<br />(FS71)</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">N-terminal cytoplasmic</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Japanese</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Adolescence</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#dfna2.REF.kamada.2006.455">Kamada et al [2006]</a>, <a class="bk_pop" href="#dfna2.REF.ishikawa.2014.557">Ishikawa et al [2014]</a></td></tr><tr><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.827G&#x0003e;C</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Trp276Ser</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">P-loop</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Dutch, Japanese</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Childhood</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#dfna2.REF.coucke.1999.1321">Coucke et al [1999]</a>, <a class="bk_pop" href="#dfna2.REF.van_camp.2002.15">Van Camp et al [2002]</a>, <a class="bk_pop" href="#dfna2.REF.topsakal.2005.52">Topsakal et al [2005]</a></td></tr><tr><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.853G&#x0003e;T</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Gly285Cys</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">P-loop</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">North American</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Childhood</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#dfna2.REF.coucke.1999.1321">Coucke et al [1999]</a></td></tr><tr><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.853G&#x0003e;A</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Gly285Ser</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">P-loop</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Northern European, Han Chinese</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_5" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Childhood</td><td headers="hd_h_dfna2.T.kcnq4_pathogenic_variants_discus_1_1_1_6" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#dfna2.REF.kubisch.1999.437">Kubisch et al [1999]</a>, <a class="bk_pop" href="#dfna2.REF.wang.2014.e103133">Wang et al [2014]</a></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin"><i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd><dt></dt><dd><div><p class="no_margin">Reference sequences for <i>KCNQ4</i>: <a href="/nuccore/NM_004700.2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_004700<wbr style="display:inline-block"></wbr>​.2</a>, <a href="/entrez/viewer.fcgi?db=protein&#x00026;id=26638653" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_004691<wbr style="display:inline-block"></wbr>​.2</a></p></div></dd><dt>1. </dt><dd><div id="dfna2.TF.3.1"><p class="no_margin">Variant designation that does not conform to current naming conventions</p></div></dd><dt>2. </dt><dd><div id="dfna2.TF.3.2"><p class="no_margin">Pathology was high-frequency hearing impairment and tissue-specific expression was in cochlear outer hair cells and brain for all pathogenic variants described in the table.</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> The protein encoded by <i>KCNQ4</i> is 695 amino acids in length and forms a potassium channel with six transmembrane domains and a P-loop region, which forms the channel pore. A highly conserved glycine-tyrosine-glycine (GYG) signature sequence within the P-loop comprises the selectivity filter that provides discrimination of potassium ions for selective transport [<a class="bk_pop" href="#dfna2.REF.kubisch.1999.437">Kubisch et al 1999</a>]. Pathogenic variants cluster in the channel pore region and some (i.e., p.Gly285Ser and p.Gly285Cys) directly affect the selectivity filter.</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> Most <i>KCNQ4</i> pathogenic variants are <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> alterations that cause hearing loss via a <a class="def" href="/books/n/gene/glossary/def-item/dominant-negative/">dominant-negative</a> effect. The <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> reflects the consequence of defective KCNQ4 protein in the inner ear. This protein assembles as a tetramer to form a potassium channel made of four subunits. In a person with one pathogenic missense variant, half of the total amount of encoded protein is defective and consequently only one of every 16 channels comprises four normal protein subunits [<a class="bk_pop" href="#dfna2.REF.kubisch.1999.437">Kubisch et al 1999</a>]. Over time the result is hypothesized to be progressive loss in potassium recycling in the inner ear. Because potassium ions are crucial for hair cell transduction, the inability to recycle these ions results in hearing loss. Most pathogenic variants affect amino acids located within or close to the channel pore. The presence of an abnormal protein subunit interferes with the assembly and/or function of the tetrameric channel protein in the inner ear.</p><p>Some <i>KCNQ4</i> pathogenic variants are deletions that result in <a class="def" href="/books/n/gene/glossary/def-item/haploinsufficiency/">haploinsufficiency</a>. As a result, cells of the inner ear produce insufficient functional KCNQ4 protein and over time auditory function is compromised.</p><p><b>Evidence for <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> heterogeneity.</b>
<i>GJB3</i> was suggested as a deafness-associated <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> at the DFNA2 locus based on two different <i>GJB3</i> sequence variants identified in two small Chinese families [<a class="bk_pop" href="#dfna2.REF.xia.1998.370">Xia et al 1998</a>]. Individuals from both families had bilateral sensorineural hearing loss (SNHL) characterized by a gently downsloping audiogram from normal hearing thresholds below 1,000 Hz to moderate hearing loss in the high frequencies.</p><p>The evidence associating <i>GJB3</i> with hearing loss is neither substantial nor convincing:</p><ul><li class="half_rhythm"><div>In both families, other individuals with normal hearing had the reported pathogenic variants in <i>GJB3</i>, a finding inconsistent with complete <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a>, which is observed in virtually all types of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> SNHL.</div></li><li class="half_rhythm"><div>It is doubtful that <i>KCNQ4</i> pathogenic variants have been excluded in these two families, which were reported in 1998, as <i>KCNQ4</i> pathogenic variants were not implicated in <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> SNHL until 1999.</div></li><li class="half_rhythm"><div>No other families with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> SNHL have been reported to segregate <i>GJB3</i> pathogenic variants.</div></li><li class="half_rhythm"><div>Specific pathogenic variants in <i>GJB3</i> cause erythrokeratodermia variabilis.</div></li></ul></div><div id="dfna2.References"><h2 id="_dfna2_References_">References</h2><div id="dfna2.Literature_Cited"><h3>Literature Cited</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.akita.2001.355">Akita J, Abe S, Shinkawa H, Kimberling WJ, Usami S. Clinical and genetic features of nonsyndromic autosomal dominant sensorineural hearing loss: KCNQ4 is a gene responsible in Japanese. <span><span class="ref-journal">J Hum Genet. </span>2001;<span class="ref-vol">46</span>:355–61.</span> [<a href="/pubmed/11450843" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11450843</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.coucke.1999.1321">Coucke PJ, Van Hauwe P, Kelley PM, Kunst H, Schatteman I, Van Velzen D, Meyers J, Ensink RJ, Verstreken M, Declau F, Marres H, Kastury K, Bhasin S, McGuirt WT, Smith RJ, Cremers CW, Van de Heyning P, Willems PJ, Smith SD, Van Camp G. Mutations in the KCNQ4 gene are responsible for autosomal dominant deafness in four DFNA2 families. <span><span class="ref-journal">Hum Mol Genet. </span>1999;<span class="ref-vol">8</span>:1321–8.</span> [<a href="/pubmed/10369879" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10369879</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.de_leenheer.2002a.41">De Leenheer EM, Ensink RJ, Kunst HP, Marres HA, Talebizadeh Z, Declau F, Smith SD, Usami S, Van de Heyning PH, Van Camp G, Huygen PL, Cremers CW. DFNA2/KCNQ4 and its manifestations. <span><span class="ref-journal">Adv Otorhinolaryngol. </span>2002a;<span class="ref-vol">61</span>:41–6.</span> [<a href="/pubmed/12408061" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12408061</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.de_leenheer.2002b.267">De Leenheer EM, Huygen PL, Coucke PJ, Admiraal RJ, van Camp G, Cremers CW. Longitudinal and cross-sectional phenotype analysis in a new, large Dutch DFNA2/KCNQ4 family. <span><span class="ref-journal">Ann Otol Rhinol Laryngol. </span>2002b;<span class="ref-vol">111</span>:267–74.</span> [<a href="/pubmed/11915881" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11915881</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.ensink.2000.62">Ensink RJ, Huygen PL, Van Hauwe P, Coucke P, Cremers CW, Van Camp G. A Dutch family with progressive sensorineural hearing impairment linked to the DFNA2 region. <span><span class="ref-journal">Eur Arch Otorhinolaryngol. </span>2000;<span class="ref-vol">257</span>:62–7.</span> [<a href="/pubmed/10784363" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10784363</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.ishikawa.2014.557">Ishikawa K, Naito T, Nishio SY, Iwasa Y, Nakamura K, Usami S, Ichimura K. A Japanese family showing high-frequency hearing loss with KCNQ4 and TECTA mutations. <span><span class="ref-journal">Acta Otolaryngol. </span>2014;<span class="ref-vol">134</span>:557–63.</span> [<a href="/pubmed/24655070" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24655070</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.kamada.2006.455">Kamada F, Kure S, Kudo T, Suzuki Y, Oshima T, Ichinohe A, Kojima K, Niihori T, Kanno J, Narumi Y, Narisawa A, Kato K, Aoki Y, Ikeda K, Kobayashi T, Matsubara Y. A novel KCNQ4 one-base deletion in a large pedigree with hearing loss: implication for the genotype-phenotype correlation. <span><span class="ref-journal">J Hum Genet. </span>2006;<span class="ref-vol">51</span>:455–60.</span> [<a href="/pubmed/16596322" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 16596322</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.kubisch.1999.437">Kubisch C, Schroeder BC, Friedrich T, Lutjohann B, El-Amraoui A, Marlin S, Petit C, Jentsch TJ. KCNQ4, a novel potassium channel expressed in sensory outer hair cells, is mutated in dominant deafness. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">96</span>:437–46.</span> [<a href="/pubmed/10025409" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10025409</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.marres.1997.573">Marres H, van Ewijk M, Huygen P, Kunst H, van Camp G, Coucke P, Willems P, Cremers C. Inherited nonsyndromic hearing loss. An audiovestibular study in a large family with autosomal dominant progressive hearing loss related to DFNA2. <span><span class="ref-journal">Arch Otolaryngol Head Neck Surg. </span>1997;<span class="ref-vol">123</span>:573–7.</span> [<a href="/pubmed/9193215" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9193215</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.rahbari.2016.126">Rahbari R, Wuster A, Lindsay SJ, Hardwick RJ, Alexandrov LB, Turki SA, Dominiczak A, Morris A, Porteous D, Smith B, Stratton MR, Hurles ME, et al.  Timing, rates and spectra of human germline mutation. <span><span class="ref-journal">Nat Genet. </span>2016;<span class="ref-vol">48</span>:126–33.</span> [<a href="/pmc/articles/PMC4731925/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4731925</span></a>] [<a href="/pubmed/26656846" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26656846</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.sloanheggen.2016.441">Sloan-Heggen CM, Bierer AO, Shearer AE, Kolbe DL, Nishimura CJ, Frees KL, Ephraim SS, Shibata SB, Booth KT, Campbell CA, Ranum PT, Weaver AE, Black-Ziegelbein EA, Wang D, Azaiez H, Smith RJ. Comprehensive genetic testing in the clinical evaluation of 1119 patients with hearing loss. <span><span class="ref-journal">Hum Genet. </span>2016;<span class="ref-vol">135</span>:441–50.</span> [<a href="/pmc/articles/PMC4796320/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4796320</span></a>] [<a href="/pubmed/26969326" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26969326</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.su.2007.20">Su CC, Yang JJ, Shieh JC, Su MC, Li SY. Identification of novel mutations in the KCNQ4 gene of patients with nonsyndromic deafness from Taiwan. <span><span class="ref-journal">Audiol Neurootol. </span>2007;<span class="ref-vol">12</span>:20–6.</span> [<a href="/pubmed/17033161" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17033161</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.talebizadeh.1999.493">Talebizadeh Z, Kelley PM, Askew JW, Beisel KW, Smith SD. Novel mutation in the KCNQ4 gene in a large kindred with dominant progressive hearing loss. <span><span class="ref-journal">Hum Mutat. </span>1999;<span class="ref-vol">14</span>:493–501.</span> [<a href="/pubmed/10571947" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10571947</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.topsakal.2005.52">Topsakal V, Pennings RJ, te Brinke H, Hamel B, Huygen PL, Kremer H, Cremers CW. Phenotype determination guides swift genotyping of a DFNA2/KCNQ4 family with a hot spot mutation (W276S). <span><span class="ref-journal">Otol Neurotol. </span>2005;<span class="ref-vol">26</span>:52–8.</span> [<a href="/pubmed/15699719" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15699719</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.van_camp.2002.15">Van Camp G, Coucke PJ, Akita J, Fransen E, Abe S, De Leenheer EM, Huygen PL, Cremers CW, Usami S. A mutational hot spot in the KCNQ4 gene responsible for autosomal dominant hearing impairment. <span><span class="ref-journal">Hum Mutat. </span>2002;<span class="ref-vol">20</span>:15–9.</span> [<a href="/pubmed/12112653" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12112653</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.van_hauwe.2000.184">Van Hauwe P, Coucke PJ, Ensink RJ, Huygen P, Cremers CW, Van Camp G. Mutations in the KCNQ4 K+ channel gene, responsible for autosomal dominant hearing loss, cluster in the channel pore region. <span><span class="ref-journal">Am J Med Genet. </span>2000;<span class="ref-vol">93</span>:184–7.</span> [<a href="/pubmed/10925378" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10925378</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.wang.2014.e103133">Wang H, Zhao Y, Yi Y, Gao Y, Liu Q, Wang D, Li Q, Lan L, Li N, Guan J, Yin Z, Han B, Zhao F, Zong L, Xiong W, Yu L, Song L, Yi X, Yang L, Petit C, Wang Q. Targeted high-throughput sequencing identifies pathogenic mutations in KCNQ4 in two large Chinese families with autosomal dominant hearing loss. <span><span class="ref-journal">PLoS One. </span>2014;<span class="ref-vol">9</span>:e103133.</span> [<a href="/pmc/articles/PMC4130520/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4130520</span></a>] [<a href="/pubmed/25116015" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 25116015</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="dfna2.REF.xia.1998.370">Xia JH, Liu CY, Tang BS, Pan Q, Huang L, Dai HP, Zhang BR, Xie W, Hu DX, Zheng D, Shi XL, Wang DA, Xia K, Yu KP, Liao XD, Feng Y, Yang YF, Xiao JY, Xie DH, Huang JZ. Mutations in the gene encoding gap junction protein beta-3 associated with autosomal dominant hearing impairment. <span><span class="ref-journal">Nat Genet. </span>1998;<span class="ref-vol">20</span>:370–3.</span> [<a href="/pubmed/9843210" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9843210</span></a>]</div></li></ul></div></div><div id="dfna2.Chapter_Notes"><h2 id="_dfna2_Chapter_Notes_">Chapter Notes</h2><div id="dfna2.Author_Notes"><h3>Author Notes</h3><p><a href="http://www.medicine.uiowa.edu/morl/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Molecular Otolaryngology Research Laboratories home page</a></p><p><a href="http://hereditaryhearingloss.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Hereditary Hearing Loss home page</a></p><p><a href="http://deafnessvariationdatabase.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Deafness Variation Database</a>. The Deafness Variation Database (DVD) collates data from major public databases to provide a single classification for each variant based on collected evidence; it is curated by experts in hereditary hearing loss with the goal of providing a single-source guide to variant interpretation focused on deafness.</p></div><div id="dfna2.Revision_History"><h3>Revision History</h3><ul><li class="half_rhythm"><div>10 May 2018 (bp) Comprehensive update posted live</div></li><li class="half_rhythm"><div>20 August 2015 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>20 June 2013 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>17 February 2011 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>4 April 2008 (me) Review posted live</div></li><li class="half_rhythm"><div>19 December 2007 (rjhs) Original submission</div></li></ul></div></div><div id="bk_toc_contnr"></div></div></div>
            <div class="post-content"><div><div class="half_rhythm"><a href="/books/about/copyright/">Copyright</a> © 1993-2019, University of Washington, Seattle. GeneReviews is
a registered trademark of the University of Washington, Seattle. All rights
reserved.<p class="small">GeneReviews® chapters are owned by the University of Washington. Permission is
hereby granted to reproduce, distribute, and translate copies of content materials for
noncommercial research purposes only, provided that (i) credit for source (<a href="http://www.genereviews.org/" ref="pagearea=meta&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">http://www.genereviews.org/</a>) and copyright (© 1993-2019 University of
Washington) are included with each copy; (ii) a link to the original material is provided
whenever the material is published elsewhere on the Web; and (iii) reproducers,
distributors, and/or translators comply with the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>. No further modifications are allowed. For clarity, excerpts
of GeneReviews chapters for use in lab reports and clinic notes are a permitted
use.</p><p class="small">For more information, see the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>.</p><p class="small">For questions regarding permissions or whether a specified use is allowed,
contact: <a href="mailto:dev@null" data-email="ude.wu@tssamda" class="oemail">ude.wu@tssamda</a>.</p></div><div class="small"><span class="label">Bookshelf ID: NBK1209</span><span class="label">PMID: <a href="/pubmed/20301388" title="PubMed record of this page" ref="pagearea=meta&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">20301388</a></span></div><div style="margin-top:2em" class="bk_noprnt"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/depdc5-epilepsy/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/dfnx1/" title="Next page in this title">Next &gt;</a></div></div></div></div>
            
        </div>

        <!-- Custom content below content -->
        <div class="col4">
            
        </div>
        
        
        <!-- Book content -->
        
        <!-- Custom contetnt below bottom nav -->
        <div class="col5">
            
        </div>
    </div>

    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            <div id="ncbi_share_book"><a href="#" class="ncbi_share" data-ncbi_share_config="popup:false,shorten:true" ref="id=NBK1209&amp;db=books">Share</a></div>

        </div>
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Views</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PDF_download" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/NBK1209/?report=reader">PubReader</a></li><li><a href="/books/NBK1209/?report=printable">Print View</a></li><li><a data-jig="ncbidialog" href="#_ncbi_dlg_citbx_NBK1209" data-jigconfig="width:400,modal:true">Cite this Page</a><div id="_ncbi_dlg_citbx_NBK1209" style="display:none" title="Cite this Page"><div class="bk_tt">Smith RJH, Hildebrand M. DFNA2 Nonsyndromic Hearing Loss. 2008 Apr 4 [Updated 2018 May 10]. In: Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. <span class="bk_cite_avail"></span></div></div></li><li><a href="/books/NBK1209/pdf/Bookshelf_NBK1209.pdf">PDF version of this page</a> (468K)</li><li><a href="#" class="toggle-glossary-link" title="Enable/disable links to the glossary">Disable Glossary Links</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>In this GeneReview</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="page-toc" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="#dfna2.Summary" ref="log$=inpage&amp;link_id=inpage">Summary</a></li><li><a href="#dfna2.Diagnosis" ref="log$=inpage&amp;link_id=inpage">Diagnosis</a></li><li><a href="#dfna2.Clinical_Characteristics" ref="log$=inpage&amp;link_id=inpage">Clinical Characteristics</a></li><li><a href="#dfna2.Genetically_Related_Allelic_Disord" ref="log$=inpage&amp;link_id=inpage">Genetically Related (Allelic) Disorders</a></li><li><a href="#dfna2.Differential_Diagnosis" ref="log$=inpage&amp;link_id=inpage">Differential Diagnosis</a></li><li><a href="#dfna2.Management" ref="log$=inpage&amp;link_id=inpage">Management</a></li><li><a href="#dfna2.Genetic_Counseling" ref="log$=inpage&amp;link_id=inpage">Genetic Counseling</a></li><li><a href="#dfna2.Resources" ref="log$=inpage&amp;link_id=inpage">Resources</a></li><li><a href="#dfna2.Molecular_Genetics" ref="log$=inpage&amp;link_id=inpage">Molecular Genetics</a></li><li><a href="#dfna2.References" ref="log$=inpage&amp;link_id=inpage">References</a></li><li><a href="#dfna2.Chapter_Notes" ref="log$=inpage&amp;link_id=inpage">Chapter Notes</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>GeneReviews Links</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="source-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/n/gene/advanced/"><i>GeneReviews</i> Advanced Search</a></li><li><a href="/books/n/gene/glossary/"><i>GeneReviews</i> Glossary</a></li><li><a href="/books/n/gene/resource_mats/">Resource Materials</a> <span class="bk_hlight1">NEW FEATURE</span></li><li><a href="/books/n/gene/updates/">New in <i>GeneReviews</i></a></li><li><a href="/books/n/gene/authors/">Author List</a></li><li><a href="/books/n/gene/prospective_authors/">For Current/Prospective Authors</a></li><li><a href="/books/n/gene/GRpersonnel/"><i>GeneReviews</i> Personnel</a></li><li><a href="/books/n/gene/howto_linkin/">Download/Link to <i>GeneReviews</i></a></li><li><a href="/books/n/gene/contact_us/">Contact Us</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Tests in GTR by Gene</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="document-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=9132[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">KCNQ4</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Variations in ClinVar</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="document-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=NBK1209+AND+genereviews[submitter]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Variations from this GeneReview in ClinVar</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Related information</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="discovery_db_links" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=omim&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_omim&amp;IdsFromResult=1492988" ref="log$=recordlinks">OMIM</a><div class="brieflinkpop offscreen_noflow">Related OMIM records</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pmc&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pmc_refs&amp;IdsFromResult=1492988" ref="log$=recordlinks">PMC</a><div class="brieflinkpop offscreen_noflow">PubMed Central citations</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pubmed&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pubmed_refs&amp;IdsFromResult=1492988" ref="log$=recordlinks">PubMed</a><div class="brieflinkpop offscreen_noflow">Links to PubMed</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=gene&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_gene&amp;IdsFromResult=1492988" ref="log$=recordlinks">Gene</a><div class="brieflinkpop offscreen_noflow">Locus Links</div></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/25719194" ref="ordinalpos=1&amp;linkpos=1&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">TBC1D24</i>-Related Disorders</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">TBC1D24</i>-Related Disorders<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Mucha BE, Hennekam RCM, Sisodiya S, Campeau PM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301595" ref="ordinalpos=1&amp;linkpos=2&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, Mitochondrial</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, Mitochondrial<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Usami S, Nishio S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301708" ref="ordinalpos=1&amp;linkpos=3&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, DFNA3</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Nonsyndromic Hearing Loss and Deafness, DFNA3<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Smith RJH, Ranum PT. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/23776385" ref="ordinalpos=1&amp;linkpos=4&amp;log$=relatedarticles&amp;logdbfrom=pubmed">Genetics of hearing loss: focus on DFNA2.</a><span class="source">[Appl Clin Genet. 2012]</span><div class="brieflinkpop offscreen_noflow">Genetics of hearing loss: focus on DFNA2.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Dominguez LM, Dodson KM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Appl Clin Genet. 2012; 5:97-104. Epub 2012 Oct 18.</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/18797286" ref="ordinalpos=1&amp;linkpos=5&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> KCNQ4 mutations associated with nonsyndromic progressive sensorineural hearing loss.</a><span class="source">[Curr Opin Otolaryngol Head Nec...]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> KCNQ4 mutations associated with nonsyndromic progressive sensorineural hearing loss.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Nie L. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Curr Opin Otolaryngol Head Neck Surg. 2008 Oct; 16(5):441-4. </em></div></div></li></ul><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=20301388" ref="ordinalpos=1&amp;log$=relatedreviews_seeall&amp;logdbfrom=pubmed">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=20301388" ref="ordinalpos=1&amp;log$=relatedarticles_seeall&amp;logdbfrom=pubmed">See all...</a></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Recent Activity</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="Shutter"></a></div><div class="portlet_content"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class=""><div class="action"><a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a></div><ul id="activity"><li class="ra_rcd ralinkpopper two_line"><a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5e042b5175f66d18aa1aa2e0">DFNA2 Nonsyndromic Hearing Loss - GeneReviews®</a><div class="ralinkpop offscreen_noflow">DFNA2 Nonsyndromic Hearing Loss - GeneReviews®<div class="brieflinkpopdesc"></div></div><div class="tertiary"></div></li></ul><p class="HTOn">Your browsing activity is empty.</p><p class="HTOff">Activity recording is turned off.</p><p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p><a class="seemore" href="/sites/myncbi/recentactivity">See more...</a></div></div></div>
        
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script type="text/javascript" src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script type="text/javascript">
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-12-25T22:38:57-05:00&amp;Snapshot=%2Fprojects%2Fbooks%2FPBooks@5.22&amp;Host=portal101&amp;ncbi_phid=CE881E49E04174E10000000008F50360&amp;ncbi_session=CE881E49E042B511_2293SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK1209%2F&amp;Db=pmc&amp;folderID=134&amp;Ncbi_App=bookshelf&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        </div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=books&amp;ncbi_pdid=book-part&amp;ncbi_acc=NBK1209&amp;ncbi_domain=gene&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/NBK1209/&amp;ncbi_pagename=DFNA2 Nonsyndromic Hearing Loss - GeneReviews® - NCBI Bookshelf&amp;ncbi_bookparttype=chapter&amp;ncbi_app=bookshelf" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE881E49E042B511_2293SID /projects/books/PBooks@5.22 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/js/3879255/4121861/3501987/4008961/3893018/3821238/4062932/4123007/4087685/4076480/3921943/3400083/3426610.js" snapshot="books"></script></body>
</html>