<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    
    <head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <!-- AppResources meta begin -->
        <meta name="paf-app-resources" content="" />
                 <script type="text/javascript">var ncbi_startTime = new Date();</script>
            
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="" />

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="books" /><meta name="ncbi_pdid" content="book-part" /><meta name="ncbi_acc" content="NBK274565" /><meta name="ncbi_domain" content="gene" /><meta name="ncbi_report" content="record" /><meta name="ncbi_type" content="fulltext" /><meta name="ncbi_objectid" content="" /><meta name="ncbi_pcid" content="/NBK274565/" /><meta name="ncbi_pagename" content="16p12.2 Recurrent Deletion - GeneReviews® - NCBI Bookshelf" /><meta name="ncbi_bookparttype" content="chapter" /><meta name="ncbi_app" content="bookshelf" />
        <!-- Logger end -->
        
        <title>16p12.2 Recurrent Deletion - GeneReviews® - NCBI Bookshelf</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css" /><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,FOLLOW,NOARCHIVE" /><meta name="citation_inbook_title" content="GeneReviews® [Internet]" /><meta name="citation_title" content="16p12.2 Recurrent Deletion" /><meta name="citation_publisher" content="University of Washington, Seattle" /><meta name="citation_date" content="2018/09/13" /><meta name="citation_author" content="Santhosh Girirajan" /><meta name="citation_author" content="Lucilla Pizzo" /><meta name="citation_author" content="John Moeschler" /><meta name="citation_author" content="Jill Rosenfeld" /><meta name="citation_pmid" content="25719193" /><meta name="citation_fulltext_html_url" content="https://www.ncbi.nlm.nih.gov/books/NBK274565/" /><meta name="citation_keywords" content="16p12.1 Microdeletion" /><meta name="citation_keywords" content="16p12.1 Microdeletion" /><meta name="citation_keywords" content="16p12.2 Recurrent Deletion" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="DC.Title" content="16p12.2 Recurrent Deletion" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="University of Washington, Seattle" /><meta name="DC.Contributor" content="Santhosh Girirajan" /><meta name="DC.Contributor" content="Lucilla Pizzo" /><meta name="DC.Contributor" content="John Moeschler" /><meta name="DC.Contributor" content="Jill Rosenfeld" /><meta name="DC.Date" content="2018/09/13" /><meta name="DC.Identifier" content="https://www.ncbi.nlm.nih.gov/books/NBK274565/" /><meta name="description" content="16p12.2 recurrent deletion is characterized by variable clinical findings that do not constitute a recognizable syndrome. Of note, the significant bias in ascertainment of individuals undergoing clinical chromosomal microarray analysis (i.e., children with intellectual disability and developmental delay; individuals with schizophrenia) makes it difficult to accurately associate specific phenotypes with the 16p12.2 recurrent deletion. Findings commonly observed in children (probands) with this deletion include: developmental delay, cognitive impairment (ranging from mild to profound), growth impairment (including short stature), cardiac malformations, epilepsy, and psychiatric and/or behavioral problems. Other findings can include: hearing loss, dental abnormalities, renal and genital anomalies (the latter in males), and cleft palate ± cleft lip." /><meta name="og:title" content="16p12.2 Recurrent Deletion" /><meta name="og:type" content="book" /><meta name="og:description" content="16p12.2 recurrent deletion is characterized by variable clinical findings that do not constitute a recognizable syndrome. Of note, the significant bias in ascertainment of individuals undergoing clinical chromosomal microarray analysis (i.e., children with intellectual disability and developmental delay; individuals with schizophrenia) makes it difficult to accurately associate specific phenotypes with the 16p12.2 recurrent deletion. Findings commonly observed in children (probands) with this deletion include: developmental delay, cognitive impairment (ranging from mild to profound), growth impairment (including short stature), cardiac malformations, epilepsy, and psychiatric and/or behavioral problems. Other findings can include: hearing loss, dental abnormalities, renal and genital anomalies (the latter in males), and cleft palate ± cleft lip." /><meta name="og:url" content="https://www.ncbi.nlm.nih.gov/books/NBK274565/" /><meta name="og:site_name" content="NCBI Bookshelf" /><meta name="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" /><meta name="twitter:card" content="summary" /><meta name="twitter:site" content="@ncbibooks" /><meta name="bk-non-canon-loc" content="/books/n/gene/mdel16p12_2/" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/books/NBK274565/" /><link rel="stylesheet" href="/corehtml/pmc/css/figpopup.css" type="text/css" media="screen" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books_print.min.css" type="text/css" media="print" /><style type="text/css">p a.figpopup{display:inline !important} .bk_tt {font-family: monospace}  .first-line-outdent .bk_ref {display: inline}  .body-content h2, .body-content .h2  {border-bottom: 1px solid #97B0C8} .body-content h2.inline {border-bottom: none} a.page-toc-label , .jig-ncbismoothscroll a {text-decoration:none;border:0 !important} .temp-labeled-list  .graphic {display:inline-block !important} .temp-labeled-list  img{width:100%}</style><script type="text/javascript" src="/corehtml/pmc/js/jquery.hoverIntent.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"> </script><script type="text/javascript">window.name="mainwindow";</script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/book-toc.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/books.min.js"> </script><meta name="book-collection" content="NONE" />

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="CE8A37F7E04089E1000000000456019B.m_12" />
<meta name='referrer' content='origin-when-cross-origin'/><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4098876/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/4008682/3881636/3579733/4062871/12930/3964959/3854974/36029/4128070/9685/3549676/3609192/3609193/3609213/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3411343/3882866.css" media="print" /></head>
    <body class="book-part">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo"><h1 class="res_name"><a href="/books/" title="Bookshelf home">Bookshelf</a></h1><h2 class="res_tagline"></h2></div>
    <div class="search"><form method="get" action="/books/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="books" selected="selected" class="last" data-ac_dict="bookshelf-search">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books" data-ac_dict="bookshelf-search">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search Books. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'bookshelf-search',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'no'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/books/browse/">Browse Titles</a>
                    </li><li>
                        <a href="/books/advanced/">Advanced</a>
                    </li><li class="help">
                        <a href="/books/NBK3833/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <!-- Custom content 1 -->
<div class="col1">
    
</div>

<div class="container">
    <div id="maincontent" class="content eight_col col">
        <!-- Custom content in the left column above book nav -->
        <div class="col2">
            
        </div>
        
        <!-- Book content -->
        

        <!-- Custom content between navigation and content -->
        <div class="col3">
            
        </div>
        
        <div class="document">
            <div class="pre-content"><div><div class="bk_prnt"><p class="small">NCBI Bookshelf. A service of the National Library of Medicine, National Institutes of Health.</p><p>Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. </p></div><div class="iconblock clearfix whole_rhythm no_top_margin bk_noprnt"><a class="img_link icnblk_img" title="All GeneReviews" href="/books/n/gene/"><img class="source-thumb" src="/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" alt="Cover of GeneReviews®" height="100px" width="80px" /></a><div class="icnblk_cntnt eight_col"><h2>GeneReviews<sup>®</sup> [Internet].</h2><a data-jig="ncbitoggler" href="#__NBK274565_dtls__">Show details</a><div style="display:none" class="ui-widget" id="__NBK274565_dtls__"><div>Adam MP, Ardinger HH, Pagon RA, et al., editors.</div><div>Seattle (WA): <a href="http://www.washington.edu" ref="pagearea=page-banner&amp;targetsite=external&amp;targetcat=link&amp;targettype=publisher">University of Washington, Seattle</a>; 1993-2019.</div></div><div class="half_rhythm"><ul class="inline_list"><li style="margin-right:1em"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a></li></ul></div><div class="bk_noprnt"><form method="get" action="/books/n/gene/" id="bk_srch"><div class="bk_search"><label for="bk_term" class="offscreen_noflow">Search term</label><input type="text" title="Search GeneReviews" id="bk_term" name="term" value="" data-jig="ncbiclearbutton" /> <input type="submit" class="jig-ncbibutton" value="Search GeneReviews" submit="false" style="padding: 0.1em 0.4em;" /></div></form><div><ul class="inline_list"><li><a href="/books/n/gene/advanced/">GeneReviews Advanced Search</a></li><li style="margin-left:.5em"><a href="/books/n/gene/helpadvsearch/">Help</a></li></ul></div></div></div><div class="icnblk_cntnt two_col"><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/del16p11_2/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/dup17q12/" title="Next page in this title">Next &gt;</a></div></div></div></div></div>
            <div class="main-content lit-style" itemscope="itemscope" itemtype="http://schema.org/CreativeWork"><div class="meta-content fm-sec"><h1 id="_NBK274565_"><span class="title" itemprop="name">16p12.2 Recurrent Deletion</span></h1><div itemprop="alternativeHeadline" class="subtitle whole_rhythm">Synonym: 16p12.1 Microdeletion</div><p class="contrib-group"><span itemprop="author">Santhosh Girirajan</span>, MBBS, PhD, <span itemprop="author">Lucilla Pizzo</span>, MS, <span itemprop="author">John Moeschler</span>, MD, MS, and <span itemprop="author">Jill Rosenfeld</span>, MS.</p><a data-jig="ncbitoggler" href="#__NBK274565_ai__" style="border:0;text-decoration:none">Author Information</a><div style="display:none" class="ui-widget" id="__NBK274565_ai__"><div class="contrib half_rhythm"><span itemprop="author">Santhosh Girirajan</span>, MBBS, PhD<div class="affiliation small">Assistant Professor, Department of Biochemistry and Molecular Biology<br />Department of Anthropology<br />Pennsylvania State University<br />University Park, Pennsylvania<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.usp@74gxs" class="oemail">ude.usp@74gxs</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Lucilla Pizzo</span>, MS<div class="affiliation small">Department of Biochemistry and Molecular Biology<br />Department of Anthropology<br />Pennsylvania State University<br />University Park, Pennsylvania<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.usp@313pml" class="oemail">ude.usp@313pml</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">John Moeschler</span>, MD, MS<div class="affiliation small">Professor of Pediatrics, Geisel School of Medicine<br />Dartmouth College<br />Section Chief, Medical Genetics<br />Children's Hospital at Dartmouth<br />Dartmouth Hitchcock Medical Center<br />Lebanon, New Hampshire<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="gro.kcochctih@relhcseom.b.nhoj" class="oemail">gro.kcochctih@relhcseom.b.nhoj</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Jill Rosenfeld</span>, MS<div class="affiliation small">Assistant Professor, Department of Molecular and Human Genetics<br />Baylor College of Medicine<br />Houston, Texas<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.mcb@yrkom" class="oemail">ude.mcb@yrkom</a></div></div></div></div><p class="small">Initial Posting: <span itemprop="datePublished">February 26, 2015</span>; Last Update: <span itemprop="dateModified">September 13, 2018</span>.</p><p><em>Estimated reading time: 21 minutes</em></p></div><div class="jig-ncbiinpagenav body-content whole_rhythm" data-jigconfig="allHeadingLevels: ['h2'],smoothScroll: false" itemprop="text"><div id="mdel16p12_2.Summary" itemprop="description"><h2 id="_mdel16p12_2_Summary_">Summary</h2><div><h4 class="inline">Clinical characteristics.</h4><p>16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> is characterized by variable clinical findings that do not constitute a recognizable syndrome. Of note, the significant bias in ascertainment of individuals undergoing clinical <a class="def" href="/books/n/gene/glossary/def-item/chromosomal-microarray/">chromosomal microarray</a> analysis (i.e., children with intellectual disability and developmental delay; individuals with schizophrenia) makes it difficult to accurately associate specific phenotypes with the 16p12.2 recurrent deletion. Findings commonly observed in children (probands) with this deletion include: developmental delay, cognitive impairment (ranging from mild to profound), growth impairment (including short stature), cardiac malformations, epilepsy, and psychiatric and/or behavioral problems. Other findings can include: hearing loss, dental abnormalities, renal and genital anomalies (the latter in males), and cleft palate &#x000b1; cleft lip.</p></div><div><h4 class="inline">Diagnosis/testing.</h4><p>The diagnosis of 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> is established by identification of a 520-kb <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> deletion on <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> 16p12.2 on <a class="def" href="/books/n/gene/glossary/def-item/chromosomal-microarray/">chromosomal microarray</a> analysis or other <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> analyses.</p></div><div><h4 class="inline">Management.</h4><p><i>Treatment of manifestations:</i> Treatment is directed to specific problems identified and may include developmental therapies; routine treatment of cardiac malformations, epilepsy, psychiatric and behavioral problems, hearing loss, and other malformations (e.g., orofacial clefting; renal, genitourinary, and dental anomalies).</p><p><i>Surveillance:</i> Periodic: developmental evaluations; monitoring of cardiac, renal, urologic, and/or dental abnormalities, as needed; reevaluation by a clinical geneticist.</p><p><i>Evaluation of relatives at risk:</i> Older and younger sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> should be tested for a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> to allow for close assessment/monitoring of developmental milestones and monitoring for neuropsychiatric manifestations in children with the deletion.</p></div><div><h4 class="inline">Genetic counseling.</h4><p>The 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> manner. The majority (~95%) of individuals with this recurrent deletion have inherited the deletion from a parent (who may or may not have clinical features related to the recurrent deletion). If a parent is <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> for the 16p12.2 recurrent deletion, the risk that the sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> would inherit the deletion is 50%; however, the risk that sibs would be <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> is less than 50% because of reduced <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> for the deletion. Children with a family history of neurodevelopmental and psychiatric disease are more likely to present with severe clinical features of the deletion. If a 16p12.2 recurrent deletion has been identified in a family member, prenatal testing for pregnancies at increased risk is possible; however, it is not possible to reliably predict <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> based on the laboratory finding of a 16p12.2 recurrent deletion.</p></div></div><div id="mdel16p12_2.Diagnosis"><h2 id="_mdel16p12_2_Diagnosis_">Diagnosis</h2><p>No formal diagnostic criteria have been established for 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>.</p><p>Because of the variable clinical presentation of 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>, the diagnosis is made by detection of 16p12.2 recurrent deletion on <a class="def" href="/books/n/gene/glossary/def-item/chromosomal-microarray/">chromosomal microarray</a> analysis (CMA) or other <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> analyses.</p><div id="mdel16p12_2.Suggestive_Findings"><h3>Suggestive Findings</h3><p>The 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> <b>should be considered</b> in individuals with the following clinical findings:</p><ul><li class="half_rhythm"><div>Developmental delays</div></li><li class="half_rhythm"><div>Mild to moderate intellectual disability</div></li><li class="half_rhythm"><div>Speech delays</div></li><li class="half_rhythm"><div>Psychiatric and behavioral abnormalities including autism, bipolar disorder, depression, and schizophrenia</div></li><li class="half_rhythm"><div>Mild <a class="def" href="/books/n/gene/glossary/def-item/dysmorphic/">dysmorphic</a> facial features without a consistent pattern</div></li><li class="half_rhythm"><div>Congenital cardiac defects</div></li><li class="half_rhythm"><div>Sleep disturbance</div></li><li class="half_rhythm"><div>Epilepsy</div></li><li class="half_rhythm"><div>A positive family history of learning disorders or psychiatric issues</div></li></ul><p>Of note, most individuals with the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> are identified by CMA performed in the context of evaluation for developmental delay, intellectual disability, and/or autism spectrum disorder.</p></div><div id="mdel16p12_2.Establishing_the_Diagnosis"><h3>Establishing the Diagnosis</h3><p>The diagnosis of the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> is established by detection of the 520-kb <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> deletion at <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> 16p12.2 (see <a class="figpopup" href="/books/NBK274565/table/mdel16p12_2.T.genomic_testing_used_in_16/?report=objectonly" target="object" rid-figpopup="figmdel16p122Tgenomictestingusedin16" rid-ob="figobmdel16p122Tgenomictestingusedin16">Table 1</a> and <a href="#mdel16p12_2.Molecular_Genetics">Molecular Genetics</a>).</p><p>For this <i>GeneReview</i>, the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> is defined as the presence of a recurrent 520-kb deletion at the approximate position of 21,948,445-22,430,805 in the reference genome (NCBI Build GRCh37/hg19).</p><p>ISCN nomenclature for this <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> is: seq[GRCh37] del(16)(p12.2) chr16:g.21,948,445-22,430,805del. Note: Since this deletion is recurrent and mediated by segmental duplications, the unique genetic sequence that is deleted is the same in all individuals with the syndrome; however, the reported size of the deletion may: (1) be larger if adjacent segmental duplications are included in the size and (2) vary based on the design of the microarray used to detect it (see <a href="#mdel16p12_2.Molecular_Pathogenesis">Molecular Pathogenesis</a>).</p><p>Note: The <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> of significantly larger or smaller deletions within this region may be clinically distinct from the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> (see <a href="#mdel16p12_2.Genetically_Related_Allelic">Genetically Related Disorders</a>).</p><p>Although seven genes of interest (<i>UQCRC2</i>, <i>PDZD9</i>, <i>MOSMO</i>, <i>VWA3A</i>, <i>EEF2K</i>, <i>POLR3E</i>, and <i>CDR2</i>) are within the 520-kb <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>, no single <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> for which pathogenic variants are causative has been identified (see <a href="#mdel16p12_2.Molecular_Genetics">Molecular Genetics</a> for genes of interest in the deleted region).</p><p><b>Genomic testing methods</b> that determine the copy number of sequences can include <b><a class="def" href="/books/n/gene/glossary/def-item/chromosomal-microarray/">chromosomal microarray</a> (CMA)</b> or <b>targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> analysis</b>. Note: The 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> cannot be identified by routine analysis of G-banded chromosomes or other conventional <a class="def" href="/books/n/gene/glossary/def-item/cytogenetic/">cytogenetic</a> banding techniques.</p><p><b>CMA</b> using oligonucleotide or SNP arrays can detect the <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> in a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>. The ability to size the deletion depends on the type of microarray used and the density of probes in the 16p12.2 region.</p><p>Note: (1) Most individuals with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> are identified by CMA performed in the context of evaluation for developmental delay, intellectual disability, or autism spectrum disorder. (2) Prior to 2010 several CMA platforms did not include coverage for this region and thus may not have detected this deletion. (3) The deletion would not have been detected using BAC arrays.</p><p><b>Targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> analysis.</b> <a class="def" href="/books/n/gene/glossary/def-item/fish/">FISH</a> analysis, <a class="def" href="/books/n/gene/glossary/def-item/quantitative-pcr/">quantitative PCR</a> (qPCR), multiplex ligation-dependent probe amplification (MLPA), or other targeted quantitative methods may be used to test relatives of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> who is known to have the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>.</p><p>Note: (1) Targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> testing is not appropriate for an individual in whom the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> was not detected by CMA designed to target this region. (2) It is not possible to size the deletion routinely by use of targeted methods.</p><div id="mdel16p12_2.T.genomic_testing_used_in_16" class="table"><h3><span class="label">Table 1. </span></h3><div class="caption"><p>Genomic Testing Used in 16p12.2 Recurrent Deletion</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK274565/table/mdel16p12_2.T.genomic_testing_used_in_16/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__mdel16p12_2.T.genomic_testing_used_in_16_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_1" rowspan="2" scope="col" colspan="1" headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">Deletion&#x000a0;<sup>1</sup></th><th id="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_2" rowspan="2" scope="col" colspan="1" headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_2" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">Test Method</th><th id="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_3" colspan="2" scope="colgroup" rowspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">Test Sensitivity</th></tr><tr><th headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_3" id="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_2_1" colspan="1" scope="colgroup" rowspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">Proband</th><th headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_3" id="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_2_2" scope="col" rowspan="1" colspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">At-risk family members</th></tr></thead><tbody><tr><td headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_1" rowspan="2" scope="row" colspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">520-Mb <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> at 16p12.2<br />ISCN: seq[GRCh37] del(16)(p12.2) chr16:g.21,948,445-22,430,805del&#x000a0;<sup>2</sup><br />ClinGen ID: ISCA-<a href="https://www.ncbi.nlm.nih.gov/projects/dbvar/clingen/clingen_region.cgi?id=ISCA-37409" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">37409</a></td><td headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_2" rowspan="1" colspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">CMA&#x000a0;<sup>3</sup></td><td headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_3 hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_2_1" rowspan="1" colspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">100%</td><td headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_3 hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_2_2" rowspan="1" colspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">100%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_2" colspan="1" scope="row" rowspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">Targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> analysis&#x000a0;<sup>4</sup></td><td headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_3 hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_2_1" rowspan="1" colspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">NA&#x000a0;<sup>5</sup></td><td headers="hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_1_3 hd_h_mdel16p12_2.T.genomic_testing_used_in_16_1_1_2_2" rowspan="1" colspan="1" style="background-color:rgb(255,255,255);text-align:left;vertical-align:middle;">100%&#x000a0;<sup>6</sup></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="mdel16p12_2.TF.1.1"><p class="no_margin">See <a href="#mdel16p12_2.Molecular_Genetics">Molecular Genetics</a> for details of the <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> and genes of interest included in the region.</p></div></dd><dt>2. </dt><dd><div id="mdel16p12_2.TF.1.2"><p class="no_margin">Standardized ISCN annotation and interpretation for <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> variants from the <a href="http://www.clinicalgenome.org/about/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Clinical Genome Resource (ClinGen) project</a> (formerly the International Standards for Cytogenomic Arrays (ISCA) Consortium). Genomic coordinates represent the minimum <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> size associated with the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> as designated by ClinGen. Deletion coordinates may vary slightly based on array design used by the testing laboratory. Note that the size of the deletion as calculated from these genomic positions may differ from the expected deletion size due to the presence of segmental duplications near breakpoints. The <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> of significantly larger or smaller deletions within this region may be clinically distinct from the 16p12.2 recurrent deletion (see <a href="#mdel16p12_2.Genetically_Related_Allelic">Genetically Related Disorders</a>).</p></div></dd><dt>3. </dt><dd><div id="mdel16p12_2.TF.1.3"><p class="no_margin">Chromosomal microarray analysis (CMA) using oligonucleotide arrays or SNP arrays. CMA designs in current clinical use target the 16p12.2 region. Note: The 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> may not have been detectable by older oligonucleotide or BAC platforms.</p></div></dd><dt>4. </dt><dd><div id="mdel16p12_2.TF.1.4"><p class="no_margin">Targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> analysis methods can include <a class="def" href="/books/n/gene/glossary/def-item/fish/">FISH</a>, <a class="def" href="/books/n/gene/glossary/def-item/quantitative-pcr/">quantitative PCR</a> (qPCR), and multiplex ligation-dependent probe amplification (MLPA) as well as other targeted quantitative methods.</p></div></dd><dt>5. </dt><dd><div id="mdel16p12_2.TF.1.5"><p class="no_margin">Targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> analysis is not appropriate for an individual in whom the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> was not detected by CMA designed to target this region.</p></div></dd><dt>6. </dt><dd><div id="mdel16p12_2.TF.1.6"><p class="no_margin">Targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> analysis may be used to test at-risk relatives of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> who is known to have the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>.</p></div></dd></dl></div></div></div><p><b>Evaluating at-risk relatives.</b> <a class="def" href="/books/n/gene/glossary/def-item/fish/">FISH</a>, qPCR, or other quantitative methods of targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> analysis can be used to identify the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> in at-risk relatives of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>. Testing of parental samples is important in determining <a class="def" href="/books/n/gene/glossary/def-item/recurrence-risk/">recurrence risk</a> (see <a href="#mdel16p12_2.Genetic_Counseling">Genetic Counseling</a>).</p></div></div><div id="mdel16p12_2.Clinical_Characteristics"><h2 id="_mdel16p12_2_Clinical_Characteristics_">Clinical Characteristics</h2><div id="mdel16p12_2.Clinical_Description"><h3>Clinical Description</h3><p>Due to the <a class="def" href="/books/n/gene/glossary/def-item/variable-expressivity/">variable expressivity</a> of <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> 16p12.2, this variant was not known prior to the use of <a class="def" href="/books/n/gene/glossary/def-item/chromosomal-microarray/">chromosomal microarray</a> testing in genetic diagnosis. The significant bias in ascertainment of children with intellectual disability and developmental delay [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.cooper.2011.838">Cooper et al 2011</a>] and of individuals with schizophrenia [<a class="bk_pop" href="#mdel16p12_2.REF.rees.2014.1669">Rees et al 2014</a>] undergoing clinical chromosomal microarray analysis makes it difficult to accurately associate specific phenotypes with the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> (<a class="figpopup" href="/books/NBK274565/table/mdel16p12_2.T.clinical_features_in_proba/?report=objectonly" target="object" rid-figpopup="figmdel16p122Tclinicalfeaturesinproba" rid-ob="figobmdel16p122Tclinicalfeaturesinproba">Table 2</a>).</p><div id="mdel16p12_2.T.clinical_features_in_proba" class="table"><h3><span class="label">Table 2. </span></h3><div class="caption"><p>Clinical Features in Probands with 16p12.2 Recurrent Deletion</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK274565/table/mdel16p12_2.T.clinical_features_in_proba/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__mdel16p12_2.T.clinical_features_in_proba_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Finding</th><th id="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Frequency</th><th id="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">%</th></tr></thead><tbody><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Developmental delay</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">76/113</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">67%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Speech delay</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">68/92</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">74%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Intellectual disability</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">46/59</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">78%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Craniofacial features</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">50/85</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">59%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Musculoskeletal features</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">30/57</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">53%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Growth retardation</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">26/74</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">35%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Microcephaly</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">25/81</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">31%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Congenital cardiac defect</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">13/34</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">38%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Epilepsy</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">27/71</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">38%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Psychiatric/behavioral disorders</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">9/16</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">56%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Autism</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">31/67</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">46%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Hearing loss</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">12/59</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">20%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Hypotonia</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">24/71</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">34%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Genital problems</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">12/47</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">25%</td></tr><tr><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Sacral dimple or tethered cord</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">4/24</td><td headers="hd_h_mdel16p12_2.T.clinical_features_in_proba_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">17%</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin"><a class="bk_pop" href="#mdel16p12_2.REF.de_jong.2010.1265">de Jong et al [2010]</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al [2010]</a>, <a class="bk_pop" href="#mdel16p12_2.REF.dalessandro.2014.83">D'Alessandro et al [2014]</a>, <a class="bk_pop" href="#mdel16p12_2.REF.rai.2015.394">Rai &#x00026; Sharif [2015]</a>, <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al [2018]</a></p></div></dd><dt></dt><dd><div><p class="no_margin">Some probands had additional genetic abnormalities identified that likely contributed to their phenotypes; frequencies therefore likely represent an ascertainment bias.</p></div></dd></dl></div></div></div><p>A study including 23 unrelated probands with 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> from a (postnatal) clinical genetic testing cohort yielded the following phenotypic findings [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>]. A more recent study [<a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al 2018</a>] analyzed 141 children with 16p12.2 deletion (median age 7 years) referred for clinical genetic testing, further extending the clinical spectrum observed in the initial reports. Because of the nature of these clinical populations, some of the phenotypes may reflect an ascertainment bias.</p><p><b>Developmental delay.</b> In the report of <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al [2010]</a>, developmental delay ranging from mild to profound was present in all individuals with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>. All individuals older than age 12 months showed speech delay; some remained nonverbal into childhood and adolescence. Motor milestones could also be affected, and many individuals were noted to have global delays. Among the individuals assessed for developmental milestones by <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al [2018]</a>, 67% showed developmental delay or psychomotor retardation while 74% exhibited speech delay.</p><p>In contrast, individuals with the <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> who were specifically ascertained for heart defects did not show delayed development [<a class="bk_pop" href="#mdel16p12_2.REF.dalessandro.2014.83">D'Alessandro et al 2014</a>]. Similarly, normal early language development was described in the report of a child age one year with the recurrent deletion [<a class="bk_pop" href="#mdel16p12_2.REF.rai.2015.394">Rai &#x00026; Sharif 2015</a>].</p><p><b>Cognitive development.</b>
<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al [2010]</a> described a spectrum of cognitive abilities in individuals with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>, ranging from normal in <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> parents to mild impairments to profoundly <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a>, nonverbal individuals. However, parents with the recurrent deletion were more likely to manifest learning disabilities than parents who did not have the recurrent deletion. Analysis of cognitive and psychiatric phenotypes by <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al [2018]</a> identified learning difficulties in school in 53% of parents with the 16p12.2 deletion. Individuals from the general population with the 16p12.2 deletion were found to have decreased cognitive function (verbal IQ scores and logical memory) compared to controls who did not have the deletion [<a class="bk_pop" href="#mdel16p12_2.REF.stefansson.2014.361">Stefansson et al 2014</a>].</p><p><b>Dysmorphic features.</b> While <a class="def" href="/books/n/gene/glossary/def-item/dysmorphic/">dysmorphic</a> features are reported in a majority of individuals with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>, no consistent pattern was evident [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>].</p><p><b>Growth.</b> Approximately two fifths of pediatric probands had growth retardation; three were specifically noted to have short stature [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>]. These features were also observed by <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al [2018]</a>, with 35% of individuals exhibiting intrauterine growth delay, short stature, and/or delayed growth.</p><p>Microcephaly was present in seven of 22 pediatric probands [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>], including two who had otherwise normal growth parameters [<a class="bk_pop" href="#mdel16p12_2.REF.de_jong.2010.1265">de Jong et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.rai.2015.394">Rai &#x00026; Sharif 2015</a>]. Twenty-five of the 81 children with 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> analyzed by <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al [2018]</a> exhibited microcephaly, and six showed increased head circumference z-scores, corresponding to a macrocephaly <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>.</p><p><b>Cardiac malformations.</b> Recurrent deletions of 16p12.2 may be a risk factor for cardiac malformations. While not all individuals with these deletions have had echocardiography, 14 children had cardiac defects, including hypoplastic left heart (found in 4 individuals, 2 of whom also had heterotaxy), ventricular septal defect, patent foramen ovale, absence of posterior pericardium, bicuspid aortic valve, aortic valve stenosis, patent ductus arteriosus, and tetralogy of Fallot [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.dalessandro.2014.83">D'Alessandro et al 2014</a>, <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al 2018</a>]. However, 16p12.2 recurrent deletions are unlikely to be a major cause of left-sided cardiac lesions [<a class="bk_pop" href="#mdel16p12_2.REF.dalessandro.2014.83">D'Alessandro et al 2014</a>].</p><p><b>Epilepsy.</b> Approximately 40% of probands with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> experienced seizures and/or had abnormal findings on EEG. Seizure types included West syndrome, Lennox-Gastaut syndrome, staring spells, epilepsy with myoclonus, and febrile seizures [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al 2018</a>].</p><p><b>Psychiatric and behavior problems.</b> Psychiatric and/or behavioral problems were identified in more than half (9/16) of individuals with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> who were assessed for these features. Autistic features or stereotypies were specifically noted in three individuals, poor attention was noted in three, and aggression was noted in two [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>]. Further analysis in a larger population identified a diagnosis of autism in 31/67 children (67%) with the deletion [<a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al 2018</a>].</p><p>Recurrent deletions of 16p12.2 have also been reported to be significantly enriched among individuals with schizophrenia, with an associated odds ratio of 2.72 (95% confidence interval, 1.48-5.02) relative to a control population [<a class="bk_pop" href="#mdel16p12_2.REF.rees.2014.1669">Rees et al 2014</a>]. Additionally, one control identified to have a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> was retrospectively diagnosed with major depressive disorder, and parents with a 16p12.2 recurrent deletion were also significantly more likely to have mild learning disability or psychiatric issues (e.g. depression, bipolar disorder) than parents without the deletion [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>]. <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al [2018]</a> showed that among parents with the 16p12.2 deletion, 53% (17/32) had exhibited learning difficulties in school, 52% (11/21) exhibited depressive behaviors, 13% (2/15) alcohol or drug addiction, 19% (3/16) schizophrenic-like behaviors, 28% epilepsy, and 13% (2/16) bipolar disorder.</p><p><b>Other neurologic features.</b> Hearing loss including unilateral, bilateral, sensorineural, and unspecified has been described in eight individuals [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al 2018</a>].</p><p>Hypotonia was reported in 34%-45% of individuals assessed for this feature [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.rai.2015.394">Rai &#x00026; Sharif 2015</a>].</p><p>Either a sacral dimple or tethered cord was present in four of 24 individuals assessed [<a class="bk_pop" href="#mdel16p12_2.REF.de_jong.2010.1265">de Jong et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.dalessandro.2014.83">D'Alessandro et al 2014</a>, <a class="bk_pop" href="#mdel16p12_2.REF.rai.2015.394">Rai &#x00026; Sharif 2015</a>].</p><p>Abnormal brain imaging was reported in 56%-63% of individuals, with features including cerebellar and cerebral atrophy, decreased white matter, unspecified periventricular changes, and agenesis of the corpus callosum [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al 2018</a>].</p><p>One individual with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> had postnatal onset of hydrocephalus due to cervicomedullary spinal stenosis [<a class="bk_pop" href="#mdel16p12_2.REF.rai.2015.394">Rai &#x00026; Sharif 2015</a>], and an additional individual reportedly had <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> hydrocephalus without additional clinical details available [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>].</p><p><b>Other <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> anomalies.</b> Aside from heart defects, other congenital anomalies have only occasionally been reported, and some of these may be attributable to other unknown genetic factors. Out of 26 reported probands with 16p12.2 recurrent deletions [<a class="bk_pop" href="#mdel16p12_2.REF.de_jong.2010.1265">de Jong et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.dalessandro.2014.83">D'Alessandro et al 2014</a>, <a class="bk_pop" href="#mdel16p12_2.REF.rai.2015.394">Rai &#x00026; Sharif 2015</a>]:</p><ul><li class="half_rhythm"><div>Four children had absence of canine teeth with duplication of incisors bilaterally, pegged incisors, crowded teeth, and/or dental caries.</div></li><li class="half_rhythm"><div>Three children had renal abnormalities (small kidneys, horseshoe kidney, or hydronephrosis).</div></li><li class="half_rhythm"><div>Three males had genital anomalies including chordee, hypospadias, and cryptorchidism [<a class="bk_pop" href="#mdel16p12_2.REF.de_jong.2010.1265">de Jong et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>].</div></li><li class="half_rhythm"><div>Two had cleft palate, one with cleft lip.</div></li><li class="half_rhythm"><div>Two had clubfoot or bowed legs.</div></li><li class="half_rhythm"><div>Single individuals have been reported with craniosynostosis, inguinal hernia, and tracheal agenesis, although the last was in an individual who had an additional rare <a class="def" href="/books/n/gene/glossary/def-item/copy-number-variant/">copy number variant</a> [<a class="bk_pop" href="#mdel16p12_2.REF.de_jong.2010.1265">de Jong et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>].</div></li></ul></div><div id="mdel16p12_2.GenotypePhenotype_Correlatio"><h3>Genotype-Phenotype Correlations</h3><p>Probands with 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> manifesting abnormal phenotypes are significantly more likely than controls to have a second rare, unrelated, large (&#x0003e;500 kb) <a class="def" href="/books/n/gene/glossary/def-item/copy-number-variant/">copy number variant</a> (CNV) [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2012.1321">Girirajan et al 2012</a>]. Such individuals had distinct or more severe phenotypes &#x02013; compared to the classic phenotypes associated with the known second pathogenic CNV &#x02013; suggesting that the 16p12.2 recurrent deletion is an independent risk factor for neurodevelopmental phenotypes in association with other large, pathogenic CNVs [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>]. Furthermore, <a class="def" href="/books/n/gene/glossary/def-item/exome-sequencing/">exome sequencing</a> analysis of 26 families with the deletion showed that probands present an enrichment of rare (&#x0003c;0.1%) likely pathogenic variants affecting functionally intolerant genes ("other hits") compared to their parent with 16p12.2 recurrent deletion, likely contributing to the more severe clinical manifestation with early-onset features [<a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al 2018</a>].</p></div><div id="mdel16p12_2.Penetrance"><h3>Penetrance</h3><p>Penetrance for 16p12.2 recurrent deletions is incomplete.</p><p>The 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> was documented to be enriched in a population undergoing clinical <a class="def" href="/books/n/gene/glossary/def-item/chromosomal-microarray/">chromosomal microarray</a> analysis [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.cooper.2011.838">Cooper et al 2011</a>] and among individuals with schizophrenia [<a class="bk_pop" href="#mdel16p12_2.REF.rees.2014.1669">Rees et al 2014</a>]. Further studies also identified the deletion in apparently healthy parents of probands and controls. On further follow up, parents with the deletion were found to be more likely to have clinical findings such as seizures, mild intellectual disability, and/or psychiatric issues, suggesting that the 16p12.2 recurrent deletion is a risk factor for abnormal neurodevelopmental phenotypes with reduced <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> and <a class="def" href="/books/n/gene/glossary/def-item/variable-expressivity/">variable expressivity</a>.</p><p>Based on data from children undergoing clinical <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> microarray analysis and adult controls, estimates for intellectual disability / developmental delay and/or <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> malformations in those with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> were 12.3% (95% confidence interval, 7.91%-18.8%) [<a class="bk_pop" href="#mdel16p12_2.REF.rosenfeld.2013.478">Rosenfeld et al 2013</a>] and 13% (95% confidence interval, 5.7%-30%) [<a class="bk_pop" href="#mdel16p12_2.REF.kirov.2014.378">Kirov et al 2014</a>].</p><p><a class="bk_pop" href="#mdel16p12_2.REF.kirov.2014.378">Kirov et al [2014]</a> estimated the <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> of schizophrenia among individuals with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> at 3.1% (95% confidence interval, 1.2%-8.3%).</p><p>In addition, individuals with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> are more likely to have a family history of neuropsychiatric phenotypes, suggesting <a class="def" href="/books/n/gene/glossary/def-item/segregation/">segregation</a> of neuropsychiatric risk factors other than 16p12.2 recurrent deletion. Of note, ten (~25%) of 42 probands with 16p12.2 recurrent deletion also had another large (&#x0003e;500-kb) CNV elsewhere in the genome. The "second hit" was frequently <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> or transmitted from the parent who did not have the 16p12.2 recurrent deletion. These observations suggest that the deletion confers risk for neuropsychiatric features and that the <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> and expressivity of the deletion-associated <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> depends on the presence of a second large CNV or, potentially, other genetic modifiers elsewhere in the genome. Probands with a strong family history of psychiatric and neurodevelopmental disease present a more heterogeneous and severe manifestation of the deletion and a higher burden of additional likely pathogenic variants. Family history of psychiatric and neurodevelopmental disease should be interrogated to assess prognosis in the children with 16p12.2 deletions [<a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al 2018</a>].</p><p>The proportion of males is higher among all individuals with a CNV associated with reduced <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> and <a class="def" href="/books/n/gene/glossary/def-item/variable-expressivity/">variable expressivity</a> (like a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>) as compared to those with <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> CNVs (like <a href="/books/n/gene/sms/">Smith-Magenis syndrome</a>, where the proportion of males is ~50%), suggesting that penetrance may be higher in males than females [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2012.1321">Girirajan et al 2012</a>, <a class="bk_pop" href="#mdel16p12_2.REF.jacquemont.2014.415">Jacquemont et al 2014</a>]. Among 140 individuals analyzed by Pizzo et al, 68% were males and 32% were females [<a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al 2018</a>].</p></div><div id="mdel16p12_2.Anticipation"><h3>Anticipation</h3><p>While almost all reports describe identification of the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> in individuals who are more severely <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> than a parent with the deletion, this is more likely to reflect an ascertainment bias than genetic <a class="def" href="/books/n/gene/glossary/def-item/anticipation/">anticipation</a>. Changes in the size of a 16p12.2 recurrent deletion on transmission from one generation to the next have not been described [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>].</p></div><div id="mdel16p12_2.Nomenclature"><h3>Nomenclature</h3><p>The chromosomal location of the <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> originally described at 16p12.1 (from coordinates ~21,850,000-~22,370,000, genome build hg18/NCB136) has changed to 16p12.2 (from coordinates ~21,948,445-~22,430,805, genome build hg19/GRCh37). Clinical reports and descriptions now use the 16p12.2 location/nomenclature.</p></div><div id="mdel16p12_2.Prevalence"><h3>Prevalence</h3><p>The estimated frequency of 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> &#x02013; on the order of 0.19% of individuals undergoing clinical microarray-based testing [<a class="bk_pop" href="#mdel16p12_2.REF.rosenfeld.2013.478">Rosenfeld et al 2013</a>] &#x02013; is similar to that of <a href="/books/n/gene/sms/">Smith-Magenis syndrome</a>, suggesting that Smith-Magenis syndrome and 16p12.2 recurrent deletion associated with an abnormal <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> may have a similar incidence of approximately 1:15,000 live births [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>].</p><p>However, this estimate does not account for healthy or mildly <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals who are <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> for the <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a>.</p><ul><li class="half_rhythm"><div>Control studies show the frequency of the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> to be 0.050%-0.072%, or approximately one in 1,400-2,000 individuals [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.rosenfeld.2013.478">Rosenfeld et al 2013</a>, <a class="bk_pop" href="#mdel16p12_2.REF.kirov.2014.378">Kirov et al 2014</a>, <a class="bk_pop" href="#mdel16p12_2.REF.rees.2014.1669">Rees et al 2014</a>].</div></li><li class="half_rhythm"><div>One study estimated the prevalence of the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> in the general population (including healthy and <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals) at 0.057% (95% confidence interval, 0.032%-0.10%) [<a class="bk_pop" href="#mdel16p12_2.REF.kirov.2014.378">Kirov et al 2014</a>].</div></li></ul></div></div><div id="mdel16p12_2.Genetically_Related_Allelic"><h2 id="_mdel16p12_2_Genetically_Related_Allelic_">Genetically Related (Allelic) Disorders</h2><p><b>16p12.2 recurrent <a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a>.</b> The reciprocal recurrent duplication has not been shown to be enriched in a patient population [<a class="bk_pop" href="#mdel16p12_2.REF.cooper.2011.838">Cooper et al 2011</a>] and is therefore likely a <a class="def" href="/books/n/gene/glossary/def-item/benign-variant/">benign variant</a>.</p><p><b>16p11.2-p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>.</b> A <a class="def" href="/books/n/gene/glossary/def-item/deletion-syndrome/">deletion syndrome</a> encompassing a larger portion the 16p region has been described. Common features include characteristic <a class="def" href="/books/n/gene/glossary/def-item/dysmorphic/">dysmorphic</a> features, <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> anomalies, feeding difficulties, frequent ear infections, and cognitive and developmental delays. The commonly deleted region is ~7.1 Mb, from coordinates ~21,500,000 to ~28,600,000 (genome build <a href="http://genome.ucsc.edu/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">UCSC</a> hg19), including the recurrently deleted 16p12.2 region [<a class="bk_pop" href="#mdel16p12_2.REF.ballif.2007.1071">Ballif et al 2007</a>, <a class="bk_pop" href="#mdel16p12_2.REF.battaglia.2009.1200">Battaglia et al 2009</a>, <a class="bk_pop" href="#mdel16p12_2.REF.hempel.2009.2106">Hempel et al 2009</a>, <a class="bk_pop" href="#mdel16p12_2.REF.okamoto.2014.213">Okamoto et al 2014</a>].</p><p>No phenotypes other than those discussed in this <i>GeneReview</i> are known to be associated with copy number variants (CNVs) in the genes located within the 16p12.2 520-kb <a class="def" href="/books/n/gene/glossary/def-item/critical-region/">critical region</a>.</p></div><div id="mdel16p12_2.Differential_Diagnosis"><h2 id="_mdel16p12_2_Differential_Diagnosis_">Differential Diagnosis</h2><p>The differential diagnosis of the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> is broad due to the variable spectrum and presence of relatively common abnormal phenotypes that occur in <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals including developmental delay, learning problems, and neuropsychiatric disorders. All manifestations of the 16p12.2 recurrent deletion can also be seen in individuals with other <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> disorders.</p></div><div id="mdel16p12_2.Management"><h2 id="_mdel16p12_2_Management_">Management</h2><div id="mdel16p12_2.Evaluations_Following_Initia"><h3>Evaluations Following Initial Diagnosis</h3><p>To establish the extent of disease and needs of an individual diagnosed with the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>, the following evaluations are recommended if they have not already been completed:</p><ul><li class="half_rhythm"><div>Consultation with a clinical geneticist and/or genetic counselor</div></li><li class="half_rhythm"><div>Measurement of height and weight</div></li><li class="half_rhythm"><div>Broad review of all organ systems</div></li><li class="half_rhythm"><div>Developmental assessment with cognitive and behavioral testing</div></li></ul><p>To consider:</p><ul><li class="half_rhythm"><div>Consultation with a neurologist and EEG testing if history suggests the possibility of seizures</div></li><li class="half_rhythm"><div>Evaluation and echocardiogram by a cardiologist</div></li></ul></div><div id="mdel16p12_2.Treatment_of_Manifestations"><h3>Treatment of Manifestations</h3><p>Because manifestations of 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> are variable, treatment should be targeted to the specific problems identified. Early diagnosis and treatment facilitate the best outcome. Referral to other appropriate medical specialists is recommended based on specific signs and symptoms. Specialists may include a developmental/behavioral pediatrician, pediatric neurologist, and/or clinical geneticist.</p><p><b>Developmental and cognitive delays.</b> Initiate developmental therapies promptly when indicated. Cognitive testing in older children may help identify any specific learning or cognitive disabilities that could be addressed by therapies or specialized education plans.</p><p><b>Cardiac malformations.</b> Individuals with 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> and cardiac malformations should have standard treatment for the malformation, including surgical correction and prophylactic antibiotic treatments as indicated.</p><p><b>Epilepsy.</b> A history suggestive of possible seizure activity should prompt referral to a neurologist for additional testing, including brain imaging and EEG, and consideration of medical intervention.</p><p><b>Psychiatric and/or behavior problems.</b> In individuals of any age, signs of psychiatric or behavioral problems should prompt referral to specialists including developmental/behavioral pediatricians, psychologists, or psychiatrists so that formal assessments may be performed and appropriate interventions (medical, social, behavioral, and/or educational) may be made.</p><p><b>Hearing loss.</b> Standard treatment should be provided for hearing loss. See <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness</a>.</p><p><b>Other malformations.</b> Standard treatment should be provided for any other malformations, including orofacial clefting and renal, genitourinary, or dental anomalies.</p></div><div id="mdel16p12_2.Surveillance"><h3>Surveillance</h3><p>Periodic:</p><ul><li class="half_rhythm"><div>Developmental evaluations because of the increased incidence of developmental delay, intellectual disability, autism spectrum disorders, and other behavioral features</div></li><li class="half_rhythm"><div>Monitoring of cardiac, renal, urologic, and/or dental abnormalities, as needed</div></li><li class="half_rhythm"><div>Reevaluation by a clinical geneticist who can apprise the family of new recommendations for monitoring for medical or mental health concerns.</div></li></ul></div><div id="mdel16p12_2.Evaluation_of_Relatives_at_R"><h3>Evaluation of Relatives at Risk</h3><p>Older and younger sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> should be tested for a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> to encourage close assessment/monitoring of developmental milestones and monitoring for neuropsychiatric and <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> manifestations in children with the deletion.</p><p>See <a href="#mdel16p12_2.Related_Genetic_Counseling_I">Genetic Counseling</a> for issues related to testing of at-risk relatives for <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> purposes.</p></div><div id="mdel16p12_2.Therapies_Under_Investigatio"><h3>Therapies Under Investigation</h3><p>Search <a href="http://clinicaltrials.gov/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">ClinicalTrials.gov</a> in the US and <a href="http://www.clinicaltrialsregister.eu/ctr-search/search" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">EU Clinical Trials Register</a> in Europe for access to information on clinical studies for a wide range of diseases and conditions. Note: There may not be clinical trials for this disorder.</p></div></div><div id="mdel16p12_2.Genetic_Counseling"><h2 id="_mdel16p12_2_Genetic_Counseling_">Genetic Counseling</h2><p>
<i>Genetic counseling is the process of providing individuals and families with
information on the nature, inheritance, and implications of genetic disorders to help them
make informed medical and personal decisions. The following section deals with genetic
risk assessment and the use of family history and genetic testing to clarify genetic
status for family members. This section is not meant to address all personal, cultural, or
ethical issues that individuals may face or to substitute for consultation with a genetics
professional</i>. &#x02014;ED.</p><div id="mdel16p12_2.Mode_of_Inheritance"><h3>Mode of Inheritance</h3><p>The 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> manner, with an estimated 95% of deletions inherited from a parent and 5% occurring <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a>.</p><p>A single, apparently <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> was reported by <a class="bk_pop" href="#mdel16p12_2.REF.rosenfeld.2013.478">Rosenfeld et al [2013]</a>. However, additional testing to confirm parentage was not performed. Seven more <i>de novo</i> 16p12.2 recurrent deletions were identified by <a class="bk_pop" href="#mdel16p12_2.REF.pizzo.2018">Pizzo et al [2018]</a>, corresponding to a <i>de novo</i> rate of 7.6% (7/92) in this study.</p></div><div id="mdel16p12_2.Risk_to_Family_Members"><h3>Risk to Family Members</h3><p><b>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>Evaluation of the parents by <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing that will detect the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> present in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> is recommended.</div></li><li class="half_rhythm"><div>Note: Germline <a class="def" href="/books/n/gene/glossary/def-item/mosaicism/">mosaicism</a>, <a class="def" href="/books/n/gene/glossary/def-item/somatic-mosaicism/">somatic mosaicism</a>, and balanced chromosomal rearrangements involving the 16p12.2 region have not been reported with 16p12.2 recurrent deletions.</div></li><li class="half_rhythm"><div>The family history of some individuals diagnosed with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> may appear to be negative because of reduced <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> and <a class="def" href="/books/n/gene/glossary/def-item/variable-expressivity/">variable expressivity</a>. Therefore, an apparently negative family history cannot be confirmed unless appropriate <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing followed by detailed clinical evaluation has been performed on the parents of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>. Even in individuals with a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> deletion, eliciting family history is important as neuropsychiatric disease can be caused by pathogenic variants other than a 16p12.2 recurrent deletion.</div></li></ul><p><b>Sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div class="half_rhythm">The risk to the sibs of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> depends on the genetic status of the parents.</div></li><li class="half_rhythm"><div class="half_rhythm">If the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> identified in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> is not identified in a parent, the risk to sibs is slightly greater than that of the general population (though still &#x0003c;1%) because of the theoretic possibility of parental <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a> for the deletion.</div></li><li class="half_rhythm"><div class="half_rhythm">If one of the parents has the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>, the risk to each sib of inheriting the deletion is 50%. However, it is not possible to reliably predict the expressivity of 16p12.2 recurrent deletion in sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</div><div class="half_rhythm">The presence of neuropsychiatric phenotypes in a parent with the <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> may indicate higher <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> within a family (see <a href="#mdel16p12_2.Penetrance">Penetrance</a>).</div></li></ul><p><b>Offspring of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>Offspring of an individual with a 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> have a 50% chance of inheriting the deletion.</div></li><li class="half_rhythm"><div>The risk to offspring of being <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> is less than 50% because of reduced <a class="def" href="/books/n/gene/glossary/def-item/penetrance/">penetrance</a> (see <a href="#mdel16p12_2.Penetrance">Penetrance</a>).</div></li></ul><p><b>Other family members.</b> The risk to other family members depends on the genetic status of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s parents: if a parent has the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>, his or her family members may also have the deletion.</p></div><div id="mdel16p12_2.Related_Genetic_Counseling_I"><h3>Related Genetic Counseling Issues</h3><p>See Management, <a href="#mdel16p12_2.Evaluation_of_Relatives_at_R">Evaluation of Relatives at Risk</a> for information on evaluating at-risk relatives for the purpose of early diagnosis and treatment.</p><p><b>Family planning</b></p><ul><li class="half_rhythm"><div>The optimal time for determination of genetic risk and discussion of the availability of prenatal testing is before pregnancy. Similarly, decisions about testing to determine the genetic status of at-risk asymptomatic family members are best made before pregnancy.</div></li><li class="half_rhythm"><div>It is appropriate to offer <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> (including discussion of potential risks to offspring and reproductive options) to young adults who are at risk of having a child with the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>.</div></li></ul><p><b>DNA banking</b> is the storage of DNA (typically extracted from white blood cells) for possible future use. Because it is likely that testing methodology and our understanding of genes, allelic variants, and diseases will improve in the future, consideration should be given to banking DNA of <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals.</p></div><div id="mdel16p12_2.Prenatal_Testing_and_Preimpl"><h3>Prenatal Testing and Preimplantation Genetic Diagnosis</h3><p><b>Pregnancies known to be at increased risk for the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>.</b> Once the 16p12.2 recurrent deletion has been identified in an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> family member, prenatal testing for a pregnancy at increased risk and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> for 16p12.2 recurrent deletion are possible. Note: It is not possible to reliably predict the expressivity of 16p12.2 recurrent deletion based on the results of prenatal testing. However, a family history of neuropsychiatric and neurodevelopmental features is reported to correlate with increased severity in the manifestation of the deletion.</p><p>Prenatal testing using targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> analysis that will detect the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> found in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> may be offered when:</p><ul><li class="half_rhythm"><div>A parent has the <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>; or</div></li><li class="half_rhythm"><div>The parents do not have the <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> but have had a child with the 16p12.2 recurrent deletion. In this instance, the <a class="def" href="/books/n/gene/glossary/def-item/recurrence-risk/">recurrence risk</a> associated with the possibility of parental <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a> or other predisposing genetic mechanisms is probably slightly greater than that of the general population (though still &#x0003c;1%).</div></li></ul><p>Differences in perspective may exist among medical professionals and within families regarding the use of prenatal testing, particularly if the testing is being considered for the purpose of pregnancy termination rather than early diagnosis. While most centers would consider decisions regarding prenatal testing to be the choice of the parents, discussion of these issues is appropriate.</p><p><b>Pregnancies not known to be at increased risk for the 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>.</b> CMA performed in a pregnancy not known to be at increased risk may detect the recurrent deletion.</p><p>Note: Regardless of whether a pregnancy is known or not known to be at increased risk for 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a>, prenatal test results cannot reliably predict the <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>.</p></div></div><div id="mdel16p12_2.Resources"><h2 id="_mdel16p12_2_Resources_">Resources</h2><p>
<i>GeneReviews staff has selected the following disease-specific and/or umbrella
support organizations and/or registries for the benefit of individuals with this disorder
and their families. GeneReviews is not responsible for the information provided by other
organizations. For information on selection criteria, click <a href="/books/n/gene/app4/">here</a>.</i></p><ul><li class="half_rhythm"><div><b>National Library of Medicine Genetics Home Reference</b></div><div><a href="https://ghr.nlm.nih.gov/condition/16p122-microdeletion" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">16p12.2 microdeletion</a></div></li><li class="half_rhythm"><div><b>Unique: The Rare Chromosome Disorder Support Group</b></div><div>G1 The Stables</div><div>Station Road West</div><div>Oxted Surrey RH8 9EE</div><div>United Kingdom</div><div><b>Phone:</b> +44 (0) 1883 723356</div><div><b>Email:</b> info@rarechromo.org; rarechromo@aol.com</div><div><a href="http://www.rarechromo.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.rarechromo.org</a></div></li></ul></div><div id="mdel16p12_2.Molecular_Genetics"><h2 id="_mdel16p12_2_Molecular_Genetics_">Molecular Genetics</h2><p><i>Information in the Molecular Genetics and OMIM tables may differ from that elsewhere in the GeneReview: tables may contain more recent information. &#x02014;</i>ED.</p><div id="mdel16p12_2.molgen.TA" class="table"><h3><span class="label">Table A.</span></h3><div class="caption"><p>16p12.2 Recurrent Deletion: Genes and Databases</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK274565/table/mdel16p12_2.molgen.TA/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__mdel16p12_2.molgen.TA_lrgtbl__"><table class="no_bottom_margin"><tbody><tr><th id="hd_b_mdel16p12_2.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;">Gene</th><th id="hd_b_mdel16p12_2.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;">Chromosome Locus</th><th id="hd_b_mdel16p12_2.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;">Protein</th><th id="hd_b_mdel16p12_2.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;">ClinVar</th></tr><tr><td headers="hd_b_mdel16p12_2.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><i>Not applicable</i></td><td headers="hd_b_mdel16p12_2.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;">16p12.2</td><td headers="hd_b_mdel16p12_2.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;">Not applicable</td><td headers="hd_b_mdel16p12_2.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div id="mdel16p12_2.TFA.1"><p class="no_margin">Data are compiled from the following standard references: <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> from
<a href="http://www.genenames.org/index.html" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HGNC</a>;
<a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> from
<a href="http://www.omim.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">OMIM</a>;
protein from <a href="http://www.uniprot.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">UniProt</a>.
For a description of databases (Locus Specific, HGMD, ClinVar) to which links are provided, click
<a href="/books/n/gene/app1/">here</a>.</p></div></dd></dl></div></div></div><div id="mdel16p12_2.molgen.TB" class="table"><h3><span class="label">Table B.</span></h3><div class="caption"><p>OMIM Entries for 16p12.2 Recurrent Deletion (<a href="/omim/136570" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">View All in OMIM</a>) </p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK274565/table/mdel16p12_2.molgen.TB/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__mdel16p12_2.molgen.TB_lrgtbl__"><table><tbody><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/136570" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">136570</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">CHROMOSOME 16p12.1 DELETION SYNDROME, 520-KB</td></tr></tbody></table></div></div><div id="mdel16p12_2.Molecular_Pathogenesis"><h3>Molecular Pathogenesis</h3><p>The 520-kb <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> at <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> 16p12.2 (coordinates ~21,948,445-~22,430,805, genome build <a href="http://genome.ucsc.edu/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">UCSC</a> hg19/GRCh37) results in the deletion of seven genes: <i>UQCRC2</i>, <i>PDZD9</i>, <i>MOSMO</i>, <i>VWA3A</i>, <i>EEF2K</i>, <i>POLR3E</i>, and <i>CDR2</i> (RefSeq genes, UCSC hg19/GRCh37). While the pericentromeric region of chromosome 16 has many segmental duplications that can mediate recurrent deletions, this 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> specifically refers to a deletion with these specific coordinates.</p><p>The 16p12.2 <a class="def" href="/books/n/gene/glossary/def-item/recurrent-deletion/">recurrent deletion</a> is mediated by <a class="def" href="/books/n/gene/glossary/def-item/recombination/">recombination</a> between flanking 68-kb low-copy segmental duplications with 99.5% sequence identity [<a class="bk_pop" href="#mdel16p12_2.REF.antonacci.2010.745">Antonacci et al 2010</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>] that contains six additional genes. Based on the number and orientation of segmental duplications, two common haplotypes have been identified in the general population [<a class="bk_pop" href="#mdel16p12_2.REF.antonacci.2010.745">Antonacci et al 2010</a>]. Haplotype S2 has the 68-kb segmental duplications in direct orientation flanking the 16p12.2 deletion region, suggesting that it would predispose to the recurrent 16p12.2 deletion. Consistent with this hypothesis, the S2 haplotype is enriched among individuals with a 16p12.2 recurrent deletion. The population frequency of the S2 haplotype varies by ancestry:</p><ul><li class="half_rhythm"><div>African (97.5%)</div></li><li class="half_rhythm"><div>European (83.1%)</div></li><li class="half_rhythm"><div>Asian (71.6%)</div></li></ul><p>Therefore, African and European populations likely have a higher risk for 16p12.2 recurrent deletions than Asian populations [<a class="bk_pop" href="#mdel16p12_2.REF.antonacci.2010.745">Antonacci et al 2010</a>].</p><p><b>Genes of interest in this region.</b> The underlying mechanism for how the heterozygosity of one or more of the seven genes in 16p12.2 results in disease is unknown; however, these genes have been studied to a variable extent:</p><ul><li class="half_rhythm"><div><i>UQCRC2</i> is a fundamental protein for the assembly of the mitochondrial respiratory chain complex.</div></li><li class="half_rhythm"><div><i>CDR2</i> is an onco-neural antigen and has been associated with cerebellar ataxia.</div></li><li class="half_rhythm"><div><i>POLR3E</i> is a component of the RNA polymerase that synthesizes small RNAs.</div></li><li class="half_rhythm"><div><i>EEF2K</i> is a kinase involved in the regulation of protein synthesis elongation and has been recently associated with learning and memory, synaptic plasticity, and the short-term antidepressant action of ketamine. <i>eef2k</i>-knockout mice have been described to have defects in the reproductive system, and those carrying a <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a>-<a class="def" href="/books/n/gene/glossary/def-item/inactivating/">inactivating</a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> leading to 0.5% residual activity showed learning memory deficits and altered brain activity.</div></li><li class="half_rhythm"><div><i>MOSMO</i> is a negative regulator of hedgehog signaling.</div></li></ul><p>Functional information on <i>PDZD9</i> and <i>VWA3A</i> is limited.</p></div></div><div id="mdel16p12_2.References"><h2 id="_mdel16p12_2_References_">References</h2><div id="mdel16p12_2.Literature_Cited"><h3>Literature Cited</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.antonacci.2010.745">Antonacci F, Kidd JM, Marques-Bonet T, Teague B, Ventura M, Girirajan S, Alkan C, Campbell CD, Vives L, Malig M, Rosenfeld JA, Ballif BC, Shaffer LG, Graves TA, Wilson RK, Schwartz DC, Eichler EE. A large and complex structural polymorphism at 16p12.1 underlies microdeletion disease risk. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:745–50.</span> [<a href="/pmc/articles/PMC2930074/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2930074</span></a>] [<a href="/pubmed/20729854" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20729854</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.ballif.2007.1071">Ballif BC, Hornor SA, Jenkins E, Madan-Khetarpal S, Surti U, Jackson KE, Asamoah A, Brock PL, Gowans GC, Conway RL, Graham JM Jr, Medne L, Zackai EH, Shaikh TH, Geoghegan J, Selzer RR, Eis PS, Bejjani BA, Shaffer LG. Discovery of a previously unrecognized microdeletion syndrome of 16p11.2-p12.2. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1071–3.</span> [<a href="/pubmed/17704777" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17704777</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.battaglia.2009.1200">Battaglia A, Novelli A, Bernardini L, Igliozzi R, Parrini B. Further characterization of the new microdeletion syndrome of 16p11.2-p12.2. <span><span class="ref-journal">Am J Med Genet A. </span>2009;<span class="ref-vol">149A</span>:1200–4.</span> [<a href="/pubmed/19449418" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19449418</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.cooper.2011.838">Cooper GM, Coe BP, Girirajan S, Rosenfeld JA, Vu TH, Baker C, Williams C, Stalker H, Hamid R, Hannig V, Abdel-Hamid H, Bader P, McCracken E, Niyazov D, Leppig K, Thiese H, Hummel M, Alexander N, Gorski J, Kussmann J, Shashi V, Johnson K, Rehder C, Ballif BC, Shaffer LG, Eichler EE. A copy number variation morbidity map of developmental delay. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:838–46.</span> [<a href="/pmc/articles/PMC3171215/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3171215</span></a>] [<a href="/pubmed/21841781" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 21841781</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.dalessandro.2014.83">D'Alessandro LC, Werner P, Xie HM, Hakonarson H, White PS, Goldmuntz E. The prevalence of 16p12.1 microdeletion in patients with left-sided cardiac lesions. <span><span class="ref-journal">Congenit Heart Dis. </span>2014;<span class="ref-vol">9</span>:83–6.</span> [<a href="/pmc/articles/PMC4575124/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4575124</span></a>] [<a href="/pubmed/23682798" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23682798</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.de_jong.2010.1265">de Jong EM, Douben H, Eussen BH, Felix JF, Wessels MW, Poddighe PJ, Nikkels PG, de Krijger RR, Tibboel D, de Klein A. 5q11.2 deletion in a patient with tracheal agenesis. <span><span class="ref-journal">Eur J Hum Genet. </span>2010;<span class="ref-vol">18</span>:1265–8.</span> [<a href="/pmc/articles/PMC2987473/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2987473</span></a>] [<a href="/pubmed/20551993" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20551993</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.girirajan.2005.820">Girirajan S, Elsas LJ 2nd, Devriendt K, Elsea SH. RAI1 variations in Smith-Magenis syndrome patients without 17p11.2 deletions. <span><span class="ref-journal">J Med Genet. </span>2005;<span class="ref-vol">42</span>:820–8.</span> [<a href="/pmc/articles/PMC1735950/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1735950</span></a>] [<a href="/pubmed/15788730" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15788730</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.girirajan.2009a.247">Girirajan S, Elsea SH. Abnormal maternal behavior, altered sociability, and impaired serotonin metabolism in Rai1-transgenic mice. <span><span class="ref-journal">Mamm Genome. </span>2009a;<span class="ref-vol">20</span>:247–55.</span> [<a href="/pubmed/19319603" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19319603</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.girirajan.2009b.224">Girirajan S, Elsea SH. Distorted Mendelian transmission as a function of genetic background in Rai1-haploinsufficient mice. <span><span class="ref-journal">Eur J Med Genet. </span>2009b;<span class="ref-vol">52</span>:224–8.</span> [<a href="/pubmed/19116176" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19116176</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.girirajan.2008.941">Girirajan S, Patel N, Slager RE, Tokarz ME, Bucan M, Wiley JL, Elsea SH. How much is too much? Phenotypic consequences of Rai1 overexpression in mice. <span><span class="ref-journal">Eur J Hum Genet. </span>2008;<span class="ref-vol">16</span>:941–54.</span> [<a href="/pubmed/18285828" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 18285828</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.girirajan.2012.1321">Girirajan S, Rosenfeld JA, Coe BP, Parikh S, Friedman N, Goldstein A, Filipink RA, McConnell JS, Angle B, Meschino WS, Nezarati MM, Asamoah A, Jackson KE, Gowans GC, Martin JA, Carmany EP, Stockton DW, Schnur RE, Penney LS, Martin DM, Raskin S, Leppig K, Thiese H, Smith R, Aberg E, Niyazov DM, Escobar LF, El-Khechen D, Johnson KD, Lebel RR, Siefkas K, Ball S, Shur N, McGuire M, Brasington CK, Spence JE, Martin LS, Clericuzio C, Ballif BC, Shaffer LG, Eichler EE. Phenotypic heterogeneity of genomic disorders and rare copy-number variants. <span><span class="ref-journal">N Engl J Med. </span>2012;<span class="ref-vol">367</span>:1321–31.</span> [<a href="/pmc/articles/PMC3494411/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3494411</span></a>] [<a href="/pubmed/22970919" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 22970919</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.girirajan.2010.203">Girirajan S, Rosenfeld JA, Cooper GM, Antonacci F, Siswara P, Itsara A, Vives L, Walsh T, McCarthy SE, Baker C, Mefford HC, Kidd JM, Browning SR, Browning BL, Dickel DE, Levy DL, Ballif BC, Platky K, Farber DM, Gowans GC, Wetherbee JJ, Asamoah A, Weaver DD, Mark PR, Dickerson J, Garg BP, Ellingwood SA, Smith R, Banks VC, Smith W, McDonald MT, Hoo JJ, French BN, Hudson C, Johnson JP, Ozmore JR, Moeschler JB, Surti U, Escobar LF, El-Khechen D, Gorski JL, Kussmann J, Salbert B, Lacassie Y, Biser A, McDonald-McGinn DM, Zackai EH, Deardorff MA, Shaikh TH, Haan E, Friend KL, Fichera M, Romano C, Gecz J, Delisi LE, Sebat J, King MC, Shaffer LG, Eichler EE. A recurrent 16p12.1 microdeletion supports a two-hit model for severe developmental delay. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:203–9.</span> [<a href="/pmc/articles/PMC2847896/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2847896</span></a>] [<a href="/pubmed/20154674" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20154674</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.girirajan.2006.417">Girirajan S, Vlangos CN, Szomju BB, Edelman E, Trevors CD, Dupuis L, Nezarati M, Bunyan DJ, Elsea SH. Genotype-phenotype correlation in Smith-Magenis syndrome: evidence that multiple genes in 17p11.2 contribute to the clinical spectrum. <span><span class="ref-journal">Genet Med. </span>2006;<span class="ref-vol">8</span>:417–27.</span> [<a href="/pubmed/16845274" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 16845274</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.hempel.2009.2106">Hempel M, Rivera Brugues N, Wagenstaller J, Lederer G, Weitensteiner A, Seidel H, Meitinger T, Strom TM. Microdeletion syndrome 16p11.2-p12.2: clinical and molecular characterization. <span><span class="ref-journal">Am J Med Genet A. </span>2009;<span class="ref-vol">149A</span>:2106–12.</span> [<a href="/pubmed/19676056" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19676056</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.jacquemont.2014.415">Jacquemont S, Coe BP, Hersch M, Duyzend MH, Krumm N, Bergmann S, Beckmann JS, Rosenfeld JA, Eichler EE. A higher mutational burden in females supports a "female protective model" in neurodevelopmental disorders. <span><span class="ref-journal">Am J Hum Genet. </span>2014;<span class="ref-vol">94</span>:415–25.</span> [<a href="/pmc/articles/PMC3951938/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3951938</span></a>] [<a href="/pubmed/24581740" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24581740</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.kirov.2014.378">Kirov G, Rees E, Walters JT, Escott-Price V, Georgieva L, Richards AL, Chambert KD, Davies G, Legge SE, Moran JL, McCarroll SA, O'Donovan MC, Owen MJ. The penetrance of copy number variations for schizophrenia and developmental delay. <span><span class="ref-journal">Biol Psychiatry. </span>2014;<span class="ref-vol">75</span>:378–85.</span> [<a href="/pmc/articles/PMC4229045/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4229045</span></a>] [<a href="/pubmed/23992924" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23992924</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.okamoto.2014.213">Okamoto N, Fujii T, Tanaka J, Saito K, Matsui T, Harada N. A clinical study of patients with pericentromeric deletion and duplication within 16p12.2-p11.2. <span><span class="ref-journal">Am J Med Genet A. </span>2014;<span class="ref-vol">164A</span>:213–9.</span> [<a href="/pubmed/24259393" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24259393</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.pizzo.2018">Pizzo L, Jensen M, Polyak A, Rosenfeld JA, Mannik K, Krishnan A, McCready E, Pichon O, Le Caignec C, Van Dijck A, Pope K, Voorhoeve E, Yoon J, Stankiewicz P, Cheung SW, Pazuchanics D, Huber E, Kumar V, Kember RL, Mari F, Curr&#x000f3; A, Castiglia L, Galesi O, Avola E, Mattina T, Fichera M, Mandar&#x000e0; L, Vincent M, Nizon M, Mercier S, B&#x000e9;n&#x000e9;teau C, Blesson S, Martin-Coignard D, Mosca-Boidron AL, Caberg JH, Bucan M, Zeesman S, Nowaczyk MJM, Lefebvre M, Faivre L, Callier P, Skinner C, Keren B, Perrine C, Prontera P, Marle N, Renieri A, Reymond A, Kooy F, Isidor B, Schwartz C, Romano C, Sistermans E, Amor DJ, Andrieux J, Girirajan S. Rare variants in the genetic background modulate cognitive and developmental phenotypes in individuals carrying disease-associated variants. <span><span class="ref-journal">Genet Med. </span>2018.</span> In press. [<a href="/pmc/articles/PMC6405313/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC6405313</span></a>] [<a href="/pubmed/30190612" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 30190612</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.rai.2015.394">Rai B, Sharif F. Cervicomedullary spinal stenosis and ventriculomegaly in a child with developmental delay due to chromosome 16p12.1 microdeletion syndrome. <span><span class="ref-journal">J Child Neurol. </span>2015;<span class="ref-vol">30</span>:394–6.</span> [<a href="/pubmed/24813870" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24813870</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.rees.2014.1669">Rees E, Walters JT, Chambert KD, O'Dushlaine C, Szatkiewicz J, Richards AL, Georgieva L, Mahoney-Davies G, Legge SE, Moran JL, Genovese G, Levinson D, Morris DW, Cormican P, Kendler KS, O'Neill FA, Riley B, Gill M, Corvin A, Sklar P, Hultman C, Pato C, Pato M, Sullivan PF, Gejman PV, McCarroll SA, O'Donovan MC, Owen MJ, Kirov G. CNV analysis in a large schizophrenia sample implicates deletions at 16p12.1 and SLC1A1 and duplications at 1p36.33 and CGNL1. <span><span class="ref-journal">Hum Mol Genet. </span>2014;<span class="ref-vol">23</span>:1669–76.</span> [<a href="/pmc/articles/PMC3929090/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3929090</span></a>] [<a href="/pubmed/24163246" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24163246</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.rosenfeld.2013.478">Rosenfeld JA, Coe BP, Eichler EE, Cuckle H, Shaffer LG. Estimates of penetrance for recurrent pathogenic copy-number variations. <span><span class="ref-journal">Genet Med. </span>2013;<span class="ref-vol">15</span>:478–81.</span> [<a href="/pmc/articles/PMC3664238/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3664238</span></a>] [<a href="/pubmed/23258348" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23258348</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="mdel16p12_2.REF.stefansson.2014.361">Stefansson H, Meyer-Lindenberg A, Steinberg S, Magnusdottir B, Morgen K, Arnarsdottir S, Bjornsdottir G, Walters GB, Jonsdottir GA, Doyle OM, Tost H, Grimm O, Kristjansdottir S, Snorrason H, Davidsdottir SR, Gudmundsson LJ, Jonsson GF, Stefansdottir B, Helgadottir I, Haraldsson M, Jonsdottir B, Thygesen JH, Schwarz AJ, Didriksen M, Stensb&#x000f8;l TB, Brammer M, Kapur S, Halldorsson JG, Hreidarsson S, Saemundsen E, Sigurdsson E, Stefansson K. CNVs conferring risk of autism or schizophrenia affect cognition in controls. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">505</span>:361–6.</span> [<a href="/pubmed/24352232" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24352232</span></a>]</div></li></ul></div></div><div id="mdel16p12_2.Chapter_Notes"><h2 id="_mdel16p12_2_Chapter_Notes_">Chapter Notes</h2><div id="mdel16p12_2.Author_Notes"><h3>Author Notes</h3><p><a href="https://autism.bx.psu.edu/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Santhosh Girirajan's Laboratory</a></p><p>I am interested in understanding the causes and consequences of genome structure and function as related to human neurodevelopmental disorders such as intellectual disability and developmental delay, autism, schizophrenia, and epilepsy. My graduate work on Smith-Magenis syndrome (SMS) trained me in understanding how all individuals carrying a <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> encompassing the retinoic acid induced 1 <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> (<i>RAI1</i>) have invariable phenotypes [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2005.820">Girirajan et al 2005</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2006.417">Girirajan et al 2006</a>]. Further, analysis of mouse models altering <i>Rai1</i> gene dosage confirmed the role of <i>RAI1</i> in SMS [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2008.941">Girirajan et al 2008</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2009a.247">Girirajan &#x00026; Elsea 2009a</a>, <a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2009b.224">Girirajan &#x00026; Elsea 2009b</a>]. As a postdoctoral fellow in Evan Eichler's lab, I was involved in identifying and studying another class of CNVs where the causal gene is not known and individuals carrying the same deletion or <a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a> manifest a wide variety of phenotypes. I was able to demonstrate that a second variant in addition to the primary <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> lesion can explain the observed phenotypic variability. Originally described for 16p12.1 deletion [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2010.203">Girirajan et al 2010</a>], the "two-hit" model was successfully replicated to explain the phenotypic heterogeneity in other recently identified CNVs such as 16p11.2 deletion, 15q13.3 deletion,17q12 deletion, 15q11.2 deletion, and 1q21.1 deletion [<a class="bk_pop" href="#mdel16p12_2.REF.girirajan.2012.1321">Girirajan et al 2012</a>].</p></div><div id="mdel16p12_2.Revision_History"><h3>Revision History</h3><ul><li class="half_rhythm"><div>13 September 2018 (sw) Comprehensive update posted live</div></li><li class="half_rhythm"><div>26 February 2015 (me) Review posted live</div></li><li class="half_rhythm"><div>9 September 2014 (sg) Original submission</div></li></ul></div></div><div id="bk_toc_contnr"></div></div></div>
            <div class="post-content"><div><div class="half_rhythm"><a href="/books/about/copyright/">Copyright</a> © 1993-2019, University of Washington, Seattle. GeneReviews is
a registered trademark of the University of Washington, Seattle. All rights
reserved.<p class="small">GeneReviews® chapters are owned by the University of Washington. Permission is
hereby granted to reproduce, distribute, and translate copies of content materials for
noncommercial research purposes only, provided that (i) credit for source (<a href="http://www.genereviews.org/" ref="pagearea=meta&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">http://www.genereviews.org/</a>) and copyright (© 1993-2019 University of
Washington) are included with each copy; (ii) a link to the original material is provided
whenever the material is published elsewhere on the Web; and (iii) reproducers,
distributors, and/or translators comply with the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>. No further modifications are allowed. For clarity, excerpts
of GeneReviews chapters for use in lab reports and clinic notes are a permitted
use.</p><p class="small">For more information, see the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>.</p><p class="small">For questions regarding permissions or whether a specified use is allowed,
contact: <a href="mailto:dev@null" data-email="ude.wu@tssamda" class="oemail">ude.wu@tssamda</a>.</p></div><div class="small"><span class="label">Bookshelf ID: NBK274565</span><span class="label">PMID: <a href="/pubmed/25719193" title="PubMed record of this page" ref="pagearea=meta&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">25719193</a></span></div><div style="margin-top:2em" class="bk_noprnt"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/del16p11_2/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/dup17q12/" title="Next page in this title">Next &gt;</a></div></div></div></div>
            
        </div>

        <!-- Custom content below content -->
        <div class="col4">
            
        </div>
        
        
        <!-- Book content -->
        
        <!-- Custom contetnt below bottom nav -->
        <div class="col5">
            
        </div>
    </div>

    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            <div id="ncbi_share_book"><a href="#" class="ncbi_share" data-ncbi_share_config="popup:false,shorten:true" ref="id=NBK274565&amp;db=books">Share</a></div>

        </div>
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Views</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PDF_download" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/NBK274565/?report=reader">PubReader</a></li><li><a href="/books/NBK274565/?report=printable">Print View</a></li><li><a data-jig="ncbidialog" href="#_ncbi_dlg_citbx_NBK274565" data-jigconfig="width:400,modal:true">Cite this Page</a><div id="_ncbi_dlg_citbx_NBK274565" style="display:none" title="Cite this Page"><div class="bk_tt">Girirajan S, Pizzo L, Moeschler J, et al. 16p12.2 Recurrent Deletion. 2015 Feb 26 [Updated 2018 Sep 13]. In: Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. <span class="bk_cite_avail"></span></div></div></li><li><a href="/books/NBK274565/pdf/Bookshelf_NBK274565.pdf">PDF version of this page</a> (193K)</li><li><a href="#" class="toggle-glossary-link" title="Enable/disable links to the glossary">Disable Glossary Links</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>In this GeneReview</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="page-toc" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="#mdel16p12_2.Summary" ref="log$=inpage&amp;link_id=inpage">Summary</a></li><li><a href="#mdel16p12_2.Diagnosis" ref="log$=inpage&amp;link_id=inpage">Diagnosis</a></li><li><a href="#mdel16p12_2.Clinical_Characteristics" ref="log$=inpage&amp;link_id=inpage">Clinical Characteristics</a></li><li><a href="#mdel16p12_2.Genetically_Related_Allelic" ref="log$=inpage&amp;link_id=inpage">Genetically Related (Allelic) Disorders</a></li><li><a href="#mdel16p12_2.Differential_Diagnosis" ref="log$=inpage&amp;link_id=inpage">Differential Diagnosis</a></li><li><a href="#mdel16p12_2.Management" ref="log$=inpage&amp;link_id=inpage">Management</a></li><li><a href="#mdel16p12_2.Genetic_Counseling" ref="log$=inpage&amp;link_id=inpage">Genetic Counseling</a></li><li><a href="#mdel16p12_2.Resources" ref="log$=inpage&amp;link_id=inpage">Resources</a></li><li><a href="#mdel16p12_2.Molecular_Genetics" ref="log$=inpage&amp;link_id=inpage">Molecular Genetics</a></li><li><a href="#mdel16p12_2.References" ref="log$=inpage&amp;link_id=inpage">References</a></li><li><a href="#mdel16p12_2.Chapter_Notes" ref="log$=inpage&amp;link_id=inpage">Chapter Notes</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>GeneReviews Links</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="source-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/n/gene/advanced/"><i>GeneReviews</i> Advanced Search</a></li><li><a href="/books/n/gene/glossary/"><i>GeneReviews</i> Glossary</a></li><li><a href="/books/n/gene/resource_mats/">Resource Materials</a> <span class="bk_hlight1">NEW FEATURE</span></li><li><a href="/books/n/gene/updates/">New in <i>GeneReviews</i></a></li><li><a href="/books/n/gene/authors/">Author List</a></li><li><a href="/books/n/gene/prospective_authors/">For Current/Prospective Authors</a></li><li><a href="/books/n/gene/GRpersonnel/"><i>GeneReviews</i> Personnel</a></li><li><a href="/books/n/gene/howto_linkin/">Download/Link to <i>GeneReviews</i></a></li><li><a href="/books/n/gene/contact_us/">Contact Us</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Related information</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="discovery_db_links" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=omim&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_omim&amp;IdsFromResult=3396338" ref="log$=recordlinks">OMIM</a><div class="brieflinkpop offscreen_noflow">Related OMIM records</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pmc&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pmc_refs&amp;IdsFromResult=3396338" ref="log$=recordlinks">PMC</a><div class="brieflinkpop offscreen_noflow">PubMed Central citations</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pubmed&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pubmed_refs&amp;IdsFromResult=3396338" ref="log$=recordlinks">PubMed</a><div class="brieflinkpop offscreen_noflow">Links to PubMed</div></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301696" ref="ordinalpos=1&amp;linkpos=1&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 22q11.2 Deletion Syndrome</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 22q11.2 Deletion Syndrome<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">McDonald-McGinn DM, Emanuel BS, Zackai EH. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/27929632" ref="ordinalpos=1&amp;linkpos=2&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 17q12 Recurrent Deletion Syndrome</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 17q12 Recurrent Deletion Syndrome<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Mitchel MW, Moreno-De-Luca D, Myers SM, Finucane B, Ledbetter DH, Martin CL. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301775" ref="ordinalpos=1&amp;linkpos=3&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 16p11.2 Recurrent Microdeletion</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 16p11.2 Recurrent Microdeletion<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Miller DT, Chung W, Nasir R, Shen Y, Steinman KJ, Wu BL, Hanson E. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/27656750" ref="ordinalpos=1&amp;linkpos=4&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 3q29 Recurrent Deletion</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 3q29 Recurrent Deletion<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Mulle JG, Gambello MJ, Cook EH, Rutkowski TP, Glassford M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301337" ref="ordinalpos=1&amp;linkpos=5&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 2q37 Microdeletion Syndrome</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> 2q37 Microdeletion Syndrome<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Doherty ES, Lacbawan FL. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li></ul><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=25719193" ref="ordinalpos=1&amp;log$=relatedreviews_seeall&amp;logdbfrom=pubmed">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=25719193" ref="ordinalpos=1&amp;log$=relatedarticles_seeall&amp;logdbfrom=pubmed">See all...</a></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Recent Activity</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="Shutter"></a></div><div class="portlet_content"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class=""><div class="action"><a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a></div><ul id="activity"><li class="ra_rcd ralinkpopper two_line"><a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5e0412ee75f66d18aafd9b60">16p12.2 Recurrent Deletion - GeneReviews®</a><div class="ralinkpop offscreen_noflow">16p12.2 Recurrent Deletion - GeneReviews®<div class="brieflinkpopdesc"></div></div><div class="tertiary"></div></li></ul><p class="HTOn">Your browsing activity is empty.</p><p class="HTOff">Activity recording is turned off.</p><p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p><a class="seemore" href="/sites/myncbi/recentactivity">See more...</a></div></div></div>
        
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script type="text/javascript" src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script type="text/javascript">
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-12-25T20:54:54-05:00&amp;Snapshot=%2Fprojects%2Fbooks%2FPBooks@5.22&amp;Host=portal103&amp;ncbi_phid=CE8A37F7E04089E1000000000456019B&amp;ncbi_session=CE8A37F7E0412ED1_1110SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK274565%2F&amp;Db=pmc&amp;folderID=134&amp;Ncbi_App=bookshelf&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        </div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=books&amp;ncbi_pdid=book-part&amp;ncbi_acc=NBK274565&amp;ncbi_domain=gene&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/NBK274565/&amp;ncbi_pagename=16p12.2 Recurrent Deletion - GeneReviews® - NCBI Bookshelf&amp;ncbi_bookparttype=chapter&amp;ncbi_app=bookshelf" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8A37F7E0412ED1_1110SID /projects/books/PBooks@5.22 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/js/3879255/4121861/3501987/4008961/3893018/3821238/4062932/4123007/4087685/4076480/3921943/3400083/3426610.js" snapshot="books"></script></body>
</html>