<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    
    <head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <!-- AppResources meta begin -->
        <meta name="paf-app-resources" content="" />
                 <script type="text/javascript">var ncbi_startTime = new Date();</script>
            
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="" />

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="books" /><meta name="ncbi_pdid" content="book-part" /><meta name="ncbi_acc" content="NBK242617" /><meta name="ncbi_domain" content="gene" /><meta name="ncbi_report" content="record" /><meta name="ncbi_type" content="fulltext" /><meta name="ncbi_objectid" content="" /><meta name="ncbi_pcid" content="/NBK242617/" /><meta name="ncbi_pagename" content="Perrault Syndrome - GeneReviews® - NCBI Bookshelf" /><meta name="ncbi_bookparttype" content="chapter" /><meta name="ncbi_app" content="bookshelf" />
        <!-- Logger end -->
        
        <title>Perrault Syndrome - GeneReviews® - NCBI Bookshelf</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css" /><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,FOLLOW,NOARCHIVE" /><meta name="citation_inbook_title" content="GeneReviews® [Internet]" /><meta name="citation_title" content="Perrault Syndrome" /><meta name="citation_publisher" content="University of Washington, Seattle" /><meta name="citation_date" content="2018/09/06" /><meta name="citation_author" content="William G Newman" /><meta name="citation_author" content="Thomas B Friedman" /><meta name="citation_author" content="Gerard S Conway" /><meta name="citation_author" content="Leigh AM Demain" /><meta name="citation_pmid" content="25254289" /><meta name="citation_fulltext_html_url" content="https://www.ncbi.nlm.nih.gov/books/NBK242617/" /><meta name="citation_keywords" content="ATP-dependent Clp protease proteolytic subunit, mitochondrial" /><meta name="citation_keywords" content="GTPase Era, mitochondrial" /><meta name="citation_keywords" content="Peroxisomal multifunctional enzyme type 2" /><meta name="citation_keywords" content="Probable histidine--tRNA ligase, mitochondrial" /><meta name="citation_keywords" content="Probable leucine--tRNA ligase, mitochondrial" /><meta name="citation_keywords" content="Twinkle protein, mitochondrial" /><meta name="citation_keywords" content="CLPP" /><meta name="citation_keywords" content="ERAL1" /><meta name="citation_keywords" content="HARS2" /><meta name="citation_keywords" content="HSD17B4" /><meta name="citation_keywords" content="LARS2" /><meta name="citation_keywords" content="TWNK" /><meta name="citation_keywords" content="Perrault Syndrome" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="DC.Title" content="Perrault Syndrome" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="University of Washington, Seattle" /><meta name="DC.Contributor" content="William G Newman" /><meta name="DC.Contributor" content="Thomas B Friedman" /><meta name="DC.Contributor" content="Gerard S Conway" /><meta name="DC.Contributor" content="Leigh AM Demain" /><meta name="DC.Date" content="2018/09/06" /><meta name="DC.Identifier" content="https://www.ncbi.nlm.nih.gov/books/NBK242617/" /><meta name="description" content="Perrault syndrome is characterized by sensorineural hearing loss (SNHL) in males and females and ovarian dysfunction in females. SNHL is bilateral and ranges from profound with prelingual (congenital) onset to moderate with early-childhood onset. When onset is in early childhood, hearing loss can be progressive. Ovarian dysfunction ranges from gonadal dysgenesis (absent or streak gonads) manifesting as primary amenorrhea to primary ovarian insufficiency (POI) defined as cessation of menses before age 40 years. Fertility in affected males is reported as normal (although the number of reported males is limited). Neurologic features described in some individuals with Perrault syndrome include learning difficulties and developmental delay, cerebellar ataxia, and motor and sensory peripheral neuropathy." /><meta name="og:title" content="Perrault Syndrome" /><meta name="og:type" content="book" /><meta name="og:description" content="Perrault syndrome is characterized by sensorineural hearing loss (SNHL) in males and females and ovarian dysfunction in females. SNHL is bilateral and ranges from profound with prelingual (congenital) onset to moderate with early-childhood onset. When onset is in early childhood, hearing loss can be progressive. Ovarian dysfunction ranges from gonadal dysgenesis (absent or streak gonads) manifesting as primary amenorrhea to primary ovarian insufficiency (POI) defined as cessation of menses before age 40 years. Fertility in affected males is reported as normal (although the number of reported males is limited). Neurologic features described in some individuals with Perrault syndrome include learning difficulties and developmental delay, cerebellar ataxia, and motor and sensory peripheral neuropathy." /><meta name="og:url" content="https://www.ncbi.nlm.nih.gov/books/NBK242617/" /><meta name="og:site_name" content="NCBI Bookshelf" /><meta name="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" /><meta name="twitter:card" content="summary" /><meta name="twitter:site" content="@ncbibooks" /><meta name="bk-non-canon-loc" content="/books/n/gene/perrault/" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/books/NBK242617/" /><link rel="stylesheet" href="/corehtml/pmc/css/figpopup.css" type="text/css" media="screen" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books_print.min.css" type="text/css" media="print" /><style type="text/css">p a.figpopup{display:inline !important} .bk_tt {font-family: monospace}  .first-line-outdent .bk_ref {display: inline}  .body-content h2, .body-content .h2  {border-bottom: 1px solid #97B0C8} .body-content h2.inline {border-bottom: none} a.page-toc-label , .jig-ncbismoothscroll a {text-decoration:none;border:0 !important} .temp-labeled-list  .graphic {display:inline-block !important} .temp-labeled-list  img{width:100%}</style><script type="text/javascript" src="/corehtml/pmc/js/jquery.hoverIntent.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"> </script><script type="text/javascript">window.name="mainwindow";</script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/book-toc.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/books.min.js"> </script><meta name="book-collection" content="NONE" />

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="CE8CBE8AE040DDA100000000055C01F9.m_12" />
<meta name='referrer' content='origin-when-cross-origin'/><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4098876/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/4008682/3881636/3579733/4062871/12930/3964959/3854974/36029/4128070/9685/3549676/3609192/3609193/3609213/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3411343/3882866.css" media="print" /></head>
    <body class="book-part">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo"><h1 class="res_name"><a href="/books/" title="Bookshelf home">Bookshelf</a></h1><h2 class="res_tagline"></h2></div>
    <div class="search"><form method="get" action="/books/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="books" selected="selected" class="last" data-ac_dict="bookshelf-search">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books" data-ac_dict="bookshelf-search">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search Books. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'bookshelf-search',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'no'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/books/browse/">Browse Titles</a>
                    </li><li>
                        <a href="/books/advanced/">Advanced</a>
                    </li><li class="help">
                        <a href="/books/NBK3833/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <!-- Custom content 1 -->
<div class="col1">
    
</div>

<div class="container">
    <div id="maincontent" class="content eight_col col">
        <!-- Custom content in the left column above book nav -->
        <div class="col2">
            
        </div>
        
        <!-- Book content -->
        

        <!-- Custom content between navigation and content -->
        <div class="col3">
            
        </div>
        
        <div class="document">
            <div class="pre-content"><div><div class="bk_prnt"><p class="small">NCBI Bookshelf. A service of the National Library of Medicine, National Institutes of Health.</p><p>Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. </p></div><div class="iconblock clearfix whole_rhythm no_top_margin bk_noprnt"><a class="img_link icnblk_img" title="All GeneReviews" href="/books/n/gene/"><img class="source-thumb" src="/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" alt="Cover of GeneReviews®" height="100px" width="80px" /></a><div class="icnblk_cntnt eight_col"><h2>GeneReviews<sup>®</sup> [Internet].</h2><a data-jig="ncbitoggler" href="#__NBK242617_dtls__">Show details</a><div style="display:none" class="ui-widget" id="__NBK242617_dtls__"><div>Adam MP, Ardinger HH, Pagon RA, et al., editors.</div><div>Seattle (WA): <a href="http://www.washington.edu" ref="pagearea=page-banner&amp;targetsite=external&amp;targetcat=link&amp;targettype=publisher">University of Washington, Seattle</a>; 1993-2019.</div></div><div class="half_rhythm"><ul class="inline_list"><li style="margin-right:1em"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a></li></ul></div><div class="bk_noprnt"><form method="get" action="/books/n/gene/" id="bk_srch"><div class="bk_search"><label for="bk_term" class="offscreen_noflow">Search term</label><input type="text" title="Search GeneReviews" id="bk_term" name="term" value="" data-jig="ncbiclearbutton" /> <input type="submit" class="jig-ncbibutton" value="Search GeneReviews" submit="false" style="padding: 0.1em 0.4em;" /></div></form><div><ul class="inline_list"><li><a href="/books/n/gene/advanced/">GeneReviews Advanced Search</a></li><li style="margin-left:.5em"><a href="/books/n/gene/helpadvsearch/">Help</a></li></ul></div></div></div><div class="icnblk_cntnt two_col"><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/dmn/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/perry/" title="Next page in this title">Next &gt;</a></div></div></div></div></div>
            <div class="main-content lit-style" itemscope="itemscope" itemtype="http://schema.org/CreativeWork"><div class="meta-content fm-sec"><h1 id="_NBK242617_"><span class="title" itemprop="name">Perrault Syndrome</span></h1><p class="contrib-group"><span itemprop="author">William G Newman</span>, MD, PhD, <span itemprop="author">Thomas B Friedman</span>, PhD, <span itemprop="author">Gerard S Conway</span>, MD, and <span itemprop="author">Leigh AM Demain</span>, PhD.</p><a data-jig="ncbitoggler" href="#__NBK242617_ai__" style="border:0;text-decoration:none">Author Information</a><div style="display:none" class="ui-widget" id="__NBK242617_ai__"><div class="contrib half_rhythm"><span itemprop="author">William G Newman</span>, MD, PhD<div class="affiliation small">Manchester Centre for Genomic Medicine<br />University of Manchester and Manchester University NHS Foundation Trust<br />Manchester, United Kingdom<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ku.ca.retsehcnam@namwen.mailliw" class="oemail">ku.ca.retsehcnam@namwen.mailliw</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Thomas B Friedman</span>, PhD<div class="affiliation small">Laboratory of Molecular Genetics<br />National Institute on Deafness and Other Communication Disorders, NIH<br />John David Porter Neuroscience Research Center, NIH<br />Bethesda, Maryland<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="vog.hin.dcdin@namdeirf" class="oemail">vog.hin.dcdin@namdeirf</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Gerard S Conway</span>, MD<div class="affiliation small">Institute for Women's Health<br />University College London<br />London, United Kingdom<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ku.ca.lcu@yawnoc.g" class="oemail">ku.ca.lcu@yawnoc.g</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Leigh AM Demain</span>, PhD<div class="affiliation small">Manchester Centre for Genomic Medicine<br />University of Manchester and Manchester University NHS Foundation Trust<br />Manchester, United Kingdom<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ku.ca.retsehcnam@niamed.hgiel" class="oemail">ku.ca.retsehcnam@niamed.hgiel</a></div></div></div></div><p class="small">Initial Posting: <span itemprop="datePublished">September 25, 2014</span>; Last Update: <span itemprop="dateModified">September 6, 2018</span>.</p><p><em>Estimated reading time: 23 minutes</em></p></div><div class="jig-ncbiinpagenav body-content whole_rhythm" data-jigconfig="allHeadingLevels: ['h2'],smoothScroll: false" itemprop="text"><div id="perrault.Summary" itemprop="description"><h2 id="_perrault_Summary_">Summary</h2><div><h4 class="inline">Clinical characteristics.</h4><p>Perrault syndrome is characterized by sensorineural hearing loss (SNHL) in males and females and ovarian dysfunction in females. SNHL is bilateral and ranges from profound with prelingual (<a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a>) onset to moderate with early-childhood onset. When onset is in early childhood, hearing loss can be progressive. Ovarian dysfunction ranges from gonadal dysgenesis (absent or streak gonads) manifesting as primary amenorrhea to primary ovarian insufficiency (POI) defined as cessation of menses before age 40 years. Fertility in <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> males is reported as normal (although the number of reported males is limited). Neurologic features described in some individuals with Perrault syndrome include learning difficulties and developmental delay, cerebellar ataxia, and motor and sensory peripheral neuropathy.</p></div><div><h4 class="inline">Diagnosis/testing.</h4><p>The diagnosis of Perrault syndrome is based on the clinical findings of SNHL in men and women and ovarian dysfunction in women with a 46,XX <a class="def" href="/books/n/gene/glossary/def-item/karyotype/">karyotype</a>. The diagnosis is confirmed by the presence of <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in one of six genes (<i>CLPP</i>, <i>ERAL1</i>, <i>HARS2</i>, <i>HSD17B4</i>, <i>LARS2</i>, or <i>TWNK</i>); however, in approximately 60% of individuals with Perrault syndrome identified to date, a molecular diagnosis cannot be established.</p></div><div><h4 class="inline">Management.</h4><p><i>Treatment of manifestations:</i> Hearing loss should be assessed and treated by a multidisciplinary team including an audiologist and otolaryngologist. Possible interventions for those with hearing loss include special educational resources, hearing aids, vibrotactile devices, and cochlear implantation. Cochlear implantation is an option for children older than 12 months with severe-to-profound hearing loss. Primary amenorrhea is treated in adolescents in collaboration with a pediatric endocrinologist in the usual manner, first to induce puberty and then to mimic the menstrual cycle and maintain bone health. Assisted reproduction through in vitro fertilization using donor eggs is a consideration for women with gonadal dysgenesis; oocyte cryopreservation can be considered in women at risk for POI.</p><p><i>Surveillance:</i> Routine audiologic assessments when hearing loss is mild to moderate; no follow up or audiologic assessments when hearing loss is profound. For children with hearing impairment: monitor development</p><p>For women with primary amenorrhea: during induction of puberty, follow up every three months for staging of pubertal development and adjustment of estrogen dose. For women on maintenance estrogen replacement therapy: annual follow up as well as assessment of bone density approximately every five years.</p><p><i>Agents/circumstances to avoid:</i> Avoid: ototoxic medication (e.g., aminoglycosides) if alternative medications are available; exposure to loud noise, which can exacerbate hearing loss.</p><p><i>Evaluation of relatives at risk:</i> It is appropriate to evaluate the older and younger sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> in order to identify as early as possible those who would benefit from early interventions (e.g., in young children with profound hearing loss; estrogen replacement to facilitate pubertal development in females with ovarian involvement; and potential oocyte cryopreservation if POI is an issue).</p></div><div><h4 class="inline">Genetic counseling.</h4><p>Perrault syndrome is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> manner. At conception, each sib of an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individual has a 25% chance of being affected, a 50% chance of being an asymptomatic <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a>, and a 25% chance of being unaffected and not a carrier. When the pathogenic variants in the family are known, <a class="def" href="/books/n/gene/glossary/def-item/carrier-testing/">carrier testing</a> for at-risk relatives, prenatal testing for pregnancies at increased risk, and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> are possible.</p></div></div><div id="perrault.Diagnosis"><h2 id="_perrault_Diagnosis_">Diagnosis</h2><p>No formal diagnostic criteria have been published for Perrault syndrome.</p><div id="perrault.Suggestive_Findings"><h3>Suggestive Findings</h3><p>Perrault syndrome <b>should be suspected</b> in individuals with the following clinical findings and family history.</p><p><b>Clinical findings</b></p><ul><li class="half_rhythm"><div><b>Sensorineural hearing loss (SNHL) in men and women.</b> SNHL is bilateral and ranges in severity from moderate with early-childhood onset to profound with prelingual (<a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a>) onset. The hearing threshold increase can be variable. When presenting in early childhood, hearing loss can be progressive.</div><ul><li class="half_rhythm"><div>SNHL may be apparent from birth in infants who fail neonatal screening tests.</div></li><li class="half_rhythm"><div>In older children SNHL can be demonstrated on an audiogram, which will show similar hearing thresholds for both bone and air conduction.</div></li></ul></li><li class="half_rhythm"><div><b>Ovarian dysfunction in women with a 46,XX <a class="def" href="/books/n/gene/glossary/def-item/karyotype/">karyotype</a>.</b> The spectrum of ovarian dysfunction extends across a continuum from primary ovarian insufficiency (POI) to ovarian dysgenesis.</div><ul><li class="half_rhythm"><div>POI is defined as cessation of menses before age 40 years, with raised levels of follicle stimulating hormone (FSH) and reduced serum estrogen concentration.</div></li><li class="half_rhythm"><div>Ovarian dysgenesis is a developmental disorder characterized by loss of germ and supportive cells (e.g., granulosa and theca cells, respectively) in the gonads. The ovaries are dysplastic, streak, or absent. Serum concentration of estrogen is decreased with a consequent elevation in serum concentration of the two gonadotropins, lutenizing hormone (LH) and follicle stimulating hormone (FSH) (i.e., with hypergonadotropic hypogonadism). The uterus is rudimentary and prepubertal on ultrasound examination.</div></li></ul></li></ul><p><b>Family history</b> is consistent with <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> inheritance including the possibility of parental <a class="def" href="/books/n/gene/glossary/def-item/consanguinity/">consanguinity</a>.</p></div><div id="perrault.Establishing_the_Diagnosis"><h3>Establishing the Diagnosis</h3><p>The diagnosis of Perrault syndrome <b>is established</b>:</p><ul><li class="half_rhythm"><div>Either by clinical findings, family history, and exclusion of other possible diagnoses with findings similar to Perrault syndrome (see <a href="#perrault.Differential_Diagnosis">Differential Diagnosis</a>);</div></li><li class="half_rhythm"><div>Or by identification of <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in one of the six associated genes (see <a class="figpopup" href="/books/NBK242617/table/perrault.T.molecular_genetic_testing_use/?report=objectonly" target="object" rid-figpopup="figperraultTmoleculargenetictestinguse" rid-ob="figobperraultTmoleculargenetictestinguse">Table 1</a>) in a person with <a href="#perrault.Suggestive_Findings">Suggestive Findings</a>.</div></li></ul><div id="perrault.Clinical_Findings_and_Family_Hi"><h4>Clinical Findings and Family History</h4><ul><li class="half_rhythm"><div><b>SNHL</b> that is bilateral and ranges in severity from moderate with early-childhood onset to profound with prelingual (<a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a>) onset</div></li><li class="half_rhythm"><div><b>Ovarian dysfunction</b> in women with a 46,XX <a class="def" href="/books/n/gene/glossary/def-item/karyotype/">karyotype</a></div></li></ul><p>Note: Sensorineural hearing loss is usually the initial manifestation of Perrault syndrome. The diagnosis will not be considered, based on clinical findings alone, in males who do not have an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> sister. The initial diagnosis will not be made in females based on clinical findings alone until delayed pubertal development is noted, usually in the teenage years.</p></div><div id="perrault.Molecular_Genetic_Testing"><h4>Molecular Genetic Testing</h4><p>The diagnosis of Perrault syndrome is molecularly confirmed by the presence of <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in one of six genes: <i>CLPP</i>, <i>ERAL1</i>, <i>HARS2</i>, <i>HSD17B4</i>, <i>LARS2</i>, and <i>TWNK</i> (see <a class="figpopup" href="/books/NBK242617/table/perrault.T.molecular_genetic_testing_use/?report=objectonly" target="object" rid-figpopup="figperraultTmoleculargenetictestinguse" rid-ob="figobperraultTmoleculargenetictestinguse">Table 1</a>).</p><p>Note: To date <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in these six genes do not account for all individuals with clinically confirmed Perrault syndrome (<a class="figpopup" href="/books/NBK242617/table/perrault.T.molecular_genetic_testing_use/?report=objectonly" target="object" rid-figpopup="figperraultTmoleculargenetictestinguse" rid-ob="figobperraultTmoleculargenetictestinguse">Table 1</a>) [<a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>].</p><p>Due to the heterogeneous nature of this disorder, <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> approaches can include <b><a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted testing</b> (through a <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a>) or <b>comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing</b> (which does not require the clinician to determine which gene[s] are likely involved).</p><ul><li class="half_rhythm"><div class="half_rhythm"><b>A <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a></b> that includes <i>CLPP</i>, <i>ERAL1</i>, <i>HARS2</i>, <i>HSD17B4</i>, <i>LARS2</i>, and <i>TWNK</i> and other genes of interest (see <a href="#perrault.Differential_Diagnosis">Differential Diagnosis</a>) is most likely to identify the genetic cause of the condition at the most reasonable cost while limiting identification of variants of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a> and pathogenic variants in genes that do not explain the underlying <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>. Note: (1) The genes included in the panel and the diagnostic <a class="def" href="/books/n/gene/glossary/def-item/sensitivity/">sensitivity</a> of the testing used for each <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> vary by laboratory and are likely to change over time. (2) Some multigene panels may include genes not associated with the condition discussed in this <i>GeneReview</i>. (3) In some laboratories, panel options may include a custom laboratory-designed panel and/or custom phenotype-focused <a class="def" href="/books/n/gene/glossary/def-item/exome/">exome</a> analysis that includes genes specified by the clinician. (4) Methods used in a panel may include <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a>, <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>, and/or other non-sequencing-based tests.</div><div class="half_rhythm">For an introduction to multigene panels click <a href="/books/n/gene/app5/#app5.Multigene_Panels">here</a>. More detailed information for clinicians ordering genetic tests can be found <a href="/books/n/gene/app5/#app5.Multigene_Panels_FAQs">here</a>.</div></li><li class="half_rhythm"><div class="half_rhythm"><b>Comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing</b> is the best option when the diagnosis of Perrault syndrome has not been considered because an individual has atypical phenotypic features. <b>Exome sequencing</b> is the most commonly used genomic testing method; <b><a class="def" href="/books/n/gene/glossary/def-item/genome-sequencing/">genome sequencing</a></b> is also possible.</div><div class="half_rhythm">For an introduction to comprehensive <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> testing click <a href="/books/n/gene/app5/#app5.Comprehensive_Genomic_Testing">here</a>. More detailed information for clinicians ordering genomic testing can be found <a href="/books/n/gene/app5/#app5.Comprehensive_Genomic_Testing_1">here</a>.</div></li></ul><div id="perrault.T.molecular_genetic_testing_use" class="table"><h3><span class="label">Table 1. </span></h3><div class="caption"><p>Molecular Genetic Testing Used in Perrault Syndrome</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.molecular_genetic_testing_use/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.molecular_genetic_testing_use_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_1" rowspan="2" scope="col" colspan="1" headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_1" style="text-align:left;vertical-align:middle;">Gene&#x000a0;<sup>1,&#x000a0;2</sup></th><th id="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_2" rowspan="2" scope="col" colspan="1" headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_2" style="text-align:left;vertical-align:middle;">Proportion of Perrault Syndrome Attributed to Pathogenic Variants in This Gene</th><th id="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3" colspan="2" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">Proportion of Pathogenic Variants&#x000a0;<sup>3</sup> Detectable by Test Method</th></tr><tr><th headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3" id="hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_1" colspan="1" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">Sequence analysis&#x000a0;<sup>4</sup></th><th headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3" id="hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a>&#x000a0;<sup>5</sup></th></tr></thead><tbody><tr><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CLPP</i></td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">7/42</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">7/7</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">None reported</td></tr><tr><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ERAL1</i></td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">3/42</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">3/3</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">None reported</td></tr><tr><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>HARS2</i></td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">3/42</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">3/3</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">None reported</td></tr><tr><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>HSD17B4</i></td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">3/42</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">3/3</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">None reported</td></tr><tr><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>LARS2</i></td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">8/42</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">8/8</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">None reported</td></tr><tr><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TWNK</i></td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">5/42</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">5/5</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">None reported</td></tr><tr><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Unknown&#x000a0;<sup>6</sup></td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">13/42</td><td headers="hd_h_perrault.T.molecular_genetic_testing_use_1_1_1_3 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_1 hd_h_perrault.T.molecular_genetic_testing_use_1_1_2_2" colspan="2" rowspan="1" style="text-align:left;vertical-align:middle;">NA</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="perrault.TF.1.1"><p class="no_margin">Genes are listed in alphabetic order.</p></div></dd><dt>2. </dt><dd><div id="perrault.TF.1.2"><p class="no_margin">See <a href="/books/NBK242617/#perrault.molgen.TA">Table A. Genes and Databases</a> for <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> and protein.</p></div></dd><dt>3. </dt><dd><div id="perrault.TF.1.3"><p class="no_margin">See <a href="#perrault.Molecular_Genetics">Molecular Genetics</a> for information on allelic variants detected in this <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>.</p></div></dd><dt>4. </dt><dd><div id="perrault.TF.1.4"><p class="no_margin">Sequence analysis detects variants that are benign, likely benign, of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a>, likely pathogenic, or pathogenic. Pathogenic variants may include small intragenic deletions/insertions and <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a>, <a class="def" href="/books/n/gene/glossary/def-item/nonsense-variant/">nonsense</a>, and <a class="def" href="/books/n/gene/glossary/def-item/splice-site/">splice site</a> variants; typically, <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> or whole-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> deletions/duplications are not detected. For issues to consider in interpretation of <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> results, click <a href="/books/n/gene/app2/">here</a>.</p></div></dd><dt>5. </dt><dd><div id="perrault.TF.1.5"><p class="no_margin">Gene-targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a> detects intragenic deletions or duplications. Methods used may include: <a class="def" href="/books/n/gene/glossary/def-item/quantitative-pcr/">quantitative PCR</a>, long-range PCR, multiplex ligation-dependent probe amplification (MLPA), and a <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>-targeted microarray designed to detect single-<a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> deletions or duplications.</p></div></dd><dt>6. </dt><dd><div id="perrault.TF.1.6"><p class="no_margin">When <a class="def" href="/books/n/gene/glossary/def-item/multigene-panel/">multigene panel</a> testing that includes most/all of the known genes is used, approximately 60% of individuals with Perrault syndrome have not had pathogenic variants [<a class="bk_pop" href="#perrault.REF.lerat.2016.1354">Lerat et al 2016</a>, <a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>]. Failure to detect pathogenic variants in one of these genes suggests either the presence of a variant in a <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> region not sequenced (e.g., enhancer, promoter, <a class="def" href="/books/n/gene/glossary/def-item/intron/">intron</a>), pathogenic variants in an as-yet-unidentified gene, or inaccurate clinical diagnosis (see <a href="#perrault.Differential_Diagnosis">Differential Diagnosis</a>).</p></div></dd></dl></div></div></div></div></div></div><div id="perrault.Clinical_Characteristics"><h2 id="_perrault_Clinical_Characteristics_">Clinical Characteristics</h2><div id="perrault.Clinical_Description"><h3>Clinical Description</h3><p>Perrault syndrome is characterized by sensorineural hearing loss (SNHL) in males and females and ovarian dysfunction in females.</p><p>Significant inter- and intrafamilial phenotypic variability has been observed [<a class="bk_pop" href="#perrault.REF.jenkinson.2012.974">Jenkinson et al 2012</a>]. Of note, the variable age of onset and degree of deafness do not depend on the sex of the <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individual.</p><p>SNHL is bilateral and ranges from profound with prelingual (<a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a>) onset to moderate with early-childhood onset. When onset is in early childhood, hearing loss can be progressive. There is no evidence of impaired vestibular function.</p><p>Affected females have gonadal dysfunction. Although the ovarian findings in Perrault syndrome were originally described as primary ovarian failure due to absent or streak gonads, subsequent reports identified a spectrum of ovarian dysfunction ranging from gonadal dysgenesis presenting as primary amenorrhea (also known as primary ovarian failure) to primary ovarian insufficiency (POI) (presenting as secondary amenorrhea) which is defined as cessation of menses before age 40 years. One woman with Perrault syndrome had children prior to the onset of ovarian insufficiency [<a class="bk_pop" href="#perrault.REF.jenkinson.2013.605">Jenkinson et al 2013</a>].</p><p>Fertility in <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> males is usually reported as normal, although the number of reported affected males is limited. Males with variants in <i>CLPP</i> have been noted to be azoospermic [<a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>].</p><p><b>Other features.</b> Some individuals have been reported to have additional clinical features. No consistent pattern has been observed with these additional features, which have been reported in more than one individual [<a class="bk_pop" href="#perrault.REF.jenkinson.2012.974">Jenkinson et al 2012</a>].</p><p>Neurologic features are present in some individuals with Perrault syndrome. Members of families with <i>CLPP</i>-related Perrault syndrome and <i>LARS2</i>-related Perrault syndrome have been reported with or without neurologic features [<a class="bk_pop" href="#perrault.REF.jenkinson.2013.605">Jenkinson et al 2013</a>, <a class="bk_pop" href="#perrault.REF.pierce.2013.614">Pierce et al 2013</a>, <a class="bk_pop" href="#perrault.REF.kosaki.2018.404">Kosaki et al 2018</a>]:</p><ul><li class="half_rhythm"><div>Learning difficulties and developmental delay [<a class="bk_pop" href="#perrault.REF.jenkinson.2012.974">Jenkinson et al 2012</a>, <a class="bk_pop" href="#perrault.REF.lerat.2016.1354">Lerat et al 2016</a>, <a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>]</div></li><li class="half_rhythm"><div>Cerebellar ataxia [<a class="bk_pop" href="#perrault.REF.jenkinson.2012.974">Jenkinson et al 2012</a>, <a class="bk_pop" href="#perrault.REF.lerat.2016.1354">Lerat et al 2016</a>, <a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>]</div></li><li class="half_rhythm"><div>Motor and sensory peripheral neuropathy [<a class="bk_pop" href="#perrault.REF.jenkinson.2012.974">Jenkinson et al 2012</a>, <a class="bk_pop" href="#perrault.REF.lerat.2016.1354">Lerat et al 2016</a>, <a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>]</div></li></ul><p>Skeletal features reported in some individuals include high-arched palate, positive thumb and wrist signs, and marfanoid habitus [<a class="bk_pop" href="#perrault.REF.zerkaoui.2017.200">Zerkaoui et al 2017</a>]</p></div><div id="perrault.Phenotype_Correlations_by_Gene"><h3>Phenotype Correlations by Gene</h3><p>Sensorineural hearing loss (SNHL) in Perrault syndrome resulting from <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in <i>ERAL1</i>, <i>HARS2</i>, <i>HSD17B4</i>, or <i>LARS2</i> can be <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> and profound or progressive with varying degrees of severity; onset is usually in early childhood and a range of frequencies are affected.</p><p><b><i>CLPP</i>.</b> In families reported to date with <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> <i>CLPP</i> pathogenic variants<i>,</i> SNHL is severe to profound with <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> or early childhood onset [<a class="bk_pop" href="#perrault.REF.jenkinson.2013.605">Jenkinson et al 2013</a>, <a class="bk_pop" href="#perrault.REF.ahmed.2015.149">Ahmed et al 2015</a>, <a class="bk_pop" href="#perrault.REF.lerat.2016.1354">Lerat et al 2016</a>, <a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>].</p><p><b><i>TWNK</i>.</b> All individuals reported to date with <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> <i>TWNK</i> variants have had associated neurologic features including ataxia and peripheral neuropathy [<a class="bk_pop" href="#perrault.REF.morino.2014.2054">Morino et al 2014</a>, <a class="bk_pop" href="#perrault.REF.lerat.2016.1354">Lerat et al 2016</a>, <a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>, <a class="bk_pop" href="#perrault.REF.o_dak.2017.25">O&#x00142;dak et al 2017</a>].</p></div><div id="perrault.GenotypePhenotype_Correlations"><h3>Genotype-Phenotype Correlations</h3><p>Low-frequency SNHL resulting in an upsloping audiogram has been associated with the <i>LARS2</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> <a class="figpopup" href="/books/NBK242617/table/perrault.T.lars2_pathogenic_variants_dis/?report=objectonly" target="object" rid-figpopup="figperraultTlars2pathogenicvariantsdis" rid-ob="figobperraultTlars2pathogenicvariantsdis">c.1565C&#x0003e;A</a> in either the <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a> state or in a <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> state in <a class="def" href="/books/n/gene/glossary/def-item/trans/"><i>trans</i></a> to another pathogenic <i>LARS2</i> variant. Other variants in <i>LARS2</i> have not been associated with low-frequency SNHL [<a class="bk_pop" href="#perrault.REF.pierce.2013.614">Pierce et al 2013</a>, <a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>].</p></div><div id="perrault.Nomenclature"><h3>Nomenclature</h3><p>Perrault syndrome has also been referred to as ovarian dysgenesis with sensorineural deafness or XX gonadal dysgenesis with deafness.</p></div><div id="perrault.Prevalence"><h3>Prevalence</h3><p>Perrault syndrome is rare; approximately 100 <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals have been reported to date [<a class="bk_pop" href="#perrault.REF.lerat.2016.1354">Lerat et al 2016</a>]. However, underascertainment is likely as males without an affected sister will be diagnosed with nonsyndromic deafness rather than Perrault syndrome. For example, the authors are aware of males with SNHL and prepubertal girls with SNHL with variants in Perrault syndrome-related genes.</p></div></div><div id="perrault.Genetically_Related_Allelic_Dis"><h2 id="_perrault_Genetically_Related_Allelic_Dis_">Genetically Related (Allelic) Disorders</h2><p>No phenotypes other than those discussed in this <i>GeneReview</i> are known to be associated with pathogenic variants in <i>ERAL1</i> or <i>HARS2.</i> Other phenotypes associated with <a class="def" href="/books/n/gene/glossary/def-item/germline/">germline</a> pathogenic variants in <i>CLPP</i>, <i>HSD17B4</i>, <i>LARS2</i>, and <i>TWNK</i> are summarized in <a class="figpopup" href="/books/NBK242617/table/perrault.T.allelic_disorders/?report=objectonly" target="object" rid-figpopup="figperraultTallelicdisorders" rid-ob="figobperraultTallelicdisorders">Table 2</a>.</p><div id="perrault.T.allelic_disorders" class="table"><h3><span class="label">Table 2. </span></h3><div class="caption"><p>Allelic Disorders</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.allelic_disorders/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.allelic_disorders_lrgtbl__"><table><thead><tr><th id="hd_h_perrault.T.allelic_disorders_1_1_1_1" rowspan="2" scope="col" colspan="1" style="text-align:left;vertical-align:middle;">Gene</th><th id="hd_h_perrault.T.allelic_disorders_1_1_1_2" rowspan="2" scope="col" colspan="1" style="text-align:left;vertical-align:middle;">Disorder</th><th id="hd_h_perrault.T.allelic_disorders_1_1_1_3" rowspan="2" scope="col" colspan="1" style="text-align:left;vertical-align:middle;">Reference</th></tr></thead><tbody><tr><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>CLPP</i></td><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Sensorineural hearing loss, epilepsy, &#x00026; leukoencephalopathy</td><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#perrault.REF.theunissen.2016.203">Theunissen et al [2016]</a></td></tr><tr><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>HSD17B4</i></td><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">D-bifunctional protein deficiency</td><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#perrault.REF.van_grunsven.1999.99">van Grunsven et al [1999]</a></td></tr><tr><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>LARS2</i></td><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Lethal infantile multisystem failure</td><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#perrault.REF.riley.2016.49">Riley et al [2016]</a></td></tr><tr><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_1" rowspan="3" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><i>TWNK</i></td><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/sca-io/">Infantile-onset spinocerebellar ataxia</a></td><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="bk_pop" href="#perrault.REF.morino.2014.2054">Morino et al [2014]</a></td></tr><tr><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Mitochondrial DNA maintenance defect presenting w/encephalohepatopathy</td><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_3" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/books/n/gene/mtdna-md-ov/">Mitochondrial DNA Maintenance Defects Overview</a></td></tr><tr><td headers="hd_h_perrault.T.allelic_disorders_1_1_1_2" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;">Mitochondrial DNA maintenance defect presenting w/ophthalmoplegia</td></tr></tbody></table></div></div></div><div id="perrault.Differential_Diagnosis"><h2 id="_perrault_Differential_Diagnosis_">Differential Diagnosis</h2><p>For individuals with a clinical diagnosis of Perrault syndrome in whom a molecular basis has not been identified, other causes of sensorineural hearing loss and ovarian dysfunction need to be excluded before a clinical diagnosis of Perrault syndrome can made with confidence.</p><p><b>Sensorineural hearing loss</b> (SNHL) is genetically heterogeneous. See <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a> for a detailed differential diagnosis.</p><p><b>XX gonadal dysgenesis</b>
<b>and primary ovarian insufficiency</b> are genetically heterogeneous.</p><ul><li class="half_rhythm"><div><b>XX gonadal dysgenesis.</b> For individuals with primary ovarian failure, defined by primary amenorrhea with low estrogen and raised gonadotropins, Turner syndrome (45, X) or other abnormalities of the X <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> should be excluded by <a class="def" href="/books/n/gene/glossary/def-item/karyotype/">karyotype</a> analysis or <a class="def" href="/books/n/gene/glossary/def-item/chromosomal-microarray/">chromosomal microarray</a> (also known as array CGH). Hearing loss is present in approximately 50% of women with Turner syndrome [<a class="bk_pop" href="#perrault.REF.king.2007.831">King et al 2007</a>], but tends to be mild to moderate at higher frequencies [<a class="bk_pop" href="#perrault.REF.oliveira.2013.226">Oliveira et al 2013</a>].</div><ul><li class="half_rhythm"><div>Testing of genes in which pathogenic variants have been reported to cause ovarian dysgenesis (including <i>BMP15, FSHR,</i>
<i>MCM9</i>, <i>PSMC3IP</i>, and <i>SOHLH1</i>) is appropriate (OMIM <a href="https://omim.org/phenotypicSeries/PS233300" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">PS233300</a>).</div></li><li class="half_rhythm"><div>Other causes of primary ovarian failure include 17&#x003b1;-hydroxylase deficiency and 17,20-lysase deficiency (OMIM <a href="https://omim.org/entry/202110" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">202110</a>); which can be excluded by measurement of 11-deoxycorticosterone and androstenedione levels.</div></li></ul></li><li class="half_rhythm"><div><b>Primary ovarian insufficiency (POI).</b> Multiple genetic causes of POI are known. Analysis of a limited number of genes is available by routine clinical testing.</div><ul><li class="half_rhythm"><div>Premutation carriers of an expanded <i>FMR1</i> <a class="def" href="/books/n/gene/glossary/def-item/allele/">allele</a> are at increased risk for ovarian insufficiency; see <a href="/books/n/gene/fragilex/"><i>FMR1</i>-Related Disorders</a>.</div></li><li class="half_rhythm"><div>Many females with <a href="/books/n/gene/bpes/">BPES</a> (<i>b</i>lepharophimosis, <i>p</i>tosis, <i>e</i>picanthus inversus <i>s</i>yndrome), caused by pathogenic variants or deletions of <i>FOXL2</i>, have POI. BPES can be distinguished from Perrault syndrome by the presence of marked blepharophimosis and ptosis in BPES.</div></li><li class="half_rhythm"><div>Ovarian antibodies are increased in polyglandular autoimmune syndrome type 1 (OMIM <a href="https://omim.org/entry/240300" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri"><u>240300</u></a>) and type 2 (OMIM <a href="https://omim.org/entry/269200" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">269200</a>).</div></li></ul></li></ul><p><b>SNHL and POI</b></p><ul><li class="half_rhythm"><div><i>RMND1</i>-associated mitochondrial disease, which has a wide phenotypic range including SNHL, hypotonia, developmental delay, lactic acidemia, and renal dysfunction [<a class="bk_pop" href="#perrault.REF.ng.2016.768">Ng et al 2016</a>] can present with features consistent with a diagnosis of Perrault syndrome [<a class="bk_pop" href="#perrault.REF.demain.2018.276">Demain et al 2018</a>].</div></li><li class="half_rhythm"><div>The authors described a novel and likely rare cause of Perrault syndrome in a female who did not have pathogenic variants in any of the known Perrault syndrome-related genes [<a class="bk_pop" href="#perrault.REF.faridi.2017.328">Faridi et al 2017</a>]. She manifested SNHL and POI as a result of inheriting <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a> pathogenic variants in each of two distinct unlinked genes: <i>CLDN14</i> and <i>SGO2.</i>
<i>CLDN14</i> is a well-known cause of <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> SNHL. Her POI was attributed to <a class="def" href="/books/n/gene/glossary/def-item/inactivating/">inactivating</a> variants in <i>SGO2</i>, a <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> not known to cause any human disorder but essential for meiosis and strongly implicated in infertility by studies in murine models. The variants of both <i>CLDN14</i> and <i>SGO2</i> were segregating in her multigenerational family; her parents were documented to be <a class="def" href="/books/n/gene/glossary/def-item/heterozygous/">heterozygous</a> for variants in both genes. Although relatives either manifested SNHL or were heterozygous for the <i>CLDN14</i> variant, she was the only female who had both SNHL and POI.</div></li></ul></div><div id="perrault.Management"><h2 id="_perrault_Management_">Management</h2><div id="perrault.Evaluations_Following_Initial_D"><h3>Evaluations Following Initial Diagnosis</h3><p>To establish the extent of disease and needs in an individual diagnosed with Perrault syndrome, the evaluations summarized in <a class="figpopup" href="/books/NBK242617/table/perrault.T.recommended_evaluations_follo/?report=objectonly" target="object" rid-figpopup="figperraultTrecommendedevaluationsfollo" rid-ob="figobperraultTrecommendedevaluationsfollo">Table 3</a> (if not performed as part of the evaluation that led to the diagnosis) are recommended.</p><div id="perrault.T.recommended_evaluations_follo" class="table"><h3><span class="label">Table 3. </span></h3><div class="caption"><p>Recommended Evaluations Following Initial Diagnosis in Individuals with Perrault Syndrome</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.recommended_evaluations_follo/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.recommended_evaluations_follo_lrgtbl__"><table class="no_bottom_margin"><tbody><tr><td scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Organ System</b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Evaluation</b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Comment</b></td></tr><tr><td scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>ENT&#x000a0;<sup>1</sup></b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Audiologic assessment</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">To define the degree &#x00026; frequency range of hearing impairment by audiometry &#x00026; physiologic tests (summarized in <a href="/books/n/gene/deafness-overview/">Hereditary Hearing Loss and Deafness Overview</a>)</td></tr><tr><td scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Neurologic&#x000a0;<sup>1</sup></b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Neurologic assessment</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Determine if ataxia, peripheral neuropathy, and/or learning disability is present</td></tr><tr><td rowspan="2" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><b>Endocrine&#x000a0;<sup>2</sup></b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Serum estrogen &#x00026; gonadotropin (LH &#x00026; FSH) concentrations</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">In women w/relatively intact ovarian function, serum anti-m&#x000fc;llerian hormone concentrations may provide evidence of incipient ovarian failure&#x000a0;<sup>3</sup></td></tr><tr><td colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Pelvic imaging</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Ultrasound scan or magnetic resonance imaging to define the presence of ovaries &#x00026; antral follicle count</td></tr><tr><td scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Miscellaneous/</b><br /><b>Other&#x000a0;<sup>1</sup></b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Consultation w/a clinical geneticist &#x00026;/or genetic counselor</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="perrault.TF.3.1"><p class="no_margin">Men and women</p></div></dd><dt>2. </dt><dd><div id="perrault.TF.3.2"><p class="no_margin">Women only</p></div></dd><dt>3. </dt><dd><div id="perrault.TF.3.3"><p class="no_margin"><a class="bk_pop" href="#perrault.REF.de_vos.2010.911">De Vos et al [2010]</a></p></div></dd></dl></div></div></div></div><div id="perrault.Treatment_of_Manifestations"><h3>Treatment of Manifestations</h3><div id="perrault.T.treatment_of_manifestations_i" class="table"><h3><span class="label">Table 4. </span></h3><div class="caption"><p>Treatment of Manifestations in Individuals with Perrault Syndrome</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.treatment_of_manifestations_i/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.treatment_of_manifestations_i_lrgtbl__"><table class="no_bottom_margin"><tbody><tr><td scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Manifestation</b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Treatment</b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Considerations/Other</b></td></tr><tr><td scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Hearing loss</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Possible interventions:
<ul><li class="half_rhythm"><div>Hearing aids</div></li><li class="half_rhythm"><div>Vibrotactile devices</div></li><li class="half_rhythm"><div>Cochlear implantation&#x000a0;<sup>1</sup></div></li></ul>
</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Assessment &#x00026; treatment by multidisciplinary team incl: audiologist, otolaryngologist, speech therapist</div></li><li class="half_rhythm"><div>Provide for any special educational needs</div></li><li class="half_rhythm"><div>Early intervention in young children w/profound hearing loss improves cognitive &#x00026; language development.</div></li></ul>
</td></tr><tr><td rowspan="2" scope="row" colspan="1" style="text-align:left;vertical-align:middle;">Ovarian insufficiency</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">In adolescents presenting w/primary amenorrhea, induction of puberty w/incremental doses of estrogen</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>In consultation w/pediatric endocrinologist</div></li><li class="half_rhythm"><div>If puberty is complete, administer cyclic estrogen &#x00026; progesterone to mimic menstrual cycle &#x00026; trigger withdrawal bleeding.</div></li><li class="half_rhythm"><div>Estrogen replacement therapy (if no contraindications) until age &#x02265;50 yrs to reduce risks of cardiovascular disease &#x00026; osteoporosis</div></li></ul>
</td></tr><tr><td colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Assisted reproduction through in vitro fertilization</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>For women w/gonadal dysgenesis: consider assisted reproduction through in vitro fertilization using donor eggs.</div></li><li class="half_rhythm"><div>For women at risk for ovarian insufficiency: consider oocyte cryopreservation if ovarian function is sufficiently well preserved to allow for successful harvesting of oocytes.</div></li><li class="half_rhythm"><div>Consider use of donor eggs.</div></li><li class="half_rhythm"><div>Before considering pregnancy, assess uterine size.</div></li></ul>
</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="perrault.TF.4.1"><p class="no_margin">Cochlear implantation can be considered in children age &#x0003e;12 months with severe-to-profound hearing loss.</p></div></dd></dl></div></div></div></div><div id="perrault.Surveillance"><h3>Surveillance</h3><div id="perrault.T.recommended_surveillance_for" class="table"><h3><span class="label">Table 5. </span></h3><div class="caption"><p>Recommended Surveillance for Individuals with Perrault Syndrome</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.recommended_surveillance_for/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.recommended_surveillance_for_lrgtbl__"><table><tbody><tr><td scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Organ System</b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Evaluation</b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Frequency</b></td></tr><tr><td scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Hearing</b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Routine audiologic assessment for possible progressive hearing impairment</div></li><li class="half_rhythm"><div>Audiologic surveillance not required for individuals w/profound hearing loss</div></li></ul>
</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Annual</td></tr><tr><td scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><b>Musculoskeletal</b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Assess bone density in women on maintenance estrogen replacement therapy</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Every ~5 yrs</td></tr><tr><td rowspan="2" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><b>Endocrine</b></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Before puberty: Clinical staging of puberty</div></li><li class="half_rhythm"><div>During induction of puberty: adjustment of estrogen dose</div></li></ul>
</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Every 3 mos</td></tr><tr><td colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Women on maintenance estrogen replacement therapy: assessment of withdrawal bleeding &#x00026; well being</td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Annual</td></tr></tbody></table></div></div></div><div id="perrault.AgentsCircumstances_to_Avoid"><h3>Agents/Circumstances to Avoid</h3><p>For individuals with hearing loss, avoid:</p><ul><li class="half_rhythm"><div>Ototoxic medication such as aminoglycosides if alternatives are available;</div></li><li class="half_rhythm"><div>Exposure to loud noise, which may contribute to deterioration of hearing.</div></li></ul></div><div id="perrault.Evaluation_of_Relatives_at_Risk"><h3>Evaluation of Relatives at Risk</h3><p>It is appropriate to evaluate the older and younger sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> in order to identify as early as possible those who would benefit from early interventions (e.g., in young children with profound hearing loss, estrogen replacement to facilitate pubertal development in females with ovarian involvement, and potential oocyte cryopreservation if primary ovarian insufficiency is an issue). See <a href="#perrault.Treatment_of_Manifestations">Treatment of Manifestations</a>.</p><ul><li class="half_rhythm"><div>If the pathogenic variants in the family are known, <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> can be used to clarify the genetic status of at-risk sibs.</div></li><li class="half_rhythm"><div>If the pathogenic variants in the family are not known, screening of sibs should include audiologic assessment in males and females and baseline measurements of serum LH and FSH in females.</div></li></ul><p>See <a href="#perrault.Related_Genetic_Counseling_Issu">Genetic Counseling</a> for issues related to testing of at-risk relatives for <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> purposes.</p></div><div id="perrault.Therapies_Under_Investigation"><h3>Therapies Under Investigation</h3><p>Search <a href="http://clinicaltrials.gov/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">ClinicalTrials.gov</a> in the US and <a href="http://www.ClinicalTrialsRegister.eu" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.ClinicalTrialsRegister.eu</a> in Europe for information on clinical studies for a wide range of diseases and conditions. Note: There may not be clinical trials for this disorder.</p></div></div><div id="perrault.Genetic_Counseling"><h2 id="_perrault_Genetic_Counseling_">Genetic Counseling</h2><p>
<i>Genetic counseling is the process of providing individuals and families with
information on the nature, inheritance, and implications of genetic disorders to help them
make informed medical and personal decisions. The following section deals with genetic
risk assessment and the use of family history and genetic testing to clarify genetic
status for family members. This section is not meant to address all personal, cultural, or
ethical issues that individuals may face or to substitute for consultation with a genetics
professional</i>. &#x02014;ED.</p><div id="perrault.Mode_of_Inheritance"><h3>Mode of Inheritance</h3><p>Perrault syndrome is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a> manner.</p></div><div id="perrault.Risk_to_Family_Members"><h3>Risk to Family Members</h3><p><b>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>The parents of an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> child are obligate heterozygotes (i.e., carriers of one Perrault syndrome-related <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>).</div></li><li class="half_rhythm"><div>Heterozygotes (carriers) are asymptomatic and are not at risk of developing the disorder.</div></li></ul><p><b>Sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>At conception, each sib of an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individual has a 25% chance of being affected, a 50% chance of being an asymptomatic <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a>, and a 25% chance of being unaffected and not a carrier.</div></li><li class="half_rhythm"><div>Heterozygotes (carriers) are asymptomatic and are not at risk of developing the disorder.</div></li></ul><p><b>Offspring of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</b> The unaffected offspring of an individual with Perrault syndrome are obligate heterozygotes (carriers) for a <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>.</p><p><b>Other family members.</b> Each sib of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s parents is at a 50% risk of being a <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a> of a Perrault syndrome-related <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>.</p></div><div id="perrault.Carrier_Heterozygote_Detection"><h3>Carrier (Heterozygote) Detection</h3><p>Carrier testing for at-risk relatives requires prior identification of the pathogenic variants causing Perrault syndrome in the family.</p></div><div id="perrault.Related_Genetic_Counseling_Issu"><h3>Related Genetic Counseling Issues</h3><p>See Management, <a href="#perrault.Evaluation_of_Relatives_at_Risk">Evaluation of Relatives at Risk</a> for information on evaluating at-risk relatives for the purpose of early diagnosis and treatment.</p><p><b>Family planning</b></p><ul><li class="half_rhythm"><div>The increased risk for primary ovarian insufficiency and infertility in females with Perrault syndrome should be addressed when discussing family planning.</div></li><li class="half_rhythm"><div>The optimal time for determination of genetic risk, clarification of <a class="def" href="/books/n/gene/glossary/def-item/carrier/">carrier</a> status, and discussion of the availability of prenatal testing is before pregnancy.</div></li><li class="half_rhythm"><div>It is appropriate to offer <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> (including discussion of potential risks to offspring and reproductive options) to young adults who are <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a>, are carriers, or are at risk of being carriers.</div></li></ul><p><b>DNA banking</b> is the storage of DNA (typically extracted from white blood cells) for possible future use. Because it is likely that testing methodology and our understanding of genes, allelic variants, and diseases will improve in the future, consideration should be given to banking DNA of <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals.</p></div><div id="perrault.Prenatal_Testing_and_Preimplant"><h3>Prenatal Testing and Preimplantation Genetic Diagnosis</h3><p>Once <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> <i>CLPP</i>, <i>ERAL1</i>, <i>HARS2</i>, <i>HSD17B4</i>, <i>LARS2</i>, or <i>TWNK</i> pathogenic variants have been identified in an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> family member, prenatal testing for a pregnancy at increased risk and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> are possible.</p><p>Differences in perspective may exist among medical professionals and within families regarding the use of prenatal testing, particularly if the testing is being considered for the purpose of pregnancy termination rather than early diagnosis. While most centers would consider decisions regarding prenatal testing to be the choice of the parents, discussion of these issues is appropriate.</p></div></div><div id="perrault.Resources"><h2 id="_perrault_Resources_">Resources</h2><p>
<i>GeneReviews staff has selected the following disease-specific and/or umbrella
support organizations and/or registries for the benefit of individuals with this disorder
and their families. GeneReviews is not responsible for the information provided by other
organizations. For information on selection criteria, click <a href="/books/n/gene/app4/">here</a>.</i></p><ul><li class="half_rhythm"><div><b>Action on Hearing Loss</b></div><div>United Kingdom</div><div><b>Phone:</b> 44 020 7296 8000; 0808 808 0123</div><div><b>Email:</b> informationline@hearingloss.org.uk</div><div><a href="https://www.actiononhearingloss.org.uk/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.actiononhearingloss.org.uk</a></div></li><li class="half_rhythm"><div><b>Alexander Graham Bell Association for the Deaf and Hard of Hearing</b></div><div>3417 Volta Place Northwest</div><div>Washington DC 20007</div><div><b>Phone:</b> 866-337-5220 (toll-free); 202-337-5220; 202-337-5221 (TTY)</div><div><b>Fax:</b> 202-337-8314</div><div><b>Email:</b> info@agbell.org</div><div><a href="http://www.agbell.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Listening and Spoken Language Knowledge Center</a></div></li><li class="half_rhythm"><div><b>American Society for Deaf Children (ASDC)</b></div><div>800 Florida Avenue Northeast</div><div>Suite 2047</div><div>Washington DC 20002-3695</div><div><b>Phone:</b> 800-942-2732 (Toll-free Parent Hotline); 866-895-4206 (toll free voice/TTY)</div><div><b>Fax:</b> 410-795-0965</div><div><b>Email:</b> info@deafchildren.org; asdc@deafchildren.org</div><div><a href="http://www.deafchildren.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.deafchildren.org</a></div></li><li class="half_rhythm"><div><b>National Association of the Deaf (NAD)</b></div><div>8630 Fenton Street</div><div>Suite 820</div><div>Silver Spring MD 20910</div><div><b>Phone:</b> 301-587-1788; 301-587-1789 (TTY)</div><div><b>Fax:</b> 301-587-1791</div><div><b>Email:</b> nad.info@nad.org</div><div><a href="http://www.nad.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.nad.org</a></div></li><li class="half_rhythm"><div><b>RESOLVE: The National Infertility Association</b></div><div>7918 Jones Branch Drive</div><div>Suite 300</div><div>McLean VA 22102</div><div><b>Phone:</b> 703-556-7172</div><div><b>Fax:</b> 703-506-3266</div><div><b>Email:</b> info@resolve.org</div><div><a href="http://www.resolve.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.resolve.org</a></div></li></ul></div><div id="perrault.Molecular_Genetics"><h2 id="_perrault_Molecular_Genetics_">Molecular Genetics</h2><p><i>Information in the Molecular Genetics and OMIM tables may differ from that elsewhere in the GeneReview: tables may contain more recent information. &#x02014;</i>ED.</p><div id="perrault.molgen.TA" class="table"><h3><span class="label">Table A.</span></h3><div class="caption"><p>Perrault Syndrome: Genes and Databases</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.molgen.TA/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.molgen.TA_lrgtbl__"><table class="no_bottom_margin"><tbody><tr><th id="hd_b_perrault.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;">Gene</th><th id="hd_b_perrault.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;">Chromosome Locus</th><th id="hd_b_perrault.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;">Protein</th><th id="hd_b_perrault.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;">Locus-Specific Databases</th><th id="hd_b_perrault.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;">HGMD</th><th id="hd_b_perrault.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;">ClinVar</th></tr><tr><td headers="hd_b_perrault.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/8192" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>CLPP</i></a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=8192" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">19p13<wbr style="display:inline-block"></wbr>​.3</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/Q16740" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">ATP-dependent Clp protease proteolytic subunit, mitochondrial</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"></td><td headers="hd_b_perrault.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=CLPP" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">CLPP</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=CLPP[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">CLPP</a></td></tr><tr><td headers="hd_b_perrault.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/26284" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>ERAL1</i></a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=26284" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">17q11<wbr style="display:inline-block"></wbr>​.2</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/O75616" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">GTPase Era, mitochondrial</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"></td><td headers="hd_b_perrault.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=ERAL1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">ERAL1</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=ERAL1[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">ERAL1</a></td></tr><tr><td headers="hd_b_perrault.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/23438" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>HARS2</i></a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=23438" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">5q31<wbr style="display:inline-block"></wbr>​.3</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/P49590" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Probable histidine--tRNA ligase, mitochondrial</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"></td><td headers="hd_b_perrault.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=HARS2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HARS2</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=HARS2[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HARS2</a></td></tr><tr><td headers="hd_b_perrault.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/3295" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>HSD17B4</i></a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=3295" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">5q23<wbr style="display:inline-block"></wbr>​.1</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/P51659" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Peroxisomal multifunctional enzyme type 2</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://databases.lovd.nl/shared/genes/HSD17B4" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HSD17B4 database</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=HSD17B4" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HSD17B4</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=HSD17B4[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HSD17B4</a></td></tr><tr><td headers="hd_b_perrault.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/23395" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>LARS2</i></a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=23395" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">3p21<wbr style="display:inline-block"></wbr>​.31</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/Q15031" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Probable leucine--tRNA ligase, mitochondrial</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"></td><td headers="hd_b_perrault.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=LARS2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">LARS2</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=LARS2[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">LARS2</a></td></tr><tr><td headers="hd_b_perrault.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/56652" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>TWNK</i></a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=56652" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">10q24<wbr style="display:inline-block"></wbr>​.31</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/Q96RR1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Twinkle protein, mitochondrial</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"></td><td headers="hd_b_perrault.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=TWNK" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">TWNK</a></td><td headers="hd_b_perrault.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=TWNK[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">TWNK</a></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div id="perrault.TFA.1"><p class="no_margin">Data are compiled from the following standard references: <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> from
<a href="http://www.genenames.org/index.html" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HGNC</a>;
<a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> from
<a href="http://www.omim.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">OMIM</a>;
protein from <a href="http://www.uniprot.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">UniProt</a>.
For a description of databases (Locus Specific, HGMD, ClinVar) to which links are provided, click
<a href="/books/n/gene/app1/">here</a>.</p></div></dd></dl></div></div></div><div id="perrault.molgen.TB" class="table"><h3><span class="label">Table B.</span></h3><div class="caption"><p>OMIM Entries for Perrault Syndrome (<a href="/omim/233400,600783,601119,601860,604544,606075,607435,614129,614926,615300,616138,617565" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">View All in OMIM</a>) </p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.molgen.TB/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.molgen.TB_lrgtbl__"><table><tbody><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/233400" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">233400</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">PERRAULT SYNDROME 1; PRLTS1</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/600783" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">600783</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">HISTIDYL-tRNA SYNTHETASE 2; HARS2</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/601119" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">601119</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">CASEINOLYTIC MITOCHONDRIAL MATRIX PEPTIDASE PROTEOLYTIC SUBUNIT; CLPP</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/601860" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">601860</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">17-@BETA-HYDROXYSTEROID DEHYDROGENASE IV; HSD17B4</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/604544" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">604544</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">LEUCYL-tRNA SYNTHETASE 2; LARS2</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/606075" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">606075</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">TWINKLE mtDNA HELICASE; TWNK</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/607435" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">607435</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">ERA G-PROTEIN-LIKE 1; ERAL1</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/614129" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">614129</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">PERRAULT SYNDROME 3; PRLTS3</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/614926" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">614926</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">PERRAULT SYNDROME 2; PRLTS2</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/615300" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">615300</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">PERRAULT SYNDROME 4; PRLTS4</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/616138" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">616138</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">PERRAULT SYNDROME 5; PRLTS5</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/617565" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">617565</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">PERRAULT SYNDROME 6; PRLTS6</td></tr></tbody></table></div></div><div id="perrault.Molecular_Pathogenesis"><h3>Molecular Pathogenesis</h3><p>To date <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in one of six genes &#x02013; <i>CLPP, ERAL1, HARS2, HSD17B4, LARS2</i>, and <i>TWNK</i> &#x02013; are known to cause Perrault syndrome. However, many cases of Perrault syndrome are not molecularly defined.</p><p><i>CLPP</i> and <i>ERAL1</i> have roles in the formation of the mitochondrial ribosome; <i>HARS2</i> and <i>LARS2</i> are important for the translation of mitochondrial proteins; and <i>TWNK</i> maintains mitochondrial DNA. It is likely that defects of mitochondrial translation and protein homeostasis in the inner ear and ovary underlie the pathogenesis of Perrault syndrome.</p><div id="perrault.CLPP"><h4><i>CLPP</i></h4><p><b>Gene structure.</b>
<i>CLPP</i> is predicted to generate four protein-coding transcripts. The longest transcript of 1194 bp (<a href="/nuccore/NM_006012.2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_006012.2</a>) has six exons. For a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information, see <a href="/books/NBK242617/#perrault.molgen.TA">Table A</a>, <b>Gene</b>.</p><p><b>Pathogenic variants.</b> Initially three unrelated families &#x02013;all of Pakistani ethnicity &#x02013; were reported [<a class="bk_pop" href="#perrault.REF.jenkinson.2013.605">Jenkinson et al 2013</a>]. Affected individuals in each family were <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a> for the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> c.433A&#x0003e;C, c.440G&#x0003e;C, or c.270+4A&#x0003e;G. Fewer than ten additional families from across the world have subsequently been reported, with <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants. Six of ten <i>CLPP</i> pathogenic missense variants localized to a region of <i>CLPP</i> from amino acid residue 142 to 162, with a cluster around residues 144-147 [<a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>].</p><p><a class="bk_pop" href="#perrault.REF.theunissen.2016.203">Theunissen et al [2016]</a> reported three <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals who were <a class="def" href="/books/n/gene/glossary/def-item/compound-heterozygous/">compound heterozygous</a> for at least one <a class="def" href="/books/n/gene/glossary/def-item/loss-of-function/">loss-of-function</a> <i>CLPP</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> and had more severe phenotypes (see <a class="figpopup" href="/books/NBK242617/table/perrault.T.clpp_pathogenic_variants_disc/?report=objectonly" target="object" rid-figpopup="figperraultTclpppathogenicvariantsdisc" rid-ob="figobperraultTclpppathogenicvariantsdisc">Table 6</a>).</p><div id="perrault.T.clpp_pathogenic_variants_disc" class="table"><h3><span class="label">Table 6. </span></h3><div class="caption"><p><i>CLPP</i> Pathogenic Variants Discussed in This <i>GeneReview</i></p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.clpp_pathogenic_variants_disc/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.clpp_pathogenic_variants_disc_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change</th><th id="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change</th><th id="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.270+4A&#x0003e;G</td><td headers="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"></td><td headers="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_3" rowspan="3" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/nuccore/NM_006012.2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_006012<wbr style="display:inline-block"></wbr>​.2</a><br /><a href="/protein/NP_006003.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_006003<wbr style="display:inline-block"></wbr>​.1</a></td></tr><tr><td headers="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.433A&#x0003e;C</td><td headers="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Thr145Pro</td></tr><tr><td headers="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.440G&#x0003e;C</td><td headers="hd_h_perrault.T.clpp_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Cys147Ser</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Note on variant classification: Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin">Note on nomenclature: <i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b>
<i>CLPP</i> encodes ATP-dependent Clp protease proteolytic subunit, mitochondrial. The longest transcript encodes the longest isoform of CLPP with 277 amino acids (<a href="/protein/NP_006003.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_006003.1</a>).</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> Crystal-structure modeling suggests that many of the pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants would alter the structure of the CLPP barrel chamber that captures unfolded or misfolded proteins and exposes them to proteolysis [<a class="bk_pop" href="#perrault.REF.jenkinson.2013.605">Jenkinson et al 2013</a>].</p></div><div id="perrault.ERAL1"><h4><i>ERAL1</i></h4><p><b>Gene structure.</b>
<i>ERAL1</i> is predicted to generate three protein-coding transcripts. The longest transcript of 1923 bp (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_005702.3" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_005702.3</a>) is generated from ten exons. See <a href="/books/NBK242617/#perrault.molgen.TA">Table A</a>, <b>Gene</b> for a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information.</p><p><b>Pathogenic variants.</b> Three families from a genetically isolated population of Dutch ancestry have been reported [<a class="bk_pop" href="#perrault.REF.chatzispyrou.2017.2541">Chatzispyrou et al 2017</a>]. All <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals were <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a> for the <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variant c.707A&#x0003e;T.</p><div id="perrault.T.eral1_pathogenic_variants_dis" class="table"><h3><span class="label">Table 7. </span></h3><div class="caption"><p><i>ERAL1</i> Pathogenic Variants Discussed in This <i>GeneReview</i></p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.eral1_pathogenic_variants_dis/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.eral1_pathogenic_variants_dis_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_perrault.T.eral1_pathogenic_variants_dis_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change</th><th id="hd_h_perrault.T.eral1_pathogenic_variants_dis_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change</th><th id="hd_h_perrault.T.eral1_pathogenic_variants_dis_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_perrault.T.eral1_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.707A&#x0003e;T</td><td headers="hd_h_perrault.T.eral1_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Asn236Ile</td><td headers="hd_h_perrault.T.eral1_pathogenic_variants_dis_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_005702.2" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_005702<wbr style="display:inline-block"></wbr>​.2</a><br /><a href="https://www.ncbi.nlm.nih.gov/protein/NP_005693.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_005693<wbr style="display:inline-block"></wbr>​.1</a></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Note on variant classification: Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin">Note on nomenclature: <i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b>
<i>ERAL1</i> encodes GTPase Era, mitochondrial, an ERA-like protein 1 chaperone of mitochondrial 12s rRNA, which is essential for assembly of the mitochondrial 28s ribosomal subunit. The longest coding isoform of ERAL1 comprises 437 amino acids (<a href="https://www.ncbi.nlm.nih.gov/protein/NP_001304914" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_001304914</a>).</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> The c.707A&#x0003e;T variant is predicted to interfere with GTP binding in ERAL1 and, therefore, its interaction with mitochondrial 12S RNA. Affected individuals had reduced levels of mitochondrial 12S rRNA and of the 28S mitochondrial subunit.</p></div><div id="perrault.HARS2"><h4><i>HARS2</i></h4><p><b>Gene structure.</b>
<i>HARS2</i> is predicted to be expressed as a number of transcript variants. The longest transcript of 2515 bp (<a href="/nuccore/NM_012208.3" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_012208.3</a>) is generated from 13 exons.</p><p>For a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information, see <a href="/books/NBK242617/#perrault.molgen.TA">Table A</a>, <b>Gene</b>.</p><p><b>Pathogenic variants.</b> Three families with Perrault syndrome with pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants in <i>HARS2</i> have been reported. Two unrelated Moroccan families were reported with both <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a> for c.1010A&#x0003e;G, which is suspected to be a <a class="def" href="/books/n/gene/glossary/def-item/founder-variant/">founder variant</a> due to shared haplotypes between affected individuals [<a class="bk_pop" href="#perrault.REF.lerat.2016.1354">Lerat et al 2016</a>]. In a single family of European descent affected individuals were compound heterozygotes for c.598C&#x0003e;G and c.1102G&#x0003e;T [<a class="bk_pop" href="#perrault.REF.pierce.2011.6543">Pierce et al 2011</a>].</p><div id="perrault.T.hars2_pathogenic_variants_dis" class="table"><h3><span class="label">Table 8. </span></h3><div class="caption"><p><i>HARS2</i> Pathogenic Variants Discussed in This <i>GeneReview</i></p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.hars2_pathogenic_variants_dis/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.hars2_pathogenic_variants_dis_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change</th><th id="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change</th><th id="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.598C&#x0003e;G&#x000a0;<sup>1</sup></td><td headers="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Leu200Val&#x000a0;<sup>1</sup></td><td headers="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_3" rowspan="3" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/nuccore/NM_012208.3" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_012208<wbr style="display:inline-block"></wbr>​.3</a><br /><a href="/protein/NP_036340.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_036340<wbr style="display:inline-block"></wbr>​.1</a></td></tr><tr><td headers="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1102G&#x0003e;T</td><td headers="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Val368Leu</td></tr><tr><td headers="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1010A&#x0003e;G</td><td headers="hd_h_perrault.T.hars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Tyr337Cys</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Note on variant classification: Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin">Note on nomenclature: <i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd><dt>1. </dt><dd><div id="perrault.TF.8.1"><p class="no_margin">Created an alternate <a class="def" href="/books/n/gene/glossary/def-item/splice-site/">splice site</a> leading to <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> of 12 codons [<a class="bk_pop" href="#perrault.REF.pierce.2011.6543">Pierce et al 2011</a>]</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b>
<i>HARS2</i> encodes probable histidine--tRNA ligase, mitochondrial. Variable protein-coding <a class="def" href="/books/n/gene/glossary/def-item/isoforms/">isoforms</a> of HARS2 have been reported. The canonic transcript variant <a href="/nuccore/NM_012208.3" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_012208.3</a> encodes the 506 amino acids of the isoform <a href="/protein/NP_036340.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_036340.1</a>.</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> The pathogenic variants in <i>HARS2</i> reduced the aminoacylation activity of HARS2 [<a class="bk_pop" href="#perrault.REF.pierce.2011.6543">Pierce et al 2011</a>].</p></div><div id="perrault.HSD17B4"><h4><i>HSD17B4</i></h4><p><b>Gene structure.</b>
<i>HSD17B4</i> is predicted to encode multiple protein-coding transcripts. The canonic transcript comprises 2710 bp encoded by 24 exons (<a href="/nuccore/NM_000414.3" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_000414.3</a>). For a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information, see <a href="/books/NBK242617/#perrault.molgen.TA">Table A</a>, <b>Gene</b>.</p><p><b>Pathogenic variants.</b> Three families with Perrault syndrome with pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants in <i>HSD17B4</i> have been reported. Two <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> sisters were compound heterozygotes for a <a class="def" href="/books/n/gene/glossary/def-item/nonsense-variant/">nonsense</a> and a missense variant, c.1704T&#x0003e;A and c.650A&#x0003e;G, respectively [<a class="bk_pop" href="#perrault.REF.pierce.2010.282">Pierce et al 2010</a>]. A single individual with PS was reported as <a class="def" href="/books/n/gene/glossary/def-item/compound-heterozygous/">compound heterozygous</a> for the variants <i>HSD17B4</i> c.46G&#x0003e;A and c.244G&#x0003e;T [<a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>]. Affected individuals of a family were <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a> for the variant <a class="figpopup" href="/books/NBK242617/table/perrault.T.hsd17b4_pathogenic_variants_d/?report=objectonly" target="object" rid-figpopup="figperraultThsd17b4pathogenicvariantsd" rid-ob="figobperraultThsd17b4pathogenicvariantsd">c.298G&#x0003e;T</a> [<a class="bk_pop" href="#perrault.REF.chen.2017.91">Chen et al 2017</a>]. The variant c.46G&#x0003e;A is associated with D-bifunctional protein deficiency [<a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>] (see <a class="figpopup" href="/books/NBK242617/table/perrault.T.hsd17b4_pathogenic_variants_d/?report=objectonly" target="object" rid-figpopup="figperraultThsd17b4pathogenicvariantsd" rid-ob="figobperraultThsd17b4pathogenicvariantsd">Table 9</a>, footnote 1).</p><div id="perrault.T.hsd17b4_pathogenic_variants_d" class="table"><h3><span class="label">Table 9. </span></h3><div class="caption"><p><i>HSD17B4</i> Pathogenic Variants Discussed in This <i>GeneReview</i></p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.hsd17b4_pathogenic_variants_d/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.hsd17b4_pathogenic_variants_d_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change</th><th id="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change</th><th id="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.46G&#x0003e;A&#x000a0;<sup>1</sup></td><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Gly16Ser</td><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_3" rowspan="5" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/nuccore/NM_000414.3" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_000414<wbr style="display:inline-block"></wbr>​.3</a><br /><a href="/protein/NP_000405.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_000405<wbr style="display:inline-block"></wbr>​.1</a></td></tr><tr><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.244G&#x0003e;T</td><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Val82Phe</td></tr><tr><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.298G&#x0003e;T</td><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Ala100Ser</td></tr><tr><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.650A&#x0003e;G</td><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Tyr217Cys</td></tr><tr><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1704T&#x0003e;A</td><td headers="hd_h_perrault.T.hsd17b4_pathogenic_variants_d_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Tyr568Ter</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Note on variant classification: Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin">Note on nomenclature: <i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd><dt>1. </dt><dd><div id="perrault.TF.9.1"><p class="no_margin">The most common <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> causing D-bifunctional protein deficiency (see <a class="figpopup" href="/books/NBK242617/table/perrault.T.allelic_disorders/?report=objectonly" target="object" rid-figpopup="figperraultTallelicdisorders" rid-ob="figobperraultTallelicdisorders">Table 2</a>)</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b>
<i>HSD17B4</i> encodes the peroxisomal multifunctional enzyme type 2 protein (HSD17B4), also known as bifunctional enzyme, hydroxysteroid (17-beta) dehydrogenase 4 and D-bifunctional protein (DBP). It is involved in fatty acid beta-oxidation and steroid metabolism. The canonic isoform (<a href="/protein/NP_000405.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_000405.1</a>) has 736 amino acid residues.</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> The <i>HSD17B4</i> pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants in individuals with PS are predicted to destabilize the dehydrogenase <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a> of DBP or interrupt co-factor binding [<a class="bk_pop" href="#perrault.REF.pierce.2010.282">Pierce et al 2010</a>, <a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>]. Biallelic <a class="def" href="/books/n/gene/glossary/def-item/loss-of-function/">loss-of-function</a> or pathogenic missense variants in <i>HSD17B4</i> may cause the allelic disorder D-bifunctional protein deficiency (see <a class="figpopup" href="/books/NBK242617/table/perrault.T.allelic_disorders/?report=objectonly" target="object" rid-figpopup="figperraultTallelicdisorders" rid-ob="figobperraultTallelicdisorders">Table 2</a>), which is more severe than Perrault syndrome and usually fatal in childhood. The variants associated with Perrault syndrome are thought to be less deleterious than those causing D-bifunctional protein deficiency, thus resulting in the milder <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>.</p></div><div id="perrault.LARS2"><h4><i>LARS2</i></h4><p><b>Gene structure.</b> The longest <i>LARS2</i> transcript of 4203 bp (<a href="/nuccore/NM_015340.3" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_015340.3</a>) is generated from 21 exons. For a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information, see <a href="/books/NBK242617/#perrault.molgen.TA">Table A</a>, <b>Gene</b>.</p><p><b>Pathogenic variants.</b> A number of individuals with <a class="def" href="/books/n/gene/glossary/def-item/biallelic/">biallelic</a> pathogenic variants in <i>LARS2</i> have been reported. Initially two individuals with Perrault syndrome were reported [<a class="bk_pop" href="#perrault.REF.pierce.2013.614">Pierce et al 2013</a>]: one with a <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a> pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variant c.1565C&#x0003e;A, and another with a <a class="def" href="/books/n/gene/glossary/def-item/compound-heterozygous/">compound heterozygous</a> <a class="def" href="/books/n/gene/glossary/def-item/frameshift-variant/">frameshift variant</a> c.1077delT and missense variant c.1886C&#x0003e;T (<a class="figpopup" href="/books/NBK242617/table/perrault.T.lars2_pathogenic_variants_dis/?report=objectonly" target="object" rid-figpopup="figperraultTlars2pathogenicvariantsdis" rid-ob="figobperraultTlars2pathogenicvariantsdis">Table 10</a>). The transversion c.1565C&#x0003e;A has been identified as a homozygous variant in two additional families and as a compound heterozygous variant <i>in trans</i> to c.351G&#x0003e;C in another family [<a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>, <a class="bk_pop" href="#perrault.REF.zerkaoui.2017.200">Zerkaoui et al 2017</a>]. All four families with the variant c.1565C&#x0003e;A had low-frequency SNHL. The variant c.1886C&#x0003e;T has been reported in a second family <i>in trans</i> to the variant c.1358G&#x0003e;A. In a single family biallelic compound heterozygous variants in <i>LARS2</i> c.880G&#x0003e;A and c.1556C&#x0003e;T were associated with a neurologic <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>.</p><div id="perrault.T.lars2_pathogenic_variants_dis" class="table"><h3><span class="label">Table 10. </span></h3><div class="caption"><p><i>LARS2</i> Pathogenic Variants Discussed in This <i>GeneReview</i></p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.lars2_pathogenic_variants_dis/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.lars2_pathogenic_variants_dis_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change</th><th id="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change<br />(Alias&#x000a0;<sup>1</sup>)</th><th id="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.351G&#x0003e;C</td><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Met117Ile</td><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_3" rowspan="7" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/nuccore/NM_015340.3" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_015340<wbr style="display:inline-block"></wbr>​.3</a><br /><a href="/protein/NP_056155.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_056155<wbr style="display:inline-block"></wbr>​.1</a></td></tr><tr><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.880G&#x0003e;A</td><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Glu294Lys</td></tr><tr><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1077delT</td><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Ile360PhefsTer15<br />(Ile360fsTer)</td></tr><tr><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1358G&#x0003e;A</td><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg453Gln</td></tr><tr><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1556C&#x0003e;T</td><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Thr519Met</td></tr><tr><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1565C&#x0003e;A</td><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Thr522Asn&#x000a0;<sup>2</sup></td></tr><tr><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1886C&#x0003e;T</td><td headers="hd_h_perrault.T.lars2_pathogenic_variants_dis_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Thr629Met</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Note on variant classification: Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin">Note on nomenclature: <i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd><dt>1. </dt><dd><div id="perrault.TF.10.1"><p class="no_margin">Variant designation that does not conform to current naming conventions</p></div></dd><dt>2. </dt><dd><div id="perrault.TF.10.2"><p class="no_margin">See <a href="#perrault.GenotypePhenotype_Correlations">Genotype-Phenotype Correlations</a>.</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b>
<i>LARS2</i> encodes mitochondrial leucyl-tRNA synthetase 2. The longest coding isoform of LARS2 is 903 amino acids (<a href="/protein/NP_056155.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_056155.1</a>).</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> The p.Thr522Asn variant occurs in a catalytic <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a> and has been shown to reduce the aminoacylation activity of <i>LARS2</i> ninefold [<a class="bk_pop" href="#perrault.REF.riley.2016.49">Riley et al 2016</a>]. Many of the other reported variants also lie in the catalytic domain and are predicted to reduce aminoacylation activity. The p.Thr629Met variant is in a site adjacent to a conserved catalytic loop [<a class="bk_pop" href="#perrault.REF.pierce.2013.614">Pierce et al 2013</a>].</p></div><div id="perrault.TWNK"><h4><i>TWNK</i></h4><p><b>Gene structure.</b> The longest <i>TWNK</i> transcript and major splice variant (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_021830.4" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_021830.4</a>) comprises five exons and encodes the protein isoform twinkle (also known as isoform A).</p><p>Transcript variant <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_001163812.1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_001163812.1</a> is a minor splice variant that encodes the distinct protein isoform known as twinky (also known as isoform B). This transcript results from the use of a downstream <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> 4 splice-donor site and leads to a 43-bp <a class="def" href="/books/n/gene/glossary/def-item/insertion/">insertion</a> between the regular exon 4 - exon 5 sequence, which causes a premature stop codon [<a href="https://www.ncbi.nlm.nih.gov/books/NBK3795/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Spelbrink et al 2001</a>]. See <a href="/books/NBK242617/#perrault.molgen.TA">Table A</a>, <b>Gene</b> for a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information.</p><p><b>Pathogenic variants.</b> Biallelic variants in <i>TWNK</i> associated with Perrault syndrome were initially reported in <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals of two unrelated families who were compound heterozygotes for c.[1172G&#x0003e;A];[1754A&#x0003e;G] and c.[1321T&#x0003e;G];[1519G&#x0003e;A] [<a class="bk_pop" href="#perrault.REF.morino.2014.2054">Morino et al 2014</a>]. Additional affected individuals and variants have been reported, including c.1196A&#x0003e;G, which was reported in two unrelated families <i>in trans</i> to the variants c.968G&#x0003e;A [<a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>] and c.1802G&#x0003e;A [<a class="bk_pop" href="#perrault.REF.o_dak.2017.25">O&#x00142;dak et al 2017</a>]. A family was also reported to be <a class="def" href="/books/n/gene/glossary/def-item/homozygous/">homozygous</a> for the variant c. 793C&#x0003e;T. To date all affected individuals have reported neurologic features [<a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>, <a class="bk_pop" href="#perrault.REF.o_dak.2017.25">O&#x00142;dak et al 2017</a>].</p><div id="perrault.T.twnk_pathogenic_variants_disc" class="table"><h3><span class="label">Table 11. </span></h3><div class="caption"><p><i>TWNK</i> Pathogenic Variants Discussed in This <i>GeneReview</i></p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK242617/table/perrault.T.twnk_pathogenic_variants_disc/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__perrault.T.twnk_pathogenic_variants_disc_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change</th><th id="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change</th><th id="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1172G&#x0003e;A</td><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg391His</td><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_3" rowspan="8" colspan="1" style="text-align:left;vertical-align:middle;"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_021830.4" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_021830<wbr style="display:inline-block"></wbr>​.4</a><br /><a href="https://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=protein&#x00026;id=39725942" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_068602<wbr style="display:inline-block"></wbr>​.2</a></td></tr><tr><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1754A&#x0003e;G</td><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Asn585Ser</td></tr><tr><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1321T&#x0003e;G</td><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Trp441Gly</td></tr><tr><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1519G&#x0003e;A</td><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Val507Ile</td></tr><tr><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.968G&#x0003e;A</td><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg323Gln</td></tr><tr><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1196A&#x0003e;G</td><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Asn399Ser</td></tr><tr><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.1802G&#x0003e;A</td><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg601Gln</td></tr><tr><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c. 793C&#x0003e;T</td><td headers="hd_h_perrault.T.twnk_pathogenic_variants_disc_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Arg265Cys</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Note on variant classification: Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin">Note on nomenclature: <i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b>
<i>TWNK</i> encodes two <a class="def" href="/books/n/gene/glossary/def-item/splicing/">splicing</a> <a class="def" href="/books/n/gene/glossary/def-item/isoforms/">isoforms</a>, twinkle and twinkie. Twinkle (<a href="https://www.ncbi.nlm.nih.gov/protein/np_001157284" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_001157284.1</a>) comprises 684 amino acids and forms hexamer complexes. Twinky (<a href="https://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=protein&#x00026;id=39725942" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_068602.2</a>) comprises 582 amino acids, shares the first 578 amino acids with twinkle, and terminates with four alternative amino acids. Twinkle is the mitochondrial DNA replicative helicase and localizes to mitochondrial nucleosomes. Twinkle contains three functional domains: a 3-prime helicase region, required for mtDNA replication; a linker region involved in hexamer formation; and a 5-primase <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a>. Twinkie has a mitochondrial localization and its function is currently unknown [<a class="bk_pop" href="#perrault.REF.spelbrink.2001.223">Spelbrink et al 2001</a>].</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> Most of the pathogenic variants associated with Perrault syndrome are located in the linker and helicase domains of twinkle. Variants located in the linker region have been predicted to interfere with hexamer formation. Variants in the helicase region are predicted to affect the helicase function or interaction of subunits within the hexamer complex [<a class="bk_pop" href="#perrault.REF.morino.2014.2054">Morino et al 2014</a>, <a class="bk_pop" href="#perrault.REF.demain.2017.302">Demain et al 2017</a>, <a class="bk_pop" href="#perrault.REF.o_dak.2017.25">O&#x00142;dak et al 2017</a>].</p></div></div></div><div id="perrault.References"><h2 id="_perrault_References_">References</h2><div id="perrault.Literature_Cited"><h3>Literature Cited</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.ahmed.2015.149">Ahmed S, Jelani M, Alrayes N, Mohamoud HS, Almramhi MM, Anshasi W, Ahmed NA, Wang J, Nasir J, Al-Aama JY. Exome analysis identified a novel missense mutation in the CLPP gene in a consanguineous Saudi family expanding the clinical spectrum of Perrault syndrome type-3. <span><span class="ref-journal">J Neurol Sci. </span>2015;<span class="ref-vol">353</span>:149–54.</span> [<a href="/pubmed/25956234" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 25956234</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.chatzispyrou.2017.2541">Chatzispyrou IA, Alders M, Guerrero-Castillo S, Zapata Perez R, Haagmans MA, Mouchiroud L, Koster J, Ofman R, Baas F, Waterham HR, Spelbrink JN, Auwerx J, Mannens MM, Houtkooper RH, Plomp AS. A homozygous missense mutation in ERAL1, encoding a mitochondrial rRNA chaperone, causes Perrault syndrome. <span><span class="ref-journal">Hum Mol Genet. </span>2017;<span class="ref-vol">26</span>:2541–50.</span> [<a href="/pmc/articles/PMC5965403/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC5965403</span></a>] [<a href="/pubmed/28449065" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 28449065</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.chen.2017.91">Chen K, Yang K, Luo SS, Chen C, Wang Y, Wang YX, Li DK, Yang YJ, Tang YL, Liu FT, Wang J, Wu JJ, Sun YM. A homozygous missense variant in HSD17B4 identified in a consanguineous Chinese Han family with type II Perrault syndrome. <span><span class="ref-journal">BMC Med Genet. </span>2017;<span class="ref-vol">18</span>:91.</span> [<a href="/pmc/articles/PMC5568266/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC5568266</span></a>] [<a href="/pubmed/28830375" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 28830375</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.demain.2017.302">Demain LA, Urquhart JE, O'Sullivan J, Williams SG, Bhaskar SS, Jenkinson EM, Lourenco CM, Heiberg A, Pearce SH, Shalev SA, Yue WW, Mackinnon S, Munro KJ, Newbury-Ecob R, Becker K, Kim MJ, O'Keefe RT, Newman WG. Expanding the genotypic spectrum of Perrault syndrome. <span><span class="ref-journal">Clin Genet. </span>2017;<span class="ref-vol">91</span>:302–12.</span> [<a href="/pubmed/26970254" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26970254</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.demain.2018.276">Demain LAM, Antunes D, O'Sullivan J, Bhaskhar SS, O'Keefe RT, Newman WG. A known pathogenic variant in the essential mitochondrial translation gene RMND1 causes a Perrault-like syndrome with renal defects. <span><span class="ref-journal">Clin Genet. </span>2018;<span class="ref-vol">94</span>:276–7.</span> [<a href="/pubmed/29671881" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 29671881</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.de_vos.2010.911">De Vos M, Devroey P, Fauser BC. Primary ovarian insufficiency. <span><span class="ref-journal">Lancet. </span>2010;<span class="ref-vol">376</span>:911–21.</span> [<a href="/pubmed/20708256" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20708256</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.faridi.2017.328">Faridi R, Rehman AU, Morell RJ, Friedman PL, Demain L, Zahra S, Khan AA, Tohlob D, Assir MZ, Beaman G, Khan SN, Newman WG, Riazuddin S, Friedman TB. Mutations of SGO2 and CLDN14 collectively cause coincidental Perrault syndrome. <span><span class="ref-journal">Clin Genet. </span>2017;<span class="ref-vol">91</span>:328–32.</span> [<a href="/pmc/articles/PMC5272805/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC5272805</span></a>] [<a href="/pubmed/27629923" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 27629923</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.jenkinson.2012.974">Jenkinson EM, Clayton-Smith J, Mehta S, Bennett C, Reardon W, Green A, Pearce SH, De Michele G, Conway GS, Cilliers D, Moreton N, Davis JR, Trump D, Newman WG. Perrault syndrome: further evidence for genetic heterogeneity. <span><span class="ref-journal">J Neurol. </span>2012;<span class="ref-vol">259</span>:974–6.</span> [<a href="/pubmed/22037954" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 22037954</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.jenkinson.2013.605">Jenkinson EM, Rehman AU, Walsh T, Clayton-Smith J, Lee K, Morell RJ, Drummond MC, Khan SN, Naeem MA, Rauf B, Billington N, Schultz JM, Urquhart JE, Lee MK, Berry A, Hanley NA, Mehta S, Cilliers D, Clayton PE, Kingston H, Smith MJ, Warner TT, Black GC, Trump D, Davis JR, Ahmad W, Leal SM, Riazuddin S, King MC, Friedman TB, Newman WG, et al.  Perrault syndrome is caused by recessive mutations in CLPP, encoding a mitochondrial ATP-dependent chambered protease. <span><span class="ref-journal">Am J Hum Genet. </span>2013;<span class="ref-vol">92</span>:605–13.</span> [<a href="/pmc/articles/PMC3617381/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3617381</span></a>] [<a href="/pubmed/23541340" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23541340</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.king.2007.831">King KA, Makishima T, Zalewski CK, Bakalov VK, Griffith AJ, Bondy CA, Brewer CC. Analysis of auditory phenotype and karyotype in 200 females with Turner syndrome. <span><span class="ref-journal">Ear Hear. </span>2007;<span class="ref-vol">28</span>:831–41.</span> [<a href="/pubmed/17982369" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17982369</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.kosaki.2018.404">Kosaki R, Horikawa R, Fujii E, Kosaki K. Biallelic mutations in LARS2 can cause Perrault syndrome type 2 with neurologic symptoms. <span><span class="ref-journal">Am J Med Genet A. </span>2018;<span class="ref-vol">176</span>:404–8.</span> [<a href="/pubmed/29205794" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 29205794</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.lerat.2016.1354">Lerat J, Jonard L, Loundon N, Christin-Maitre S, Lacombe D, Goizet C, Rouzier C, Van Maldergem L, Gherbi S, Garabedian EN, Bonnefont JP, Touraine P, Mosnier I, Munnich A, Denoyelle F, Marlin S. An application of NGS for molecular investigations in Perrault syndrome: study of 14 families and review of the literature. <span><span class="ref-journal">Hum Mutat. </span>2016;<span class="ref-vol">37</span>:1354–62.</span> [<a href="/pubmed/27650058" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 27650058</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.morino.2014.2054">Morino H, Pierce SB, Matsuda Y, Walsh T, Ohsawa R, Newby M, Hiraki-Kamon K, Kuramochi M, Lee MK, Klevit RE, Martin A, Maruyama H, King MC, Kawakami H. Mutations in Twinkle primase-helicase cause Perrault syndrome with neurologic features. <span><span class="ref-journal">Neurology. </span>2014;<span class="ref-vol">83</span>:2054–61.</span> [<a href="/pmc/articles/PMC4248451/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC4248451</span></a>] [<a href="/pubmed/25355836" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 25355836</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.ng.2016.768">Ng YS, Alston CL, Diodato D, Morris AA, Ulrick N, Kmoch S, Houstek J, Martinelli D, Haghighi A, Atiq M, Gamero MA, Garcia-Martinez E, Kratochvilova H, Santra S, Brown RM, Brown GK, Ragge N, Monavari A, Pysden K, Ravn K, Casey JP, Khan A, Chakrapani A, Vassallo G, Simons C, McKeever K, O'Sullivan S, Childs AM, Ostergaard E, Vanderver A, Goldstein A, Vogt J, Taylor RW, McFarland R. The clinical, biochemical and genetic features associated with RMND1-related mitochondrial disease. <span><span class="ref-journal">J Med Genet. </span>2016;<span class="ref-vol">53</span>:768–75.</span> [<a href="/pmc/articles/PMC5264221/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC5264221</span></a>] [<a href="/pubmed/27412952" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 27412952</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.o_dak.2017.25">O&#x00142;dak M, Ozi&#x00119;b&#x00142;o D, Pollak A, St&#x00119;pniak I, Lazniewski M, Lechowicz U, Kochanek K, Furmanek M, Tacikowska G, Plewczynski D, Wolak T, P&#x00142;oski R, Skar&#x0017c;y&#x00144;ski H. Novel neuro-audiological findings and further evidence for TWNK involvement in Perrault syndrome. <span><span class="ref-journal">J Transl Med. </span>2017;<span class="ref-vol">15</span>:25.</span> [<a href="/pmc/articles/PMC5299684/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC5299684</span></a>] [<a href="/pubmed/28178980" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 28178980</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.oliveira.2013.226">Oliveira CS, Ribeiro FM, Lago R, Alves C. Audiological abnormalities in patients with turner syndrome. <span><span class="ref-journal">Am J Audiol. </span>2013;<span class="ref-vol">22</span>:226–32.</span> [<a href="/pubmed/23824435" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23824435</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.pierce.2011.6543">Pierce SB, Chisholm KM, Lynch ED, Lee MK, Walsh T, Opitz JM, Li W, Klevit RE, King MC. Mutations in mitochondrial histidyl tRNA synthetase HARS2 cause ovarian dysgenesis and sensorineural hearing loss of Perrault syndrome. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2011;<span class="ref-vol">108</span>:6543–8.</span> [<a href="/pmc/articles/PMC3081023/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3081023</span></a>] [<a href="/pubmed/21464306" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 21464306</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.pierce.2013.614">Pierce SB, Gersak K, Michaelson-Cohen R, Walsh T, Lee MK, Malach D, Klevit RE, King MC, Levy-Lahad E. Mutations in LARS2, encoding mitochondrial leucyl-tRNA synthetase, lead to premature ovarian failure and hearing loss in Perrault syndrome. <span><span class="ref-journal">Am J Hum Genet. </span>2013;<span class="ref-vol">92</span>:614–20.</span> [<a href="/pmc/articles/PMC3617377/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3617377</span></a>] [<a href="/pubmed/23541342" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23541342</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.pierce.2010.282">Pierce SB, Walsh T, Chisholm KM, Lee MK, Thornton AM, Fiumara A, Opitz JM, Levy-Lahad E, Klevit RE, King MC. Mutations in the DBP-deficiency protein HSD17B4 cause ovarian dysgenesis, hearing loss, and ataxia of Perrault Syndrome. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:282–8.</span> [<a href="/pmc/articles/PMC2917704/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2917704</span></a>] [<a href="/pubmed/20673864" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20673864</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.riley.2016.49">Riley LG, Rudinger-Thirion J, Schmitz-Abe K, Thorburn DR, Davis RL, Teo J, Arbuckle S, Cooper ST, Campagna DR, Frugier M, Markianos K, Sue CM, Fleming MD, Christodoulou J. LARS2 variants associated with hydrops, lactic acidosis, sideroblastic anemia, and multisystem failure. <span><span class="ref-journal">JIMD Rep. </span>2016;<span class="ref-vol">28</span>:49–57.</span> [<a href="/pmc/articles/PMC5059179/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC5059179</span></a>] [<a href="/pubmed/26537577" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 26537577</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.spelbrink.2001.223">Spelbrink JN, Li FY, Tiranti V, Nikali K, Yuan QP, Tariq M, Wanrooij S, Garrido N, Comi G, Morandi L, Santoro L, Toscano A, Fabrizi GM, Somer H, Croxen R, Beeson D, Poulton J, Suomalainen A, Jacobs HT, Zeviani M, Larsson C. Human mitochondrial DNA deletions associated with mutations in the gene encoding Twinkle, a phage T7 gene 4-like protein localized in mitochondria. <span><span class="ref-journal">Nat Genet. </span>2001;<span class="ref-vol">28</span>:223–31.</span> [<a href="/pubmed/11431692" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11431692</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.theunissen.2016.203">Theunissen TE, Szklarczyk R, Gerards M, Hellebrekers DM, Mulder-Den Hartog EN, Vanoevelen J, Kamps R, de Koning B, Rutledge SL, Schmitt-Mechelke T, van Berkel CG, van der Knaap MS, de Coo IF, Smeets HJ. Specific MRI abnormalities reveal severe Perrault syndrome due to CLPP defects. <span><span class="ref-journal">Front Neurol. </span>2016;<span class="ref-vol">7</span>:203.</span> [<a href="/pmc/articles/PMC5110515/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC5110515</span></a>] [<a href="/pubmed/27899912" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 27899912</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.van_grunsven.1999.99">van Grunsven EG, van Berkel E, Mooijer PA, Watkins PA, Moser HW, Suzuki Y, Jiang LL, Hashimoto T, Hoefler G, Adamski J, Wanders RJ. Peroxisomal bifunctional protein deficiency revisited: resolution of its true enzymatic and molecular basis. <span><span class="ref-journal">Am J Hum Genet. </span>1999;<span class="ref-vol">64</span>:99–107.</span> [<a href="/pmc/articles/PMC1377707/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1377707</span></a>] [<a href="/pubmed/9915948" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9915948</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="perrault.REF.zerkaoui.2017.200">Zerkaoui M, Demain LAM, Cherkaoui Jaouad I, Ratbi I, Amjoud K, Urquhart JE, O'Sullivan J, Newman WG, Sefiani A. Marfanoid habitus is a nonspecific feature of Perrault syndrome. <span><span class="ref-journal">Clin Dysmorphol. </span>2017;<span class="ref-vol">26</span>:200–4.</span> [<a href="/pubmed/28832386" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 28832386</span></a>]</div></li></ul></div></div><div id="perrault.Chapter_Notes"><h2 id="_perrault_Chapter_Notes_">Chapter Notes</h2><div id="perrault.Author_Notes"><h3>Author Notes</h3><p>Bill Newman is Professor of Translational Genomic Medicine at the University of Manchester. His main interests are in applying new technologies in clinical practice, the characterization of rare diseases, and pharmacogenetics. Web: <a href="http://www.manchester.ac.uk/research/william.newman" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.manchester.ac.uk/research/william.newman</a></p><p>Tom Friedman is Chief of the Laboratory of Molecular Genetics at the NIDCD/NIH. His goal is to identify and study the functions of genes associated with <a class="def" href="/books/n/gene/glossary/def-item/syndromic/">syndromic</a> and nonsyndromic forms of deafness.</p><p>Gerard Conway is Professor of Reproductive Endocrinology at the Institute of Women's Health, University College London, UK. His clinical research interests are in the fields of reproductive endocrinology and disorders of development.</p><p>Leigh Demain is a Research Associate at the University of Manchester. Her research focuses on the genetic and molecular causes of Perrault syndrome.</p></div><div id="perrault.Acknowledgments"><h3>Acknowledgments</h3><p>Prof Newman&#x02019;s work is supported by Action on Hearing Loss. Prof Friedman is supported by the Intramural Program of the NIDCD.</p></div><div id="perrault.Revision_History"><h3>Revision History</h3><ul><li class="half_rhythm"><div>6 September 2018 (bp) Comprehensive update posted live</div></li><li class="half_rhythm"><div>25 September 2014 (me) Review posted live</div></li><li class="half_rhythm"><div>10 March 2014 (wn) Original submission</div></li></ul></div></div><div id="bk_toc_contnr"></div></div></div>
            <div class="post-content"><div><div class="half_rhythm"><a href="/books/about/copyright/">Copyright</a> © 1993-2019, University of Washington, Seattle. GeneReviews is
a registered trademark of the University of Washington, Seattle. All rights
reserved.<p class="small">GeneReviews® chapters are owned by the University of Washington. Permission is
hereby granted to reproduce, distribute, and translate copies of content materials for
noncommercial research purposes only, provided that (i) credit for source (<a href="http://www.genereviews.org/" ref="pagearea=meta&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">http://www.genereviews.org/</a>) and copyright (© 1993-2019 University of
Washington) are included with each copy; (ii) a link to the original material is provided
whenever the material is published elsewhere on the Web; and (iii) reproducers,
distributors, and/or translators comply with the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>. No further modifications are allowed. For clarity, excerpts
of GeneReviews chapters for use in lab reports and clinic notes are a permitted
use.</p><p class="small">For more information, see the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>.</p><p class="small">For questions regarding permissions or whether a specified use is allowed,
contact: <a href="mailto:dev@null" data-email="ude.wu@tssamda" class="oemail">ude.wu@tssamda</a>.</p></div><div class="small"><span class="label">Bookshelf ID: NBK242617</span><span class="label">PMID: <a href="/pubmed/25254289" title="PubMed record of this page" ref="pagearea=meta&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">25254289</a></span></div><div style="margin-top:2em" class="bk_noprnt"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/dmn/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/perry/" title="Next page in this title">Next &gt;</a></div></div></div></div>
            
        </div>

        <!-- Custom content below content -->
        <div class="col4">
            
        </div>
        
        
        <!-- Book content -->
        
        <!-- Custom contetnt below bottom nav -->
        <div class="col5">
            
        </div>
    </div>

    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            <div id="ncbi_share_book"><a href="#" class="ncbi_share" data-ncbi_share_config="popup:false,shorten:true" ref="id=NBK242617&amp;db=books">Share</a></div>

        </div>
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Views</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PDF_download" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/NBK242617/?report=reader">PubReader</a></li><li><a href="/books/NBK242617/?report=printable">Print View</a></li><li><a data-jig="ncbidialog" href="#_ncbi_dlg_citbx_NBK242617" data-jigconfig="width:400,modal:true">Cite this Page</a><div id="_ncbi_dlg_citbx_NBK242617" style="display:none" title="Cite this Page"><div class="bk_tt">Newman WG, Friedman TB, Conway GS, et al. Perrault Syndrome. 2014 Sep 25 [Updated 2018 Sep 6]. In: Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. <span class="bk_cite_avail"></span></div></div></li><li><a href="/books/NBK242617/pdf/Bookshelf_NBK242617.pdf">PDF version of this page</a> (493K)</li><li><a href="#" class="toggle-glossary-link" title="Enable/disable links to the glossary">Disable Glossary Links</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>In this GeneReview</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="page-toc" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="#perrault.Summary" ref="log$=inpage&amp;link_id=inpage">Summary</a></li><li><a href="#perrault.Diagnosis" ref="log$=inpage&amp;link_id=inpage">Diagnosis</a></li><li><a href="#perrault.Clinical_Characteristics" ref="log$=inpage&amp;link_id=inpage">Clinical Characteristics</a></li><li><a href="#perrault.Genetically_Related_Allelic_Dis" ref="log$=inpage&amp;link_id=inpage">Genetically Related (Allelic) Disorders</a></li><li><a href="#perrault.Differential_Diagnosis" ref="log$=inpage&amp;link_id=inpage">Differential Diagnosis</a></li><li><a href="#perrault.Management" ref="log$=inpage&amp;link_id=inpage">Management</a></li><li><a href="#perrault.Genetic_Counseling" ref="log$=inpage&amp;link_id=inpage">Genetic Counseling</a></li><li><a href="#perrault.Resources" ref="log$=inpage&amp;link_id=inpage">Resources</a></li><li><a href="#perrault.Molecular_Genetics" ref="log$=inpage&amp;link_id=inpage">Molecular Genetics</a></li><li><a href="#perrault.References" ref="log$=inpage&amp;link_id=inpage">References</a></li><li><a href="#perrault.Chapter_Notes" ref="log$=inpage&amp;link_id=inpage">Chapter Notes</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>GeneReviews Links</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="source-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/n/gene/advanced/"><i>GeneReviews</i> Advanced Search</a></li><li><a href="/books/n/gene/glossary/"><i>GeneReviews</i> Glossary</a></li><li><a href="/books/n/gene/resource_mats/">Resource Materials</a> <span class="bk_hlight1">NEW FEATURE</span></li><li><a href="/books/n/gene/updates/">New in <i>GeneReviews</i></a></li><li><a href="/books/n/gene/authors/">Author List</a></li><li><a href="/books/n/gene/prospective_authors/">For Current/Prospective Authors</a></li><li><a href="/books/n/gene/GRpersonnel/"><i>GeneReviews</i> Personnel</a></li><li><a href="/books/n/gene/howto_linkin/">Download/Link to <i>GeneReviews</i></a></li><li><a href="/books/n/gene/contact_us/">Contact Us</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Tests in GTR by Gene</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="document-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=23438[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">HARS2</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=8192[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">CLPP</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=3295[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">HSD17B4</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=26284[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">ERAL1</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=23395[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">LARS2</a></li><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=56652[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">TWNK</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Related information</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="discovery_db_links" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=omim&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_omim&amp;IdsFromResult=3299472" ref="log$=recordlinks">OMIM</a><div class="brieflinkpop offscreen_noflow">Related OMIM records</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pmc&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pmc_refs&amp;IdsFromResult=3299472" ref="log$=recordlinks">PMC</a><div class="brieflinkpop offscreen_noflow">PubMed Central citations</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pubmed&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pubmed_refs&amp;IdsFromResult=3299472" ref="log$=recordlinks">PubMed</a><div class="brieflinkpop offscreen_noflow">Links to PubMed</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=gene&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_gene&amp;IdsFromResult=3299472" ref="log$=recordlinks">Gene</a><div class="brieflinkpop offscreen_noflow">Locus Links</div></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/26970254" ref="ordinalpos=1&amp;linkpos=1&amp;log$=relatedarticles&amp;logdbfrom=pubmed">Expanding the genotypic spectrum of Perrault syndrome.</a><span class="source">[Clin Genet. 2017]</span><div class="brieflinkpop offscreen_noflow">Expanding the genotypic spectrum of Perrault syndrome.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Demain LA, Urquhart JE, O'Sullivan J, Williams SG, Bhaskar SS, Jenkinson EM, Lourenco CM, Heiberg A, Pearce SH, Shalev SA, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Clin Genet. 2017 Feb; 91(2):302-312. Epub 2016 Apr 1.</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/20301444" ref="ordinalpos=1&amp;linkpos=2&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Alström Syndrome</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Alström Syndrome<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Paisey RB, Steeds R, Barrett T, Williams D, Geberhiwot T, Gunay-Aygun M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/29205794" ref="ordinalpos=1&amp;linkpos=3&amp;log$=relatedarticles&amp;logdbfrom=pubmed">Biallelic mutations in LARS2 can cause Perrault syndrome type 2 with neurologic symptoms.</a><span class="source">[Am J Med Genet A. 2018]</span><div class="brieflinkpop offscreen_noflow">Biallelic mutations in LARS2 can cause Perrault syndrome type 2 with neurologic symptoms.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Kosaki R, Horikawa R, Fujii E, Kosaki K. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Am J Med Genet A. 2018 Feb; 176(2):404-408. Epub 2017 Dec 3.</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/25719194" ref="ordinalpos=1&amp;linkpos=4&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">TBC1D24</i>-Related Disorders</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> <i xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">TBC1D24</i>-Related Disorders<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Mucha BE, Hennekam RCM, Sisodiya S, Campeau PM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/25590127" ref="ordinalpos=1&amp;linkpos=5&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Deafness and Myopia Syndrome</a><span class="source">[GeneReviews<sup>®</sup>. 1993]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Deafness and Myopia Syndrome<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Ordonez JL, Tekin M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">GeneReviews®. 1993</em></div></div></li></ul><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=25254289" ref="ordinalpos=1&amp;log$=relatedreviews_seeall&amp;logdbfrom=pubmed">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=25254289" ref="ordinalpos=1&amp;log$=relatedarticles_seeall&amp;logdbfrom=pubmed">See all...</a></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Recent Activity</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="Shutter"></a></div><div class="portlet_content"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class=""><div class="action"><a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a></div><ul id="activity"><li class="ra_rcd ralinkpopper two_line"><a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5e041a8039a66ee2ba6e35d6">Perrault Syndrome - GeneReviews®</a><div class="ralinkpop offscreen_noflow">Perrault Syndrome - GeneReviews®<div class="brieflinkpopdesc"></div></div><div class="tertiary"></div></li></ul><p class="HTOn">Your browsing activity is empty.</p><p class="HTOff">Activity recording is turned off.</p><p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p><a class="seemore" href="/sites/myncbi/recentactivity">See more...</a></div></div></div>
        
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script type="text/javascript" src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script type="text/javascript">
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-12-25T21:27:12-05:00&amp;Snapshot=%2Fprojects%2Fbooks%2FPBooks@5.22&amp;Host=portal105&amp;ncbi_phid=CE8CBE8AE040DDA100000000055C01F9&amp;ncbi_session=CE8CBE8AE041A7F1_1372SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK242617%2F&amp;Db=pmc&amp;folderID=134&amp;Ncbi_App=bookshelf&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        </div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=books&amp;ncbi_pdid=book-part&amp;ncbi_acc=NBK242617&amp;ncbi_domain=gene&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/NBK242617/&amp;ncbi_pagename=Perrault Syndrome - GeneReviews® - NCBI Bookshelf&amp;ncbi_bookparttype=chapter&amp;ncbi_app=bookshelf" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8CBE8AE041A7F1_1372SID /projects/books/PBooks@5.22 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/js/3879255/4121861/3501987/4008961/3893018/3821238/4062932/4123007/4087685/4076480/3921943/3400083/3426610.js" snapshot="books"></script></body>
</html>