<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    
    <head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <!-- AppResources meta begin -->
        <meta name="paf-app-resources" content="" />
                 <script type="text/javascript">var ncbi_startTime = new Date();</script>
            
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="" />

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="books" /><meta name="ncbi_pdid" content="book-part" /><meta name="ncbi_acc" content="NBK5189" /><meta name="ncbi_domain" content="gene" /><meta name="ncbi_report" content="record" /><meta name="ncbi_type" content="fulltext" /><meta name="ncbi_objectid" content="" /><meta name="ncbi_pcid" content="/NBK5189/" /><meta name="ncbi_pagename" content="PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia - GeneReviews® - NCBI Bookshelf" /><meta name="ncbi_bookparttype" content="chapter" /><meta name="ncbi_app" content="bookshelf" />
        <!-- Logger end -->
        
        <title>PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia - GeneReviews® - NCBI Bookshelf</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css" /><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,FOLLOW,NOARCHIVE" /><meta name="citation_inbook_title" content="GeneReviews® [Internet]" /><meta name="citation_title" content="PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia" /><meta name="citation_publisher" content="University of Washington, Seattle" /><meta name="citation_date" content="2014/08/14" /><meta name="citation_author" content="William B Dobyns" /><meta name="citation_author" content="Soma Das" /><meta name="citation_pmid" content="20301752" /><meta name="citation_fulltext_html_url" content="https://www.ncbi.nlm.nih.gov/books/NBK5189/" /><meta name="citation_keywords" content="LIS1-Associated Lissencephaly/Subcortical Band Heterotopia" /><meta name="citation_keywords" content="LIS1-Associated Lissencephaly/Subcortical Band Heterotopia" /><meta name="citation_keywords" content="Miller-Dieker Syndrome" /><meta name="citation_keywords" content="Isolated 17-Linked Lissencephaly" /><meta name="citation_keywords" content="17-Linked Subcortical Band Heterotopia" /><meta name="citation_keywords" content="Platelet-activating factor acetylhydrolase IB subunit alpha" /><meta name="citation_keywords" content="PAFAH1B1" /><meta name="citation_keywords" content="PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="DC.Title" content="PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="University of Washington, Seattle" /><meta name="DC.Contributor" content="William B Dobyns" /><meta name="DC.Contributor" content="Soma Das" /><meta name="DC.Date" content="2014/08/14" /><meta name="DC.Identifier" content="https://www.ncbi.nlm.nih.gov/books/NBK5189/" /><meta name="description" content="PAFAH1B1-associated lissencephaly includes Miller-Dieker syndrome (MDS), isolated lissencephaly sequence (ILS), and (rarely) subcortical band heterotopia (SBH). Lissencephaly and SBH are cortical malformations caused by deficient neuronal migration during embryogenesis. Lissencephaly refers to a &quot;smooth brain&quot; with absent gyri (agyria) or abnormally wide gyri (pachygyria). SBH refers to a band of heterotopic gray matter located just beneath the cortex and separated from it by a thin zone of normal white matter. MDS is characterized by lissencephaly, typical facial features, and severe neurologic abnormalities. ILS is characterized by lissencephaly and its direct sequelae: developmental delay, intellectual disability, and seizures." /><meta name="og:title" content="PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia" /><meta name="og:type" content="book" /><meta name="og:description" content="PAFAH1B1-associated lissencephaly includes Miller-Dieker syndrome (MDS), isolated lissencephaly sequence (ILS), and (rarely) subcortical band heterotopia (SBH). Lissencephaly and SBH are cortical malformations caused by deficient neuronal migration during embryogenesis. Lissencephaly refers to a &quot;smooth brain&quot; with absent gyri (agyria) or abnormally wide gyri (pachygyria). SBH refers to a band of heterotopic gray matter located just beneath the cortex and separated from it by a thin zone of normal white matter. MDS is characterized by lissencephaly, typical facial features, and severe neurologic abnormalities. ILS is characterized by lissencephaly and its direct sequelae: developmental delay, intellectual disability, and seizures." /><meta name="og:url" content="https://www.ncbi.nlm.nih.gov/books/NBK5189/" /><meta name="og:site_name" content="NCBI Bookshelf" /><meta name="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" /><meta name="twitter:card" content="summary" /><meta name="twitter:site" content="@ncbibooks" /><meta name="bk-non-canon-loc" content="/books/n/gene/chrom17-lis/" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/books/NBK5189/" /><link rel="stylesheet" href="/corehtml/pmc/css/figpopup.css" type="text/css" media="screen" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/bookshelf/2.22/css/books_print.min.css" type="text/css" media="print" /><style type="text/css">p a.figpopup{display:inline !important} .bk_tt {font-family: monospace}  .first-line-outdent .bk_ref {display: inline}  .body-content h2, .body-content .h2  {border-bottom: 1px solid #97B0C8} .body-content h2.inline {border-bottom: none} a.page-toc-label , .jig-ncbismoothscroll a {text-decoration:none;border:0 !important} .temp-labeled-list  .graphic {display:inline-block !important} .temp-labeled-list  img{width:100%}</style><script type="text/javascript" src="/corehtml/pmc/js/jquery.hoverIntent.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"> </script><script type="text/javascript">window.name="mainwindow";</script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/book-toc.min.js"> </script><script type="text/javascript" src="/corehtml/pmc/js/bookshelf/2.22/books.min.js"> </script><meta name="book-collection" content="NONE" />

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="CE8BED77E0408B710000000005E2021E.m_12" />
<meta name='referrer' content='origin-when-cross-origin'/><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4098876/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/4008682/3881636/3579733/4062871/12930/3964959/3854974/36029/4128070/9685/3549676/3609192/3609193/3609213/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/css/3411343/3882866.css" media="print" /></head>
    <body class="book-part">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo"><h1 class="res_name"><a href="/books/" title="Bookshelf home">Bookshelf</a></h1><h2 class="res_tagline"></h2></div>
    <div class="search"><form method="get" action="/books/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="books" selected="selected" class="last" data-ac_dict="bookshelf-search">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books" data-ac_dict="bookshelf-search">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search Books. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'bookshelf-search',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'no'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/books/browse/">Browse Titles</a>
                    </li><li>
                        <a href="/books/advanced/">Advanced</a>
                    </li><li class="help">
                        <a href="/books/NBK3833/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <!-- Custom content 1 -->
<div class="col1">
    
</div>

<div class="container">
    <div id="maincontent" class="content eight_col col">
        <!-- Custom content in the left column above book nav -->
        <div class="col2">
            
        </div>
        
        <!-- Book content -->
        

        <!-- Custom content between navigation and content -->
        <div class="col3">
            
        </div>
        
        <div class="document">
            <div class="pre-content"><div><div class="bk_prnt"><p class="small">NCBI Bookshelf. A service of the National Library of Medicine, National Institutes of Health.</p><p>Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. </p></div><div class="iconblock clearfix whole_rhythm no_top_margin bk_noprnt"><a class="img_link icnblk_img" title="All GeneReviews" href="/books/n/gene/"><img class="source-thumb" src="/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-gene-lrg.png" alt="Cover of GeneReviews®" height="100px" width="80px" /></a><div class="icnblk_cntnt eight_col"><h2>GeneReviews<sup>®</sup> [Internet].</h2><a data-jig="ncbitoggler" href="#__NBK5189_dtls__">Show details</a><div style="display:none" class="ui-widget" id="__NBK5189_dtls__"><div>Adam MP, Ardinger HH, Pagon RA, et al., editors.</div><div>Seattle (WA): <a href="http://www.washington.edu" ref="pagearea=page-banner&amp;targetsite=external&amp;targetcat=link&amp;targettype=publisher">University of Washington, Seattle</a>; 1993-2019.</div></div><div class="half_rhythm"><ul class="inline_list"><li style="margin-right:1em"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a></li></ul></div><div class="bk_noprnt"><form method="get" action="/books/n/gene/" id="bk_srch"><div class="bk_search"><label for="bk_term" class="offscreen_noflow">Search term</label><input type="text" title="Search GeneReviews" id="bk_term" name="term" value="" data-jig="ncbiclearbutton" /> <input type="submit" class="jig-ncbibutton" value="Search GeneReviews" submit="false" style="padding: 0.1em 0.4em;" /></div></form><div><ul class="inline_list"><li><a href="/books/n/gene/advanced/">GeneReviews Advanced Search</a></li><li style="margin-left:.5em"><a href="/books/n/gene/helpadvsearch/">Help</a></li></ul></div></div></div><div class="icnblk_cntnt two_col"><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/otc-def/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/papr/" title="Next page in this title">Next &gt;</a></div></div></div></div></div>
            <div class="main-content lit-style" itemscope="itemscope" itemtype="http://schema.org/CreativeWork"><div class="meta-content fm-sec"><h1 id="_NBK5189_"><span class="title" itemprop="name"><i>PAFAH1B1</i>-Associated Lissencephaly/Subcortical Band Heterotopia</span></h1><div itemprop="alternativeHeadline" class="subtitle whole_rhythm">Synonym: <i>LIS1</i>-Associated Lissencephaly/Subcortical Band Heterotopia</div><p class="contrib-group"><span itemprop="author">William B Dobyns</span>, MD and <span itemprop="author">Soma Das</span>, PhD.</p><a data-jig="ncbitoggler" href="#__NBK5189_ai__" style="border:0;text-decoration:none">Author Information</a><div style="display:none" class="ui-widget" id="__NBK5189_ai__"><div class="contrib half_rhythm"><span itemprop="author">William B Dobyns</span>, MD<div class="affiliation small">Department of Pediatrics<br />Division of Genetic Medicine<br />University of Washington<br />Principal Investigator, Seattle Children&#x02019;s Research Institute<br />Seattle, Washington<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.wu@dbw" class="oemail">ude.wu@dbw</a></div></div></div><div class="contrib half_rhythm"><span itemprop="author">Soma Das</span>, PhD<div class="affiliation small">Department of Human Genetics<br />University of Chicago<br />Chicago, Illinois<div><span class="email-label">Email: </span><a href="mailto:dev@null" data-email="ude.ogacihcu.dsb@sads" class="oemail">ude.ogacihcu.dsb@sads</a></div></div></div></div><p class="small">Initial Posting: <span itemprop="datePublished">March 3, 2009</span>; Last Update: <span itemprop="dateModified">August 14, 2014</span>.</p><p><em>Estimated reading time: 27 minutes</em></p></div><div class="jig-ncbiinpagenav body-content whole_rhythm" data-jigconfig="allHeadingLevels: ['h2'],smoothScroll: false" itemprop="text"><div id="chrom17-lis.Summary" itemprop="description"><h2 id="_chrom17-lis_Summary_">Summary</h2><div><h4 class="inline">Clinical characteristics.</h4><p><i>PAFAH1B1</i>-associated lissencephaly includes Miller-Dieker syndrome (MDS), <a class="def" href="/books/n/gene/glossary/def-item/isolated/">isolated</a> lissencephaly sequence (ILS), and (rarely) subcortical band heterotopia (SBH). Lissencephaly and SBH are cortical malformations caused by deficient neuronal migration during embryogenesis. Lissencephaly refers to a "smooth brain" with absent gyri (agyria) or abnormally wide gyri (pachygyria). SBH refers to a band of heterotopic gray matter located just beneath the cortex and separated from it by a thin zone of normal white matter. MDS is characterized by lissencephaly, typical facial features, and severe neurologic abnormalities. ILS is characterized by lissencephaly and its direct sequelae: developmental delay, intellectual disability, and seizures.</p></div><div><h4 class="inline">Diagnosis/testing.</h4><p>MDS is caused by either small cytogenetically visible deletions or <a class="def" href="/books/n/gene/glossary/def-item/fish/">FISH</a>-detectable microdeletions of 17p13.3 that include <i>PAFAH1B1</i> (formerly <i>LIS1</i>) and <i>YWHAE</i>, and intervening genes. ILS is caused by smaller submicroscopic deletions of <i>PAFAH1B1</i>; intragenic deletions or duplications of <i>PAFAH1B1</i>; or sequence variants of <i>PAFAH1B1</i>.</p></div><div><h4 class="inline">Management.</h4><p><i>Treatment of manifestations:</i> Poor feeding may require nasogastric tube feedings in newborns and later placement of a gastrostomy tube. Seizures require prompt and aggressive management based on the specific seizure type and frequency; response to treatment is similar to that in children with seizures due to other causes.</p><p><i>Surveillance:</i> Whenever any unusual spells or any developmental regression is noted, a child neurology consultation should be performed and EEG considered.</p></div><div><h4 class="inline">Genetic counseling.</h4><p>Approximately 80% of individuals with MDS have a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> involving 17p13.3 and approximately 20% have inherited a deletion from a parent who carries a balanced <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> rearrangement. If neither parent has a structural chromosome rearrangement detectable by high-resolution chromosome analysis, the risk to sibs is negligible. If a parent has a balanced structural chromosome rearrangement, the risk to sibs for MDS depends on the specific rearrangement. Although <a class="def" href="/books/n/gene/glossary/def-item/germline-mosaicism/">germline mosaicism</a> in a parent could lead to <a class="def" href="/books/n/gene/glossary/def-item/familial/">familial</a> recurrence of ILS, all <i>PAFAH1B1</i> intragenic pathogenic variants reported to date have been <i>de novo</i>; thus, the risk to sibs for ILS is negligible if neither parent has mosaicism for the <i>PAFAH1B1</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> present in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>. Prenatal testing for pregnancies at increased risk for MDS is possible if the familial chromosome rearrangement is known.</p></div></div><div id="chrom17-lis.GeneReview_Scope"><h2 id="_chrom17-lis_GeneReview_Scope_"><i>GeneReview</i> Scope</h2><div id="chrom17-lis.Tc" class="table"><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK5189/table/chrom17-lis.Tc/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__chrom17-lis.Tc_lrgtbl__"><table><thead><tr><th id="hd_h_chrom17-lis.Tc_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>PAFAH1B1</i>-Associated Lissencephaly/Subcortical Band Heterotopia: Included Phenotypes</th></tr></thead><tbody><tr><td headers="hd_h_chrom17-lis.Tc_1_1_1_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><ul><li class="half_rhythm"><div>Miller-Dieker syndrome</div></li><li class="half_rhythm"><div>Isolated 17-linked lissencephaly</div></li><li class="half_rhythm"><div>17-linked subcortical band heterotopia</div></li></ul></td></tr></tbody></table></div></div></div><div id="chrom17-lis.Diagnosis"><h2 id="_chrom17-lis_Diagnosis_">Diagnosis</h2><div id="chrom17-lis.Clinical_Diagnosis"><h3>Clinical Diagnosis</h3><p>Together, lissencephaly and subcortical band heterotopia (SBH) comprise the "agyria-pachygyria-band" spectrum of cortical malformations that are caused by deficient neuronal migration during embryogenesis [<a class="bk_pop" href="#chrom17-lis.REF.barkovich.1991.139">Barkovich et al 1991</a>, <a class="bk_pop" href="#chrom17-lis.REF.norman.1995">Norman et al 1995</a>]. The term lissencephaly refers to a "smooth brain" with absent (agyria) or abnormally wide gyri (pachygyria).</p><div id="chrom17-lis.MRI_Findings"><h4>MRI Findings</h4><p><b>Lissencephaly</b></p><ul><li class="half_rhythm"><div>Cerebral gyri are absent or abnormally broad.</div></li><li class="half_rhythm"><div>The cerebral cortex is abnormally thick (12-20 mm; normal: 3-4 mm) [<a class="bk_pop" href="#chrom17-lis.REF.barkovich.1991.139">Barkovich et al 1991</a>].</div></li><li class="half_rhythm"><div>Associated findings in the most common ("classic") form of lissencephaly include:</div><ul><li class="half_rhythm"><div>Enlarged lateral ventricles, especially posteriorly</div></li><li class="half_rhythm"><div>Mild hypoplasia of the corpus callosum (the anterior portion often appears flattened)</div></li><li class="half_rhythm"><div>Cavum septi pellucidi et vergae</div></li><li class="half_rhythm"><div>Normal brain stem and cerebellum in most individuals, mild cerebellar vermis hypoplasia in a few</div></li></ul></li></ul><p><b>Subcortical band heterotopia (SBH)</b></p><ul><li class="half_rhythm"><div>A subcortical band of heterotopic gray matter, present just beneath the cortex, is separated from it by a thin zone of normal white matter [<a class="bk_pop" href="#chrom17-lis.REF.barkovich.1994.609">Barkovich et al 1994</a>]. SBH is most often symmetric, but may be asymmetric.</div></li><li class="half_rhythm"><div>In <i>PAFAH1B1</i>-associated SBH, the subcortical bands are restricted to the parietal and occipital lobes (diffuse or frontal predominant SBH are associated with mutation of <i>DCX</i> (see <a href="/books/n/gene/dcx/"><i>DCX-</i>Related Disorders</a>).</div></li><li class="half_rhythm"><div>The gyral pattern is normal or demonstrates mildly simplified shallow sulci; a normal cortical ribbon is present.</div></li></ul><p>Lissencephaly and SBH are graded by anterior-posterior gradient and severity (<a class="figpopup" href="/books/NBK5189/table/chrom17-lis.T.grading_system_for_classic/?report=objectonly" target="object" rid-figpopup="figchrom17lisTgradingsystemforclassic" rid-ob="figobchrom17lisTgradingsystemforclassic">Table 1</a> and <a class="figpopup" href="/books/NBK5189/figure/chrom17-lis.F1/?report=objectonly" target="object" rid-figpopup="figchrom17lisF1" rid-ob="figobchrom17lisF1">Figure 1</a>). When the lissencephaly or SBH is more severe posteriorly, it is referred to as a posterior to anterior (p&#x0003e;a) gradient. When more severe anteriorly, it is referred to as an anterior to posterior (a&#x0003e;p) gradient. <i>PAFAH1B1</i> abnormalities give rise to diffuse or p&#x0003e;a gradients, whereas abnormalities of <i>DCX</i> generally give rise to diffuse or a&#x0003e;p gradients [<a class="bk_pop" href="#chrom17-lis.REF.pilz.1998a.29">Pilz et al 1998a</a>, <a class="bk_pop" href="#chrom17-lis.REF.dobyns.1999.270">Dobyns et al 1999</a>].</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figchrom17lisF1" co-legend-rid="figlgndchrom17lisF1"><a href="/books/NBK5189/figure/chrom17-lis.F1/?report=objectonly" target="object" title="Figure 1. " class="img_link icnblk_img figpopup" rid-figpopup="figchrom17lisF1" rid-ob="figobchrom17lisF1"><img class="small-thumb" src="/books/NBK5189/bin/chrom17-lis-Image001.gif" src-large="/books/NBK5189/bin/chrom17-lis-Image001.jpg" alt="Figure 1. . Brain MRIs of lissencephaly, ranging from grade 1 (the most severe) to grade 6 (subcortical band heterotopia, the least severe)." /></a><div class="icnblk_cntnt" id="figlgndchrom17lisF1"><h4 id="chrom17-lis.F1"><a href="/books/NBK5189/figure/chrom17-lis.F1/?report=objectonly" target="object" rid-ob="figobchrom17lisF1">Figure 1. </a></h4><p class="float-caption no_bottom_margin">Brain MRIs of lissencephaly, ranging from grade 1 (the most severe) to grade 6 (subcortical band heterotopia, the least severe). (P) indicates posterior; (A) indicates anterior. Lissencephaly that is more severe posteriorly than anteriorly (best observed <a href="/books/NBK5189/figure/chrom17-lis.F1/?report=objectonly" target="object" rid-ob="figobchrom17lisF1">(more...)</a></p></div></div><div id="chrom17-lis.T.grading_system_for_classic" class="table"><h3><span class="label">Table 1. </span></h3><div class="caption"><p>Grading System for Classic Lissencephaly and SBH</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK5189/table/chrom17-lis.T.grading_system_for_classic/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__chrom17-lis.T.grading_system_for_classic_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Gradient</th><th id="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Grade of Severity</th></tr></thead><tbody><tr><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">1a=p&#x000a0;<sup>1</sup></td><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Complete agyria</td></tr><tr><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">2p&#x0003e;a or 2a&#x0003e;p</td><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Diffuse agyria with a few undulations at the frontal or occipital poles</td></tr><tr><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">3p&#x0003e;a or 3a&#x0003e;p</td><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Mixed agyria and pachygyria</td></tr><tr><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">4p&#x0003e;a or 4a&#x0003e;p</td><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Diffuse pachygyria, or mixed pachygyria and normal or simplified gyri</td></tr><tr><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">5a&#x0003e;p&#x000a0;<sup>2</sup></td><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Mixed pachygyria and subcortical band heterotopia</td></tr><tr><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">6p&#x0003e;a or 6a&#x0003e;p</td><td headers="hd_h_chrom17-lis.T.grading_system_for_classic_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Subcortical band heterotopia only</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt>1. </dt><dd><div id="chrom17-lis.TF.1.1"><p class="no_margin">With severe grade 1 lissencephaly, it is difficult to determine if a gradient is present.</p></div></dd><dt>2. </dt><dd><div id="chrom17-lis.TF.1.2"><p class="no_margin">The reverse (5p&#x0003e;a) has not been observed.</p></div></dd></dl></div></div></div></div><div id="chrom17-lis.Histologic_Findings"><h4>Histologic Findings</h4><p><b>Classic lissencephaly.</b> The cortex is abnormally thick and poorly organized with four apparent layers consisting of the following [<a class="bk_pop" href="#chrom17-lis.REF.crome.1956.335">Crome 1956</a>, <a class="bk_pop" href="#chrom17-lis.REF.forman.2005.847">Forman et al 2005</a>]:</p><ul><li class="half_rhythm"><div>Poorly defined marginal zone with increased cellularity</div></li><li class="half_rhythm"><div>Superficial cortical gray zone with diffusely scattered neurons</div></li><li class="half_rhythm"><div>Relatively neuron-sparse zone</div></li><li class="half_rhythm"><div>Deep cortical gray zone with neurons often oriented in columns. Neurons may be oriented with dendrites extending toward the pial surface or inverted, extending toward the ventricle.</div></li></ul><p><b>SBH.</b> The architecture of the cortex has not been studied in individuals with <i>PAFAH1B1</i> pathogenic variants (see <a href="/books/n/gene/dcx/"><i>DCX</i>-Related Disorders</a>)</p></div></div><div id="chrom17-lis.Testing"><h3>Testing</h3><p><b>Chromosome analysis</b></p><ul><li class="half_rhythm"><div>High-resolution <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> studies at the 450-band level or higher identify cytogenetically visible deletions or other structural rearrangements of 17p13.3 in approximately 70% of individuals with Miller-Dieker syndrome (MDS), but not in individuals with <a class="def" href="/books/n/gene/glossary/def-item/isolated/">isolated</a> lissencephaly sequence (ILS) or SBH [<a class="bk_pop" href="#chrom17-lis.REF.dobyns.1991.584">Dobyns et al 1991</a>].</div></li><li class="half_rhythm"><div>Rarely, individuals with ILS have a balanced reciprocal <a class="def" href="/books/n/gene/glossary/def-item/translocation/">translocation</a> disrupting <i>PAFAH1B1</i>.</div></li><li class="half_rhythm"><div>Rarely, individuals with posterior-predominant SBH have mosaic deletions of 17p13.3 visible on <a class="def" href="/books/n/gene/glossary/def-item/cytogenetic/">cytogenetic</a> analysis.</div></li></ul><div id="chrom17-lis.Molecular_Genetic_Testing"><h4>Molecular Genetic Testing</h4><p><b>Gene.</b> Abnormalities of <i>PAFAH1B1</i> (<i>LIS1</i>)* cause <a class="def" href="/books/n/gene/glossary/def-item/isolated/">isolated</a> <i>PAFAH1B1</i>-associated lissencephaly/SBH [<a class="bk_pop" href="#chrom17-lis.REF.pilz.1998b.2029">Pilz et al 1998b</a>, <a class="bk_pop" href="#chrom17-lis.REF.pilz.1999.1757">Pilz et al 1999</a>, <a class="bk_pop" href="#chrom17-lis.REF.fogli.1999.154">Fogli et al 1999</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2000.3019">Cardoso et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2002.4">Cardoso et al 2002</a>, <a class="bk_pop" href="#chrom17-lis.REF.sicca.2003.1042">Sicca et al 2003</a>, <a class="bk_pop" href="#chrom17-lis.REF.saillour.2009.1007">Saillour et al 2009</a>].</p><p>* Note: <i>PAFAH1B1</i> is the official <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> designation; however, the former gene designation (<i>LIS1</i>) is still in common use in medical genetics.</p><p><b>Clinical testing.</b> Abnormalities of <i>PAFAH1B1</i> range from single-nucleotide variants to contiguous <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> deletions. Clinical testing for <i>PAFAH1B1</i> is available for the detection of these various abnormalities.</p><div id="chrom17-lis.T.molecular_genetic_testing" class="table"><h3><span class="label">Table 2. </span></h3><div class="caption"><p>Molecular Genetic Testing Used in <i>PAFAH1B1-</i>Related Lissencephaly/SBH</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK5189/table/chrom17-lis.T.molecular_genetic_testing/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__chrom17-lis.T.molecular_genetic_testing_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_1" rowspan="2" scope="col" colspan="1" headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_1" style="text-align:left;vertical-align:middle;">Gene&#x000a0;<sup>1</sup></th><th id="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_2" rowspan="2" colspan="2" scope="colgroup" headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_2" style="text-align:left;vertical-align:middle;">Test Method</th><th id="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3" colspan="3" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">Proportion of Probands with a Pathogenic Variant Detectable by This Method&#x000a0;<sup>2</sup></th></tr><tr><th headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3" id="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_1" colspan="1" scope="colgroup" rowspan="1" style="text-align:left;vertical-align:middle;">MDS&#x000a0;<sup>3</sup></th><th headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3" id="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">ILS</th><th headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3" id="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">SBH</th></tr></thead><tbody><tr><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_1" rowspan="3" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><i>PAFAH1B1</i></td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_2" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Deletion / <a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a> analysis&#x000a0;<sup>4</sup></td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">See footnotes 5, 6</td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3 hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">100%</td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3 hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">~54%</td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3 hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">4 individuals&#x000a0;<sup>7</sup></td></tr><tr><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">See footnote 8</td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3 hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">0</td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3 hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">~68%&#x000a0;<sup>9</sup></td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3 hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Not described</td></tr><tr><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_2" colspan="2" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">Sequence analysis&#x000a0;<sup>1o</sup></td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3 hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_1" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">0</td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3 hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">~32%&#x000a0;<sup>11</sup></td><td headers="hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_1_3 hd_h_chrom17-lis.T.molecular_genetic_testing_1_1_2_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">7 individuals&#x000a0;<sup>12</sup></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Data from <a class="bk_pop" href="#chrom17-lis.REF.pilz.1998a.29">Pilz et al [1998a]</a>, <a class="bk_pop" href="#chrom17-lis.REF.pilz.1998b.2029">Pilz et al [1998b]</a>, <a class="bk_pop" href="#chrom17-lis.REF.pilz.1999.1757">Pilz et al [1999]</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2002.4">Cardoso et al [2002]</a>, <a class="bk_pop" href="#chrom17-lis.REF.d_agostino.2002.2507">D&#x02019;Agostino et al [2002]</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2003.918">Cardoso et al [2003]</a>, <a class="bk_pop" href="#chrom17-lis.REF.sicca.2003.1042">Sicca et al [2003]</a>, <a class="bk_pop" href="#chrom17-lis.REF.mei.2008.355">Mei et al [2008]</a>, <a class="bk_pop" href="#chrom17-lis.REF.haverfield.2009.911">Haverfield et al [2009]</a></p></div></dd><dt>1. </dt><dd><div id="chrom17-lis.TF.2.1"><p class="no_margin">See <a href="/books/NBK5189/#chrom17-lis.molgen.TA">Table A. Genes and Databases</a> for <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> and protein. See <a href="#chrom17-lis.Molecular_Genetics">Molecular Genetics</a> for information on allelic variants detected in this <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>.</p></div></dd><dt>2. </dt><dd><div id="chrom17-lis.TF.2.2"><p class="no_margin">Does not take into account individuals with pathogenic variants in <i>DCX</i>, <i>TUBA1A</i>, or other lissencephaly-associated genes, and those in whom no pathogenic variants have been identified.</p></div></dd><dt>3. </dt><dd><div id="chrom17-lis.TF.2.3"><p class="no_margin">As currently defined, MDS is associated with deletions that include both <i>PAFAH1B1</i> and <i>YWHAE</i> (a region of about 1.3 Mb harboring many genes) in 17p13.3 [<a class="bk_pop" href="#chrom17-lis.REF.pilz.1998a.29">Pilz et al 1998a</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2003.918">Cardoso et al 2003</a>].</p></div></dd><dt>4. </dt><dd><div id="chrom17-lis.TF.2.4"><p class="no_margin">Testing that identifies <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> or whole-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> deletions/duplications not detectable by <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> of the coding and flanking <a class="def" href="/books/n/gene/glossary/def-item/intronic/">intronic</a> regions of <a class="def" href="/books/n/gene/glossary/def-item/genomic/">genomic</a> DNA. Included in the variety of methods that may be used are: <a class="def" href="/books/n/gene/glossary/def-item/quantitative-pcr/">quantitative PCR</a>, long-range PCR, multiplex ligation-dependent probe amplification (MLPA), targeted array CGH and <a class="def" href="/books/n/gene/glossary/def-item/chromosomal-microarray/">chromosomal microarray</a> (CMA) that includes this gene/<a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> segment.</p></div></dd><dt>5. </dt><dd><div id="chrom17-lis.TF.2.5"><p class="no_margin">Detecting deletions of <i>PAFAH1B1</i> and adjacent genes/regions.</p></div></dd><dt>6. </dt><dd><div id="chrom17-lis.TF.2.6"><p class="no_margin">Chromosomal microarray analysis as well as <a class="def" href="/books/n/gene/glossary/def-item/fish/">FISH</a> testing can be performed using a probe containing <i>PAFAH1B1</i> (e.g., PAC95H6). Chromosomal microarray analysis will detect microdeletions/contiguous <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> deletions that involve <i>PAFAH1B1</i>, and may detect partial deletions of <i>PAFAH1B1</i> depending on the size of the <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> and probe density of the array. FISH analysis will detect microdeletions/contiguous gene deletions that involve the entire gene but will not detect partial deletions of <i>PAFAH1B1</i>.</p></div></dd><dt>7. </dt><dd><div id="chrom17-lis.TF.2.7"><p class="no_margin">Mosaic deletions of 17p13.3 involving <i>PAFAH1B1</i> have been observed in four individuals with SBH [WB Dobyns, personal communication].</p></div></dd><dt>8. </dt><dd><div id="chrom17-lis.TF.2.8"><p class="no_margin">Quantitative <a class="def" href="/books/n/gene/glossary/def-item/pcr/">PCR</a>, long-range PCR, multiplex ligation-dependent probe amplification (MLPA), and targeted array CGH may be used to detect partial deletions of <i>PAFAH1B1</i> that may be intragenic and may affect single or multiple exons, the promoter, or the 5&#x02019; untranslated region.</p></div></dd><dt>9. </dt><dd><div id="chrom17-lis.TF.2.9"><p class="no_margin">Intragenic deletions and duplications of <i>PAFAH1B1</i> that range in size from a single <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> to multiple exons to the entire <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> are present in approximately 14% of individuals with <i>PAFAH1B1</i>-related ILS [<a class="bk_pop" href="#chrom17-lis.REF.mei.2008.355">Mei et al 2008</a>, <a class="bk_pop" href="#chrom17-lis.REF.haverfield.2009.911">Haverfield et al 2009</a>].</p></div></dd><dt>10. </dt><dd><div id="chrom17-lis.TF.2.10"><p class="no_margin">Sequence analysis detects variants that are benign, likely benign, of <a class="def" href="/books/n/gene/glossary/def-item/uncertain-significance/">uncertain significance</a>, likely pathogenic, or pathogenic. Pathogenic variants may include small intragenic deletions/insertions and <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a>, <a class="def" href="/books/n/gene/glossary/def-item/nonsense-variant/">nonsense</a>, and <a class="def" href="/books/n/gene/glossary/def-item/splice-site/">splice site</a> variants; typically, <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> or whole-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> deletions/duplications are not detected. For issues to consider in interpretation of <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> results, click <a href="/books/n/gene/app2/">here</a>.</p></div></dd><dt>11. </dt><dd><div id="chrom17-lis.TF.2.11"><p class="no_margin">Germline <i>PAFAH1B1</i> pathogenic variants are present in approximately 32% of individuals with <i>PAFAH1B1</i>-related ILS. Rarely, mosaic pathogenic variants have been identified.</p></div></dd><dt>12. </dt><dd><div id="chrom17-lis.TF.2.12"><p class="no_margin">On occasion, <i>PAFAH1B1</i> pathogenic variants are identified in individuals with SBH. To date, pathogenic variants have been identified in seven individuals: four who had mosaic pathogenic variants and three with apparent <a class="def" href="/books/n/gene/glossary/def-item/germline/">germline</a> pathogenic variants [<a class="bk_pop" href="#chrom17-lis.REF.pilz.1999.1757">Pilz et al 1999</a>; <a class="bk_pop" href="#chrom17-lis.REF.d_agostino.2002.2507">D&#x02019;Agostino et al 2002</a>; <a class="bk_pop" href="#chrom17-lis.REF.sicca.2003.1042">Sicca et al 2003</a>; <a class="bk_pop" href="#chrom17-lis.REF.uyanik.2007.442">Uyanik et al 2007</a>; <a class="bk_pop" href="#chrom17-lis.REF.mineyko.2010.738">Mineyko et al 2010</a>; <a class="bk_pop" href="#chrom17-lis.REF.pagnamenta.2012.70">Pagnamenta et al 2012</a>; WB Dobyns &#x00026; S Das, personal communication].</p></div></dd></dl></div></div></div><p><b>Issues with interpretation of test results</b></p><ul><li class="half_rhythm"><div>Normal results of <a class="def" href="/books/n/gene/glossary/def-item/fish/">FISH</a> analysis using a <i>PAFAH1B1</i>-derived probe such as PAC95H6 do <b>not exclude</b> the presence of the following:</div><ul><li class="half_rhythm"><div>A submicroscopic <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> of 17p13 that includes partial deletions of <i>PAFAH1B1</i>. Partial deletions of <i>PAFAH1B1</i> may affect single or multiple exons, the promoter or the 5&#x02019; untranslated region (UTR).</div></li><li class="half_rhythm"><div>An intragenic <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>PAFAH1B1</i></div></li></ul></li><li class="half_rhythm"><div>Normal results of <a class="def" href="/books/n/gene/glossary/def-item/chromosomal-microarray/">chromosomal microarray</a> analysis do <b>not exclude</b> the presence of <i>PAFAH1B1</i> intragenic deletions or duplications that affect single or multiple exons. Targeted <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a> that contains probes to <i>PAFAH1B1</i> can detect smaller <i>PAFAH1B1</i> intragenic deletions/duplications.</div></li></ul></div></div><div id="chrom17-lis.Testing_Strategy"><h3>Testing Strategy</h3><div id="chrom17-lis.Probands_with_Isolated_Lisse"><h4>Probands with Isolated Lissencephaly</h4><p><b>Sequential single-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> testing.</b> One strategy for the diagnosis of an individual suspected of having lissencephaly with a p&#x0003e;a gradient or lissencephaly with an unknown or uncertain gradient is sequential single-gene testing.</p><ul><li class="half_rhythm"><div>Deletion/<a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a> analysis that detects all large deletions involving <i>PAFAH1B1</i> as well as small <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> deletions and duplications not detected by <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> should be performed first.</div></li><li class="half_rhythm"><div>If no <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> or <a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a> involving <i>PAFAH1B1</i> is identified, <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> of <i>PAFAH1B1</i> should be considered next.</div></li></ul><p><b>Lissencephaly with a p&#x0003e;a gradient</b></p><ul><li class="half_rhythm"><div>If no <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>PAFAH1B1</i> is identified by <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a>/<a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a> or <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a>, <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> of the following genes may be considered next: <i>DCX,</i>
<i>DYNC1H1, KIF2A, TUBA1A</i> (codon R402) and <i>TUBG1</i> (see <a href="#chrom17-lis.Differential_Diagnosis">Differential Diagnosis</a>).</div></li><li class="half_rhythm"><div>Chromosome analysis. Only two individuals with balanced reciprocal translocations disrupting <i>PAFAH1B1</i> have been observed (&#x0003c;1%); both had the ILS <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>.</div></li></ul><p><b>Lissencephaly with an unknown or uncertain gradient</b></p><ul><li class="half_rhythm"><div>If no <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>PAFAH1B1</i> is identified by <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a>/<a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a> or <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a>, <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> of the following genes may be considered next: <i>DCX</i>, <i>ACTB, ACTG1,</i>
<i>DYNC1H1, KIF2A, RELN, TUBA1A</i> (codon R402)<i>, TUBG1</i> and <i>VLDLR</i> (see <a href="#chrom17-lis.Differential_Diagnosis">Differential Diagnosis</a>).</div></li></ul><p><b>Multigene panel.</b> An alternative to single-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> is a panel in which a number of genes that lead to lissencephaly (see <a href="#chrom17-lis.Differential_Diagnosis">Differential Diagnosis</a>) are represented. These panels vary by methods used and genes included; thus, the ability of a panel to detect a causative variant or variants in any given individual also varies. For an introduction to multigene panels click <a href="/books/n/gene/app5/#app5.Multigene_Panels">here</a>. More detailed information for clinicians ordering genetic tests can be found <a href="/books/n/gene/app5/#app5.Multigene_Panels_FAQs">here</a>.</p><p><b>Lissencephaly with an a&#x0003e;p gradient</b> (see <a href="#chrom17-lis.Differential_Diagnosis">Differential Diagnosis</a>)</p><ul><li class="half_rhythm"><div>Perform <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> of <i>DCX</i>, <i>ACTB, ACTG1, PAFAH1B1, RELN</i> and <i>VLDLR</i>. While <i>DCX</i> is the most indicated in this category and can be sequenced first, the other genes can result in an overlapping <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a>. Note: <i>PAFAH1B1</i> abnormalities are generally not associated with an a&#x0003e;p gradient but when severe can be difficult to differentiate.</div></li></ul></div><div id="chrom17-lis.Probands_with_SBH"><h4>Probands with SBH</h4><p><b>Sequential single-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> testing.</b> One strategy for the diagnosis of an individual suspected of having SBH is sequential single-gene testing.</p><ul><li class="half_rhythm"><div>Sequence analysis of <i>DCX</i> should be considered first, as this detects pathogenic variants in up to 80% of females [<a class="bk_pop" href="#chrom17-lis.REF.des_portes.1998.1063">des Portes et al 1998</a>, <a class="bk_pop" href="#chrom17-lis.REF.gleeson.1999.146">Gleeson et al 1999</a>, <a class="bk_pop" href="#chrom17-lis.REF.matsumoto.2001.5">Matsumoto et al 2001</a>] and 30% of males [<a class="bk_pop" href="#chrom17-lis.REF.d_agostino.2002.2507">D&#x02019;Agostino et al 2002</a>].</div></li><li class="half_rhythm"><div>If no <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> is detected in <i>DCX</i> through <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a>, <a class="def" href="/books/n/gene/glossary/def-item/deletion-duplication-analysis/">deletion/duplication analysis</a> should be considered next. Deletion/duplication analysis of <i>DCX</i> detects intragenic deletions in approximately 5%-9% of females [<a class="bk_pop" href="#chrom17-lis.REF.mei.2007.446">Mei et al 2007</a>, <a class="bk_pop" href="#chrom17-lis.REF.haverfield.2009.911">Haverfield et al 2009</a>].</div></li><li class="half_rhythm"><div>If no pathogenic variants are detected in <i>DCX</i>, <a class="def" href="/books/n/gene/glossary/def-item/sequence-analysis/">sequence analysis</a> of <i>PAFAH1B1</i> may be considered.</div></li><li class="half_rhythm"><div>If no pathogenic variants are identified, <a class="def" href="/books/n/gene/glossary/def-item/fish/">FISH</a> testing using a <i>PAFAH1B1</i>-derived probe may be considered.</div></li></ul><p><b>Multigene panel.</b> An alternative to single-<a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a> is a panel in which a number of genes that lead to SBH (see <a href="#chrom17-lis.Differential_Diagnosis">Differential Diagnosis</a>) are represented. These panels vary by methods used and genes included; thus, the ability of a panel to detect a causative variant or variants in any given individual also varies. For an introduction to multigene panels click <a href="/books/n/gene/app5/#app5.Multigene_Panels">here</a>. More detailed information for clinicians ordering genetic tests can be found <a href="/books/n/gene/app5/#app5.Multigene_Panels_FAQs">here</a>.</p></div></div></div><div id="chrom17-lis.Clinical_Characteristics"><h2 id="_chrom17-lis_Clinical_Characteristics_">Clinical Characteristics</h2><div id="chrom17-lis.Clinical_Description"><h3>Clinical Description</h3><p>The phenotypes associated with mutation of <i>PAFAH1B1</i> comprise a spectrum of severity that can be separated into Miller-Dieker syndrome (MDS), posterior predominant ILS (or <a class="def" href="/books/n/gene/glossary/def-item/isolated/">isolated</a> lissencephaly) and, on rare occasion, posterior predominant subcortical band heterotopia (SBH).</p><p><b>MDS</b> consists of severe lissencephaly (grade 1-2) (see <a class="figpopup" href="/books/NBK5189/table/chrom17-lis.T.grading_system_for_classic/?report=objectonly" target="object" rid-figpopup="figchrom17lisTgradingsystemforclassic" rid-ob="figobchrom17lisTgradingsystemforclassic">Table 1</a> and <a class="figpopup" href="/books/NBK5189/figure/chrom17-lis.F1/?report=objectonly" target="object" rid-figpopup="figchrom17lisF1" rid-ob="figobchrom17lisF1">Figure 1</a>), characteristic facial changes, other more variable malformations, and severe neurologic and developmental abnormalities [<a class="bk_pop" href="#chrom17-lis.REF.dobyns.1991.584">Dobyns et al 1991</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2003.918">Cardoso et al 2003</a>]. The facial changes consist of high and prominent forehead, bitemporal hollowing, short nose with upturned nares, protuberant upper lip with downturned vermilion border, and small jaw (<a class="figpopup" href="/books/NBK5189/figure/chrom17-lis.F2/?report=objectonly" target="object" rid-figpopup="figchrom17lisF2" rid-ob="figobchrom17lisF2">Figure 2</a>). Other malformations seen on occasion include omphalocele and <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> heart defects.</p><div class="iconblock whole_rhythm clearfix ten_col fig" id="figchrom17lisF2" co-legend-rid="figlgndchrom17lisF2"><a href="/books/NBK5189/figure/chrom17-lis.F2/?report=objectonly" target="object" title="Figure 2. " class="img_link icnblk_img figpopup" rid-figpopup="figchrom17lisF2" rid-ob="figobchrom17lisF2"><img class="small-thumb" src="/books/NBK5189/bin/chrom17-lis-Image002.gif" src-large="/books/NBK5189/bin/chrom17-lis-Image002.jpg" alt="Figure 2. " /></a><div class="icnblk_cntnt" id="figlgndchrom17lisF2"><h4 id="chrom17-lis.F2"><a href="/books/NBK5189/figure/chrom17-lis.F2/?report=objectonly" target="object" rid-ob="figobchrom17lisF2">Figure 2. </a></h4><p class="float-caption no_bottom_margin">Two children with Miller-Dieker syndrome showing typical facial features Photographs obtained with consent of the families.</p></div></div><p><b>ILS</b> consists of more variable lissencephaly (grades 2-4) (see <a class="figpopup" href="/books/NBK5189/table/chrom17-lis.T.grading_system_for_classic/?report=objectonly" target="object" rid-figpopup="figchrom17lisTgradingsystemforclassic" rid-ob="figobchrom17lisTgradingsystemforclassic">Table 1</a> and <a class="figpopup" href="/books/NBK5189/figure/chrom17-lis.F1/?report=objectonly" target="object" rid-figpopup="figchrom17lisF1" rid-ob="figobchrom17lisF1">Figure 1</a>), minor facial changes, rare malformations outside of the brain, and similar neurologic and developmental handicaps as in MDS [<a class="bk_pop" href="#chrom17-lis.REF.dobyns.1992.1375">Dobyns et al 1992</a>].</p><p><b>In both MDS and ILS</b>, the pregnancy may be complicated by polyhydramnios. Affected newborns may appear normal or may have mild to moderate hypotonia, poor feeding, and transient elevations in bilirubin likely related to feeding difficulties [<a class="bk_pop" href="#chrom17-lis.REF.dobyns.1991.584">Dobyns et al 1991</a>, <a class="bk_pop" href="#chrom17-lis.REF.dobyns.1992.1375">Dobyns et al 1992</a>]. At birth, the occipitofrontal circumference (OFC) is typically normal (between the mean and 2 SD below the mean). However, postnatal head growth is slow; most children develop microcephaly by age one year. Prior to the onset of seizures, most infants have mild delay in development and mild hypotonia including poor head control. Some have difficulty with feeding.</p><p>Children with lissencephaly have epileptic encephalopathies that typically evolve from infantile spasms (West syndrome) to Lennox-Gastaut syndrome of mixed epilepsy with a slow spike and wave pattern on EEG. Overall, seizures occur in more than 90% of children with lissencephaly with onset usually before age six months. Approximately 80% have infantile spasms, although the EEG does not always show the typical hypsarrhythmia pattern. The onset of infantile spasms may be associated with a decline in function. After the first months of life, most children have mixed seizure disorders including persisting infantile spasms, focal motor and generalized tonic seizures [<a class="bk_pop" href="#chrom17-lis.REF.guerrini.2005.32">Guerrini 2005</a>], complex partial seizures, atypical absences, and atonic and myoclonic seizures. Some children with lissencephaly have characteristic EEG changes, including diffuse high-amplitude fast rhythms that are considered to be highly specific for this malformation [<a class="bk_pop" href="#chrom17-lis.REF.quirk.1993.193">Quirk et al 1993</a>].</p><p>The developmental prognosis is poor for all children with MDS and for the majority with <a class="def" href="/books/n/gene/glossary/def-item/isolated/">isolated</a> lissencephaly. Even with good seizure control, the best developmental level achieved by children with MDS or isolated lissencephaly (excluding the few with partial lissencephaly) is the equivalent of about age three to five months. This may include brief visual tracking, rolling over, limited creeping, and very rarely, sitting. With poor seizure control, children with lissencephaly may function at or below the level of a newborn. A few individuals with less severe (grade 4) lissencephaly, especially partial posterior lissencephaly or pachygyria, have a better developmental outcome [<a class="bk_pop" href="#chrom17-lis.REF.leventer.2001.416">Leventer et al 2001</a>].</p><p>During the first years, neurologic examination typically demonstrates brief visual tracking and response to sounds, axial hypotonia, and mild distal spasticity. Infants often demonstrate abnormal arching (opisthotonus). Later, distal spasticity becomes more prominent, although hypotonia remains. Rarely, <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals develop moderate spastic quadriplegia and scoliosis.</p><p>Feeding often improves during the first few months of life, but typically worsens again with seizure onset during the first year of life, and then again at several years of age for various reasons.</p><p>Children with lissencephaly have poor control of their airway, which predisposes to aspiration pneumonia, the most common terminal event.</p><p>The prognosis differs somewhat between MDS and <a class="def" href="/books/n/gene/glossary/def-item/isolated/">isolated</a> lissencephaly. In MDS, death occurs within the first two years in many children, and only a few reach age ten years. The oldest known individual with MDS died at age 17 years. In isolated lissencephaly, approximately 50% live to age ten years, and very few reach age 20 years. The oldest known individual lived to age 30 years. These estimates apply only to individuals with typical lissencephaly affecting the entire brain (the large majority of those with lissencephaly). In general, life expectancy is related to the severity of the lissencephaly on neuroimaging [<a class="bk_pop" href="#chrom17-lis.REF.de_wit.2011.417">de Wit et al 2011</a>].</p><p><b>Seven individuals with SBH</b> associated with <i>PAFAH1B1</i> pathogenic variants have been reported/identified, three with <a class="def" href="/books/n/gene/glossary/def-item/germline/">germline</a> pathogenic variants and four with mosaic pathogenic variants [<a class="bk_pop" href="#chrom17-lis.REF.pilz.1999.1757">Pilz et al 1999</a>; <a class="bk_pop" href="#chrom17-lis.REF.d_agostino.2002.2507">D&#x02019;Agostino et al 2002</a>; <a class="bk_pop" href="#chrom17-lis.REF.sicca.2003.1042">Sicca et al 2003</a>; <a class="bk_pop" href="#chrom17-lis.REF.uyanik.2007.442">Uyanik et al 2007</a>; <a class="bk_pop" href="#chrom17-lis.REF.mineyko.2010.738">Mineyko et al 2010</a>; <a class="bk_pop" href="#chrom17-lis.REF.pagnamenta.2012.70">Pagnamenta et al 2012</a>; WB Dobyns &#x00026; S Das, unpublished review]. Two other unreported individuals have had mosaic deletions of <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> 17p13.3 [WB Dobyns, personal observation].</p><p>SBH is characterized by normal facial appearance, epilepsy, and intellectual disability. Four of the seven individuals with <i>PAFAH1B1</i>-related SBH had frequent seizures. One had an IQ of 107 at age seven years that declined to 60 by age 13 years, most likely as a result of severe seizures; one had an IQ of approximately 60; one had severe intellectual disability; two had global developmental delay.</p><p>In general individuals with SBH live into adult life. No reliable data regarding life span exist; it is likely to be shortened in those with severe intellectual disability, intractable epilepsy, or both.</p></div><div id="chrom17-lis.GenotypePhenotype_Correlatio"><h3>Genotype-Phenotype Correlations</h3><p><b>MDS.</b> The most severely <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals with MDS have large cytogenetically visible deletions of 17p13.3 and unbalanced <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> rearrangements associated with <a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a> of another chromosome segment.</p><p>The next-most severe abnormality is <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> of 17p13.3 that includes both <i>PAFAH1B1</i> and <i>YWHAE</i> [<a class="bk_pop" href="#chrom17-lis.REF.cardoso.2003.918">Cardoso et al 2003</a>], the latter located telomeric to <i>PAFAH1B1</i> in the MDS <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> region.</p><p><b>ILS.</b> Smaller deletions, including intragenic deletions and duplications of <i>PAFAH1B1</i>, cause ILS that is less severe than MDS, ranging in severity from grade 2 to 4.</p><ul><li class="half_rhythm"><div>Most individuals with a telomeric <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> including the 5&#x02019; end of <i>PAFAH1B1</i> have grade 2 to 3 lissencephaly.</div></li><li class="half_rhythm"><div>Most individuals with a <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> of the 3&#x02019; end of <i>PAFAH1B1</i> have grade 3 to 4 lissencephaly [<a class="bk_pop" href="#chrom17-lis.REF.chong.1997.147">Chong et al 1997</a>; unpublished data].</div></li><li class="half_rhythm"><div>The vast majority of individuals with intragenic deletions and duplications of <i>PAFAH1B1</i> have grade 3 lissencephaly [<a class="bk_pop" href="#chrom17-lis.REF.haverfield.2009.911">Haverfield et al 2009</a>].</div></li></ul><p>Intragenic pathogenic variants in <i>PAFAH1B1</i> usually result in ILS grade 3 or 4. In general:</p><ul><li class="half_rhythm"><div>Intragenic pathogenic variants that predict premature termination of the PAFAH1B1 protein tend to result in a more severe lissencephaly <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> than <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants in <i>PAFAH1B1</i> [<a class="bk_pop" href="#chrom17-lis.REF.cardoso.2000.3019">Cardoso et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.leventer.2001.416">Leventer et al 2001</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2002.4">Cardoso et al 2002</a>].</div></li><li class="half_rhythm"><div>Pathogenic variants near the beginning of the <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> in the coiled-coil <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a> that result in truncation/<a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> may cause a more severe lissencephaly <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> than the similar variants that occur in other downstream regions of the gene [<a class="bk_pop" href="#chrom17-lis.REF.cardoso.2000.3019">Cardoso et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2002.4">Cardoso et al 2002</a>].</div></li></ul><p>Note: These generalizations notwithstanding, severity of the <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> does not always appear to correspond to location and type of <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>, as a more severe phenotype has been observed in some individuals with pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants and more severe grades (2 and 3) of lissencephaly have been observed in individuals with truncation/<a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> variants in the coiled-coil <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a> toward the 3&#x02019; end of <i>PAFAH1B1</i> [<a class="bk_pop" href="#chrom17-lis.REF.uyanik.2007.442">Uyanik et al 2007</a>].</p><p><b>SBH.</b> Two <i>PAFAH1B1</i> pathogenic <a class="def" href="/books/n/gene/glossary/def-item/missense/">missense</a> variants, one <a class="def" href="/books/n/gene/glossary/def-item/frameshift-variant/">frameshift variant</a>, and <a class="def" href="/books/n/gene/glossary/def-item/somatic-mosaicism/">somatic mosaicism</a> for a <i>PAFAH1B1</i> missense, <a class="def" href="/books/n/gene/glossary/def-item/nonsense-variant/">nonsense</a>, <a class="def" href="/books/n/gene/glossary/def-item/in-frame/">in-frame</a>, and <a class="def" href="/books/n/gene/glossary/def-item/splice-site/">splice site</a> variant have been identified in seven individuals with a milder <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> of SBH [<a class="bk_pop" href="#chrom17-lis.REF.pilz.1999.1757">Pilz et al 1999</a>; <a class="bk_pop" href="#chrom17-lis.REF.d_agostino.2002.2507">D&#x02019;Agostino et al 2002</a>; <a class="bk_pop" href="#chrom17-lis.REF.sicca.2003.1042">Sicca et al 2003</a>; <a class="bk_pop" href="#chrom17-lis.REF.uyanik.2007.442">Uyanik et al 2007</a>; <a class="bk_pop" href="#chrom17-lis.REF.mineyko.2010.738">Mineyko et al 2010</a>; <a class="bk_pop" href="#chrom17-lis.REF.pagnamenta.2012.70">Pagnamenta et al 2012</a>; WB Dobyns &#x00026; S Das, unpublished review].</p><p>Somatic <a class="def" href="/books/n/gene/glossary/def-item/mosaicism/">mosaicism</a> for a <i>PAFAH1B1</i> <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> also results in the milder <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> SBH (seen in two individuals [WB Dobyns, unpublished observation]).</p></div><div id="chrom17-lis.Prevalence"><h3>Prevalence</h3><p>Classic lissencephaly is rare. Birth prevalence is estimated to range from 11.7 to 40 per million births [personal communication with Metropolitan Atlanta Congenital Defects Program Personnel, National Center on Birth Defects and Developmental Disabilities, Centers for Disease Control and Prevention, Atlanta, GA, 2002]. Even the latter is likely to be an underestimate as the CDC program ascertains only hospitalized children in the first several years of life.</p></div></div><div id="chrom17-lis.Genetically_Related_Allelic"><h2 id="_chrom17-lis_Genetically_Related_Allelic_">Genetically Related (Allelic) Disorders</h2><p>No phenotypes other than those discussed in this <i>GeneReview</i> are known to be associated with mutation of <i>PAFAH1B1</i>.</p></div><div id="chrom17-lis.Differential_Diagnosis"><h2 id="_chrom17-lis_Differential_Diagnosis_">Differential Diagnosis</h2><p>The greatest difficulty in diagnosis of lissencephaly and subcortical band heterotopia (SBH) is recognizing the malformation. Several types of lissencephaly have been described, although they have overlapping features (<a class="figpopup" href="/books/NBK5189/table/chrom17-lis.T.types_of_lissencephaly/?report=objectonly" target="object" rid-figpopup="figchrom17lisTtypesoflissencephaly" rid-ob="figobchrom17lisTtypesoflissencephaly">Table 3</a>). The most common are classic lissencephaly (including SBH) and cobblestone complex.</p><div id="chrom17-lis.T.types_of_lissencephaly" class="table"><h3><span class="label">Table 3. </span></h3><div class="caption"><p>Types of Lissencephaly</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK5189/table/chrom17-lis.T.types_of_lissencephaly/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__chrom17-lis.T.types_of_lissencephaly_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Type</th><th id="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Description</th><th id="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Genes</th><th id="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Inheritance Pattern</th></tr></thead><tbody><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_1" rowspan="3" scope="row" colspan="1" style="text-align:left;vertical-align:middle;">Classic lissencephaly</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_2" rowspan="3" colspan="1" style="text-align:left;vertical-align:middle;">Very thick cortex (15-20 mm), normal corpus callosum, &#x00026; cerebellar vermis (or mild hypoplasia)</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>PAFAH1B1</i></td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">AD</td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><i>DCX</i></td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a></td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;"><i>TUBA1A</i></td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">AD</td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_1" rowspan="2" scope="row" colspan="1" style="text-align:left;vertical-align:middle;">Lissencephaly with cerebellar hypoplasia, tubulin type</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_2" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Diffuse or posterior predominant lissencephaly w/moderate to severe cerebellar hypoplasia</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>TUBA1A</i></td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_4" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">AD</td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;"><i>TUBB2B</i></td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_1" rowspan="2" scope="row" colspan="1" style="text-align:left;vertical-align:middle;">Lissencephaly with cerebellar hypoplasia, reelinopathy type</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_2" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">Mild frontal lissencephaly w/severe cerebellar hypoplasia</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>RELN</i></td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_4" rowspan="2" colspan="1" style="text-align:left;vertical-align:middle;">AR</td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;"><i>VLDLR</i></td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_1" rowspan="7" scope="row" colspan="1" style="text-align:left;vertical-align:middle;">Cobblestone cortical malformation (lissencephaly)</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_2" rowspan="7" colspan="1" style="text-align:left;vertical-align:middle;">Pebbled brain surface, moderately thick 5-10 mm cortex (unless thinned by hydrocephalus), diffuse or patchy white matter abnormality, brain stem &#x00026; cerebellar hypoplasia</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>FKTN</i></td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_4" rowspan="7" colspan="1" style="text-align:left;vertical-align:middle;">AR</td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;"><i>FKRP</i></td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;"><i>LARGE</i></td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;"><i>POMGnT1</i></td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;"><i>POMT1</i></td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;"><i>POMT2</i></td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" colspan="1" scope="col" rowspan="1" style="text-align:left;vertical-align:middle;"><i>ADGRG1</i> (<i>GPR56</i>)</td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Lissencephaly with agenesis of the corpus callosum</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Lissencephaly w/total or severe partial agenesis of the corpus callosum</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><i>ARX</i></td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;"><a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a></td></tr><tr><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_1" scope="row" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Microlissencephaly</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Birth OFC -3 SD or small, thick cortex</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">NA</td><td headers="hd_h_chrom17-lis.T.types_of_lissencephaly_1_1_1_4" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">NA</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">AD = <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a></p></div></dd><dt></dt><dd><div><p class="no_margin">AR = <a class="def" href="/books/n/gene/glossary/def-item/autosomal-recessive/">autosomal recessive</a></p></div></dd><dt></dt><dd><div><p class="no_margin">NA = not applicable</p></div></dd><dt></dt><dd><div><p class="no_margin">OFC = occipital frontal circumference</p></div></dd></dl></div></div></div><p>Several different cortical malformations that are sometimes mistaken for lissencephaly have been described, including severe <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> microcephaly with reduced number of gyri, cobblestone malformations as seen in Walker-Warburg and other syndromes, polymicrogyria, and polymicrogyria-like variants associated with pathogenic variants of tubulin genes. This leads to incorrect diagnosis, counseling, and testing.</p><p>Clinical features can help distinguish children who have lissencephaly from those who have other brain malformations. Children with lissencephaly usually have normal or slightly small OFC at birth (&#x0003e; -3 SD) and diffuse hypotonia except for mildly increased tone at the wrists and ankles. Children with severe <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> (i.e., primary) microcephaly and gyral abnormalities have smaller birth OFC (&#x02264; -3 SD) and may be hypotonic or spastic. Infants with polymicrogyria, especially when the frontal lobes are involved, frequently have spastic quadriparesis.</p><p>The differential diagnosis of <b>classic lissencephaly</b> includes the <i>PAFAH1B1</i>-related malformations, <a href="/books/n/gene/dcx/"><i>DCX</i>-related malformations</a>, <i>TUBA1A</i>-related malformations and the rare Baraitser-Winter syndrome (BWS) [<a class="bk_pop" href="#chrom17-lis.REF.dobyns.1991.584">Dobyns et al 1991</a>, <a class="bk_pop" href="#chrom17-lis.REF.dobyns.1992.1375">Dobyns et al 1992</a>, <a class="bk_pop" href="#chrom17-lis.REF.pilz.1998a.29">Pilz et al 1998a</a>, <a class="bk_pop" href="#chrom17-lis.REF.pilz.1998b.2029">Pilz et al 1998b</a>, <a class="bk_pop" href="#chrom17-lis.REF.dobyns.1999.270">Dobyns et al 1999</a>, <a class="bk_pop" href="#chrom17-lis.REF.matsumoto.2001.5">Matsumoto et al 2001</a>, <a class="bk_pop" href="#chrom17-lis.REF.ross.2001.256">Ross et al 2001</a>, <a class="bk_pop" href="#chrom17-lis.REF.rossi.2003.287">Rossi et al 2003</a>, <a class="bk_pop" href="#chrom17-lis.REF.poirier.2007.1055">Poirier et al 2007</a>]. These disorders are distinguished by <a class="def" href="/books/n/gene/glossary/def-item/mode-of-inheritance/">mode of inheritance</a>, grade and gradient of lissencephaly or SBH (<a class="figpopup" href="/books/NBK5189/table/chrom17-lis.T.grading_system_for_classic/?report=objectonly" target="object" rid-figpopup="figchrom17lisTgradingsystemforclassic" rid-ob="figobchrom17lisTgradingsystemforclassic">Table 1</a>), presence of other <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> anomalies, and results of <a class="def" href="/books/n/gene/glossary/def-item/molecular-genetic-testing/">molecular genetic testing</a>.</p><ul><li class="half_rhythm"><div>Classic lissencephaly associated with deletions or intragenic pathogenic variants in <i>PAFAH1B1</i> is more common than classic lissencephaly associated with pathogenic variants in <i>DCX</i>, located on the X <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> [<a class="bk_pop" href="#chrom17-lis.REF.pilz.1998b.2029">Pilz et al 1998b</a>], and classic lissencephaly associated with pathogenic variants in <i>TUBA1A</i> codon Arg402.</div></li><li class="half_rhythm"><div>SBH in females associated with pathogenic variants in <i>DCX</i> is far more common than SBH in females with <i>PAFAH1B1</i> pathogenic variants [<a class="bk_pop" href="#chrom17-lis.REF.pilz.1999.1757">Pilz et al 1999</a>, <a class="bk_pop" href="#chrom17-lis.REF.d_agostino.2002.2507">D&#x02019;Agostino et al 2002</a>, <a class="bk_pop" href="#chrom17-lis.REF.sicca.2003.1042">Sicca et al 2003</a>].</div></li><li class="half_rhythm"><div><i>PAFAH1B1</i>-related malformations are characterized by a p&#x0003e;a gradient (see <a class="figpopup" href="/books/NBK5189/table/chrom17-lis.T.grading_system_for_classic/?report=objectonly" target="object" rid-figpopup="figchrom17lisTgradingsystemforclassic" rid-ob="figobchrom17lisTgradingsystemforclassic">Table 1</a>). <i>DCX</i>-related malformations are associated with an a&#x0003e;p gradient [<a class="bk_pop" href="#chrom17-lis.REF.pilz.1998b.2029">Pilz et al 1998b</a>, <a class="bk_pop" href="#chrom17-lis.REF.dobyns.1999.270">Dobyns et al 1999</a>]. However, in severe classic lissencephaly or SBH the gradient may be difficult to discern.</div></li><li class="half_rhythm"><div><i>TUBA1A</i>-related malformations, like <i>PAFAH1B1</i>-related malformations, are characterized by a p&#x0003e;a gradient. The lissencephaly <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> associated with <i>PAFAH1B1</i> and <i>TUBA1A</i> Arg402 pathogenic variants may be indistinguishable. Pathogenic variants elsewhere in <i>TUBA1A</i> result in a spectrum of malformations that vary from polymicrogyria-like cortical malformation with cerebellar hypoplasia and callosal dysgenesis to the most severe forms of lissencephaly with cerebellar hypoplasia (also called 2-layer lissencephaly) [<a class="bk_pop" href="#chrom17-lis.REF.poirier.2007.1055">Poirier et al 2007</a>, <a class="bk_pop" href="#chrom17-lis.REF.kumar.2010.2817">Kumar et al 2010</a>, <a class="bk_pop" href="#chrom17-lis.REF.cushion.2013.536">Cushion et al 2013</a>].</div></li><li class="half_rhythm"><div>Baraitser-Winter syndrome is characterized by: lissencephaly with an a&#x0003e;p gradient similar to that of <i>DCX</i>-related malformations; trigonocephaly; shallow orbits; ptosis; and colobomas of the iris, choroid, or both [<a class="bk_pop" href="#chrom17-lis.REF.ramer.1995.403">Ramer et al 1995</a>, <a class="bk_pop" href="#chrom17-lis.REF.rossi.2003.287">Rossi et al 2003</a>].</div></li></ul><p>Lissencephaly with agenesis of the corpus callosum is typically associated with pathogenic variants in <i>ARX</i>. The XLAG (<a class="def" href="/books/n/gene/glossary/def-item/x-linked/">X-linked</a> lissencephaly with abnormal genitalia) <a class="def" href="/books/n/gene/glossary/def-item/phenotype/">phenotype</a> in severely <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals with a 46,XY <a class="def" href="/books/n/gene/glossary/def-item/karyotype/">karyotype</a> differs significantly from the phenotype associated with pathogenic variants in either <i>PAFAH1B1</i> or <i>DCX</i>. XLAG is characterized by <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> or postnatal microcephaly, neonatal-onset intractable epilepsy, poor temperature regulation, chronic diarrhea, and abnormal genitalia [<a class="bk_pop" href="#chrom17-lis.REF.kato.2003.r89">Kato &#x00026; Dobyns 2003</a>, <a class="bk_pop" href="#chrom17-lis.REF.kato.2004.147">Kato et al 2004</a>].</p><p>The cobblestone cortical malformation (lissencephaly) syndromes (Walker-Warburg syndrome, muscle-eye-brain disease, and <a href="/books/n/gene/fcmd/">Fukuyama congenital muscular dystrophy</a>) differ clinically in a number of ways, including the frequent presence of hydrocephalus and cerebellar hypoplasia, multiple different eye anomalies, and <a class="def" href="/books/n/gene/glossary/def-item/congenital/">congenital</a> muscular dystrophy manifest by hypotonia and elevated serum creatine kinase concentrations.</p></div><div id="chrom17-lis.Management"><h2 id="_chrom17-lis_Management_">Management</h2><div id="chrom17-lis.Evaluations_Following_Initia"><h3>Evaluations Following Initial Diagnosis</h3><p>To establish the extent of disease and needs in an individual diagnosed with <i>PAFAH1B1</i>-associated lissencephaly/ subcortical band heterotopia (SBH), the following are recommended:</p><ul><li class="half_rhythm"><div>Evaluation of:</div><ul><li class="half_rhythm"><div>Growth</div></li><li class="half_rhythm"><div>Feeding and nutrition</div></li><li class="half_rhythm"><div>Respiratory status</div></li><li class="half_rhythm"><div>Development</div></li><li class="half_rhythm"><div>Seizures</div></li></ul></li><li class="half_rhythm"><div>Clinical genetics consultation</div></li></ul><p>MRI should be interpreted carefully to provide as much prognostic information as possible. Note: Although most <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals have severe to profound intellectual disability, a minority have less extensive lissencephaly that results in only moderate intellectual disability, and a few have limited malformations that allow near-normal development. In the latter, the lissencephaly or SBH is typically less severe and less extensive on MRI. The resolution of brain CT scan is not usually sufficient to allow this.</p></div><div id="chrom17-lis.Treatment_of_Manifestations"><h3>Treatment of Manifestations</h3><p>Parents seem best able to deal with this severe disorder when accurate information regarding the prognosis is given as soon as possible after the diagnosis is recognized. For those with severe lissencephaly, it is usually appropriate to discuss limitations of care, such as "do not resuscitate" (DNR) orders, in the event of severe illnesses.</p><p>Poor feeding in newborns is usually managed by nasogastric tube feedings, as the feeding problems often improve during the first weeks of life. But they often worsen again with intercurrent illnesses and with advancing age and size. At least 50% of children with <i>PAFAH1B1</i>-related lissencephaly (but not SBH) eventually have a gastrostomy tube placed for feeding.</p><p>A large majority of children with lissencephaly have seizures, including frequent infantile spasms, which can be difficult to control. Management of seizures in children with ILS or SBH is based on the specific seizure type and frequency. In general, seizures should be treated promptly and aggressively by specialists, as poor seizure control frequently results in decline in function and health. Specifically, poor seizure control worsens feeding and increases the likelihood that a gastrostomy tube will be needed, and increases the risk for pneumonia.</p></div><div id="chrom17-lis.Surveillance"><h3>Surveillance</h3><p>Whenever any unusual spells or any developmental regression is noted, a child neurology consultation should be performed, and EEG considered.</p></div><div id="chrom17-lis.Evaluation_of_Relatives_at_R"><h3>Evaluation of Relatives at Risk</h3><p>See <a href="#chrom17-lis.Related_Genetic_Counseling_I">Genetic Counseling</a> for issues related to testing of at-risk relatives for <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> purposes.</p></div><div id="chrom17-lis.Therapies_Under_Investigatio"><h3>Therapies Under Investigation</h3><p>Search <a href="http://clinicaltrials.gov/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">ClinicalTrials.gov</a> in the US and <a href="http://www.ClinicalTrialsRegister.eu" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.ClinicalTrialsRegister.eu</a> in Europe for access to information on clinical studies for a wide range of diseases and conditions. Note: There may not be clinical trials for this disorder.</p></div></div><div id="chrom17-lis.Genetic_Counseling"><h2 id="_chrom17-lis_Genetic_Counseling_">Genetic Counseling</h2><p>
<i>Genetic counseling is the process of providing individuals and families with
information on the nature, inheritance, and implications of genetic disorders to help them
make informed medical and personal decisions. The following section deals with genetic
risk assessment and the use of family history and genetic testing to clarify genetic
status for family members. This section is not meant to address all personal, cultural, or
ethical issues that individuals may face or to substitute for consultation with a genetics
professional</i>. &#x02014;ED.</p><div id="chrom17-lis.Mode_of_Inheritance"><h3>Mode of Inheritance</h3><p><i>PAFAH1B1</i>-associated lissencephaly/subcortical band heterotopia (SBH) is inherited in an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> manner, although most deletions and all intragenic pathogenic variants reported to date have been <i>de novo.</i></p></div><div id="chrom17-lis.Risk_to_Family_Members"><h3>Risk to Family Members</h3><p><b>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>Approximately 80% of probands with Miller-Dieker syndrome (MDS) have a <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> involving 17p13.3.</div><ul><li class="half_rhythm"><div><a class="def" href="/books/n/gene/glossary/def-item/fish/">FISH</a> testing should be used to evaluate both parents for a possible cryptic <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/translocation/">translocation</a> involving 17p13.3.</div></li></ul></li><li class="half_rhythm"><div>Approximately 20% of probands with MDS have inherited a <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> from a parent who carries a balanced <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> rearrangement [<a class="bk_pop" href="#chrom17-lis.REF.dobyns.1991.584">Dobyns et al 1991</a>; Dobyns, unpublished data]</div><ul><li class="half_rhythm"><div>Parents of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with a structural unbalanced <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> constitution (e.g., <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a> or <a class="def" href="/books/n/gene/glossary/def-item/duplication/">duplication</a>) are at increased risk for balanced chromosome rearrangements and should be offered chromosome analysis.</div></li></ul></li><li class="half_rhythm"><div>Most <i>PAFAH1B1</i> intragenic pathogenic variants are <i>de novo.</i></div><ul><li class="half_rhythm"><div>Parental studies are recommended to exclude <a class="def" href="/books/n/gene/glossary/def-item/mosaicism/">mosaicism</a> in one parent.</div></li></ul></li></ul><p><b>Sibs of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a></b></p><ul><li class="half_rhythm"><div>Proband with a <a class="def" href="/books/n/gene/glossary/def-item/cytogenetic/">cytogenetic</a> <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a></div><ul><li class="half_rhythm"><div>If neither parent has a structural <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> rearrangement detectable by high-resolution chromosome analysis, the risk to sibs is negligible.</div></li><li class="half_rhythm"><div>If a parent has a balanced structural <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> rearrangement, the risk to sibs is increased and depends on the specific rearrangement, and possibly other variables.</div></li></ul></li><li class="half_rhythm"><div>Proband with an intragenic <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in <i>PAFAH1B1</i></div><ul><li class="half_rhythm"><div>The risk to sibs is negligible if neither parent has <a class="def" href="/books/n/gene/glossary/def-item/mosaicism/">mosaicism</a> for the <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> present in the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</div></li></ul></li></ul><p><b>Offspring of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>.</b> Individuals with <i>PAFAH1B1</i>-associated lissencephaly/SBH do not reproduce.</p><p><b>Other family members.</b> The risk to other family members depends on the genetic status of the <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a>'s parents. If a parent has a <a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> rearrangement, his or her family members are at risk and can be offered chromosome analysis.</p></div><div id="chrom17-lis.Related_Genetic_Counseling_I"><h3>Related Genetic Counseling Issues</h3><p><b>Predictive testing</b> for at-risk asymptomatic adult family members requires prior identification of the <i>PAFAH1B1</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> in the family.</p><p><b>Considerations in families with an apparent <a class="def" href="/books/n/gene/glossary/def-item/de-novo/"><i>de novo</i></a> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a>.</b> When neither parent of a <a class="def" href="/books/n/gene/glossary/def-item/proband/">proband</a> with an <a class="def" href="/books/n/gene/glossary/def-item/autosomal-dominant/">autosomal dominant</a> condition has the pathogenic variant or <a class="def" href="/books/n/gene/glossary/def-item/deletion/">deletion</a>, it is likely that the proband has a <i>de novo</i> pathogenic variant. However, possible non-medical explanations including <a class="def" href="/books/n/gene/glossary/def-item/alternate-paternity/">alternate paternity</a> or maternity (e.g., with assisted reproduction) or undisclosed adoption could also be explored.</p><p><b>Family planning</b></p><ul><li class="half_rhythm"><div>The optimal time for determination of genetic risk and discussion of the availability of prenatal testing is before pregnancy.</div></li><li class="half_rhythm"><div>It is appropriate to offer <a class="def" href="/books/n/gene/glossary/def-item/genetic-counseling/">genetic counseling</a> (including discussion of potential risks to offspring and reproductive options) to young adults who are <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> or at risk.</div></li></ul><p><b>DNA banking</b> is the storage of DNA (typically extracted from white blood cells) for possible future use. Because it is likely that testing methodology and our understanding of genes, allelic variants, and diseases will improve in the future, consideration should be given to banking DNA of <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals.</p></div><div id="chrom17-lis.Prenatal_Testing_and_Preimpl"><h3>Prenatal Testing and Preimplantation Genetic Diagnosis</h3><p>Once the <i>PAFAH1B1</i> <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> has been identified in an <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> family member, prenatal testing for a pregnancy at increased risk and <a class="def" href="/books/n/gene/glossary/def-item/preimplantation-genetic-diagnosis/">preimplantation genetic diagnosis</a> are possible.</p><p><b>Fetal ultrasound examination.</b> Prenatal testing by level 2 ultrasound examination does <b>NOT</b> detect lissencephaly, as normal fetal brains have a smooth surface until late in gestation.</p></div></div><div id="chrom17-lis.Resources"><h2 id="_chrom17-lis_Resources_">Resources</h2><p>
<i>GeneReviews staff has selected the following disease-specific and/or umbrella
support organizations and/or registries for the benefit of individuals with this disorder
and their families. GeneReviews is not responsible for the information provided by other
organizations. For information on selection criteria, click <a href="/books/n/gene/app4/">here</a>.</i></p><ul><li class="half_rhythm"><div><b>National Institute of Neurological Disorders and Stroke (NINDS)</b></div><div>PO Box 5801</div><div>Bethesda MD 20824</div><div><b>Phone:</b> 800-352-9424 (toll-free); 301-496-5751; 301-468-5981 (TTY)</div><div><a href="https://www.ninds.nih.gov/Disorders/All-Disorders/Lissencephaly-Information-Page" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Lissencephaly Information Page</a></div></li><li class="half_rhythm"><div><b>American Epilepsy Society (AES)</b></div><div><a href="http://www.aesnet.org" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.aesnet.org</a></div></li><li class="half_rhythm"><div><b>Epilepsy Foundation</b></div><div>8301 Professional Place East</div><div>Suite 200</div><div>Landover MD 20785-7223</div><div><b>Phone:</b> 800-332-1000 (toll-free)</div><div><b>Email:</b> ContactUs@efa.org</div><div><a href="https://www.epilepsy.com/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">www.epilepsy.com</a></div></li></ul></div><div id="chrom17-lis.Molecular_Genetics"><h2 id="_chrom17-lis_Molecular_Genetics_">Molecular Genetics</h2><p><i>Information in the Molecular Genetics and OMIM tables may differ from that elsewhere in the GeneReview: tables may contain more recent information. &#x02014;</i>ED.</p><div id="chrom17-lis.molgen.TA" class="table"><h3><span class="label">Table A.</span></h3><div class="caption"><p>PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia: Genes and Databases</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK5189/table/chrom17-lis.molgen.TA/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__chrom17-lis.molgen.TA_lrgtbl__"><table class="no_bottom_margin"><tbody><tr><th id="hd_b_chrom17-lis.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;">Gene</th><th id="hd_b_chrom17-lis.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;">Chromosome Locus</th><th id="hd_b_chrom17-lis.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;">Protein</th><th id="hd_b_chrom17-lis.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;">Locus-Specific Databases</th><th id="hd_b_chrom17-lis.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;">HGMD</th><th id="hd_b_chrom17-lis.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;">ClinVar</th></tr><tr><td headers="hd_b_chrom17-lis.molgen.TA_1_1_1_1" rowspan="1" colspan="1" style="vertical-align:top;"><a href="/gene/5048" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=gene"><i>PAFAH1B1</i></a></td><td headers="hd_b_chrom17-lis.molgen.TA_1_1_1_2" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/genome/gdv/?context=gene&#x00026;acc=5048" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">17p13<wbr style="display:inline-block"></wbr>​.3</a></td><td headers="hd_b_chrom17-lis.molgen.TA_1_1_1_3" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.uniprot.org/uniprot/P43034" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Platelet-activating factor acetylhydrolase IB subunit alpha</a></td><td headers="hd_b_chrom17-lis.molgen.TA_1_1_1_4" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://databases.lovd.nl/shared/genes/PAFAH1B1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">PAFAH1B1 database</a></td><td headers="hd_b_chrom17-lis.molgen.TA_1_1_1_5" rowspan="1" colspan="1" style="vertical-align:top;"><a href="http://www.hgmd.cf.ac.uk/ac/gene.php?gene=PAFAH1B1" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">PAFAH1B1</a></td><td headers="hd_b_chrom17-lis.molgen.TA_1_1_1_6" rowspan="1" colspan="1" style="vertical-align:top;"><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=PAFAH1B1[gene]" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">PAFAH1B1</a></td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div id="chrom17-lis.TFA.1"><p class="no_margin">Data are compiled from the following standard references: <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> from
<a href="http://www.genenames.org/index.html" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">HGNC</a>;
<a class="def" href="/books/n/gene/glossary/def-item/chromosome/">chromosome</a> <a class="def" href="/books/n/gene/glossary/def-item/locus/">locus</a> from
<a href="http://www.omim.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">OMIM</a>;
protein from <a href="http://www.uniprot.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">UniProt</a>.
For a description of databases (Locus Specific, HGMD, ClinVar) to which links are provided, click
<a href="/books/n/gene/app1/">here</a>.</p></div></dd></dl></div></div></div><div id="chrom17-lis.molgen.TB" class="table"><h3><span class="label">Table B.</span></h3><div class="caption"><p>OMIM Entries for PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia (<a href="/omim/247200,601545,607432" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">View All in OMIM</a>) </p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK5189/table/chrom17-lis.molgen.TB/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__chrom17-lis.molgen.TB_lrgtbl__"><table><tbody><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/247200" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">247200</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">MILLER-DIEKER LISSENCEPHALY SYNDROME; MDLS</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/601545" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">601545</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">PLATELET-ACTIVATING FACTOR ACETYLHYDROLASE, ISOFORM 1B, ALPHA SUBUNIT; PAFAH1B1</td></tr><tr><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">
<a href="/omim/607432" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=omim">607432</a></td><td rowspan="1" colspan="1" style="text-align:left;vertical-align:top;">LISSENCEPHALY 1; LIS1</td></tr></tbody></table></div></div><p><b>Gene structure.</b>
<i>PAFAH1B1</i> (often referred to as <i>LIS1</i>) is approximately 92 kb in size and consists of 11 exons. The <a class="def" href="/books/n/gene/glossary/def-item/coding-region/">coding region</a> is 1230 bp, with the AUG start codon located in <a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> 2, and it encodes a protein of 410 amino acids called PAFAH1B1 (platelet-activating factor acetylhydrolase IB alpha subunit, also known as brain isoform 1b) [<a class="bk_pop" href="#chrom17-lis.REF.reiner.1993.717">Reiner et al 1993</a>]. There are two alternative transcripts (5.5 kb and 7.5 kb) that differ in the length at their respective 3&#x02019; UTR regions; both transcripts are expressed ubiquitously [<a class="bk_pop" href="#chrom17-lis.REF.hattori.1994.216">Hattori et al 1994</a>, <a class="bk_pop" href="#chrom17-lis.REF.lo_nigro.1997.157">Lo Nigro et al 1997</a>]. For a detailed summary of <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a> and protein information, see <a href="/books/NBK5189/#chrom17-lis.molgen.TA">Table A</a>, <b>Gene</b>.</p><p><b>Benign variants.</b> To date, several different polymorphisms have been identified in <i>PAFAH1B1</i> (not all have been published). The polymorphic changes that have been identified are thought to be benign, as they have been found either in individuals in whom a clearly deleterious variant is also present or in individuals without the disease. Most represent rare variants, with the exception of two polymorphisms in the 3&#x02019; UTR (c.*3G&#x0003e;T and c.*17C&#x0003e;T; see <a class="figpopup" href="/books/NBK5189/table/chrom17-lis.T.selected_pafah1b1_lis1_rec/?report=objectonly" target="object" rid-figpopup="figchrom17lisTselectedpafah1b1lis1rec" rid-ob="figobchrom17lisTselectedpafah1b1lis1rec">Table 4</a>) that occur at frequencies of approximately 10% and 40% respectively [<a class="bk_pop" href="#chrom17-lis.REF.koch.1996.527">Koch et al 1996</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2000.3019">Cardoso et al 2000</a>].</p><p><b>Pathogenic variants.</b> More than 130 different <i>PAFAH1B1</i> (formerly <i>LIS1</i>) pathogenic variants have been described [<a class="bk_pop" href="#chrom17-lis.REF.cardoso.2000.3019">Cardoso et al 2000</a>; <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2002.4">Cardoso et al 2002</a>; <a class="bk_pop" href="#chrom17-lis.REF.uyanik.2007.442">Uyanik et al 2007</a>; <a class="bk_pop" href="#chrom17-lis.REF.saillour.2009.1007">Saillour et al 2009</a>; Author, unpublished]. Most are <a class="def" href="/books/n/gene/glossary/def-item/private/">private</a> variants; however, a few recurrent variants have been described, including c.162delA and c.162dupA, which occur in a small stretch of adenine residues, and c.1050delG and c.1050dupG, which occur in a small stretch of guanine residues. Pathogenic variants appear to be evenly distributed throughout the <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>. No predominant common <a class="def" href="/books/n/gene/glossary/def-item/pathogenic-variant/">pathogenic variant</a> has been identified in any population.</p><p>Intragenic deletions and duplications of <i>PAFAH1B1</i> that range from single-<a class="def" href="/books/n/gene/glossary/def-item/exon/">exon</a> deletions or duplications to deletions of the entire <a class="def" href="/books/n/gene/glossary/def-item/coding-region/">coding region</a> have been described; to date, approximately 35 partial deletions and duplications of <i>PAFAH1B1</i> have been identified [<a class="bk_pop" href="#chrom17-lis.REF.mei.2008.355">Mei et al 2008</a>, <a class="bk_pop" href="#chrom17-lis.REF.haverfield.2009.911">Haverfield et al 2009</a>, <a class="bk_pop" href="#chrom17-lis.REF.saillour.2009.1007">Saillour et al 2009</a>]. These partial deletions and duplications appear to be spread throughout the <a class="def" href="/books/n/gene/glossary/def-item/gene/">gene</a>. Microdeletions of the 17p13.3 region that delete all of <i>PAFAH1B1</i> have been described in approximately 40% of individuals with ILS and differ with regard to size and break points. 17p13.3 microdeletions have been identified in <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> individuals of different ethnic groups.</p><div id="chrom17-lis.T.selected_pafah1b1_lis1_rec" class="table"><h3><span class="label">Table 4. </span></h3><div class="caption"><p>Selected <i>PAFAH1B1</i> (<i>LIS1</i>) Recurrent Variants</p></div><p class="large-table-link" style="display:none"><span class="right"><a href="/books/NBK5189/table/chrom17-lis.T.selected_pafah1b1_lis1_rec/?report=objectonly" target="object">View in own window</a></span></p><div class="large_tbl" id="__chrom17-lis.T.selected_pafah1b1_lis1_rec_lrgtbl__"><table class="no_bottom_margin"><thead><tr><th id="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_1" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Variant Classification</th><th id="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_2" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">DNA Nucleotide Change<br />(Alias&#x000a0;<sup>1</sup>)</th><th id="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_3" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Predicted Protein Change</th><th id="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_4" scope="col" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">Reference Sequences</th></tr></thead><tbody><tr><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_1" rowspan="2" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><b>Benign</b></td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.*3G&#x0003e;T<br />(1233*3G&#x0003e;T)</td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">--</td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_4" rowspan="8" colspan="1" style="text-align:left;vertical-align:middle;"><a href="/nuccore/164419737" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NM_000430<wbr style="display:inline-block"></wbr>​.3</a><br /><a href="/protein/4557741" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">NP_000421<wbr style="display:inline-block"></wbr>​.1</a></td></tr><tr><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.*17C&#x0003e;T<br />(1233*17C&#x0003e;T)</td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">--</td></tr><tr><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_1" rowspan="6" scope="row" colspan="1" style="text-align:left;vertical-align:middle;"><b>Pathogenic</b></td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_2" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">c.162delA</td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Lys54AsnfsTer15</td></tr><tr><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.162dupA<br />(c.162_163insA)</td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Trp55MetfsTer6</td></tr><tr><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.1050delG</td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Lys351SerfsTer4</td></tr><tr><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.1050dupG<br />(c.1050_1051insG)</td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">p.Lys351GlufsTer8</td></tr><tr><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.569-10T&#x0003e;C<br />(IVS6-10T&#x0003e;C)</td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">--</td></tr><tr><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_2" colspan="1" scope="row" rowspan="1" style="text-align:left;vertical-align:middle;">c.1002+1G&#x0003e;A<br />(IVS9+1G&#x0003e;A)</td><td headers="hd_h_chrom17-lis.T.selected_pafah1b1_lis1_rec_1_1_1_3" rowspan="1" colspan="1" style="text-align:left;vertical-align:middle;">--</td></tr></tbody></table></div><div><div><dl class="temp-labeled-list small"><dt></dt><dd><div><p class="no_margin">Note on variant classification: Variants listed in the table have been provided by the authors. <i>GeneReviews</i> staff have not independently verified the classification of variants.</p></div></dd><dt></dt><dd><div><p class="no_margin">Note on nomenclature: <i>GeneReviews</i> follows the standard naming conventions of the Human Genome Variation Society (<a href="http://varnomen.hgvs.org/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">varnomen<wbr style="display:inline-block"></wbr>​.hgvs.org</a>). See <a href="/books/n/gene/app3/">Quick Reference</a> for an explanation of nomenclature.</p></div></dd><dt>1. </dt><dd><div id="chrom17-lis.TF.4.1"><p class="no_margin">Variant designation that does not conform to current naming conventions</p></div></dd></dl></div></div></div><p><b>Normal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> Platelet-activating factor acetylhydrolase IB subunit alpha (PAFAH1B1) is a 45-kd protein that contains 410 amino acids and is highly conserved among species. The main functional domains of this protein are a LisH motif at the N terminus, followed by a coiled-coil region and seven WD40 repeats at the C terminus [<a class="bk_pop" href="#chrom17-lis.REF.reiner.1993.717">Reiner et al 1993</a>, <a class="bk_pop" href="#chrom17-lis.REF.cardoso.2000.3019">Cardoso et al 2000</a>]. The LisH <a class="def" href="/books/n/gene/glossary/def-item/domain/">domain</a>, coiled-coil domain, and WD40 repeats are important for dimerization, involved in protein/protein interactions, and essential for PAFAH1B1 function [<a class="bk_pop" href="#chrom17-lis.REF.sapir.1999.1011">Sapir et al 1999</a>, <a class="bk_pop" href="#chrom17-lis.REF.efimov.2000.681">Efimov &#x00026; Morris 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.sweeney.2000.263">Sweeney et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.ahn.2001.9903">Ahn &#x00026; Morris 2001</a>, <a class="bk_pop" href="#chrom17-lis.REF.cahana.2001.6429">Cahana et al 2001</a>, <a class="bk_pop" href="#chrom17-lis.REF.kim.2004.987">Kim et al 2004</a>].The PAFAH1B1 protein has two main functions: (1) it forms a trimeric complex with the PAFAH1B2 and PAFAH1B3 proteins to regulate the level of platelet activating factor in the brain [<a class="bk_pop" href="#chrom17-lis.REF.albrecht.1996.579">Albrecht et al 1996</a>, <a class="bk_pop" href="#chrom17-lis.REF.bix.1998.307">Bix &#x00026; Clark 1998</a>]; (2) more importantly, it has been shown to play a central role in the organization of the cytoskeleton by the interaction with proteins including tubulin (the major component of microtubules) as well as proteins associated with the centrosome and involved in microtubule dynamics (e.g., cyoplasmic dynein, dynactin, NUDE<b>,</b> and NUDEL), in turn affecting neuronal proliferation and migration [<a class="bk_pop" href="#chrom17-lis.REF.faulkner.2000.784">Faulkner et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.feng.2000.665">Feng et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.niethammer.2000.697">Niethammer et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.smith.2000.767">Smith et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.suzuki.2007.591">Suzuki et al 2007</a>, <a class="bk_pop" href="#chrom17-lis.REF.torisawa.2011.1959">Torisawa et al 2011</a>, <a class="bk_pop" href="#chrom17-lis.REF.egan.2012.971">Egan et al 2012</a>, <a class="bk_pop" href="#chrom17-lis.REF.huang.2012.975">Huang et al 2012</a>, <a class="bk_pop" href="#chrom17-lis.REF.dix.2013.479">Dix et al 2013</a>] and playing an important role in mitotic spindle regulation [<a class="bk_pop" href="#chrom17-lis.REF.wang.2013.3522">Wang et al 2013</a>, <a class="bk_pop" href="#chrom17-lis.REF.moon.2014.449">Moon et al 2014</a>]. PAFAH1B1 also interacts with components of the Reelin signaling pathway [<a class="bk_pop" href="#chrom17-lis.REF.assadi.2003.270">Assadi et al 2003</a>].</p><p><b>Abnormal <a class="def" href="/books/n/gene/glossary/def-item/gene-product/">gene product</a>.</b> Pathogenic variants in <i>PAFAH1B1</i> result in a reduction in the amount of correctly folded PAFAH1B1 protein [<a class="bk_pop" href="#chrom17-lis.REF.sapir.1999.1011">Sapir et al 1999</a>]. Modeling studies of <i>PAFAH1B1</i> pathogenic variants have shown that they abolish the binding of PAFAH1B1 with protein partners such as PAFAH1B2, PAFAH1B3, NUDE, or NUDEL [<a class="bk_pop" href="#chrom17-lis.REF.sapir.1999.1011">Sapir et al 1999</a>, <a class="bk_pop" href="#chrom17-lis.REF.feng.2000.665">Feng et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.sasaki.2000.681">Sasaki et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.sweeney.2000.263">Sweeney et al 2000</a>]. This in turn interferes with functions such as neuroblast proliferation and migration. <i>PAFAH1B1</i> RNA interference studies indicate that an abnormal <i>PAFAH1B1</i> gene product disrupts the production of neurons in the developing brain as well as their migration [<a class="bk_pop" href="#chrom17-lis.REF.tsai.2005.935">Tsai et al 2005</a>]. Both non-radial cell migration (inhibitory interneurons) and radial migration (excitatory projection neurons) appear to be <a class="def" href="/books/n/gene/glossary/def-item/affected/">affected</a> [<a class="bk_pop" href="#chrom17-lis.REF.fleck.2000.2439">Fleck et al 2000</a>, <a class="bk_pop" href="#chrom17-lis.REF.pancoast.2005.400">Pancoast et al 2005</a>].</p></div><div id="chrom17-lis.References"><h2 id="_chrom17-lis_References_">References</h2><div id="chrom17-lis.Literature_Cited"><h3>Literature Cited</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.ahn.2001.9903">Ahn C, Morris NR. Nudf, a fungal homolog of the human LIS1 protein, functions as a dimmer in vivo. <span><span class="ref-journal">J Biol Chem. </span>2001;<span class="ref-vol">276</span>:9903–9.</span> [<a href="/pubmed/11134054" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11134054</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.albrecht.1996.579">Albrecht U, Abu-Issa R, Ratz R, Hattori M, Aoki J, Arai H, Inoue K, Eichele G. Platelet-activating factor acetylhydrolase expression and activity suggest a link between neuronal migration and platelet-activating factor. <span><span class="ref-journal">Dev Biol. </span>1996;<span class="ref-vol">180</span>:579–93.</span> [<a href="/pubmed/8954729" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 8954729</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.assadi.2003.270">Assadi AH, Zhang G, Beffert U, McNeil RS, Renfro AL, Niu S, Quattrocchi CC, Antalffy BA, Sheldon M, Armstrong DD, Wynshaw-Boris A, Herz J, D'Arcangelo G, Clark GD. Interaction of reelin signaling and Lis1 in brain development. <span><span class="ref-journal">Nat Genet. </span>2003;<span class="ref-vol">35</span>:270–6.</span> [<a href="/pubmed/14578885" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 14578885</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.barkovich.1994.609">Barkovich AJ, Guerrini R, Battaglia G, Kalifa G, N'Guyen T, Parmeggiani A, Santucci M, Giovanardi-Rossi P, Granata T, D'Incerti L. Band heterotopia: correlation of outcome with magnetic resonance imaging parameters. <span><span class="ref-journal">Ann Neurol. </span>1994;<span class="ref-vol">36</span>:609–17.</span> [<a href="/pubmed/7524438" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 7524438</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.barkovich.1991.139">Barkovich AJ, Koch TK, Carrol CL. The spectrum of lissencephaly: report of ten patients analyzed by magnetic resonance imaging. <span><span class="ref-journal">Ann Neurol. </span>1991;<span class="ref-vol">30</span>:139–46.</span> [<a href="/pubmed/1897907" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 1897907</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.bix.1998.307">Bix GJ, Clark GD. Platelet-activating factor receptor stimulation disrupts neuronal migration in vitro. <span><span class="ref-journal">J Neurosci. </span>1998;<span class="ref-vol">18</span>:307–18.</span> [<a href="/pmc/articles/PMC6793386/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC6793386</span></a>] [<a href="/pubmed/9412509" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9412509</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.cahana.2001.6429">Cahana A, Escamez T, Nowakowski RS, Hayes NL, Giacobini M, von Holst A, Shmueli O, Sapir T, McConnell SK, Wurst W, Martinez S, Reiner O. Targeted mutagenesis of LIS1 disrupts cortical development and LIS1 homodimerization. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2001;<span class="ref-vol">98</span>:6429–34.</span> [<a href="/pmc/articles/PMC33485/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC33485</span></a>] [<a href="/pubmed/11344260" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11344260</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.cardoso.2002.4">Cardoso C, Leventer RJ, Dowling JJ, Ward HL, Chung J, Petras KS, Roseberry JA, Weiss AM, Das S, Martin CL, Pilz DT, Dobyns WB, Ledbetter DH. Clinical and molecular basis of classical lissencephaly: mutations in the LIS1 gene. <span><span class="ref-journal">Hum Mutat. </span>2002;<span class="ref-vol">19</span>:4–15.</span> [<a href="/pubmed/11754098" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11754098</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.cardoso.2000.3019">Cardoso C, Leventer RJ, Matsumoto N, Kuc JA, Ramocki MB, Mewborn SK, Dudlicek LL, May LF, Mills PL, Das S, Pilz DT, Dobyns WB, Ledbetter DH. The location and type of mutation predict malformation severity in isolated lissencephaly caused by abnormalities within the LIS1 gene. <span><span class="ref-journal">Hum Mol Genet. </span>2000;<span class="ref-vol">9</span>:3019–28.</span> [<a href="/pubmed/11115846" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11115846</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.cardoso.2003.918">Cardoso C, Leventer RJ, Ward HL, Toyo-Oka K, Chung J, Gross A, Martin CL, Allanson J, Pilz DT, Olney AH, Mutchinick OM, Hirotsune S, Wynshaw-Boris A, Dobyns WB, Ledbetter DH. Refinement of a 400-kb critical region allows genotypic differentiation between isolated lissencephaly, Miller-Dieker syndrome, and other phenotypes secondary to deletions of 17p13.3. <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">72</span>:918–30.</span> [<a href="/pmc/articles/PMC1180354/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1180354</span></a>] [<a href="/pubmed/12621583" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12621583</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.chong.1997.147">Chong SS, Pack SD, Roschke AV, Tanigami A, Carrozzo R, Smith AC, Dobyns WB, Ledbetter DH. A revision of the lissencephaly and Miller-Dieker syndrome critical regions in chromosome 17p13.3. <span><span class="ref-journal">Hum Mol Genet. </span>1997;<span class="ref-vol">6</span>:147–55.</span> [<a href="/pubmed/9063734" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9063734</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.crome.1956.335">Crome L. Pachygyria. <span><span class="ref-journal">J Pathol Bacteriol. </span>1956;<span class="ref-vol">71</span>:335–52.</span> [<a href="/pubmed/13398879" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 13398879</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.cushion.2013.536">Cushion TD, Dobyns WB, Mullins JG, Stoodley N, Chung SK, Fry AE, Hehr U, Gunny R, Aylsworth AS, Prabhakar P, Uyanik G, Rankin J, Rees MI, Pilz DT. Overlapping cortical malformations and mutations in TUBB2B and TUBA1A. <span><span class="ref-journal">Brain. </span>2013 Feb;<span class="ref-vol">136</span>(Pt 2):536–48.</span> [<a href="/pubmed/23361065" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23361065</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.d_agostino.2002.2507">D&#x02019;Agostino MD, Bernasconi A, Das S, Bastos A, Valerio RM, Palmini A, Costa da Costa J, Scheffer IE, Berkovic S, Guerrini R, Dravet C, Ono J, Gigli G, Federico A, Booth F, Bernardi B, Volpi L, Tassinari CA, Guggenheim MA, Ledbetter DH, Gleeson JG, Lopes-Cendes I, Vossler DG, Malaspina E, Franzoni E, Sartori RJ, Mitchell MH, Mercho S, Dubeau F, Andermann F, Dobyns WB, Andermann E. Subcortical band heterotopia (SBH) in males: clinical, imaging and genetic findings in comparison with females. <span><span class="ref-journal">Brain. </span>2002;<span class="ref-vol">125</span>:2507–22.</span> [<a href="/pubmed/12390976" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12390976</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.de_wit.2011.417">de Wit MC, de Rijk-van Andel J, Halley DJ, Poddighe PJ, Arts WF, de Coo IF, Mancini GM. Long-term follow-up of type 1 lissencephaly: survival is related to neuroimaging abnormalities. <span><span class="ref-journal">Dev Med Child Neurol. </span>2011;<span class="ref-vol">53</span>:417–421.</span> [<a href="/pubmed/21410694" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 21410694</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.des_portes.1998.1063">des Portes V, Francis F, Pinard JM, Desguerre I, Moutard ML, Snoeck I, Meiners LC, Capron F, Cusmai R, Ricci S, Motte J, Echenne B, Ponsot G, Dulac O, Chelly J, Beldjord C. Doublecortin is the major gene causing X-linked subcortical laminar heterotopia (SCLH). <span><span class="ref-journal">Hum Mol Genet. </span>1998;<span class="ref-vol">7</span>:1063–70.</span> [<a href="/pubmed/9618162" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9618162</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.dix.2013.479">Dix CI, Soundararajan HC, Dzhindzhev NS, Begum F, Suter B, Ohkura H, Stephens E, Bullock SL. Lissencephaly-1 promotes the recruitment of dynein and dynactin to transported mRNAs. <span><span class="ref-journal">J Cell Biol. </span>2013;<span class="ref-vol">202</span>:479–94.</span> [<a href="/pmc/articles/PMC3734092/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3734092</span></a>] [<a href="/pubmed/23918939" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 23918939</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.dobyns.1991.584">Dobyns WB, Curry CJ, Hoyme HE, Turlington L, Ledbetter DH. Clinical and molecular diagnosis of Miller-Dieker syndrome. <span><span class="ref-journal">Am J Hum Genet. </span>1991;<span class="ref-vol">48</span>:584–94.</span> [<a href="/pmc/articles/PMC1682996/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC1682996</span></a>] [<a href="/pubmed/1671808" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 1671808</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.dobyns.1992.1375">Dobyns WB, Elias ER, Newlin AC, Pagon RA, Ledbetter DH. Causal heterogeneity in isolated lissencephaly. <span><span class="ref-journal">Neurology. </span>1992;<span class="ref-vol">42</span>:1375–88.</span> [<a href="/pubmed/1620349" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 1620349</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.dobyns.1999.270">Dobyns WB, Truwit CL, Ross ME, Matsumoto N, Pilz DT, Ledbetter DH, Gleeson JG, Walsh CA, Barkovich AJ. Differences in the gyral pattern distinguish chromosome 17-linked and X- linked lissencephaly. <span><span class="ref-journal">Neurology. </span>1999;<span class="ref-vol">53</span>:270–7.</span> [<a href="/pubmed/10430413" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10430413</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.efimov.2000.681">Efimov VP, Morris NR. The LIS1-related NUDF protein of Aspergillus nidulans interacts with the coiled-coil domain of the NUDE/R011 protein. <span><span class="ref-journal">J Cell Biol. </span>2000;<span class="ref-vol">150</span>:681–8.</span> [<a href="/pmc/articles/PMC2175200/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2175200</span></a>] [<a href="/pubmed/10931877" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10931877</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.egan.2012.971">Egan MJ, Tan K, Reck-Peterson SL. Lis1 is an initiation factor for dynein-driven organelle transport. <span><span class="ref-journal">J Cell Biol. </span>2012;<span class="ref-vol">197</span>:971–82.</span> [<a href="/pmc/articles/PMC3384415/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3384415</span></a>] [<a href="/pubmed/22711696" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 22711696</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.faulkner.2000.784">Faulkner NE, Dujardin DL, Tai CY, Vaughan KT, O&#x02019;Connell CB, Wang Y, Vallee RB. A role for the lissencephaly gene LIS1 in mitosis and cytoplasmic dynein function. <span><span class="ref-journal">Nat Cell Biol. </span>2000;<span class="ref-vol">2</span>:784–91.</span> [<a href="/pubmed/11056532" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11056532</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.feng.2000.665">Feng Y, Olson EC, Stukenberg PT, Flanagan LA, Kirschner MW, Walsh CA. LIS1 regulates CNS lamination by interacting with mNudE, a central component of the centrosome. <span><span class="ref-journal">Neuron. </span>2000;<span class="ref-vol">28</span>:665–79.</span> [<a href="/pubmed/11163258" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11163258</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.fleck.2000.2439">Fleck MW, Hirotsune S, Gambello MJ, Phillips-Tansey E, Suares G, Mervis RF, Wynshaw-Boris A, McBain CJ. Hippocampal abnormalities and enhanced excitability in a murine model of human lissencephaly. <span><span class="ref-journal">J Neurosci. </span>2000;<span class="ref-vol">20</span>:2439–50.</span> [<a href="/pmc/articles/PMC6772237/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC6772237</span></a>] [<a href="/pubmed/10729324" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10729324</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.fogli.1999.154">Fogli A, Guerrini R, Moro F, Fernandez-Alvarez E, Livet MO, Renieri A, Cioni M, Pilz DT, Veggiotti P, Rossi E, Ballabio A, Carrozzo R. Intracellular levels of the LIS1 protein correlate with clinical and neuroradiological findings in patients with classical lissencephaly. <span><span class="ref-journal">Ann Neurol. </span>1999;<span class="ref-vol">45</span>:154–61.</span> [<a href="/pubmed/9989616" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9989616</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.forman.2005.847">Forman MS, Squier W, Dobyns WB, Golden JA. Genotypically defined lissencephalies show distinct pathologies. <span><span class="ref-journal">J Neuropathol Exp Neurol. </span>2005;<span class="ref-vol">64</span>:847–57.</span> [<a href="/pubmed/16215456" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 16215456</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.gleeson.1999.146">Gleeson JG, Minnerath SR, Fox JW, Allen KM, Luo RF, Hong SE, Berg MJ, Kuzniecky R, Reitnauer PJ, Borgatti R, Mira AP, Guerrini R, Holmes GL, Rooney CM, Berkovic S, Scheffer I, Cooper EC, Ricci S, Cusmai R, Crawford TO, Leroy R, Andermann E, Wheless JW, Dobyns WB, Walsh CA, et al.  Characterization of mutations in the gene doublecortin in patients with double cortex syndrome. <span><span class="ref-journal">Ann Neurol. </span>1999;<span class="ref-vol">45</span>:146–53.</span> [<a href="/pubmed/9989615" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9989615</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.guerrini.2005.32">Guerrini R. Genetic malformations of the cerebral cortex and epilepsy. <span><span class="ref-journal">Epilepsia. </span>2005;<span class="ref-vol">46</span> Suppl 1:32–7.</span> [<a href="/pubmed/15816977" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15816977</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.hattori.1994.216">Hattori M, Adachi H, Tsujimoto M, Arai H, Inoue K. Miller-Dieker lissencephaly gene encodes a subunit of brain platelet-activating factor acetylhydrolase. <span><span class="ref-journal">Nature. </span>1994;<span class="ref-vol">370</span>:216–8.</span> [<a href="/pubmed/8028668" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 8028668</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.haverfield.2009.911">Haverfield EV, Whited AJ, Petras KS, Dobyns WB, Das S. Intragenic deletions and duplications of the LIS1 and DCX genes: a major disease-causing mechanism in lissencephaly and subcortical band heterotopia. <span><span class="ref-journal">Eur J Hum Genet. </span>2009;<span class="ref-vol">17</span>:911–8.</span> [<a href="/pmc/articles/PMC2986498/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2986498</span></a>] [<a href="/pubmed/19050731" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19050731</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.huang.2012.975">Huang J, Roberts AJ, Leschziner AE, Peck-Peterson SL. Lis1 acts as a &#x02018;clutch&#x02019; between the ATPase and microtubule-binding domains of the dynein motor. <span><span class="ref-journal">Cell. </span>2012;<span class="ref-vol">150</span>:975–86.</span> [<a href="/pmc/articles/PMC3438448/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3438448</span></a>] [<a href="/pubmed/22939623" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 22939623</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.kato.2004.147">Kato M, Das S, Petras K, Kitamura K, Morohashi K, Abuelo DN, Barr M, Bonneau D, Brady AF, Carpenter NJ, Cipero KL, Frisone F, Fukuda T, Guerrini R, Iida E, Itoh M, Lewanda AF, Nanba Y, Oka A, Proud VK, Saugier-Veber P, Schelley SL, Selicorni A, Shaner R, Silengo M, Stewart F, Sugiyama N, Toyama J, Toutain A, Vargas AL, Yanazawa M, Zackai EH, Dobyns WB. Mutations of ARX are associated with striking pleiotropy and consistent genotype-phenotype correlation. <span><span class="ref-journal">Hum Mutat. </span>2004;<span class="ref-vol">23</span>:147–59.</span> [<a href="/pubmed/14722918" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 14722918</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.kato.2003.r89">Kato M, Dobyns WB. Lissecenphaly and the molecular basis of neuronal migration. <span><span class="ref-journal">Hum Mol Genet. </span>2003;<span class="ref-vol">12</span>:R89–96.</span> [<a href="/pubmed/12668601" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 12668601</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.kim.2004.987">Kim MH, Cooper DR, Oleksy A, Devedjiev Y, Derewenda U, Reiner O, Otlewski J, Derewenda ZS. The structure of the N-terminal domain of the product of the lissencephaly gene LIS1 and its functional implications. <span><span class="ref-journal">Structure. </span>2004;<span class="ref-vol">12</span>:987–98.</span> [<a href="/pubmed/15274919" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15274919</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.koch.1996.527">Koch A, Tonn J, Albrecht S, Sorensen N, Wiestler OD, Pietsch T. Frequent intragenic polymorphism in the 3&#x02019; untranslated region of the lissencephaly gene 1 (LIS-1). <span><span class="ref-journal">Clin Genet. </span>1996;<span class="ref-vol">50</span>:527–8.</span> [<a href="/pubmed/9147889" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9147889</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.kumar.2010.2817">Kumar RA, Pilz DT, Babatz TD, Cushion TD, Harvey K, Topf M, Yates L, Robb S, Uyanik G, Mancini GMS, Rees MI, Harvey RJ, Dobyns WB. TUBA1A mutations cause wide spectrum lissencephaly (smooth brain) and suggest that multiple neuronal migration pathways converge on alpha tubulins. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:2817–27.</span> [<a href="/pmc/articles/PMC2893812/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2893812</span></a>] [<a href="/pubmed/20466733" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 20466733</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.leventer.2001.416">Leventer RJ, Cardoso C, Ledbetter DH, Dobyns WB. LIS1 missense mutations cause milder lissencephaly phenotypes including a child with normal IQ. <span><span class="ref-journal">Neurology. </span>2001;<span class="ref-vol">57</span>:416–22.</span> [<a href="/pubmed/11502906" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11502906</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.lo_nigro.1997.157">Lo Nigro C, Choong CS, Smith AC, Dobyns WB, Carrozzo R, Ledbetter DH. Point mutations and an intragenic deletion in LIS1, the lissencephaly causative gene in isolated lissencephaly sequence and Miller-Dieker syndrome. <span><span class="ref-journal">Hum Mol Genet. </span>1997;<span class="ref-vol">6</span>:157–64.</span> [<a href="/pubmed/9063735" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9063735</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.matsumoto.2001.5">Matsumoto N, Leventer RJ, Kuc JA, Mewborn SK, Dudlicek LL, Ramocki MB, Pilz DT, Mills PL, Das S, Ross ME, Ledbetter DH, Dobyns WB. Mutation analysis of the DCX gene and genotype/phenotype correlation in subcortical band heterotopia. <span><span class="ref-journal">Eur J Hum Genet. </span>2001;<span class="ref-vol">9</span>:5–12.</span> [<a href="/pubmed/11175293" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11175293</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.mei.2008.355">Mei D, Lewis R, Parrini E, Lazarou LP, Marini C, Pilz DT, Guerrini R. High frequency of genomic deletions&#x02014;and a duplication&#x02014;in the LIS1 gene in lissencephaly: implications for molecular diagnosis. <span><span class="ref-journal">J Med Genet. </span>2008;<span class="ref-vol">45</span>:355–61.</span> [<a href="/pubmed/18285425" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 18285425</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.mei.2007.446">Mei D, Parrini E, Pasqualetti M, Tortorella G, Franzoni E, Giussani U, Marini C, Migliarini S, Guerrini R. Multiplex ligation-dependent probe amplification detects DCX gene deletions in band heterotopia. <span><span class="ref-journal">Neurology. </span>2007;<span class="ref-vol">68</span>:446–50.</span> [<a href="/pubmed/17283321" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17283321</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.mineyko.2010.738">Mineyko A, Doja A, Hurteau J, Dobyns WD, Das S, Boycott KM. A novel missense mutation in LIS1 in a child with subcortical band heterotopia and pachygyria inherited from his mildly affected mother with somatic mosaicism. <span><span class="ref-journal">J Child Neurol. </span>2010;<span class="ref-vol">25</span>:738–41.</span> [<a href="/pubmed/19808989" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19808989</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.moon.2014.449">Moon HM, Youn YH, Pemble H, Yingling J, Wittman T, Wynshaw-Borris A. LIS1 controls mitosis and mitotic spindle organization via the LIS1-NDEL1-dynein complex. <span><span class="ref-journal">Hum Mol Genet. </span>2014;<span class="ref-vol">23</span>:449–66.</span> [<a href="/pmc/articles/PMC3869364/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3869364</span></a>] [<a href="/pubmed/24030547" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24030547</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.niethammer.2000.697">Niethammer M, Smith DS, Ayala R, Peng J, Ko J, Lee M, Morabito M, Tsai LH. NUDEL is a novel Cdk5 substrate that associates with LIS1 and cytoplasmic dynein. <span><span class="ref-journal">Neuron. </span>2000;<span class="ref-vol">28</span>:697–711.</span> [<a href="/pubmed/11163260" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11163260</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.norman.1995">Norman MG, McGillivray BC, Kalousek DK, Hill A, Poskitt KJ. <em>Congenital Malformations of the Brain: Pathological, Embryological, Clinical, Radiological, and Genetic Aspects</em>. New York, NY: Oxford University Press; 1995.</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.pagnamenta.2012.70">Pagnamenta AT, Lise S, Harrison V, Stewart H, Jayawant S, Quaghebeur G, Deng AT, Murphy VE, Sadighi Akha E, Rimmer A, Mathieson I, Knight SJ, Kini U, Taylor JC, Keays DA. Exome sequencing can detect pathogenic mosaic mutations present at low allele frequencies. <span><span class="ref-journal">J Hum Genet. </span>2012;<span class="ref-vol">57</span>:70–2.</span> [<a href="/pubmed/22129557" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 22129557</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.pancoast.2005.400">Pancoast M, Dobyns WB, Golden JA. Interneuron deficits in patients with the Miller-Dieker syndrome. <span><span class="ref-journal">Acta Neuropathol. </span>2005;<span class="ref-vol">109</span>:400–4.</span> [<a href="/pubmed/15739099" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 15739099</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.pilz.1999.1757">Pilz DT, Kuc J, Matsumoto N, Bodurtha J, Bernadi B, Tassinari CA, Dobyns WB, Ledbetter DH. Subcortical band heterotopia in rare affected males can be caused by missense mutations in DCX (XLIS) or LIS1. <span><span class="ref-journal">Hum Mol Genet. </span>1999;<span class="ref-vol">8</span>:1757–60.</span> [<a href="/pubmed/10441340" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10441340</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.pilz.1998a.29">Pilz DT, Macha ME, Precht KS, Smith AC, Dobyns WB, Ledbetter DH. Fluorescence in situ hybridization analysis with LIS1 specific probes reveals a high deletion mutation rate in isolated lissencephaly sequence. <span><span class="ref-journal">Genet Med. </span>1998a;<span class="ref-vol">1</span>:29–33.</span> [<a href="/pubmed/11261426" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11261426</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.pilz.1998b.2029">Pilz DT, Matsumoto N, Minnerath S, Mills P, Gleeson JG, Allen KM, Walsh CA, Barkovich AJ, Dobyns WB, Ledbetter DH, Ross ME. LIS1 and XLIS (DCX) mutations cause most classical lissencephaly, but different patterns of malformation. <span><span class="ref-journal">Hum Mol Genet. </span>1998b;<span class="ref-vol">7</span>:2029–37.</span> [<a href="/pubmed/9817918" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 9817918</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.poirier.2007.1055">Poirier K, Keays DA, Francis F, Saillour Y, Bahi N, Manouvrier S, Fallet-Bianco C, Pasquier L, Toutain A, Tuy FP, Bienvenu T, Joriot S, Odent S, Ville D, Desguerre I, Goldenberg A, Moutard ML, Fryns JP, van Esch H, Harvey RJ, Siebold C, Flint J, Beldjord C, Chelly J. Large spectrum of lissencephaly and pachygyria phenotypes resulting from de novo missense mutations in Tubulin Alpha 1A (TUBA1A). <span><span class="ref-journal">Hum Mutat. </span>2007;<span class="ref-vol">28</span>:1055–64.</span> [<a href="/pubmed/17584854" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17584854</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.quirk.1993.193">Quirk JA, Kendall B, Kingsley DP, Boyd SG, Pitt MC. EEG features of cortical dysplasia in children. <span><span class="ref-journal">Neuropediatrics. </span>1993;<span class="ref-vol">24</span>:193–9.</span> [<a href="/pubmed/8232776" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 8232776</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.ramer.1995.403">Ramer JC, Lin AE, Dobyns WB, Winter R, Ayme S, Pallotta R, Ladda RL. Previously apparently undescribed syndrome: shallow orbits, ptosis, coloboma, trigonocephaly, gyral malformations, and mental and growth retardation. <span><span class="ref-journal">Am J Med Genet. </span>1995;<span class="ref-vol">57</span>:403–9.</span> [<a href="/pubmed/7545868" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 7545868</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.reiner.1993.717">Reiner O, Carrozzo R, Shen Y, Wehnert M, Faustinella F, Dobyns WB, Caskey CT, Ledbetter DH. Isolation of a Miller-Dieker lissencephaly gene containing G protein beta-subunit-like repeats. <span><span class="ref-journal">Nature. </span>1993;<span class="ref-vol">364</span>:717–21.</span> [<a href="/pubmed/8355785" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 8355785</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.ross.2001.256">Ross ME, Swanson K, Dobyns WB. Lissencephaly with cerebellar hypoplasia (LCH): a heterogenous group of cortical malformations. <span><span class="ref-journal">Neuropediatrics. </span>2001;<span class="ref-vol">32</span>:256–63.</span> [<a href="/pubmed/11748497" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11748497</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.rossi.2003.287">Rossi M, Guerrini R, Dobyns WB, Andria G, Winter RM. Characterization of brain malformations in the Baraitser-Winter syndrome and review of the literature. <span><span class="ref-journal">Neuropediatrics. </span>2003;<span class="ref-vol">34</span>:287–92.</span> [<a href="/pubmed/14681753" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 14681753</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.saillour.2009.1007">Saillour Y, Carion N, Quelin C, Leger PL, Boddaert N, Elie C, Toutain A, Mercier S, Barthez MA, Milh M, Joriot S, des Portes V, Philip N, Broglin D, Roubertie A, Pitelet G, Moutard ML, Pinard JM, Cances C, Kaminska A, Chelly J, Beldjord C, Bahi-Buisson N. LIS1-related isolated lissencephaly: spectrum of mutations and relationships with malformation severity. <span><span class="ref-journal">Arch Neurol. </span>2009;<span class="ref-vol">66</span>:1007–15.</span> [<a href="/pubmed/19667223" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 19667223</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.sapir.1999.1011">Sapir T, Eisenstein M, Burgess HA, Horesh D, Cahana A, Aoki J, Hattori M, Arai H, Inoue K, Reiner O. Analysis of lissencephaly-causing LIS1 mutations. <span><span class="ref-journal">Eur J Biochem. </span>1999;<span class="ref-vol">266</span>:1011–20.</span> [<a href="/pubmed/10583396" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10583396</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.sasaki.2000.681">Sasaki S, Shionoya A, Ishida M, Gambello MJ, Yingling J, Wynshaw-Boris A, Hirotsune S. A LIS1/NUDEL/cytoplasmic dynein heavy chain complex in the developing and adult nervous system. <span><span class="ref-journal">Neuron. </span>2000;<span class="ref-vol">28</span>:681–96.</span> [<a href="/pubmed/11163259" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11163259</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.sicca.2003.1042">Sicca F, Kelemen A, Genton P, Das S, Mei D, Moro F, Dobyns WB, Guerrini R. Mosaic mutations of the LIS1 gene cause subcortical band heterotopia. <span><span class="ref-journal">Neurology. </span>2003;<span class="ref-vol">61</span>:1042–6.</span> [<a href="/pubmed/14581661" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 14581661</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.smith.2000.767">Smith DS, Niethammer M, Ayala R, Zhou Y, Gambello MJ, Wynshaw-Boris A, Tsai HL. Regulation of cytoplasmic dynein behaviour and microtubule organization by mammalian LIS1. <span><span class="ref-journal">Nat Cell Biol. </span>2000;<span class="ref-vol">2</span>:767–75.</span> [<a href="/pubmed/11056530" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 11056530</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.suzuki.2007.591">Suzuki SO, McKenney RJ, Mawatari SY, Mizuguchi M, Mikami A, Iwaki T, Goldman JE, Canoll P, Vallee RB. Expression patterns of LIS1, dynein and their interaction partners dynactin, NudE, NudEL and NudC in human gliomas suggest roles in invasion and proliferation. <span><span class="ref-journal">Acta Neuropathol. </span>2007;<span class="ref-vol">113</span>:591–9.</span> [<a href="/pubmed/17221205" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17221205</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.sweeney.2000.263">Sweeney KJ, Clark GD, Prokscha A, Dobyns WB, Eichele G. Lissencephaly associated mutations suggest a requirement for the PAFAH1B heterotrimeric complex in brain development. <span><span class="ref-journal">Mech Dev. </span>2000;<span class="ref-vol">92</span>:263–71.</span> [<a href="/pubmed/10727864" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 10727864</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.torisawa.2011.1959">Torisawa T, Nakayama A, Furuta K, Yamada M, Hirotsune S, Toyoshima YY. Functional dissection of LIS1 and NDEL1 towards understanding the molecular mechanisms of cytoplasmic dynein regulation. <span><span class="ref-journal">J Biol Chem. </span>2011;<span class="ref-vol">286</span>:1959–65.</span> [<a href="/pmc/articles/PMC3023492/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3023492</span></a>] [<a href="/pubmed/21036906" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 21036906</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.tsai.2005.935">Tsai JW, Chen Y, Kriegstein AR, Vallee RB. LIS1 RNA interference blocks neural stem cell division, morphogenesis and motility at multiple stages. <span><span class="ref-journal">J Cell Biol. </span>2005;<span class="ref-vol">170</span>:935–45.</span> [<a href="/pmc/articles/PMC2171430/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC2171430</span></a>] [<a href="/pubmed/16144905" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 16144905</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.uyanik.2007.442">Uyanik G, Morris-Rosendahl DJ, Stiegler J, Klapecki J, Gross C, Berman Y, Martin P, Dey L, Spranger S, Korenke GC, Schreyer I, Hertzberg C, Neumann TE, Burkart P, Spaich C, Meng M, Holthausen H, Ad&#x000e8;s L, Seidel J, Mangold E, Buyse G, Meinecke P, Schara U, Zeschnigk C, Muller D, Helland G, Schulze B, Wright ML, Kortge-Jung S, Hehr A, Bogdahn U, Schuierer G, Kohlhase J, Aigner L, Wolff G, Hehr U, Winkler J. Location and type of mutation in the LIS1 gene do not predict phenotypic severity. <span><span class="ref-journal">Neurology. </span>2007;<span class="ref-vol">69</span>:442–7.</span> [<a href="/pubmed/17664403" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17664403</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.wang.2013.3522">Wang S, Ketcham SA, Schon A, Goodman B, Wang Y, Yates J 3rd, Freire E, Schroer TA, Zheng Y. Nudel/NudE and Lis1 promote dynein and dynactin interaction in the context of spindle morphogenesis. <span><span class="ref-journal">Mol Biol Cell. </span>2013;<span class="ref-vol">24</span>:3522–33.</span> [<a href="/pmc/articles/PMC3826990/" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pmc">PMC free article<span class="bk_prnt">: PMC3826990</span></a>] [<a href="/pubmed/24025714" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 24025714</span></a>]</div></li></ul></div><div id="chrom17-lis.Suggested_Reading"><h3>Suggested Reading</h3><ul class="simple-list"><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.guerrini.2006.322">Guerrini R, Marini C. Genetic malformations of cortical development. <span><span class="ref-journal">Exp Brain Res. </span>2006;<span class="ref-vol">173</span>:322–33.</span> [<a href="/pubmed/16724181" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 16724181</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.kerjan.2007.623">Kerjan G, Gleeson JG. Genetic mechanisms underlying abnormal neuronal migration in classical lissencephaly. <span><span class="ref-journal">Trends Genet. </span>2007;<span class="ref-vol">23</span>:623–30.</span> [<a href="/pubmed/17997185" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17997185</span></a>]</div></li><li class="half_rhythm"><div class="bk_ref" id="chrom17-lis.REF.wynshawboris.2007.296">Wynshaw-Boris A. Lissencephaly and LIS1: insights into the molecular mechanisms of neuronal migration and development. <span><span class="ref-journal">Clin Genet. </span>2007;<span class="ref-vol">72</span>:296–304.</span> [<a href="/pubmed/17850624" ref="pagearea=cite-ref&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">PubMed<span class="bk_prnt">: 17850624</span></a>]</div></li></ul></div></div><div id="chrom17-lis.Chapter_Notes"><h2 id="_chrom17-lis_Chapter_Notes_">Chapter Notes</h2><div id="chrom17-lis.Revision_History"><h3>Revision History</h3><ul><li class="half_rhythm"><div>14 August 2014 (me) Comprehensive update posted live</div></li><li class="half_rhythm"><div>3 March 2009 (me) Review posted live</div></li><li class="half_rhythm"><div>22 August 2008 (wd) Original submission</div></li></ul></div></div><div id="bk_toc_contnr"></div></div></div>
            <div class="post-content"><div><div class="half_rhythm"><a href="/books/about/copyright/">Copyright</a> © 1993-2019, University of Washington, Seattle. GeneReviews is
a registered trademark of the University of Washington, Seattle. All rights
reserved.<p class="small">GeneReviews® chapters are owned by the University of Washington. Permission is
hereby granted to reproduce, distribute, and translate copies of content materials for
noncommercial research purposes only, provided that (i) credit for source (<a href="http://www.genereviews.org/" ref="pagearea=meta&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">http://www.genereviews.org/</a>) and copyright (© 1993-2019 University of
Washington) are included with each copy; (ii) a link to the original material is provided
whenever the material is published elsewhere on the Web; and (iii) reproducers,
distributors, and/or translators comply with the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>. No further modifications are allowed. For clarity, excerpts
of GeneReviews chapters for use in lab reports and clinic notes are a permitted
use.</p><p class="small">For more information, see the <a href="/books/n/gene/GRcopyright_permiss/">GeneReviews® Copyright Notice and Usage
Disclaimer</a>.</p><p class="small">For questions regarding permissions or whether a specified use is allowed,
contact: <a href="mailto:dev@null" data-email="ude.wu@tssamda" class="oemail">ude.wu@tssamda</a>.</p></div><div class="small"><span class="label">Bookshelf ID: NBK5189</span><span class="label">PMID: <a href="/pubmed/20301752" title="PubMed record of this page" ref="pagearea=meta&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=pubmed">20301752</a></span></div><div style="margin-top:2em" class="bk_noprnt"><a class="bk_cntns" href="/books/n/gene/">GeneReviews by Title</a><div class="pagination bk_noprnt"><a class="active page_link prev" href="/books/n/gene/otc-def/" title="Previous page in this title">&lt; Prev</a><a class="active page_link next" href="/books/n/gene/papr/" title="Next page in this title">Next &gt;</a></div></div></div></div>
            
        </div>

        <!-- Custom content below content -->
        <div class="col4">
            
        </div>
        
        
        <!-- Book content -->
        
        <!-- Custom contetnt below bottom nav -->
        <div class="col5">
            
        </div>
    </div>

    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            <div id="ncbi_share_book"><a href="#" class="ncbi_share" data-ncbi_share_config="popup:false,shorten:true" ref="id=NBK5189&amp;db=books">Share</a></div>

        </div>
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Views</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PDF_download" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/NBK5189/?report=reader">PubReader</a></li><li><a href="/books/NBK5189/?report=printable">Print View</a></li><li><a data-jig="ncbidialog" href="#_ncbi_dlg_citbx_NBK5189" data-jigconfig="width:400,modal:true">Cite this Page</a><div id="_ncbi_dlg_citbx_NBK5189" style="display:none" title="Cite this Page"><div class="bk_tt">Dobyns WB, Das S. PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia. 2009 Mar 3 [Updated 2014 Aug 14]. In: Adam MP, Ardinger HH, Pagon RA, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2019. <span class="bk_cite_avail"></span></div></div></li><li><a href="/books/NBK5189/pdf/Bookshelf_NBK5189.pdf">PDF version of this page</a> (665K)</li><li><a href="#" class="toggle-glossary-link" title="Enable/disable links to the glossary">Disable Glossary Links</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>In this GeneReview</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="page-toc" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="#chrom17-lis.Summary" ref="log$=inpage&amp;link_id=inpage">Summary</a></li><li><a href="#chrom17-lis.GeneReview_Scope" ref="log$=inpage&amp;link_id=inpage"><i>GeneReview</i> Scope</a></li><li><a href="#chrom17-lis.Diagnosis" ref="log$=inpage&amp;link_id=inpage">Diagnosis</a></li><li><a href="#chrom17-lis.Clinical_Characteristics" ref="log$=inpage&amp;link_id=inpage">Clinical Characteristics</a></li><li><a href="#chrom17-lis.Genetically_Related_Allelic" ref="log$=inpage&amp;link_id=inpage">Genetically Related (Allelic) Disorders</a></li><li><a href="#chrom17-lis.Differential_Diagnosis" ref="log$=inpage&amp;link_id=inpage">Differential Diagnosis</a></li><li><a href="#chrom17-lis.Management" ref="log$=inpage&amp;link_id=inpage">Management</a></li><li><a href="#chrom17-lis.Genetic_Counseling" ref="log$=inpage&amp;link_id=inpage">Genetic Counseling</a></li><li><a href="#chrom17-lis.Resources" ref="log$=inpage&amp;link_id=inpage">Resources</a></li><li><a href="#chrom17-lis.Molecular_Genetics" ref="log$=inpage&amp;link_id=inpage">Molecular Genetics</a></li><li><a href="#chrom17-lis.References" ref="log$=inpage&amp;link_id=inpage">References</a></li><li><a href="#chrom17-lis.Chapter_Notes" ref="log$=inpage&amp;link_id=inpage">Chapter Notes</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>GeneReviews Links</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="source-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="/books/n/gene/advanced/"><i>GeneReviews</i> Advanced Search</a></li><li><a href="/books/n/gene/glossary/"><i>GeneReviews</i> Glossary</a></li><li><a href="/books/n/gene/resource_mats/">Resource Materials</a> <span class="bk_hlight1">NEW FEATURE</span></li><li><a href="/books/n/gene/updates/">New in <i>GeneReviews</i></a></li><li><a href="/books/n/gene/authors/">Author List</a></li><li><a href="/books/n/gene/prospective_authors/">For Current/Prospective Authors</a></li><li><a href="/books/n/gene/GRpersonnel/"><i>GeneReviews</i> Personnel</a></li><li><a href="/books/n/gene/howto_linkin/">Download/Link to <i>GeneReviews</i></a></li><li><a href="/books/n/gene/contact_us/">Contact Us</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Tests in GTR by Gene</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="document-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="https://www.ncbi.nlm.nih.gov/gtr/tests/?term=5048[geneid]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri&amp;link_id=tests_in_gtr_by_gene">PAFAH1B1</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Variations in ClinVar</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="document-links" id="Shutter"></a></div><div class="portlet_content"><ul xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="simple-list"><li><a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=NBK5189+AND+genereviews[submitter]" ref="pagearea=document-links&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri">Variations from this GeneReview in ClinVar</a></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Related information</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="discovery_db_links" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=omim&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_omim&amp;IdsFromResult=1483258" ref="log$=recordlinks">OMIM</a><div class="brieflinkpop offscreen_noflow">Related OMIM records</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pmc&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pmc_refs&amp;IdsFromResult=1483258" ref="log$=recordlinks">PMC</a><div class="brieflinkpop offscreen_noflow">PubMed Central citations</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=pubmed&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_pubmed_refs&amp;IdsFromResult=1483258" ref="log$=recordlinks">PubMed</a><div class="brieflinkpop offscreen_noflow">Links to PubMed</div></li><li class="brieflinkpopper"><a class="brieflinkpopperctrl" href="/books/?Db=gene&amp;DbFrom=books&amp;Cmd=Link&amp;LinkName=books_gene&amp;IdsFromResult=1483258" ref="log$=recordlinks">Gene</a><div class="brieflinkpop offscreen_noflow">Locus Links</div></li></ul></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" id="Shutter"></a></div><div class="portlet_content"><ul><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/23633430" ref="ordinalpos=1&amp;linkpos=1&amp;log$=relatedarticles&amp;logdbfrom=pubmed">A novel inverted 17p13.3 microduplication disrupting PAFAH1B1 (LIS1) in a girl with syndromic lissencephaly.</a><span class="source">[Am J Med Genet A. 2013]</span><div class="brieflinkpop offscreen_noflow">A novel inverted 17p13.3 microduplication disrupting PAFAH1B1 (LIS1) in a girl with syndromic lissencephaly.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Classen S, Goecke T, Drechsler M, Betz B, Nickel N, Beier M, Schaper J, Karenfort M, Royer-Pokora B. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Am J Med Genet A. 2013 Jun; 161A(6):1453-8. Epub 2013 Apr 30.</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/19050731" ref="ordinalpos=1&amp;linkpos=2&amp;log$=relatedarticles&amp;logdbfrom=pubmed">Intragenic deletions and duplications of the LIS1 and DCX genes: a major disease-causing mechanism in lissencephaly and subcortical band heterotopia.</a><span class="source">[Eur J Hum Genet. 2009]</span><div class="brieflinkpop offscreen_noflow">Intragenic deletions and duplications of the LIS1 and DCX genes: a major disease-causing mechanism in lissencephaly and subcortical band heterotopia.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Haverfield EV, Whited AJ, Petras KS, Dobyns WB, Das S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Eur J Hum Genet. 2009 Jul; 17(7):911-8. Epub 2008 Dec 3.</em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/11115846" ref="ordinalpos=1&amp;linkpos=3&amp;log$=relatedarticles&amp;logdbfrom=pubmed">The location and type of mutation predict malformation severity in isolated lissencephaly caused by abnormalities within the LIS1 gene.</a><span class="source">[Hum Mol Genet. 2000]</span><div class="brieflinkpop offscreen_noflow">The location and type of mutation predict malformation severity in isolated lissencephaly caused by abnormalities within the LIS1 gene.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Cardoso C, Leventer RJ, Matsumoto N, Kuc JA, Ramocki MB, Mewborn SK, Dudlicek LL, May LF, Mills PL, Das S, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Hum Mol Genet. 2000 Dec 12; 9(20):3019-28. </em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/10369882" ref="ordinalpos=1&amp;linkpos=4&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Murine modelling of classical lissencephaly.</a><span class="source">[Neurogenetics. 1999]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Murine modelling of classical lissencephaly.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Gambello MJ, Hirotsune S, Wynshaw-Boris A. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Neurogenetics. 1999 Apr; 2(2):77-86. </em></div></div></li><li class="brieflinkpopper two_line"><a class="brieflinkpopperctrl" href="/pubmed/29628935" ref="ordinalpos=1&amp;linkpos=5&amp;log$=relatedreviews&amp;logdbfrom=pubmed"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Neurodevelopmental Genetic Diseases Associated With Microdeletions and Microduplications of Chromosome 17p13.3.</a><span class="source">[Front Genet. 2018]</span><div class="brieflinkpop offscreen_noflow"><span xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="invert">Review</span> Neurodevelopmental Genetic Diseases Associated With Microdeletions and Microduplications of Chromosome 17p13.3.<div class="brieflinkpopdesc"><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Blazejewski SM, Bennison SA, Smith TH, Toyo-Oka K. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Front Genet. 2018; 9:80. Epub 2018 Mar 23.</em></div></div></li></ul><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=20301752" ref="ordinalpos=1&amp;log$=relatedreviews_seeall&amp;logdbfrom=pubmed">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=20301752" ref="ordinalpos=1&amp;log$=relatedarticles_seeall&amp;logdbfrom=pubmed">See all...</a></div></div><div class="portlet"><div class="portlet_head"><div class="portlet_title"><h3><span>Recent Activity</span></h3></div><a name="Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="Shutter"></a></div><div class="portlet_content"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class=""><div class="action"><a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a></div><ul id="activity"><li class="ra_rcd ralinkpopper two_line"><a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5e0416a7dde089116917660a">PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia - GeneReviews®</a><div class="ralinkpop offscreen_noflow">PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia - GeneReviews®<div class="brieflinkpopdesc"></div></div><div class="tertiary"></div></li></ul><p class="HTOn">Your browsing activity is empty.</p><p class="HTOff">Activity recording is turned off.</p><p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p><a class="seemore" href="/sites/myncbi/recentactivity">See more...</a></div></div></div>
        
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script type="text/javascript" src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script type="text/javascript">
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-12-25T21:10:47-05:00&amp;Snapshot=%2Fprojects%2Fbooks%2FPBooks@5.22&amp;Host=portal104&amp;ncbi_phid=CE8BED77E0408B710000000005E2021E&amp;ncbi_session=CE8BED77E0416A61_1506SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK5189%2F&amp;Db=pmc&amp;folderID=134&amp;Ncbi_App=bookshelf&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        </div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=books&amp;ncbi_pdid=book-part&amp;ncbi_acc=NBK5189&amp;ncbi_domain=gene&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/NBK5189/&amp;ncbi_pagename=PAFAH1B1-Associated Lissencephaly/Subcortical Band Heterotopia - GeneReviews® - NCBI Bookshelf&amp;ncbi_bookparttype=chapter&amp;ncbi_app=bookshelf" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8BED77E0416A61_1506SID /projects/books/PBooks@5.22 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4176521/js/3879255/4121861/3501987/4008961/3893018/3821238/4062932/4123007/4087685/4076480/3921943/3400083/3426610.js" snapshot="books"></script></body>
</html>